# Cecil_2023_Epigenetics applied to child and adolescent mental health Progress, challenges and opportunities.

Received: 29 September 2022- Accepted: 1 December 2022

R E S E A R C H R E V I E W

DOI: 10.1002/jcv2.12133

Epigenetics applied to child and adolescent mental health:
Progress, challenges and opportunities

Charlotte A. M. Cecil1,2,3

| Alexander Neumann4,5 | Esther Walton6

1Department of Child and Adolescent Psychiatry/Psychology, Erasmus MC‐Sophia, Rotterdam, The Netherlands

2Department of Epidemiology, Erasmus MC, Rotterdam, The Netherlands

3Department of Biomedical Data Sciences, Molecular Epidemiology, Leiden University Medical Center, Leiden, The Netherlands

4Complex Genetics of Alzheimer's Disease Group, VIB Center for Molecular Neurology, VIB, Antwerp, Belgium

5Department of Biomedical Sciences, University of Antwerp, Antwerp, Belgium

6Department of Psychology, University of Bath, Bath, UK

Correspondence

Charlotte A. M. Cecil, Department of Child
and Adolescent Psychiatry/Psychology,
Erasmus MC‐Sophia, Rotterdam, The
Netherlands.
Email: c.cecil@erasmusmc.nl

Funding information

European Commission, Grant/Award Number:
101057529; Horizon 2020 Framework
Programme, Grant/Award Number: 848158;
H2020 European Research Council, Grant/
Award Number: 101039672; CLOSER, Grant/
Award Number: ES/K000357/1

Abstract

Background: Epigenetic processes are fast emerging as a promising molecular sys-

tem in the search for both biomarkers and mechanisms underlying human health

and disease risk, including psychopathology.

Methods: In this review, we discuss the application of epigenetics (specifically DNA

methylation) to research in child and adolescent mental health, with a focus on the

use of developmentally sensitive datasets, such as prospective, population‐based

cohorts. We look back at lessons learned to date, highlight current developments

in the field and areas of priority for future research. We also reflect on why

epigenetic research on child and adolescent mental health currently lags behind

other areas of epigenetic research and what we can do to overcome existing

barriers.

Results: To move the field forward, we advocate for the need of large‐scale,

harmonized, collaborative efforts that explicitly account for the time‐varying nature

of epigenetic and mental health data across development.

Conclusion: We conclude with a perspective on what the future may hold in terms

of translational applications as more robust signals emerge from epigenetic research

on child and adolescent mental health.

K E Y W O R D S
child and adolescent psychopathology, development, DNA methylation, epigenetics, population

INTRODUCTION

prevention and intervention. Although numerous pre‐ and postnatal

risk factors have already been identified (e.g., parental psychopa-

Half of mental illnesses are established before the age of 18 years,

thology, socio‐economic hardship, childhood adversities; Barker

often manifesting first in childhood as emotional, behavioural and

et al., 2018), associations with child and adolescent mental health

neurodevelopmental problems (Solmi et al., 2022). This points to

outcomes are far from straightforward, with equifinality (i.e., multiple

early life as a critical window of opportunity for timely detection,

risks associating with the same outcome) and multifinality (the same

This is an open access article under the terms of the Creative Commons Attribution License, which permits use, distribution and reproduction in any medium, pro-
vided the original work is properly cited.

© 2022 The Authors. JCPP Advances published by John Wiley & Sons Ltd on behalf of Association for Child and Adolescent Mental Health.

JCPP Advances. 2023;3:e12133.
https://doi.org/10.1002/jcv2.12133

wileyonlinelibrary.com/journal/jcv2

- 1 of 14

2 of 14 -

risk factor associating with multiple outcomes) representing the

norm rather than the exception. This leaves the puzzling question of

how, exactly, different mental health problems can emerge from what

is seemingly a plethora of common, non‐specific risk factors. The

answer to this question is largely thought to lie in the way that ex-

posures interact with other important factors, such as a child's ge-

netic predispositions and developmental status (i.e.,

timing of

exposure in relation to a child's maturation level), leading to growing

calls

for

integrative research that considers complex gene‐

environment‐development dynamics (GED; Boyce et al., 2020).

Epigenetic processes involved in gene regulation represent a

particularly attractive biological system for studying GED interplay.

Of these, DNA methylation (DNAm) is currently the most widely

investigated and best understood epigenetic process, as it is rela-

tively easy and cost‐effective to quantify on a large scale. DNAm

involves the addition of methyl molecules to DNA base pairs, typi-

cally in the context of cytosine‐guanine (CpG) dinucleotides. Studies

have shown that DNAm: (1) is partly under genetic control (Min

et al., 2021); (2) is sensitive to environmental influences beginning in

CECIL ET AL.

Key points

� Epigenetic processes, such as DNA methylation, are a

promising molecular system for understanding complex

gene‐environment‐development interactions on mental

health.

� Despite tremendous growth in the field of psychiatric

epigenetics, research focussed on child and adolescent

mental health continues to lag behind.

� Closing this gap will require a shift towards better‐

powered, harmonized, multi‐cohort studies that can

adequately capture the time‐varying nature of DNA

methylation and mental health.

� Bringing ‘timing’ at the forefront of epigenetic research

can enhance prediction and mechanistic understanding

of child and adolescent psychopathology.

utero (e.g., dietary, chemical and psychosocial exposures (Cowley

DNAm measured from easily accessible tissues (e.g., blood) may show

et al., 2018; González‐Peña et al., 2021; Rijlaarsdam et al., 2017)); (3)

limited correspondence to those in the target tissue of interest (i.e.,

is

temporally

dynamic,

playing

an

essential

role

in

brain; Bakulski et al., 2016). Even within the brain, DNAm patterns

(neuro)development (Mulder et al., 2020); and that (4) aberrations in

can vary between different regions and cell‐types (Edgar et al., 2017;

DNAm associate with a wide range of health outcomes, including

Rizzardi et al., 2019). Another important factor is time: in a recent

psychiatric disorders (Liu et al., 2018). As a result, DNAm has gained

study pooling longitudinal, epigenome‐wide data from over 2000

much interest in the search for both biomarkers and mechanisms

individuals, we found that more than half of DNAm sites change

underlying GED interplay on psychopathology. In this review, we

significantly during the first two decades of life, and can do so in a

discuss some of the complexities and unique opportunities of

non‐linear way (i.e., at different rates across development; Mulder

studying DNAm in the context of child and adolescent mental health.

et al., 2020). By comparison, studies indicate only modest variability

We provide readers with a view on major lessons learned, current

across individuals for many measured DNAm sites,

leading to a

developments, and emerging topics in this rapidly growing field.

continuing debate about the usefulness of including seemingly non‐

Considering how understudied this area is relative to, for example,

variable sites in analyses. For example, DNAm patterns in many re-

epigenetics applied to adult health, we borrow some of the concepts

gions remain static (e.g., those related to cell‐type differentiation and

and findings from other research areas, reflecting on potential im-

identity), and it is estimated that only about 20% of all methylation

plications for child and adolescent mental health. We conclude with

sites are variable (Ziller et al., 2013).

recommendations for moving the field forward and a perspective on

what the future may hold in terms of translational applications for

research and clinical practice in child and adolescent mental health.

How is this shaping where we are going?

LOOKING BACK: LESSONS LEARNED

This evidence highlights the importance of: (1) considering cross‐

tissue variability and evaluating whether peripheral‐brain concor-

dance is important

for the research question examined (e.g.,

Human epigenetic research has seen tremendous growth over recent

concordance may be less relevant in the search for biomarkers

years. Here, we describe three key lessons that we have learned from

compared to mechanistic research); (2) carefully choosing the timing

this research, and what implications they have for the application of

of DNAm assessment and taking into account (non‐linear) temporal

DNAm to child and adolescent mental health (Figure 1.1).

changes in DNAm, as these could suggest developmentally‐sensitive

periods; and (3) better characterizing DNAm sites at a population

level to define what should be considered (biologically) meaningful

Ways in which DNAm does and does not vary

variability, with potential implications for how arrays are designed in

future (Gunasekara et al., 2019).

One of the main lessons we have learned from existing research is

that DNAm is a highly dynamic process, and that different factors

contribute to this variability. Topping the list of factors is arguably

Many associations, few replicated

tissue and cell‐type: for most DNAm sites, cross‐tissue variability ex-

ceeds that of inter‐individual variability within the same tissue

One of the strongest and most robust associations to emerge from

(Hannon et al., 2015). This is particularly consequential for the study

population‐based epigenetic studies is that of DNAm with smoking

of brain‐based phenotypes (e.g., mental health traits), given that

exposure. The largest epigenome‐wide association study (EWAS)

EPIGENETICS APPLIED TO CHILD AND ADOLESCENT MENTAL HEALTH

- 3 of 14

F I G U R E 1 Graphical overview of themes discussed in this review. Created with BioRender.com

meta‐analysis to date has identified over 5000 significant DNAm

between studies (while being mindful to retain data quality and depth

sites in relation to prenatal smoking exposure in newborns (9 co-

of phenotyping); (2) embedding replication efforts to identify reliable

horts; Npooled = 5648), and over 35,000 sites in relation to own
smoking in adults (16 cohorts; Npooled = 15,907), with hypo‐
methylation of the DNAm site cg05575921 (annotated to the

markers; (3) using repositories to make analysis protocols and results

fully available to improve transparency and reusability of research

findings (e.g., EWAS Atlas and EWAS catalogue; Battram et al., 2022;

AHRR gene) showing the most pronounced difference across studies

Li et al., 2019); and (4) moving from a focus on single DNAm sites to

(Sikdar et al., 2019). Other traits that show widespread and repli-

aggregate scores (also referred to methylation risk scores [MRS]) that

cated associations with DNAm include (gestational) age and (birth)

capture broader DNAm signatures associated with a phenotype of

weight (Küpers et al., 2019; Merid et al., 2020). The picture is starkly

interest (Hüls & Czamara, 2020). Recent work from the Pregnancy

different in the realm of mental health. While many associations have

and Childhood Epigenetics (PACE) consortium (Felix et al., 2018)—

been reported in the literature, these typically show much smaller

comprising over 40 paediatric cohorts with DNAm data at 1+ time

effects, are derived from single datasets of modest sample size and

points—is powerfully illustrating how these strategies can be applied

mostly await replication. Reproducibility in mental health research

to study risk factors (Sammallahti et al., 2021) and outcomes (Car-

has been particularly hampered by heterogeneity in study charac-

amaschi et al., 2022; Neumann et al., 2020) related to child and

teristics, choice of assessments and methodology, adjustment for

adolescent mental health.

important confounders (e.g., medication use), the temporally dynamic

(and often episodic) nature of many mental health phenotypes, as

well as uneven uptake of open science practices, although standards

are constantly improving (Walton et al., 2020).

How is this shaping where we are going?

Peripheral DNAm: Growing support as a biological
marker; mixed evidence as a mediator

Early interest in DNAm in the context of mental health was largely

focussed on its potential role as a mediator of environmental (and

more recently also genetic)

influences on psychiatric outcomes.

Following in the footsteps of genetic research, several strategies are

However, increased awareness of the ‘tissue issue’ has cast more

increasingly being used to address study heterogeneity and small

doubt on the biological plausibility of peripheral DNAm as a mecha-

effect sizes, including: (1) establishing and leveraging collaborative

nism underlying psychiatric risk. While experimental models have

initiatives to maximize power, generalizability and comparability

evidenced multiple ways in which DNAm in blood and brain tissue can

4 of 14 -

CECIL ET AL.

be linked (Walton et al., 2019, 2020), it remains difficult to evaluate

blood)—a ‘timing effect’ initially observed in single cohorts (Walton

these models in humans. Soberingly, a recent large‐scale study

et al., 2017) and recently confirmed via multi‐cohort meta‐analysis

applying Mendelian randomization found limited evidence for a causal

(Neumann et al., 2020). Top DNAm sites at birth implicate, among

role of blood‐based DNAm in neuropsychiatric disorders (Min

others, genes involved in neural functions (e.g., myelination, neuro-

et al., 2021).

transmitter release). The most notable example is ST3GAL3: common

Although cross‐tissue variability makes mechanistic discoveries

variation in this gene has also been identified as a top GWAS hit for

challenging, it does not undercut the potential of DNAm as a biological

ADHD (Demontis et al., 2019; Klein et al., 2019), rare mutations of

marker for disease prediction, stratification and diagnosis. Indeed,

ST3GAL3 associate with cognitive and motor developmental delays

peripheral DNAm patterns are already being used to estimate a range

(Khamirani et al., 2021), and ST3GAL3 knockout in mice results in

of exposures, traits and health outcomes (e.g., age, smoking, BMI;

profound cognitive deficits and hyperactivity due to myelination

McCartney et al., 2018) based on algorithms trained from large data-

disruption (Rivero et al., 2021). Similar epigenetic timing effects (i.e.,

sets, and to detect certain diseases sooner and more accurately than

where prospective associations at birth show overall a stronger signal

conventional diagnostic methods, leading to improved clinical care

in EWAS results than cross‐sectional associations in childhood) have

(Chen, Zang, et al., 2020; Priesterbach‐Ackley et al., 2020). These ap-

also been observed for other neurodevelopmental phenotypes (e.g.,

plications have been slower to permeate the field of mental health,

social communication deficits (Rijlaarsdam et al., 2021)), but not for

likely due to the more limited availability of sufficiently powered

broader child mental

(e.g., general psychopathology (Rijlaarsdam

datasets and challenges with psychiatric phenotypes, such as hetero-

et al., 2022), sleep problems (Sammallahti et al., 2022)) or physical (e.g.,

geneity in clinical presentation and assessment approaches. Never-

BMI; Vehmeijer et al., 2020) health outcomes, despite studies using

theless, the prospect of methylation‐based profiling of neurodevelop-

largely overlapping data, which points to a degree of phenotypic

tnqh_9;mental and psychiatric conditions appears

increasingly

specificity. A detailed overview of epigenetic timing effects in the

possible.

How is this shaping where we are going?

context of neurodevelopmental conditions such as ADHD, outstanding

questions and research priorities in this area can be found elsewhere

(Cecil & Nigg, 2022).

Why is the discovery of epigenetic timing effects meaningful? In

addition to highlighting the dynamic nature of associations between

Large, longitudinal datasets and advanced methods are needed to

DNAm and neurodevelopmental outcomes, this finding has two

disentangle the directionality and causality of associations between

major implications: (1) it supports the potential of DNAm as an

peripheral DNAm patterns and mental health outcomes. In this

early pre‐symptomatic marker of neurodevelopmental risk, and (2)

context, developmental research is all the more important. Even if

it suggests that, to benefit from this potential marker, the timing of

associations are not causal and unlikely intervention targets, DNAm

DNAm assessment could be crucial—the DNAm risk signal captured

can still have utility as a biomarker or as a proxy for causal processes,

at birth may no longer be detectable when DNAm is measured later

thereby (indirectly) lending mechanistic insights.

in life (Walton, 2019). Timing effects may also explain some of the

CURRENT DEVELOPMENTS IN THE FIELD:
HIGHLIGHTS

seemingly inconsistent findings in the literature, as studies have

sampled DNAm at widely different ages. Advanced approaches

capable of handling large‐scale epigenetic data at repeated time

points (e.g., structured life‐course modelling, structural equation

modelling, and time‐course analyses (Brown et al., 2020; Dunn

In this section, we highlight research areas that are gaining increasing

et al., 2019; Hill et al., 2019; Simons et al., 2017)) are needed to

traction and lending new insights into the relationship between

further characterize and disentangle these timing effects, although

DNAm and (mental) health. Where possible, we refer specifically to

separating true temporal signals from technical sources of variation

findings on child and adolescent psychopathology, but also note

in longitudinal data (e.g., batch effects) will be challenging. In future,

research in adjacent fields that could be applied within a develop-

studies will also need to establish whether a signal similar to the

mental context in the future (Figure 1.2).

one observed in cord blood could be obtained from other neonatal

Epigenetic timing effects on neurodevelopmental
outcomes

peripheral tissues, such as neonatal blood spots, which are routinely

collected from heel pricks during the first week of life in many

countries and are already widely used for screening and diagnostic

purposes. Ultimately, epigenetic signals at birth could inform stra-

tegies for improved early risk detection (e.g., by integrating DNAm

While still rare, the increased availability of birth cohorts with

markers in multi‐modal assessment tools including other known risk

repeated epigenetic data in the same individuals has recently made it

factors), and shed light on biological correlates underlying neuro-

possible to explore key developmental aspects of the relationship be-

developmental risk.

tween DNAm and mental health, including (1) whether DNAm patterns

measured at birth (i.e. before symptom onset) associate with mental

health problems later on; and (2) whether these associations remain

stable or change across time. Findings have been intriguing, showing

Differentiating between biological and chronological
age: Epigenetic clocks

that DNAm profiles at birth (cord blood) associate more strongly with

certain neurodevelopmental problems, particularly ADHD symptoms,

Recent years have seen a surge of interest in the concept of bio-

than DNAm measured cross‐sectionally during childhood (whole

logical ageing and its links to health. In the context of DNAm, a

EPIGENETICS APPLIED TO CHILD AND ADOLESCENT MENTAL HEALTH

- 5 of 14

number of ‘epigenetic clocks’ have been developed that can predict

Gene‐environment (G‐E) integration

chronological age (Horvath & Raj, 2018), the pace of ageing (Belsky

et al., 2020) or declining health and mortality (Lu et al., 2019). In

Current research mainly examines genetic or environmental effects

adults, epigenetic age acceleration (i.e., residual or differences

on DNAm separately. This is problematic as environmental exposures

scores, where DNAm‐estimated age outpaces chronological age) has

may be genetically confounded, and in turn, genetic effects on DNAm

been associated with a myriad of

factors,

including socio‐

may be environmentally modulated (i.e., potentially actionable).

demographic characteristics (e.g., male sex,

low socio‐economic

Consideration of both G and E is thus essential to precisely identify

status), unhealthy behaviours (e.g., smoking, alcohol use), poor

influences on DNAm. This is supported by evidence that variation in

health outcomes (e.g., obesity, cancer, heart disease), all‐cause

DNAm is best explained by joint (i.e., additive and interactive) G‐E

mortality and, less consistently, brain outcomes (e.g., brain health,

effects, rather than G or E alone (Czamara et al., 2019, 2021). G‐E

schizophrenia and depression,

lower cognitive ability, total brain

integration can be achieved in several ways, ranging from database

volume, cortical thinning, and greater vascular lesions in old age

queries allowing one to estimate genetic effects on DNAm sites of

(Oblak et al., 2021)). For a comprehensive review of the applica-

interest when genotyping is not available (e.g., GoDMC: http://

tions of DNAm clocks, related challenges and recommendations, see

mqtldb.godmc.org.uk/; Min et al., 2021); currently limited to DNAm

Bell et al. (2019).

sites on the 450k array), to the use of polygenic risk scores (PRS; e.g.,

In contrast to adult studies, the application and significance of

as confounders or moderators of E effects) when genotyping is

epigenetic clocks during development is far less clear. One chal-

available but sample sizes are modest. Ideally, studies would inte-

lenge is methodological: most clocks are trained primarily on adult

grate genetic and environmental data directly; however, this will

samples using wide age ranges and show less accuracy in paediatric

require access to sufficiently powered datasets. To add complexity,

samples (Sanders et al., 2022). While certain clocks have been

G‐E influences should be examined and interpreted in the context of

specifically developed in paediatric samples, these focus either on

developmental timing (in line with the GED framework of psycho-

cord blood to estimate gestational age at birth, or peripheral blood/

pathology; Boyce et al., 2020), as their effects can vary with age. This

buccal cells in childhood and adolescence, complicating efforts to

is well‐exemplified by a recent study showing that glucocorticoid

characterize and integrate epigenetic age measures across these

exposure during the proliferation stage of neural progenitor hippo-

stages of development (Wang & Zhou, 2021). A second challenge is

campal cells, but not during post‐differentiation, alters DNAm pat-

more conceptual: while being ‘epigenetically’ older than one's age is

terns causing long‐lasting changes in the cells' response to future

an indicator of risk in later life, this may not necessarily be the case

stressors (i.e.,

‘priming’ effect; Provençal et al., 2020). In another

during development. For example,

low gestational age at birth

study, timing of childhood adversity was found to explain more

(indicating developmental

immaturity) is a known risk factor for

variability in DNAm than alternative models (e.g., accumulation or

poor mental and physical health outcomes (Aarnoudse‐Moens

recency of exposure), pointing again to the importance of

et al., 2009; den Dekker et al., 2016; Eves et al., 2021). In line with

developmentally‐sensitive epigenetic research (Dunn et al., 2019).

this, preliminary evidence suggests that age deceleration at birth

associates with prenatal environmental adversities (e.g., maternal

depression) and offspring internalizing problems later on (Suarez

Multi‐omics and multi‐phenotype integration

et al., 2018); whereas age acceleration in childhood and adolescence

relates to postnatal environmental adversities (e.g., trauma expo-

The extent to which statistically significant DNAm sites are also

sure) and depressive symptoms (Sumner et al., 2019). It is important

functionally relevant is often unclear. To address these challenges,

to note, however, that findings so far are based on individual

studies have begun to integrate multiple layers of biological data.

studies of modest sample size, and currently lack independent

Most commonly, this involves the use of transcriptomic data to test

replication. This, together with findings that DNAm levels can

whether DNAm sites of interest associate with gene expression

change at different

rates across development, and between

levels, either measured directly in peripheral tissues (with potentially

different tissues, adds a further layer of complexity to the study of

limited relevance to the brain), or examined indirectly in the brain

epigenetic ageing in early life (Dieckmann et al., 2021). A promising

through the use of openly accessible resources. While many of these

avenue is the use of repeated measures of DNAm from birth on-

resources exist to help researchers functionally annotate and char-

ward to estimate a pace of development clock (i.e., focussing on intra‐

acterize findings (e.g., GTEx for gene expression (The GTEX Con-

as opposed to inter‐individual change in epigenetic age) as well as to

sortium, 2020); GoDMC for genetic effects on DNAm (Min

map dynamic associations between risk exposures, epigenetic clocks

et al., 2021); blood‐brain comparison tools for cross‐tissue concor-

and child outcomes as they unfold across different developmental

dance (Edgar et al., 2017)), these often lack sample context and cell‐

stages.

Integration on multiple levels

type specific resolution. Another promising type of data integration,

which is helping to clarify links between peripheral DNAm patterns

and the brain in vivo, is the combination of DNAm and neuroimaging

(Walton et al., 2020)—although this only provides an indirect mea-

sure of peripheral‐brain associations rather than a tool for directly

As the field matures, it is becoming increasingly clear that we need to

inferring functional effects of DNAm on the brain. Besides lending

move towards integrative research to better capture the complexity

biological insights, multi‐omics integration may also help to achieve

of DNAm and its relationship to mental health.

more powerful predictive models, as effect sizes from psychiatric

6 of 14 -

CECIL ET AL.

EWASs are generally small, suggesting that DNAm patterns alone are

performs in predictive models of neurodevelopmental problems such

likely to explain only limited variance in these phenotypes. Going

as ADHD (e.g., compared to baseline models using more established

forward, multivariate approaches will also be needed to account for

risk factors), given the observed epigenetic timing effects described

the known co‐occurrence of both (1) risk factors for psychopathology

above.

(e.g., parental psychopathology and childhood maltreatment); and (2)

different domains of psychopathology themselves (e.g., internalizing

and externalizing problems). In this respect, methods that are already

DNAm and causal inference

established in the field of genetics, such as multi‐trait GWAS (Wu

et al., 2020) or genomic SEM (Grotzinger et al., 2019), could be

The past years have seen major developments in the application of

extended for use with DNAm data.

epidemiological methods to epigenetic research (Adams, 2019;

DNAm in prediction

Yousefi et al., 2022), leading to an increased appreciation of the

challenges faced in making causal inferences about the role of pe-

ripheral DNAm on (mental) health outcomes. These include (ge-

netic) confounding, reverse causation, biological constraints (e.g.,

One of the key interests in DNAm lies in its potential as a predictor of

tissue and cell‐type specificity) and more general methodological

health and disease risk. Indeed, the concept of ‘methylation‐based

issues such as missing data and representativeness. Of note, most

health profiling’ has gained increased traction in recent years and is

studies on DNAm and mental health are restricted to participants

already demonstrating some success. For example, tools such as

of European ancestry. This limits the generalizability of findings and

MethylDetectR (Hillary & Marioni, 2021) enable users to estimate a

risks accentuating health disparities, as future precision medicine

range of human traits (e.g., age, BMI), lifestyle characteristics (e.g.,

tools based on epigenetic data may disproportionately benefit

smoking and alcohol use) and biochemical variables (e.g., neurological

populations from which they were developed, underscoring the

and inflammatory proteins) based on peripheral DNAm alone. These

need for more diverse samples. Approaches that are being used to

DNAm‐based estimates have a number of potential advantages: they

overcome the above challenges include the use of prospective data,

allow users to obtain information on data that is not directly available

repeated measures to model directionality of associations, better

in their dataset (e.g., proteomics; Gadd et al., 2022), they may offer

control for known and unknown confounders (e.g., via surrogate

more reliable information on certain variables (e.g., smoking) than

variable analysis), and the application of more advanced causal

more bias‐prone traditional assessments (e.g., self‐report; Bojesen

inference analyses leveraging genetic data, such as Mendelian

et al., 2017), and importantly may perform better as predictors of

randomization (MR). In the context of epigenetics, MR typically

disease risk. For example, DNAm‐based estimates of BMI in adults

involves the use of mQTLs as genetic proxies for a particular DNAm

have been found to predict risk for diabetes more strongly than BMI

site of interest (i.e., the ‘exposure’), to test its effect on an outcome

itself (Wahl et al., 2017). Although less mature, the application of

while minimizing the potential for confounding and reverse cau-

predictive models to psychiatric epigenetics is beginning to bear fruit.

sality. Using this method, studies have shown, for example, that

For example, adult studies have reported reproducible blood‐based

DNAm patterns in blood are more likely a consequence of than a

DNAm ‘signatures’ of suicide risk (Clive et al., 2016), schizophrenia

cause for BMI, consistent with findings from longitudinal observa-

(Chen, Zang, et al., 2020), and future depression risk, showing greater

tional data (Reed et al., 2020). So far, very few epigenetic studies

explanatory power than models using genetic or clinical data alone

have applied MR to child and adolescent brain‐based phenotypes.

(Clark et al., 2020). Furthermore, a large EWAS study in adults found

One EWAS of seizures across development used MR to test the

that DNAm patterns in blood explain a substantial proportion of

directionality of associations, finding that seizures might be causal

variance in general cognitive function (g), and that a methylation‐

for changes in methylation in blood, rather than vice‐versa (Car-

based predictor derived from these results performs similarly to

amaschi et al., 2020). An extension of MR (2‐Step MR; Relton &

measured cognitive ability in predicting outcomes in independent

Davey Smith, 2012) further allows to examine the role of peripheral

samples, as well as generalizing across different age ranges and pe-

DNAm as a potential casual mediator of environmental exposures on

ripheral tissues (McCartney et al., 2022; Raffington et al., 2021).

outcomes of interest. Using this method, a study found support for

Another study employing a sequencing‐based approach recently

cord blood DNAm as a causal link between maternal vitamin B12

identified a DNAm signature of trauma exposure in early adoles-

during pregnancy and offspring later cognitive function (Caramaschi

cence, which predicted future psychiatric and health outcomes more

et al., 2017, p. 12). As more well‐powered mQTL and EWAS studies

strongly than self‐reported trauma (van den Oord et al., 2022).

continue to emerge, MR will become an increasingly feasible and

Importantly, the majority of these predictive DNAm sites were no

attractive approach for

strengthening causality in epigenetic

longer associated with outcomes when measured again in adulthood,

research on mental health. Other strategies that could be used in

which provides further support for the temporally‐dynamic nature of

future to triangulate evidence include the use of negative controls

DNAm‐mental health associations. It is noteworthy that area under

(e.g., comparing the effect of paternal vs. maternal prenatal expo-

the curve (AUC) estimates in these studies are not far off from well‐

sures on offspring DNAm and downstream outcomes when exam-

established, clinically implemented predictive models, such as the

ining in utero effects), novel techniques for modelling epigenome‐

Framingham Risk Score used to predict coronary heart disease

wide mediation (e.g., Divide‐Aggregate Composite null Test,

(Tzoulaki et al., 2009). Whether similar prediction can be achieved for

DACT; Liu et al., 2020) as well as trio genetic data to better parse

child and adolescent mental health outcomes is currently unclear. In

genetically versus environmentally‐mediated effects on the epi-

the future, it would be interesting to see how well cord blood DNAm

genome and mental health.

EPIGENETICS APPLIED TO CHILD AND ADOLESCENT MENTAL HEALTH

NEXT FRONTIERS

level, there is increased awareness that DNAm patterns can vary

throughout the day based on circadian rhythm, resulting in cyclic

- 7 of 14

What is on the horizon for epigenetic research on (child and

epigenetic oscillations (Oh & Petronis, 2021). Although disruptions in

adolescent) mental health? To push the boundaries of what is

these cycles have been implicated in ageing and disease risk, little is

currently possible, we first need to reach a fuller understanding of

known about their association with mental health outcomes, pointing

epigenetic data itself (Figure 1.3).

to an interesting avenue for future research.

Mapping the covariance structure of DNAm

New ways of addressing the ‘tissue issue’

One property of DNAm that we still know little about is its ‘internal

Clearly, tissue and cell‐type heterogeneity in DNAm remains a major

structure’ (i.e., patterns of covariance). The field of population genetics

challenge for epigenetic research, and we will need to keep improving

has made great strides in defining linkage disequilibrium (LD) in ge-

the ways we take this heterogeneity into account from study design

netic data, enabling key developments such as the imputation of

(e.g., biomarker vs. mechanistic research) to data analysis and inter-

genome‐wide data from a limited set of measured SNPs, improved

pretation. Current studies typically rely on algorithms to estimate

polygenic score analyses as well as genetic heritability and correla-

and adjust for cell‐type proportions. These algorithms, however, are

tion estimations based on summary statistics (Allegrini et al., 2022;

imperfect and account for a limited set of cell‐types. In future, such

Bulik‐Sullivan et al., 2015). Mapping the covariance structure of

panels could be expanded, enabling us to better capture develop-

DNAm would similarly open possibilities to adapt these methods to

mental changes in cell‐type composition (e.g., including multipotent

epigenetic data and propel the field forward. For example, the ability

cells found in cord blood at birth, but scarcely present in peripheral

to impute unmeasured DNAm sites would not only allow one to gain

blood later in life), as well as to extend recent methods for per-

more information from currently available arrays, but also facilitate

forming cell‐type specific EWAS from bulk tissue (Rahmani

pooling of results from samples using different arrays, as routinely

et al., 2019). It will also be important to evaluate whether prior

done in genetic studies. The feasibility of this endeavour, however, is

knowledge of cross‐tissue concordance may be used to improve

uncertain. On the one hand, DNAm sites that show a more consistent

signal in psychiatric epigenetic studies, for example, by selecting or

covariance may be under the strongest genetic control, and thus of

weighing DNAm sites based on blood‐brain correlations, or priori-

limited relevance for capturing exposure‐related DNAm patterns. On

tizing regions that show high inter‐individual variability in combina-

the other, more dynamic sites may be influenced by a range of time‐

tion with low cross‐tissue variability (Gunasekara et al., 2019).

varying, individual‐level factors (e.g., cell‐type, age, environmental,

biological and disease‐related factors) that could affect the covari-

ance structure of DNAm, making it difficult to predict.

Moving beyond (cg) DNAm

Normative modelling and the rise of
‘chronoepigenetics’

Finally, it is important to note that (cg) DNAm is only one of multiple

types of epigenetic factors, which likely play a role in neuro-

development and show potential as markers or mediators of psy-

chiatric risk. These include other types of DNAm marks found to be

The time‐varying nature of DNAm is another property that must be

enriched in the brain (e.g., hydroxymethylcytosine; Spiers et al., 2017)

better characterized. As mentioned in previous sections, longitudinal

as well as histone modifications implicated in several neuro-

studies have identified epigenetic

timing effects on neuro-

developmental and psychiatric conditions, including autism spectrum

developmental outcomes, while research on epigenetic clocks points

disorder (Tseng et al., 2022). Further, experimental studies increas-

to the dynamic nature of associations between risk exposures, bio-

ingly point to circulating microRNAs as a potential mechanism un-

logical age and health outcomes. Together, this evidence highlights

derlying intergenerational transmission of phenotypes,

including

the importance of bringing ‘timing’ at the forefront of the epigenetic

stress‐related physiological and behavioural alterations

(Lem-

research agenda, particularly for fields concerned with develop-

pradl, 2020), while population‐based studies in adults (Mens

mental questions, as in the case of child and adolescent mental

et al., 2021) are beginning to reveal their potential as biomarkers of

health. Going forward, this will mean investing in longitudinal

disease. Currently, these types of data are still rare in paediatric

epigenetic data spanning (pre)birth to adulthood. For example, birth

studies. In future, large‐scale profiling of multiple epigenetic marks

cohorts such as ALSPAC (UK) and the Generation R Study (the

during development will be needed to characterize their independent

Netherlands) are continuing to expand their epigenetic resource

and joint contribution to child and adolescent mental health.

through large‐scale profiling of DNAm patterns at four or more time

points across development. Such resources could be used to generate

normative models (as done in paediatrics [i.e., growth charts] and

increasingly in neuroimaging (Bethlehem et al., 2022; Marquand

et al., 2019)), to estimate the degree to which DNAm levels in an

individual at a given time point deviate from normative develop-

TAKING STOCK: WHY IS THE APPLICATION OF
EPIGENETICS TO CHILD AND ADOLESCENT
MENTAL HEALTH LAGGING BEHIND?

mental curves. This knowledge could also be used to weigh DNAm

As readers will have likely noticed, many of the new findings and

sites in cross‐sectional analyses based on information regarding their

developments highlighted in this review do not originate directly

known pattern of stability or change over time. At a more granular

from the field of child and adolescent mental health, but rather from

8 of 14 -

CECIL ET AL.

fields adjacent to it. This reflects a broader trend in the literature:

robustly detect subtle associations. This time of life also comes

despite tremendous growth in psychiatric (and more broadly health‐

with additional ethical safeguards, which can have an impact on

related) epigenetics over the past 2 decades, the application of

the choice of whether to even collect biological material, and

DNAm to child and adolescent mental health continues to account

which tissue to select.

Indeed, while (longitudinal) population‐

for only a fraction of this work (see Figure 2). Paradoxically, most

based paediatric cohorts typically collect blood,

(cross‐sectional)

psychiatric disorders have developmental origins, making the period

high‐risk and clinical paediatric studies more commonly assess

between pregnancy and young adulthood arguably the most relevant,

DNAm from less invasive peripheral tissues (e.g., saliva, buccal

at least from the perspective of risk prediction and aetiological un-

samples), which further limits opportunities to bridge these com-

derstanding. So, what is behind this research gap?

plementary study types. The second challenge relates to mental

Challenges

health phenotypes

themselves. Their characteristics, even in

adulthood, make them difficult targets to study (e.g., their clinical

heterogeneity,

low specificity and ‘fuzzy’ diagnostic boundaries)

compared to phenotypes that have shown more success in EWAS

Success in epigenetic research applied to complex traits has

studies, and which are easier to measure objectively (e.g., BMI,

largely rested on the ability to carry out well‐designed and

hip‐to‐waist ratio). Developmental research has an added layer of

adequately

powered

studies,

usually

involving multi‐cohort

complexity, as psychiatric phenotypes are more dynamic in early

collaboration,

to detect

robust associations. Although great

life (e.g., differing in age of onset and temporal course), and there

strides have been made in this direction, achieving the same

is less consensus on how best to measure them (e.g., type of

standard for research on child and adolescent mental health has

assessment tool and rater, including parents, teachers or based on

been especially challenging for two main reasons: (1) the stage of

self‐reports), contributing to study heterogeneity.

life and (2) the phenotypes under consideration. Development is

characterized by more rapid and drastic biopsychosocial changes

compared to adulthood, making it a particularly interesting, but

Recommendations

also difficult, period of life to study. To capture these changes,

paediatric data needs to be collected more frequently and at

What can we do to address current challenges and stimulate epige-

closer intervals than adult studies,

implying more administrative

netic research on child and adolescent mental health? With the goal

costs and an increased commitment from families. Given limited

of harmonization in mind,

it will be important to reach a better

resources, paediatric cohorts must typically select key time points

consensus on the time points and intervals of importance within this

to focus on, which may miss important developmental changes and

area of research. Previous studies have highlighted the ‘first

often do not overlap (completely) between cohorts. While this

1000 days’ as a key window for (neuro)development (Cusick &

problem is not exclusive to paediatric studies, the implications are

Georgieff, 2016), while adolescence has also been identified by many

likely greater:

for example, adult data collected at 30 versus

as a critical time point (e.g., due to factors such as pubertal devel-

35 years of age between cohorts is likely to be more comparable

opment). Such periods could be prioritized to maximize comparability

than paediatric data collected at birth versus 5 years, which

between studies, and where possible supplemented by more fine‐

clearly tap into different developmental periods. Together, the

grained and study‐specific time points of data collection. This goal

dynamic nature of development, combined with the availability of

can be achieved largely through better use of existing data. Many

data at non‐overlapping time points across studies, complicates

studies have collected, but not yet processed, biospecimens at

harmonization efforts needed to achieve adequate power

to

different time points (sometimes stored for years or decades), which

F I G U R E 2
2022)

Trends in psychiatric epigenetics: Number of published DNAm studies (2000–2021). Scopus search (access date: 6th May

EPIGENETICS APPLIED TO CHILD AND ADOLESCENT MENTAL HEALTH

- 9 of 14

could be epigenotyped to capture these developmental periods and

DNAm may capture genetic effects that are not measured by SNP

maximize correspondence with other cohorts. Researchers could also

arrays, such as rare variants, as well as gene‐environment correla-

consider linking data from large biobanks and medical records, in line

tions and interactions. Furthermore, DNAm may act as a ‘biological

with legal and ethical standards. For example, many hospitals

record’ of environmental exposures,

leading to more reliable as-

routinely collect heel‐prick samples from newborns, which could be

sessments (e.g., cg05575921 methylation vs. self‐reported smoking)

linked with electronic health data gathered across development and

and greater predictive power compared to alternative measurement

used for research purposes. Relatedly, optimal choice of peripheral

approaches (e.g., MRS of trauma exposure predicting psychiatric risk

tissue will rest on gaining a better understanding of whether certain

better than self‐reported trauma). Unlike genetic data, the time‐

tissues are more informative for a given phenotype (e.g., blood for

varying nature of DNAm also offers possibilities (and unique chal-

capturing immune‐related processes vs. the use of saliva/buccal

lenges) for tracking disease status and health over time, which could

samples to target cells originating from the same ectodermal germ

be particularly useful for early risk detection, patient stratification

layer as the brain), as well as careful consideration their respective

and response to treatment. In this respect, there is already much

drawbacks (e.g. blood collection being more invasive, while saliva and

interest in utilizing epigenetic clocks in adulthood as markers of

buccal samples require restricted eating/drinking prior to collection).

healthy ageing, which may be extended earlier in life to evaluate

Success in this field will also depend on improving outcome definition

healthy development. Whereas these epigenetic clocks may be used

and harmonization of mental health phenotypes across studies.

as broad health markers; they may not fully capture disease‐specific

Consortia are helping to lead the way in this respect. For example,

pathological mechanisms. Epigenetic predictors trained on specific

the LifeCycle project (Jaddoe et al., 2020) has undertaken a massive

outcomes could provide more nuanced information about particular

effort to harmonize environmental, biological and (mental) health

health profiles to inform diagnoses and guide decision‐making in

data across paediatric cohorts including over 250,000 children as

more concrete clinical settings. Regarding diagnosis, it is notable that

part of their EU Child Cohort Network (Nader et al., 2021). The PACE

tools relying on ‘epi‐signatures’ from peripheral blood have already

consortium has further helped to set standards for EWAS meta‐

been developed for a wide range of Mendelian neurodevelopmental

analyses involving paediatric data (Felix et al., 2018). Finally,

diseases, demonstrating utility for brain‐based disorders (Aref‐Eshghi

several new initiatives,

including the ADHD‐Epigenetics Working

et al., 2020). Whether epigenetic‐based tools could one day be used

Group of the Psychiatric Genetics Consortium (PGC) and the EU‐

to improve diagnostic accuracy of child and adolescent psychiatric

funded TEMPO project have been established to tackle the

conditions—and whether the benefits of such tools would outweigh

complexity of timing in epigenetic research on child and adolescent

potential risks and ethical concerns—is an important topic for future

mental health, in order to improve harmonization of epigenetic data

research.

during development and advance capabilities for

longitudinal

At the same time, MRS development will likely face the same

modelling of DNAm‐mental health associations.

challenge of insufficient discovery sample sizes as for PRSs in the

A LOOK TO THE FUTURE: TRANSLATIONAL
APPLICATIONS

past. Individual effect sizes of DNAm sites are not appreciably

larger than SNP effects, but sample sizes of EWAS are many

magnitudes lower compared to GWAS. Furthermore, unlike GWAS,

we have yet to reach a consensus regarding the use of standard-

ized pipelines for pre‐processing of DNAm arrays, including which

As the field matures and more robust epigenetic associations emerge,

method to choose for data normalization and batch correction—an

the question of how this information may be used clinically becomes

important step for maximising comparability between studies and

increasingly relevant. To address this question, genetic research may

reducing noise due to technical variation in EWAS meta‐analyses.

give us useful insights into where we may be heading.

As mentioned in previous sections, while it is possible to impute

Translational applications of genetic research findings have

unmeasured SNPs from genotyping arrays, this is more challenging

focussed to a large degree on PRSs. With ever increasing sample

for DNAm arrays. As such, EWAS studies and potential down-

sizes, the variance explained by PRSs are improving steadily. For

stream applications,

including MRS development, are confined to

instance, a PRS of height based on more than five million participants

measured probes, which represent only a small fraction of DNAm

explained about 40% of variance and 80% of its SNP heritability

sites on the genome. Lastly, reverse causality may limit the appli-

(Yengo et al., 2022). Many child and adolescent psychiatric disorders

cation of cross‐sectional data to develop methylation‐based pre-

are more heritable than height, but PRSs generally explain less than

dictive tools, and confounding may provide misleading therapeutic

5% of their variance (Jansen et al., 2020; Li & He, 2021). This can

targets.

likely be explained by the much lower sample sizes (typically under

In conclusion, the field is still in its infancy, and concrete trans-

100,000 participants) and higher measurement errors of the dis-

lational applications remain a distant goal. Nevertheless, DNAm

covery GWASs. The low variance explained casts doubt on the clin-

continues to hold unique potential as a biological system for

ical utility of current psychiatric PRSs. However, some have argued

biomarker discovery and mechanistic insights into the aetiology of

that they may already be sufficient to identify extreme cases, aid in

child and adolescent psychiatric disorders. Looking to the future,

differential diagnosis and improve treatment response (Fullerton &

increases in sample sizes—via collaborative science, harmonization

Nurnberger, 2019).

efforts and better use of existing data—in combination with a focus

Could adding information on DNAm bring these applications a

on developmentally‐sensitive,

longitudinal study designs will be

step closer to clinical practice? In addition to common variants,

crucial to move the field forward and leverage this potential.

10 of 14 -

A U T H O R C O N T R IBUTIONS

Charlotte A. M. Cecil: Conceptualization, Funding acquisition, Project

administration, Supervision, Visualization, Writing—original draft,

Writing – review & editing. Alexander Neumann: Conceptualization,

Writing—original draft, Writing—review & editing. Esther Walton:

Conceptualization, Writing—original draft, Writing—review & editing.

A C K N OWL E D G EM ENTS

Charlotte A. M. Cecil and Esther Walton have received funding for

this work from the European Union's Horizon 2020 Research and

Innovation Programme (EarlyCause; grant agreement No 848158).

Esther Walton received funding from the National Institute of Mental

Health of

the National

Institutes of Health (award number

R01MH113930) and from CLOSER (ES/K000357/1). Charlotte A. M.

Cecil and Alexander Neumann are also supported by the European

Union's HorizonEurope Research and Innovation Programme (FAM-

ILY; grant agreement No 101057529) and the European Research

Council (TEMPO; grant agreement No 101039672).

C O N F L IC T O F I N T EREST

The authors have declared that they have no competing or potential

conflicts of interest.

DATA AVAILABILITY STATEMENT

Data sharing not applicable to this article as no datasets were

generated or analysed during the current study.

ETHICAL CON SIDER ATIONS

Not applicable.

ORCID

Charlotte A. M. Cecil

https://orcid.org/0000-0002-2389-5922

REFERENC ES

Aarnoudse‐Moens, C. S. H., Weisglas‐Kuperus, N., van Goudoever, J. B., &
Oosterlaan, J. (2009). Meta‐analysis of neurobehavioral outcomes in
very preterm and/or very low birth weight children. Pediatrics,
124(2), 717–728. https://doi.org/10.1542/peds.2008‐2816

Adams, C. D. (2019). A brief tour of epidemiologic epigenetics and mental
health. Current Opinion in Psychology, Genetics, 27, 36–40. https://doi.
org/10.1016/j.copsyc.2018.07.010

Allegrini, A. G., Baldwin, J. R., Barkhuizen, W., & Pingault, J.‐B. (2022).
Research review: A guide to computing and implementing polygenic
scores in developmental research. Journal of Child Psychology and
Psychiatry, 63(10), 1111–1124. https://doi.org/10.1111/jcpp.13611
Aref‐Eshghi, E., Kerkhof, J., Pedro, V. P., Barat‐Houari, M., Ruiz‐Pallares,
N., Andrau, J.‐C., Lacombe, D., VanGils, J., Fergelot, P., Dubourg, C.,
Cormier‐Daire, V., Rondeau, S., Lecoquierre, F., Saugier‐Veber, P.,
Nicolas, G., Lesca, G., Chatron, N., Sanlaville, D., Vitobello, A., …
Sadikovic, B. (2020). Evaluation of DNA methylation episignatures
for diagnosis and phenotype correlations in 42 Mendelian neuro-
developmental disorders. The American Journal of Human Genetics,
106(3), 356–370. https://doi.org/10.1016/j.ajhg.2020.01.019
Bakulski, K. M., Halladay, A., Hu, V. W., Mill, J., & Fallin, M. D. (2016).
Epigenetic research in neuropsychiatric disorders: The “tissue issue”.
Current Behavioral Neuroscience Reports, 3, 264–274. https://doi.org/
10.1007/s40473‐016‐0083‐4

Barker, E. D., Walton, E., & Cecil, C. A. M. (2018). Annual research review:
DNA methylation as a mediator in the association between risk
exposure and child and adolescent psychopathology. Journal of Child
Psychology and Psychiatry, 59(4), 303–322. https://doi.org/10.1111/
jcpp.12782

CECIL ET AL.

Battram, T., Yousefi, P., Crawford, G., Prince, C., Babaei, M. S., Sharp, G.,
Hatcher, C., Vega‐Salas, M. J., Khodabakhsh, S., Whitehurst, O.,
Langdon, R., Mahoney, L., Elliott, H. R., Mancano, G., Lee, M. A.,
Watkins, S. H., Lay, A. C., Hemani, G., Gaunt, T. R., … Suderman, M.
(2022). The EWAS catalog: A database of epigenome‐wide associa-
tion studies. Wellcome Open Research, 7. https://doi.org/10.12688/
wellcomeopenres.17598.2

Bell, C. G., Lowe, R., Adams, P. D., Baccarelli, A. A., Beck, S., Bell, J. T.,
Christensen, B. C., Gladyshev, V. N., Heijmans, B. T., Horvath, S.,
Ideker, T., Issa, J.‐P. J., Kelsey, K. T., Marioni, R. E., Reik, W., Relton,
C. L., Schalkwyk, L. C., Teschendorff, A. E., Wagner, W., Zhang, K., &
Rakyan, V. K. (2019). DNA methylation aging clocks: Challenges and
recommendations. Genome Biology, 20(1), 249. https://doi.org/10.
1186/s13059‐019‐1824‐y

Belsky, D. W., Caspi, A., Arseneault, L., Baccarelli, A., Corcoran, D. L., Gao,
X., Hannon, E., Harrington, H. L., Rasmussen, L. J., Houts, R., Huffman,
K., Kraus, W. E., Kwon, D., Mill, J., Pieper, C. F., Prinz, J. A., Poulton,
R., Schwartz, J., Sugden, K., … Moffitt, T. E. (2020). Quantification of
the pace of biological aging in humans through a blood test, the
DunedinPoAm DNA methylation algorithm. Elife, 9, e54870. https://
doi.org/10.7554/eLife.54870

Bethlehem, R. A. I., Seidlitz, J., White, S. R., Vogel, J. W., Anderson, K. M.,
Adamson, C., Adler, S., Alexopoulos, G. S., Anagnostou, E., Areces‐
Gonzalez, A., Astle, D. E., Auyeung, B., Ayub, M., Bae, J., Ball, G.,
Baron‐Cohen, S., Beare, R., Bedford, S. A., Benegal, V., … Alexander‐
Bloch, A. F. (2022). Brain charts for the human lifespan. Nature,
604(7906), 525–533. https://doi.org/10.1038/s41586‐022‐04554‐y
Bojesen, S. E., Timpson, N., Relton, C., Davey Smith, G., & Nordestgaard,
B. G. (2017). AHRR (cg05575921) hypomethylation marks smoking
behaviour, morbidity and mortality. Thorax, 72(7), 646–653. https://
doi.org/10.1136/thoraxjnl‐2016‐208789

Boyce, W. T., Sokolowski, M. B., & Robinson, G. E. (2020). Genes and
environments, development and time. Proceedings of the National
Academic Science, 117(38), 23235–23241. https://doi.org/10.1073/
pnas.2016710117

Brown, G. L., Kogan, S. M., & Cho, J. (2020). Pathways linking childhood
trauma to rural, unmarried, African American father involvement
through oxytocin receptor gene methylation. Developmental Psy-
chology, 56(8), 1496–1508. https://doi.org/10.1037/dev0000929
Bulik‐Sullivan, B., Finucane, H. K., Anttila, V., Gusev, A., Day, F. R., Loh, P.‐
R., Duncan, L., Perry, J. R. B., Patterson, N., Robinson, E. B., Daly,
M. J., Price, A. L., & Neale, B. M., Psychiatric Genomics Consortium &
Genetic Consortium for Anorexia Nervosa of the Wellcome Trust
Case Control Consortium 3. (2015). An atlas of genetic correlations
across human diseases and traits. Nature Genetics, 47(11),
1236–1241. https://doi.org/10.1038/ng.3406

Caramaschi, D., Hatcher, C., Mulder, R. H., Felix, J. F., Cecil, C. A. M.,
Relton, C. L., & Walton, E. (2020). Epigenome‐wide association study
of seizures in childhood and adolescence. Clinical Epigenetics, 12(1), 8.
https://doi.org/10.1186/s13148‐019‐0793‐z

Caramaschi, D., Neumann, A., Cardenas, A., Tindula, G., Alemany, S., Zillich,
L., Pesce, G., Lahti, J. M. T., Havdahl, A., Mulder, R., Felix, J. F., Tiemeier,
H., Sirignano, L., Frank, J., Witt, S. H., Rietschel, M., Deuschle, M., Huen,
K., Eskenazi, B., … Sharp, G. (2022). Meta‐analysis of epigenome‐wide
associations between DNA methylation at birth and childhood
cognitive skills. Molecular Psychiatry, 27(4), 2126–2135. https://doi.
org/10.1038/s41380‐022‐01441‐w

Caramaschi, D., Sharp, G. C., Nohr, E. A., Berryman, K., Lewis, S. J., Davey
Smith, G., & Relton, C. L. (2017). Exploring a causal role of DNA
methylation in the relationship between maternal vitamin B12
during pregnancy and child’s IQ at age 8, cognitive performance and
educational attainment: A two‐step Mendelian randomization study.
Human Molecular Genetics, 26(15), 3001–3013. https://doi.org/10.
1093/hmg/ddx164

Cecil, C. A. M., & Nigg, J. T. (2022). Epigenetics and ADHD: Reflections on
current knowledge, research priorities and translational potential.
Molecular Diagnosis and Therapy, 26(6), 581–606. https://doi.org/10.
1007/s40291‐022‐00609‐y

Chen, X., Gole, J., Gore, A., He, Q., Lu, M., Min, J., Yuan, Z., Yang, X., Jiang,
Y., Zhang, T., Suo, C., Li, X., Cheng, L., Zhang, Z., Niu, H., Li, Z., Xie, Z.,

EPIGENETICS APPLIED TO CHILD AND ADOLESCENT MENTAL HEALTH

Shi, H., Zhang, X., … Jin, L. (2020). Non‐invasive early detection of
cancer four years before conventional diagnosis using a blood test.
Nature Communications, 11(1), 3475. https://doi.org/10.1038/
s41467‐020‐17316‐z

Chen, J., Zang, Z., Braun, U., Schwarz, K., Harneit, A., Kremer, T., Ma, R.,
Schweiger, J., Moessnang, C., Geiger, L., Cao, H., Degenhardt, F.,
Nöthen, M. M., Tost, H., Meyer‐Lindenberg, A., & Schwarz, E. (2020).
Association of a reproducible epigenetic risk profile for schizo-
phrenia with brain methylation and function. JAMA Psychiatry, 77(6),
628–636. https://doi.org/10.1001/jamapsychiatry.2019.4792
Clark, S. L., Hattab, M. W., Chan, R. F., Shabalin, A. A., Han, L. K., Zhao, M.,
Smit, J. H., Jansen, R., Milaneschi, Y., Xie, L. Y., van Grootheest, G.,
Penninx, B. W., Aberg, K. A., & van den Oord, E. J. (2020). A
methylation study of long term depression risk. Molcular Psychiatry,
25(6), 1334–1343. https://doi.org/10.1038/s41380‐019‐0516‐z
Clive, M. L., Boks, M. P., Vinkers, C. H., Osborne, L. M., Payne, J. L., Ressler,
K. J., Smith, A. K., Wilcox, H. C., & Kaminsky, Z. (2016). Discovery and
replication of a peripheral tissue DNA methylation biosignature to
augment a suicide prediction model. Clinical Epigenetics, 8(1), 113.
https://doi.org/10.1186/s13148‐016‐0279‐1

Cowley, M., Skaar, D. A., Jima, D. D., Maguire, R. L., Hudson, K. M., Park,
S. S., Sorrow, P., & Hoyo, C. (2018). Effects of cadmium exposure on
DNA methylation at imprinting control regions and genome‐wide in
mothers and newborn children. Environmental Health Perspectives,
126(3), 037003. https://doi.org/10.1289/EHP2085

Cusick, S., & Georgieff, M. (2016). The first 1,000 days of life: The brain’s

window of opportunity. UNICEF‐IRC.

Czamara, D., Eraslan, G., Page, C. M., Lahti, J., Lahti‐Pulkkinen, M.,
Hämäläinen, E., Kajantie, E., Laivuori, H., Villa, P. M., Reynolds, R. M.,
Nystad, W., Håberg, S. E., London, S. J., O’Donnell, K. J., Garg, E.,
Meaney, M. J., Entringer, S., Wadhwa, P. D., Buss, C., Jones, M. J., Lin,
D. T. S., MacIsaac, J. L., Kobor, M. S., Koen, N., Zar, H. J., Koenen,
K. C., Dalvie, S., Stein, D. J., Kondofersky, I., Müller, N. S., Theis, F. J.,
Major Depressive Disorder Working Group of the Psychiatric Ge-
nomics Consortium, Räikkönen, K., & Binder, E. B. (2019). Integrated
analysis of environmental and genetic influences on cord blood DNA
methylation in new‐borns. Nature Communications, 10, 2548. https://
doi.org/10.1038/s41467‐019‐10461‐0

Czamara, D., Tissink, E., Tuhkanen, J., Martins, J., Awaloff, Y., Drake, A. J.,
Khulan, B., Palotie, A., Winter, S. M., Nemeroff, C. B., Craighead,
W. E., Dunlop, B. W., Mayberg, H. S., Kinkead, B., Mathew, S. J.,
Iosifescu, D. V., Neylan, T. C., Heim, C. M., Lahti, J., … Binder, E. B.
(2021). Combined effects of genotype and childhood adversity shape
variability of DNA methylation across age. Translational Psychiatry,
11(1), 88. https://doi.org/10.1038/s41398‐020‐01147‐z

Demontis, D., Walters, R. K., Martin, J., Mattheisen, M., Als, T. D., Agerbo,
E., Baldursson, G., Belliveau, R., Bybjerg‐Grauholm, J., Bækvad‐
Hansen, M., Cerrato, F., Chambert, K., Churchhouse, C., Dumont, A.,
Eriksson, N., Gandal, M., Goldstein, J. I., Grasby, K. L., Grove, J., …
Neale, B. M. (2019). Discovery of the first genome‐wide significant
risk loci for attention deficit/hyperactivity disorder. Nature Genetics,
51, 63–75. https://doi.org/10.1038/s41588‐018‐0269‐7

den Dekker, H. T., Sonnenschein‐van der Voort, A. M. M., de Jongste, J. C.,
Anessi‐Maesano, I., Arshad, S. H., Barros, H., Beardsmore, C. S.,
Bisgaard, H., Phar, S. C., Craig, L., Devereux, G., van der Ent, C. K.,
Esplugues, A., Fantini, M. P., Flexeder, C., Frey, U., Forastiere, F.,
Gehring, U., Gori, D., … Duijts, L. (2016). Early growth characteristics
and the risk of reduced lung function and asthma: A meta‐analysis of
25, 000 children. The Journal of Allergy and Clinical
Immunology,
137(4), 1026–1035. https://doi.org/10.1016/j.jaci.2015.08.050
Dieckmann, L., Lahti‐Pulkkinen, M., Kvist, T., Lahti, J., De Witt, P. E.,
Cruceanu, C., Laivuori, H., Sammallahti, S., Villa, P. M., Suomalainen‐
König, S., Eriksson, J. G., Kajantie, E., Raikkönen, K., Binder, E. B., &
Czamara, D.
(2021). Characteristics of epigenetic aging across
gestational and perinatal tissues. Clinical Epigenetics, 13(1), 97.
https://doi.org/10.1186/s13148‐021‐01080‐y

- 11 of 14

Results from a prospective, longitudinal study. Biological Psychiatry,
85(10), 838–849. https://doi.org/10.1016/j.biopsych.2018.12.023

Edgar, R. D., Jones, M. J., Meaney, M. J., Turecki, G., & Kobor, M. S., 2017.
BECon: A tool for interpreting DNA methylation findings from blood
in the context of brain. Translational Psychiatry, 7(8), e1187. https://
doi.org/10.1038/tp.2017.171

Eves, R., Mendonça, M., Baumann, N., Ni, Y., Darlow, B. A., Horwood, J.,
Woodward, L. J., Doyle, L. W., Cheong, J., Anderson, P. J., Bartmann, P.,
Marlow, N., Johnson, S., Kajantie, E., Hovi, P., Nosarti, C., Indredavik,
M. S., Evensen, K.‐A. I., Räikkönen, K., … Wolke, D. (2021). Association
of very preterm birth or very low birth weight with intelligence in
adulthood: An individual participant data meta‐analysis. JAMA Pedi-
atrics, 175, e211058. https://doi.org/10.1001/jamapediatrics.202
1.1058

Felix, J. F., Joubert, B. R., Baccarelli, A. A., Sharp, G. C., Almqvist, C.,
Annesi‐Maesano, I., Arshad, H., Baïz, N., Bakermans‐Kranenburg,
M. J., Bakulski, K. M., Binder, E. B., Bouchard, L., Breton, C. V., Bru-
nekreef, B., Brunst, K. J., Burchard, E. G., Bustamante, M., Chatzi, L.,
Cheng Munthe‐Kaas, M., … London, S. J. (2018). Cohort profile:
Pregnancy and childhood epigenetics (PACE) consortium. Interna-
tional Journal of Epidemiology, 47(1), 22–23u. https://doi.org/10.
1093/ije/dyx190

Fullerton, J. M., & Nurnberger, J. I. (2019). Polygenic risk scores in psy-
chiatry: Will they be useful for clinicians? https://doi.org/10.12688/
f1000research.18491.1

Gadd, D. A., Hillary, R. F., McCartney, D. L., Zaghlool, S. B., Stevenson, A. J.,
Cheng, Y., Fawns‐Ritchie, C., Nangle, C., Campbell, A., Flaig, R.,
Harris, S. E., Walker, R. M., Shi, L., Tucker‐Drob, E. M., Gieger, C.,
Peters, A., Waldenberger, M., Graumann, J., McRae, A. F., … Marioni,
R. E. (2022). Epigenetic scores for the circulating proteome as tools
for disease prediction. Elife, 11, e71802. https://doi.org/10.7554/
eLife.71802

González‐Peña, S. M., Calvo‐Anguiano, G., Martínez‐De‐villarreal, L. E.,
Ancer‐Rodríguez, P. R., Lugo‐Trampe, J. J., Saldivar‐Rodríguez, D.,
Hernández‐Almaguer, M. D., Calzada‐Dávila, M., Guerrero‐Orjuela,
L. S., & Campos‐Acevedo, L. D. (2021). Maternal folic acid intake and
methylation status of genes associated with ventricular septal de-
fects in children: Case–control study. Nutrients, 13(6), 2071. https://
doi.org/10.3390/nu13062071

Grotzinger, A. D., Rhemtulla, M., de Vlaming, R., Ritchie, S. J., Mallard, T. T.,
Hill, W. D., Ip, H. F., Marioni, R. E., McIntosh, A. M., Deary, I. J.,
Koellinger, P. D., Harden, K. P., Nivard, M. G., & Tucker‐Drob, E. M.
(2019). Genomic structural equation modelling provides insights into
the multivariate genetic architecture of complex traits. Nature Hu-
man Behaviour, 3(5), 513–525. https://doi.org/10.1038/s41562‐019‐
0566‐x

Gunasekara, C. J., Scott, C. A., Laritsky, E., Baker, M. S., MacKay, H.,
Duryea, J. D., Kessler, N. J., Hellenthal, G., Wood, A. C., Hodges,
K. R., Gandhi, M., Hair, A. B., Silver, M. J., Moore, S. E., Prentice,
A. M., Li, Y., Chen, R., Coarfa, C., & Waterland, R. A. (2019). A
genomic atlas of systemic interindividual epigenetic variation in
humans. Genome Biology, 20(1), 105. https://doi.org/10.1186/
s13059‐019‐1708‐1

Hannon, E., Lunnon, K., Schalkwyk, L., & Mill, J. (2015). Interindividual
methylomic variation across blood, cortex, and cerebellum: Impli-
cations for epigenetic studies of neurological and neuropsychiatric
phenotypes. Epigenetics, 10(11), 1024–1032. https://doi.org/10.
1080/15592294.2015.1100786

Hillary, R. F., Marioni, R. E.

(2021). MethylDetectR: A software for
methylation‐based health profiling. Wellcome Open Research, 5.
https://doi.org/10.12688/wellcomeopenres.16458.2

Hill, J., Pickles, A., Wright, N., Quinn, J. P., Murgatroyd, C., & Sharp, H.
(2019). Mismatched prenatal and postnatal maternal depressive
symptoms and child behaviours: A sex‐dependent role for NR3C1
DNA methylation in the Wirral child health and development study.
Cells, 8. https://doi.org/10.3390/cells8090943

Dunn, E. C., Soare, T. W., Zhu, Y., Simpkin, A. J., Suderman, M. J., Klengel,
T., Smith, A. D. A. C., Ressler, K., & Relton, C. L. (2019). Sensitive
periods for the effect of childhood adversity on DNA methylation:

Horvath, S., & Raj, K. (2018). DNA methylation‐based biomarkers and the
epigenetic clock theory of ageing. Nature Reviews Genetics, 19(6),
371–384. https://doi.org/10.1038/s41576‐018‐0004‐3

12 of 14 -

Hüls, A., & Czamara, D. (2020). Methodological challenges in constructing
DNA methylation risk scores. Epigenetics, 15(1–2), 1–11. https://doi.
org/10.1080/15592294.2019.1644879

Jaddoe, V. W. V., Felix, J. F., Andersen, A.‐M. N., Charles, M.‐A., Chatzi, L.,
Corpeleijn, E., Donner, N., Elhakeem, A., Eriksson, J. G., Foong, R.,
Grote, V., Haakma, S., Hanson, M., Harris, J. R., Heude, B., Huang, R.‐
C., Inskip, H., Järvelin, M.‐R., Koletzko, B., … Rauschert, S. (2020). The
LifeCycle project‐EU child cohort network: A federated analysis
infrastructure and harmonized data of more than 250,000 children
and parents. European Journal of Epidemiology, 35(7), 709–724.
https://doi.org/10.1007/s10654‐020‐00662‐z

Jansen, A. G., Dieleman, G. C., Jansen, P. R., Verhulst, F. C., Posthuma, D., &
Polderman, T. J. C. (2020). Psychiatric polygenic risk scores as pre-
dictor for attention deficit/hyperactivity disorder and autism spec-
trum disorder in a clinical child and adolescent sample. Behavior
Genetics, 50(4), 203–212. https://doi.org/10.1007/s10519‐019‐
09965‐8

Khamirani, H. J., Zoghi, S., Faghihi, F., Dastgheib, S. A., Hassanipour, H.,
Bagher Tabei, S. M., Mohammadi, S., Masoudi, M., Poorang, S.,
Ehsani, E., & Dianatpour, M. (2021). Phenotype of ST3GAL3 deficient
patients: A case and review of the literature. European Journal of
Medical Genetics, 64(8), 104250. https://doi.org/10.1016/j.ejmg.20
21.104250

Klein, M., Walters, R. K., Demontis, D., Stein, J. L., Hibar, D. P., Adams,
H. H., Bralten, J., Roth Mota, N., Schachar, R., Sonuga‐Barke, E.,
Mattheisen, M., Neale, B. M., Thompson, P. M., Medland, S. E.,
Børglum, A. D., Faraone, S. V., Arias‐Vasquez, A., & Franke, B. (2019).
Genetic markers of ADHD‐related variations in Intracranial volume.
American Journal of Psychiatry, 176(3), 228–238. https://doi.org/10.
1176/appi.ajp.2018.18020149

Küpers, L. K., Monnereau, C., Sharp, G. C., Yousefi, P., Salas, L. A., Ghan-
tous, A., Page, C. M., Reese, S. E., Wilcox, A. J., Czamara, D., Starling,
A. P., Novoloaca, A., Lent, S., Roy, R., Hoyo, C., Breton, C. V., Allard,
C., Just, A. C., Bakulski, K. M., … Felix, J. F. (2019). Meta‐analysis of
epigenome‐wide association studies in neonates reveals widespread
differential DNA methylation associated with birthweight. Nature
Communications, 10(1), 1893. https://doi.org/10.1038/s41467‐019‐
09671‐3

Lempradl, A. (2020). Germ cell‐mediated mechanisms of epigenetic in-
heritance. Seminar Cell Developmental Biology, SI: Chromatin Dynamics
in Regeneration, 97, 116–122. https://doi.org/10.1016/j.semcdb.20
19.07.012

Li, J. J., & He, Q. (2021). Polygenic scores for ADHD: A meta‐analysis.
Research on Child and Adolescent Psychopathology, 49(3), 297–310.
https://doi.org/10.1007/s10802‐021‐00774‐4

Liu, C., Jiao, C., Wang, K., & Yuan, N. (2018). DNA methylation and psy-
chiatric disorders. In D. R. Grayson (Ed.), Progress in molecular biology
and translational science, epigenetics and psychiatric disease (pp.
175–232). Academic Press. https://doi.org/10.1016/bs.pmbts.2018.
01.006

Liu, Z., Shen, J., Barfield, R., Schwartz, J., Baccarelli, A. A., & Lin, X. (2020).
Large‐scale hypothesis testing for causal mediation effects with
applications in genome‐wide epigenetic studies. https://doi.org/10.
1101/2020.09.20.20198226

Li, M., Zou, D., Li, Z., Gao, R., Sang, J., Zhang, Y., Li, R., Xia, L., Zhang, T., Niu,
G., Bao, Y., & Zhang, Z. (2019). EWAS Atlas: A curated knowledge-
base of epigenome‐wide association studies. Nucleic Acids Research,
47(D1), D983–D988. https://doi.org/10.1093/nar/gky1027

Lu, A. T., Quach, A., Wilson, J. G., Reiner, A. P., Aviv, A., Raj, K., Hou, L.,
Baccarelli, A. A., Li, Y., Stewart, J. D., Whitsel, E. A., Assimes, T. L.,
Ferrucci, L., & Horvath, S.
(2019). DNA methylation GrimAge
strongly predicts lifespan and healthspan. Aging, 11(2), 303–327.
https://doi.org/10.18632/aging.101684

Marquand, A. F., Kia, S. M., Zabihi, M., Wolfers, T., Buitelaar, J. K., &
Beckmann, C. F. (2019). Conceptualizing mental disorders as de-
viations from normative functioning. Molecular Psychiatry, 24(10),
1415–1424. https://doi.org/10.1038/s41380‐019‐0441‐1

McCartney, D. L., Hillary, R. F., Conole, E. L. S., Banos, D. T., Gadd, D. A.,
Walker, R. M., Nangle, C., Flaig, R., Campbell, A., Murray, A. D.,

CECIL ET AL.

Maniega, S. M., Valdés‐Hernández, M., del, C., Harris, M. A., Bastin,
M. E., Wardlaw, J. M., Harris, S. E., Porteous, D. J., Tucker‐Drob,
E. M., … Marioni, R. E. (2022). Blood‐based epigenome‐wide analyses
of cognitive abilities. Genome Biology, 23(1), 26. https://doi.org/10.
1186/s13059‐021‐02596‐5

McCartney, D. L., Hillary, R. F., Stevenson, A. J., Ritchie, S. J., Walker, R. M.,
Zhang, Q., Morris, S. W., Bermingham, M. L., Campbell, A., Murray,
A. D., Whalley, H. C., Gale, C. R., Porteous, D. J., Haley, C. S., McRae,
A. F., Wray, N. R., Visscher, P. M., McIntosh, A. M., Evans, K. L., Deary,
I. J., & Marioni, R. E. (2018). Epigenetic prediction of complex traits
and death. Genome Biology, 19(1), 136. https://doi.org/10.1186/
s13059‐018‐1514‐1

Mens, M. M. J., Heshmatollah, A., Fani, L., Ikram, M. A., Ikram, M. K., &
Ghanbari, M. (2021). Circulatory microRNAs as potential biomarkers
for stroke risk. Stroke, 52(3), 945–953. https://doi.org/10.1161/
STROKEAHA.120.031543

Merid, S. K., Novoloaca, A., Sharp, G. C., Küpers, L. K., Kho, A. T., Roy, R.,
Gao, L., Annesi‐Maesano, I., Jain, P., Plusquin, M., Kogevinas, M.,
Allard, C., Vehmeijer, F. O., Kazmi, N., Salas, L. A., Rezwan, F. I.,
Zhang, H., Sebert, S., Czamara, D., … Melén, E. (2020). Epigenome‐
wide meta‐analysis of blood DNA methylation in newborns and
children identifies numerous loci related to gestational age. Genome
Medicine, 12(1), 25. https://doi.org/10.1186/s13073‐020‐0716‐9
Min, J. L., Hemani, G., Hannon, E., Dekkers, K. F., Castillo‐Fernandez, J.,
Luijk, R., Carnero‐Montoro, E., Lawson, D. J., Burrows, K., Suderman,
M., Bretherick, A. D., Richardson, T. G., Klughammer, J., Iotchkova, V.,
Sharp, G., AlKhleifat, A., Shatunov, A., Iacoangeli, A., McArdle, W. L., …
Relton, C. L. (2021). Genomic and phenotypic insights from an atlas of
genetic effects on DNA methylation. Nature Genetics, 53(9),
1311–1321. https://doi.org/10.1038/s41588‐021‐00923‐x

Mulder, R. H., Neumann, A., Cecil, C. A. M., Walton, E., Houtepen, L. C.,
Simpkin, A. J., Rijlaarsdam, J., Heijmans, B. T., Gaunt, T. R., Felix, J. F.,
Jaddoe, V. W. V., Bakermans‐Kranenburg, M. J., Tiemeier, H., Relton,
C. L., Ijzendoorn, M. H., & van Suderman, M. (2020). Epigenome‐wide
change and variation in DNA methylation from birth to late
adolescence. bioRxiv 2020.06.09.142620. https://doi.org/10.1101/
2020.06.09.142620

Nader, J. L., López‐Vicente, M., Julvez, J., Guxens, M., Cadman, T., Elha-
keem, A., Järvelin, M.‐R., Rautio, N., Miettunen, J., El Marroun, H.,
Melchior, M., Heude, B., Charles, M.‐A., Yang, T. C., McEachan, R. R.
C., Wright, J., Polanska, K., Carson, J., Lin, A., … Harris, J. R. (2021).
Cohort description: Measures of early‐life behaviour and later psy-
chopathology in the LifeCycle project – EU child cohort network.
Journal of Epidemiology. https://doi.org/10.2188/jea.JE20210241
Neumann, A., Walton, E., Alemany, S., Cecil, C., González, J. R., Jima, D. D.,
Lahti, J., Tuominen, S. T., Barker, E. D., Binder, E., Caramaschi, D.,
Carracedo, Á., Czamara, D., Evandt, J., Felix, J. F., Fuemmeler, B. F.,
Gutzkow, K. B., Hoyo, C., Julvez, J., … Tiemeier, H. (2020). Associa-
tion between DNA methylation and ADHD symptoms from birth to
school age: A prospective meta‐analysis. Translational Psychiatry, 10,
1–11. https://doi.org/10.1038/s41398‐020‐01058‐z

Oblak, L., van der Zaag, J., Higgins‐Chen, A. T., Levine, M. E., & Boks, M. P.
(2021). A systematic review of biological, social and environmental
factors associated with epigenetic clock acceleration. Ageing
Research Reviews, 69, 101348. https://doi.org/10.1016/j.arr.2021.1
01348

Oh, E. S., & Petronis, A. (2021). Origins of human disease: The chrono‐
epigenetic perspective. Nature Reviews Genetics, 22(8), 533–546.
https://doi.org/10.1038/s41576‐021‐00348‐6

Priesterbach‐Ackley, L. P., Boldt, H. B., Petersen, J. K., Bervoets, N., Scheie,
D., Ulhøi, B. P., Gardberg, M., Brännström, T., Torp, S. H., Aronica, E.,
Küsters, B., den Dunnen, W. F. A., de Vos, F. Y. F. L., Wesseling, P., de
Leng, W. W. J., & Kristensen, B. W. (2020). Brain tumour diagnostics
using a DNA methylation‐based classifier as a diagnostic support
tool. Neuropathology and Applied Neurobiology, 46(5), 478–492.
https://doi.org/10.1111/nan.12610

Provençal, N., Arloth, J., Cattaneo, A., Anacker, C., Cattane, N., Wiech-
mann, T., Röh, S., Ködel, M., Klengel, T., Czamara, D., Müller, N. S.,
Lahti, J., Räikkönen, K., Pariante, C. M., Binder, E. B., Kajantie, E.,

EPIGENETICS APPLIED TO CHILD AND ADOLESCENT MENTAL HEALTH

Hamalainen, E., Villa, P., & Laivuori, H., & PREDO Team. (2020).
Glucocorticoid exposure during hippocampal neurogenesis primes
future stress response by inducing changes in DNA methylation.
Proceedings of the National Academy of Sciences of the United States of
America, 117(38), 23280–23285. https://doi.org/10.1073/pnas.18
20842116

Raffington, L., Tanksley, P. T., Sabhlok, A., Vinnik, L., Mallard, T., King, L. S.,
Goosby, B., Harden, K. P., & Tucker‐Drob, E. M. (2021). Socially
stratified epigenetic profiles are associated with cognitive func-
tioning in children and adolescents. https://doi.org/10.1101/2021.
08.19.456979

Rahmani, E., Schweiger, R., Rhead, B., Criswell, L. A., Barcellos, L. F., Eskin,
E., Rosset, S., Sankararaman, S., & Halperin, E. (2019). Cell‐type‐
specific resolution epigenetics without the need for cell sorting or
single‐cell biology. Nature Communications, 10(1), 3417. https://doi.
org/10.1038/s41467‐019‐11052‐9

Reed, Z. E., Suderman, M. J., Relton, C. L., Davis, O. S. P., & Hemani, G.
(2020). The association of DNA methylation with body mass index:
Distinguishing between predictors and biomarkers. Clinical Epige-
netics, 12(1), 50. https://doi.org/10.1186/s13148‐020‐00841‐5
Relton, C. L., & Davey Smith, G. (2012). Two‐step epigenetic Mendelian
randomization: A strategy for establishing the causal role of epige-
International Journal of
netic processes in pathways to disease.
Epidemiology, 41(1), 161–176. https://doi.org/10.1093/ije/dyr233

Rijlaarsdam, J., Cecil, C. A. M., Relton, C. L., & Barker, E. D. (2021).
Epigenetic profiling of social communication trajectories and co‐
occurring mental health problems: A prospective, methylome‐wide
association study. Development and Psychopathology, 34(3), 1–10.
https://doi.org/10.1017/S0954579420001662

Rijlaarsdam, J., Cecil, C. A. M., Walton, E., Mesirow, M. S. C., Relton, C. L.,
Gaunt, T. R., McArdle, W., & Barker, E. D. (2017). Prenatal unhealthy
diet,
insulin‐like growth factor 2 gene (IGF2) methylation, and
attention deficit hyperactivity disorder symptoms in youth with
early‐onset conduct problems. Journal on Child Psychology and Psy-
chiatry, 58(1), 19–27. https://doi.org/10.1111/jcpp.12589

Rijlaarsdam, J., Cosin‐Tomas, M., Schellhas, L., Abrishamcar, S., Malmberg,
A., Neumann, A., Felix, J. F., Sunyer, J., Gutzkow, K. B., Grazulevi-
ciene, R., Wright, J., Kampouri, M., Zar, H. J., Stein, D. J., Heinonen,
K., Räikkönen, K., Lahti, J., Hüls, A., Caramaschi, D., Alemany, S., &
Cecil, C. A. M. (2022). DNA Methylation and general psychopa-
thology in childhood: An epigenome‐wide meta‐analysis from the
PACE consortium. Molecular Psychiatry, 1–9. https://doi.org/10.
1038/s41380‐022‐01871‐6

Rivero, O., Alhama‐Riba, J., Ku, H.‐P., Fischer, M., Ortega, G., Álmos, P.,
Diouf, D., van den Hove, D., & Lesch, K.‐P. (2021). Haploinsufficiency
of the attention‐deficit/hyperactivity disorder risk gene St3gal3 in
mice causes alterations in cognition and expression of genes
involved in myelination and Sialylation. Frontiers in Genetics, 12,
688488. https://doi.org/10.3389/fgene.2021.688488

Rizzardi, L. F., Hickey, P. F., Rodriguez DiBlasi, V., Tryggvadóttir, R., Call-
ahan, C. M., Idrizi, A., Hansen, K. D., & Feinberg, A. P. (2019).
Neuronal brain‐region‐specific DNA methylation and chromatin
accessibility are associated with neuropsychiatric trait heritability.
Nature Neuroscience, 22(2), 307–316. https://doi.org/10.1038/
s41593‐018‐0297‐8

Sammallahti, S., Cortes Hidalgo, A. P., Tuominen, S., Malmberg, A., Mulder,
R. H., Brunst, K. J., Alemany, S., McBride, N. S., Yousefi, P., Heiss, J. A.,
McRae, N., Page, C. M., Jin, J., Pesce, G., Caramaschi, D., Rifas‐Shiman,
S. L., Koen, N., Adams, C. D., Magnus, M. C., … Lahti, J. (2021). Maternal
anxiety during pregnancy and newborn epigenome‐wide DNA
methylation. Molecular Psychiatry, 26(6), 1–14. https://doi.org/10.
1038/s41380‐020‐00976‐0

Sammallahti, S., Koopman‐Verhoeff, M. E., Binter, A.‐C., Mulder, R. H.,
Cabré‐Riera, A., Kvist, T., Malmberg, A. L. K., Pesce, G., Plancoulaine,
S., Heiss, J. A., Rifas‐Shiman, S. L., Röder, S. W., Starling, A. P., Wilson,
R., Guerlich, K., Haftorn, K. L., Page, C. M., Luik, A. I., Tiemeier, H., …
Cecil, C. A. M. (2022). Longitudinal associations of DNA methylation

- 13 of 14

and sleep in children: A meta‐analysis. Clinical Epigenetics, 14(1), 83.
https://doi.org/10.1186/s13148‐022‐01298‐4

Sanders, F., Baltramonaityte, V., Donohoe, G., Davies, N. M., Dunn, E. C.,
Cecil, C. A. M., & Walton, E. (2022). Associations between methyl-
ation age and brain age in late adolescence. https://doi.org/10.1101/
2022.09.08.506972

Sikdar, S., Joehanes, R., Joubert, B. R., Xu, C.‐J., Vives‐Usano, M., Rezwan,
F. I., Felix, J. F., Ward, J. M., Guan, W., Richmond, R. C., Brody, J. A.,
Küpers, L. K., Baïz, N., Håberg, S. E., Smith, J. A., Reese, S. E., Asli-
bekyan, S., Hoyo, C., Dhingra, R., … London, S. J. (2019). Comparison
of smoking‐related DNA methylation between newborns from pre-
natal exposure and adults from personal smoking. Epigenomics,
11(13), 1487–1500. https://doi.org/10.2217/epi‐2019‐0066

Simons, R. L., Lei, M. K., Beach, S. R. H., Cutrona, C. E., & Philibert, R. A.
(2017). Methylation of the oxytocin receptor gene mediates the
effect of adversity on negative schemas and depression. Development
and Psychopathology, 29(3), 725–736. https://doi.org/10.1017/
S0954579416000420

Solmi, M., Radua, J., Olivola, M., Croce, E., Soardo, L., Salazar de Pablo, G.,
Il Shin, J., Kirkbride, J. B., Jones, P., Kim, J. H., Kim, J. Y., Carvalho,
A. F., Seeman, M. V., Correll, C. U., & Fusar‐Poli, P. (2022). Age at
onset of mental disorders worldwide: Large‐scale meta‐analysis of
192 epidemiological studies. Molecular Psychiatry, 27(1), 281–295.
https://doi.org/10.1038/s41380‐021‐01161‐7

Spiers, H., Hannon, E., Schalkwyk, L. C., Bray, N. J., & Mill, J. (2017). 5‐
hydroxymethylcytosine is highly dynamic across human fetal brain
development. BMC Genomics, 18(1), 738. https://doi.org/10.1186/
s12864‐017‐4091‐x

Suarez, A., Lahti, J., Czamara, D., Lahti‐Pulkkinen, M., Knight, A. K.,
Girchenko, P., Hämäläinen, E., Kajantie, E., Lipsanen, J., Laivuori, H.,
Villa, P. M., Reynolds, R. M., Smith, A. K., Binder, E. B., & Räikkönen,
K. (2018). The epigenetic clock at birth: Associations with maternal
Antenatal depression and child psychiatric problems. Journal of the
American Academy of Child & Adolescent Psychiatry, 57(5), 321–328.
e2. https://doi.org/10.1016/j.jaac.2018.02.011

Sumner, J. A., Colich, N. L., Uddin, M., Armstrong, D., & McLaughlin, K. A.
(2019). Early experiences of threat, but not deprivation, are asso-
ciated with accelerated biological aging in children and adolescents.
Biological Psychiatry, 85(3), 268–278. https://doi.org/10.1016/j.
biopsych.2018.09.008

The GTEX Consortium. (2020). The GTEx Consortium atlas of genetic
regulatory effects across human tissues. Science, 369(6509),
1318–1330. https://doi.org/10.1126/science.aaz1776

Tseng, C.‐E. J., McDougle, C. J., Hooker, J. M., & Zürcher, N. R. (2022).
Epigenetics of autism spectrum disorder: Histone deacetylases.
Biological Psychiatry, 91(11), 922–933. https://doi.org/10.1016/j.
biopsych.2021.11.021

Tzoulaki, I., Liberopoulos, G., & Ioannidis, J. P. A. (2009). Assessment of
claims of improved prediction beyond the Framingham risk score.
JAMA, 302(21), 2345–2352. https://doi.org/10.1001/jama.2009.
1757

van den Oord, C. L. J. D., Copeland, W. E., Zhao, M., Xie, L. Y., Aberg, K. A.,
& van den Oord, E. J. C. G. (2022). DNA methylation signatures of
childhood trauma predict psychiatric disorders and other adverse
outcomes 17 years after exposure. Molecular Psychiatry, 27(8),
3367–3373. https://doi.org/10.1038/s41380‐022‐01597‐5

Vehmeijer, F. O. L., Küpers, L. K., Sharp, G. C., Salas, L. A., Lent, S., Jima,
D. D., Tindula, G., Reese, S., Qi, C., Gruzieva, O., Page, C., Rezwan,
F. I., Melton, P. E., Nohr, E., Escaramís, G., Rzehak, P., Heiskala, A.,
Gong, T., Tuominen, S. T., … Felix, J. F. (2020). DNA methylation and
body mass index from birth to adolescence: meta‐analyses of
epigenome‐wide association studies. Genome Medicine, 12(1), 105.
https://doi.org/10.1186/s13073‐020‐00810‐w

Wahl, S., Drong, A., Lehne, B., Loh, M., Scott, W. R., Kunze, S., Tsai, P.‐C.,
Ried, J. S., Zhang, W., Yang, Y., Tan, S., Fiorito, G., Franke, L., Guar-
rera, S., Kasela, S., Kriebel, J., Richmond, R. C., Adamo, M., Afzal, U., …
Chambers, J. C. (2017). Epigenome‐wide association study of body

14 of 14 -

mass index, and the adverse outcomes of adiposity. Nature,
541(7635), 81–86. https://doi.org/10.1038/nature20784

Walton, E. (2019). Epigenome‐wide associations with attention‐deficit/
life
hyperactivity disorder in adults: The need for a longitudinal
course approach in epigenetic psychiatry. Biological Psychiatry, 86(8),
570–572. https://doi.org/10.1016/j.biopsych.2019.07.021

Walton, E., Calhoun, V. D., Heijmans, B., Thompson, P., & Cecil, C. (2020).
The rise of neuroimaging epigenetics: A systematic review of studies
examining associations between DNA methylation and brain imag-
ing. https://doi.org/10.31234/osf.io/4a8xn

Walton, E., Pingault, J.‐B., Cecil, C. a. M., Gaunt, T. R., Relton, C. L., Mill, J.,
& Barker, E. D. (2017). Epigenetic profiling of ADHD symptoms
trajectories: A prospective, methylome‐wide study. Molecular Psy-
chiatry, 22(2), 250–256. https://doi.org/10.1038/mp.2016.85
Walton, E., Relton, C. L., & Caramaschi, D. (2019). Using openly accessible
resources to strengthen causal inference in epigenetic epidemiology
of neurodevelopment and mental health. Genes, 10(3), 193. https://
doi.org/10.3390/genes10030193

Wang, J., & Zhou, W.‐H. (2021). Epigenetic clocks in the pediatric popu-
lation: When and why they tick? Chinese Medical Journal, 134(24),
2901–2910. https://doi.org/10.1097/CM9.0000000000001723
Wu, Y., Cao, H., Baranova, A., Huang, H., Li, S., Cai, L., Rao, S., Dai, M., Xie,
M., Dou, Y., Hao, Q., Zhu, L., Zhang, X., Yao, Y., Zhang, F., Xu, M., &
Wang, Q. (2020). Multi‐trait analysis for genome‐wide association
study of five psychiatric disorders. Translational Psychiatry, 10, 1–11.
https://doi.org/10.1038/s41398‐020‐00902‐6

CECIL ET AL.

Yengo, L., Vedantam, S., Marouli, E., Sidorenko, J., Bartell, E., Sakaue, S.,
Graff, M., Eliasen, A. U., Jiang, Y., Raghavan, S., Miao, J., Arias, J. D.,
Graham, S. E., Mukamel, R. E., Spracklen, C. N., Yin, X., Chen, S.‐H.,
Ferreira, T., Highland, H. H., … Hirschhorn, J. N., 23 and Me Research
Team. (2022). A saturated map of common genetic variants associ-
ated with human height. Nature, 610, 704–712. https://doi.org/10.
1038/s41586‐022‐05275‐y

Yousefi, P. D., Suderman, M., Langdon, R., Whitehurst, O., Davey Smith, G.,
& Relton, C. L. (2022). DNA methylation‐based predictors of health:
Applications and statistical considerations. Nature Reviews Genetics,
23(6), 369–383. https://doi.org/10.1038/s41576‐022‐00465‐w
Ziller, M. J., Gu, H., Müller, F., Donaghey, J., Tsai, L. T.‐Y., Kohlbacher, O.,
De Jager, P. L., Rosen, E. D., Bennett, D. A., Bernstein, B. E., Gnirke,
A., & Meissner, A. (2013). Charting a dynamic DNA methylation
landscape of the human genome. Nature, 500(7463), 477–481.
https://doi.org/10.1038/nature12433

How to cite this article: Cecil, C. A. M., Neumann, A., &

Walton, E. (2023). Epigenetics applied to child and adolescent

mental health: Progress, challenges and opportunities. JCPP

Advances, 3(1), e12133. https://doi.org/10.1002/jcv2.12133
