# Genovese_2020_Clinical Assessment, Genetics, and Treatment Approaches in Autism Spectrum Disorder (ASD).

Review
Clinical Assessment, Genetics, and Treatment
Approaches in Autism Spectrum Disorder (ASD)

Ann Genovese and Merlin G. Butler *

Department of Psychiatry & Behavioral Sciences, University of Kansas Medical Center,
Kansas City, KS 66160, USA; agenovese@kumc.edu
* Correspondence: mbutler4@kumc.edu; Tel.: +1-913-588-1800; Fax: +1-913-588-1305

Received: 1 May 2020; Accepted: 27 June 2020; Published: 2 July 2020

Abstract: Autism spectrum disorder (ASD) consists of a genetically heterogenous group of
neurobehavioral disorders characterized by impairment in three behavioral domains including
communication, social interaction, and stereotypic repetitive behaviors. ASD aﬀects more than 1%
of children in Western societies, with diagnoses on the rise due to improved recognition, screening,
clinical assessment, and diagnostic testing. We reviewed the role of genetic and metabolic factors
which contribute to the causation of ASD with the use of new genetic technology. Up to 40 percent of
individuals with ASD are now diagnosed with genetic syndromes or have chromosomal abnormalities
including small DNA deletions or duplications, single gene conditions, or gene variants and metabolic
disturbances with mitochondrial dysfunction. Although the heritability estimate for ASD is between
70 and 90%, there is a lower molecular diagnostic yield than anticipated. A likely explanation may
relate to multifactorial causation with etiological heterogeneity and hundreds of genes involved
with a complex interplay between inheritance and environmental factors inﬂuenced by epigenetics
and capabilities to identify causative genes and their variants for ASD. Behavioral and psychiatric
correlates, diagnosis and genetic evaluation with testing are discussed along with psychiatric
treatment approaches and pharmacogenetics for selection of medication to treat challenging behaviors
or comorbidities commonly seen in ASD. We emphasize prioritizing treatment based on targeted
symptoms for individuals with ASD, as treatment will vary from patient to patient based on diagnosis,
comorbidities, causation, and symptom severity.

Keywords: autism; ASD; genetics; heterogeneity; syndromes; assessment; medications; treatment; causes

1. Introduction

Leo Kanner in 1943 [1] ﬁrst introduced the term autism as a diagnostic label to deﬁne a speciﬁc
syndrome observed in young children manifested by early onset, characteristic symptomatology,
and disrupted social and emotional relationships. Since then, autism is now recognized as Autism
Spectrum Disorder (ASD), which is classiﬁed as a developmental disorder as deﬁned in DSM-5
(Diagnostic and Statistical Manual of Mental Disorders, 5th Edition) by the American Psychiatric
Association [2] and the ICD-10 (International Classiﬁcation of Diseases, 10th Revision) by the World
Health Organization [3]. Autism is characterized by signiﬁcant impairment in social communication
and atypical repetitive and/or restrictive behaviors or interests, with an onset in the early developmental
period, prior to age 3 years. The American Academy of Pediatrics [4] recommends screening all infants
and toddlers to identify early signs of autism at 18 months and again at 24 months of age. Rating or
assessment scales that have been validated for both clinical and research purposes are helpful in
establishing the diagnosis of autism. These scales include the Autism Diagnostic Interview-Revised
(ADI-R) and the Autism Diagnostic Observation Schedule, Second Edition (ADOS-2) and should be

Int. J. Mol. Sci. 2020, 21, 4726; doi:10.3390/ijms21134726

www.mdpi.com/journal/ijms

 International Journal of Molecular Sciences(cid:1)(cid:2)(cid:3)(cid:1)(cid:4)(cid:5)(cid:6)(cid:7)(cid:8)(cid:1)(cid:1)(cid:2)(cid:3)(cid:4)(cid:5)(cid:6)(cid:7)Int. J. Mol. Sci. 2020, 21, 4726

2 of 18

administered by trained specialists in conjunction with an evaluation of the child with consideration of
history and clinical presentation [5,6].

ASD aﬀects between 1 to 2% of children in United States with a growing role for genetic factors
with etiological heterogeneity. ASD can be conceptualized as a behavioral syndrome rather than a
speciﬁc categorical mental disorder [7]. The concept of “syndromic autism” (ASD associated with
morphological signs or symptoms helpful in the identiﬁcation of speciﬁc genetic disorders) stands
in contrast to “non-syndromic autism” (idiopathic ASD with no associated signs or symptoms).
Multiplex autism refers to those with a positive family history of other similarly aﬀected individuals,
which highlights the heterogeneity of ASD [8,9].

Clinical and other health concerns that may be associated with ASD include intellectual disability
(ID), electroencephalogram (EEG) abnormalities with or without epilepsy, dysmorphic features,
and abnormal MRI ﬁndings [10,11]. About 10% of children with autism are reported to have
microcephaly [12,13], which may be associated with additional ﬁndings and a poor prognosis. On the
other hand, a large-appearing head size is common in children with autism along with increased brain
volumes, particularly in the frontal lobes, but with smaller occipital lobes [14–19]. Mutations of the
phosphatase and tensin homolog (PTEN) tumor suppressor gene were reported by Butler et al. [14] in
children with autism and extreme macrocephaly. Recent studies have shown that about 20% of genes
implicated in autism are also known cancer genes, thereby stimulating an interest in not only risks for
cancer development in individuals with ASD but whether chemotherapeutic agents could play a role
in treatment of autism [20].

Tordjman et al. [7] provided a comprehensive review of diverse genetic disorders associated
with autism and considered possible common underlying mechanisms leading to a similar
cognitive-behavioral phenotype of autism, while examining relevant genetics, syndromes, epigenetics,
and environmental factors. Despite the recognition of nearly 800 susceptibility, clinically relevant,
or known genes for autism spectrum disorder collated by Butler et al. [21] and characterized by
numerous etiological studies including relevant animal models [22], it appears that no cohesive model
of causation, biomarker [23], or speciﬁc mode of transmission for the development of autism has been
ﬁrmly identiﬁed [24].

The cause of ASD is heterogenous involving genetics with multiple diﬀerent gene variants
and environmental inﬂuences triggering physiological changes in genetically sensitive individuals
along with in utero and metabolic factors including mitochondria dysfunction reported in 10 to 20%
of patients with ASD. Familial and heritability studies have shown that genetic factors contribute,
with estimates as high as 90% with tuberous sclerosis, fragile X, and Rett syndromes as examples of
single gene conditions found but accounting for less than 10% of all ASD cases [25–27]. A list of genetic
syndromes and chromosomal disorders associated with ASD is illustrated as Box 1 below.

Behavioral and psychiatric comorbidities are common in individuals on the autism spectrum,
and can have a substantial impact on overall health, quality of life, and long-term prognosis.
Approximately 30% of individuals with ASD require psychological and psychiatric treatments including
medication for behavioral problems including hyperactivity, impulsivity, inattention, aggression,
property destruction, self-injury, mood disorders, and psychotic or tic disorders [28,29], a major focus
of our report.

Int. J. Mol. Sci. 2020, 21, 4726

3 of 18

Box 1. A List of Genetic Syndromes and Chromosome Findings where Autism is a Recognized Feature.

Adenylate succinase deﬁciency

• Myotonic dystrophy

Angelman and Prader-Willi syndromes (AS–maternal or
PWS–paternal 15q11-q13 deletions)

Apert syndrome

15q11.2 BP1-BP2 microdeletion (Burnside-Butler) syndrome

CHARGE syndrome

Chromosome 15 duplications (maternal origin)

Chromosome 16p11.2 deletions

Cohen syndrome

De Lange syndrome

Down syndrome

Duchenne muscular dystrophy

Fragile X syndrome (FMR1 gene)

Hypomelanosis of Ito

Joubert syndrome

•

•

•

•

•

•

•
•

•

•

•

•

•

•

Neuroﬁbromatosis

Noonan syndrome

Oculo-auriculo-vertebral spectrum

Phelan-McDermid syndrome (22q13 deletion)

PTEN gene associated disorders with extreme
macrocephaly
(Cowden/Bannayan-Riley-Ruvalcaba syndrome)

Rett syndrome (MECP2 gene)
Shprinzten/velo-cardio-facial/DiGeorge
(22q11 deletion)

Smith-Lemli-Opitz syndrome

Smith-Magenis syndrome (17p11.2 deletion)

Sotos syndrome

Tuberous sclerosis

Turner syndrome

Untreated or poorly treated phenylketonuria (PKU)

•

•

•

•

•

•

•

•

•

•

•

•

•

•

• Mitochondrial dysfunction
• Moebius sequence

• Williams syndrome

Modiﬁed from G.B. Schaefer and N.J. Mendelsohn, “Genetics evaluation for the etiologic diagnosis of autism spectrum disorders,”
Genetics in Medicine, vol. 10, pp 4–12, 2008 [25] and from M.G. Butler and others, “Assessment and treatment in autism spectrum
disorders: A focus on genetics and psychiatry”, Autism Research and Treatment, vol. 2012, 242537, 2012 [30].

2. Diagnosis and Genetics of ASD

ASD aﬀects about 1 individual in 50–100 live births [31,32] and is on the increase with a higher
prevalence than reported for congenital brain malformations or Down syndrome. The recurrence rate
may be as high as 25–30% if a second child is also diagnosed with ASD in a family (i.e., multiplex)
compared with a sporadic pattern (simplex) form of ASD. High heritability estimates have been
reported in ASD, e.g., 70 to 90% concordance rate in monozygotic twins [33,34], indicating the potential
importance of genetics, but studies have not identiﬁed the anticipated number of pathogenic variants
to date. Those without a family history may be at a greater risk of copy number variants (CNVs) or
deletions/duplications at the chromosome level using chromosomal microarray analysis and DNA
probes for CNVs and comparative genomic hybridization [35]. Furthermore, 10% of the individuals
with autism from simplex families had CNVs, while only 3% of individuals with autism from multiplex
families with more than one family member aﬀected showed CNVs, compared with 1% seen in normally
developing children studied as controls. The majority of CNVs were of the deletion type. Single gene
conditions are found in about 20% of subjects with ASD, while epigenetics impacted by environmental
factors such as nutrition, infections, or toxins could alter the gene status through methylation, controlling
function without changing the DNA sequence [27,36,37]. Genome-wide linkage and association studies
(GWAS) have identiﬁed hundreds of ASD risk gene loci in all human chromosomes.

Autism is considered the most heritable neurodevelopmental disorder based on a large diﬀerence in
concordance rates or heritability estimates between monozygotic and dizygotic twins with monozygotic
twins having rates that are nearly three times higher than rates found in dizygotic twins [38].
Furthermore, a meta-analysis of twin studies on the heritability of ASD in more than 6000 twin pairs
was reported by Tick et al. [39]. They found that correlations for monozygotic twins were very close
to perfect at a score of 0.98 while the score for dizygotic twins was 0.53, indicating a role of shared
environmental eﬀects. Hallmayer et al. [38] concluded that susceptibility to ASD showed moderate
genetic heritability and substantially shared twin environmental components, indicating a challenge to
ﬁnd genetic causation for autism.

Int. J. Mol. Sci. 2020, 21, 4726

4 of 18

Genetic investigations have identiﬁed the role of hundreds of gene variants, but risk eﬀects are
highly variable and relate to other conditions besides autism, making it diﬃcult to ﬁnd ASD-speciﬁc
gene variants [40–42]. Many gene variants do impact on common biological pathways or interactions
and may play a potential causative role in autism, but more research is needed to address the current
challenges in translating autism genetics into clinical practice as genetic etiology and pathogenesis of
ASD remain largely unclear [43–45].

Further advances made in genetic technology and testing with improved DNA sequencing
and development of bioinformatics with searchable computer genetic variant databases have led to
discoveries and characterization of genetic defects in the potential causation of ASD. Improvements
of chromosome microarray technology with combination of probes for both copy number variants
and single nucleotide polymorphisms (SNPs) have not only led to enhanced testing capabilities
in identifying segmental deletions and duplications in the genome, but also the identiﬁcation of
pathogenic or disease-causing genes and their positions within chromosomal regions.

2.1. Genetic Factors Contributing to Autism

Advances in genetic testing and evaluation for syndromic causation of patients with ASD have
identiﬁed an etiology in up to 40%, using a three-tier clinical genetic approach described by Schaefer
and others in 2008 [25] and later in 2013 [34] to identify causes in children diagnosed with ASD.
These include fragile X, Rett, and other genetic syndromes, such as tuberous sclerosis (10–20%),
PTEN gene mutations (3%), and structural chromosomal deletions or duplications using early versions
of chromosomal microarrays (3%), and an additional 10% or higher when using high-resolution
microarray technology. Metabolic disorders such as mitochondrial dysfunctions are seen in 10 to 20%
of patients with ASD [32,34,46]. Children with ASD reported with microdeletions or duplications
involve chromosome regions 1q24.2, 2q37.3, 3p26.2, 4q34.2, 6q24.3, 7q35, 13q13.2-q22, 15q11-q13, 15q22,
16p11.2, 17p11.2, 22q11, 2q13, and Xp22 [13] and additional cytogenetic disorders associated with ASD
are found with new ultra-high-resolution microarray technology (e.g., 15q11.2 BP1-BP2 deletions) [47].
Recent GWAS ﬁndings in ASD and broad autism phenotype in 28 extended pedigrees from Canada
and the United States showed additional chromosome regions including 1p36.22, 2p13.1, 6q27, 8q24.22,
9p21.3, 9q31.2, 12p13.31, 16p13.2, and 18q21.1 [48].

These newer chromosomal SNP microarrays can identify abnormalities 100 times smaller than
can be seen with high-resolution chromosome methods including for ASD candidate genes. A report
by Shen et al. [26] on 933 patients with ASD using standard karyotype analysis, fragile X DNA testing,
and chromosomal microarrays found abnormal karyotypes in 2.2%, abnormal fragile X testing in 0.5%,
and microdeletions or microduplications in 18.2% of subjects. These included recurrent deletions or
duplications of chromosome 16p11.2 [49] and for chromosome 15q13.2q-13.3, while new studies found
chromosome 7q11, chromosome 15q11.2 BP1-BP2, and chromosome 22q11.2 [50].

Whole-exome sequencing (WES) have identiﬁed yields of up to 30% [51] but other studies show
lower results (e.g., 9.3%) in individuals with ASD [52]. The vast majority of gene variants are of
uncertain clinical signiﬁcance due in part to the rarity found in genomic normative datasets and
limitations of bioinformatics, evolutionary conservation, computational predictions, and relevance in
relationship to the normal population. Likely explanations for the lack of consistency among molecular
diagnostic testing results may relate to multifactorial causation of ASD inﬂuenced by a complex
interplay between inheritance and environmental eﬀects along with contributions by epigenetics on
gene expression. Despite considerable interest in identifying autism-speciﬁc genes, deleterious variants
have been implicated across multiple neurodevelopmental and psychiatric disorders but insuﬃcient to
date in identifying those genes that, when mutated, confer a largely ASD-speciﬁc risk [42].

An early genome-wide association study (GWAS) on 4300 aﬀected children with ASD reported
by Wang et al. [53] and 6500 controls of European ancestry found a strong association with six
single nucleotide polymorphisms (SNPs) located between cadherin 10 (CDH10) and cadherin
9 (CDH9) genes located on chromosome 5 encoding neuronal cell-adhesion molecules. Since then,

Int. J. Mol. Sci. 2020, 21, 4726

5 of 18

over 100 genetic loci have been reported to be associated with ASD [54,55], comprising genes converging
on chromatin–remodeling, synaptic function in neuronal signaling, and neurodevelopment [56,57].
Furthermore, Butler et al. [21] collated about 800 genes from the literature that were implicated as
clinically relevant, susceptible, or known in ASD. These multiple genes include several members
of the neuroligin, neurexin, GABA receptor, cadherin, and SHANK gene families. Other genes
were found to code for neurotransmitters and their receptors, transporters, oncogenes, brain-derived
hormones, epigenetics, and signaling and ubiquitin pathway proteins, along with neuronal cell-adhesion
molecules [21,58–61].

2.2. Metabolic Factors Contributing to Autism

Metabolic factors are now recognized as contributing to autism including the mitochondria.
Next-generation DNA sequencing now allows for accurate detection of mutations or gene variants at the
nuclear and mitochondrial DNA (mtDNA) level and is potentially more informative than chromosomal
microarray analysis involved in structural DNA changes. This technology is now available in the
clinical setting for individuals presenting with biochemical and mitochondrial disturbances and
autism [21,62]. Three functional pathways to ASD are potentially involved, which include genes and
pathways for chromatin remodeling, (e.g., CHD7, MECP2, DNMT3A, and PHF2), Wnt (e.g., CHD8,
PAX5, and ATRX), and other signaling super-pathways (e.g., GPCR, ERK, RET, and AKT) [50,51] and
mitochondrial dysfunction in ASD (e.g., [62]). High lactate levels are also reported in about one in ﬁve
children with ASD, further supporting the role of the mitochondria in energy metabolism and brain
development [32,46,62].

The mitochondria are intracellular organelles found in the cytoplasm which play a crucial role
in adenosine 5’-triphosphate (ATP) production through oxidative phosphorylation [62–66], the latter
process carried out by the electron transport chain made up of Complexes I, II, III, and IV situated
in the inner membrane of the mitochondria containing about 100 proteins. Genes that encode the
proteins are located in both nuclear and mitochondrial DNA [65–67] and are required for cellular energy
that can impact or inﬂuence brain development and activity. There are hundreds of nuclear genes
involved in mitochondrial function, while only 13 mitochondrial genes code for protein. Mitochondrial
disturbances include a depletion type, or reduced number of mitochondria per cell, with a decreased
quantity of mtDNA, or mtDNA mutations producing defects in biochemical reactions within the
mitochondria and individual cells [32,46,65,68].

A subset of individuals with ASD can have small mitochondrial DNA deletions/duplications
detectable with mitochondrial genome microarrays. Human mitochondrial DNA (mtDNA) is a circular,
double-stranded DNA molecule contained within the mitochondrion and inherited solely from the
mother. Each mitochondrion contains 2–10 mtDNA copies. In humans, 100–10,000 separate copies of
mtDNA are usually present per cell [63–65]. Inborn errors of metabolism may contribute signiﬁcantly
to the causation of ASD with enzyme deﬁciencies leading to an accumulation of substances that
can cause toxic eﬀects on the developing brain. A common example is phenylketonuria, leading to
excessive phenylalanine levels, intellectual disability, and ASD, if not diet controlled.

3. Clinical Assessment and Testing

3.1. Initial Clinical Evaluation

A healthcare professional interviews the parent or caregiver regarding presenting problems,
reviews a three-generation family history, developmental milestones, and abnormal behaviors of the
child, medical and surgical history, and any past or current treatments. The diagnostic evaluation is
typically performed by a developmental pediatrician or a child and adolescent psychiatrist. Physical
and mental status examinations are performed, and additional testing ordered, as appropriate. If a
positive family history for autism is found or dysmorphic (syndromic) features, then a referral is
made for clinical genetics’ evaluation. Laboratory evaluations may include genetic testing, lead levels,

Int. J. Mol. Sci. 2020, 21, 4726

6 of 18

thyroid function, lactate, pyruvate and cholesterol levels, and urine for organic acids. Referrals are
made for neurological evaluations and brain imaging when clinically indicated.

3.2. High-Resolution Microarrays and ASD

Genetic testing often begins with chromosomal microarray analysis (CMA) to identify copy
number variants (CNVs) to search for a cause of autism spectrum disorder and other related conditions.
Microarrays employ a variety of designs and range of coverage of genomic regions, which increases the
diagnostic yield as arrays have evolved over time to include better coverage and accuracy. Often the
CNVs identiﬁed are unclassiﬁed or poorly understood in their role in causation of ASD.

Neurodevelopmental disorders can cumulatively aﬀect up to 15% of children [69]. While the
etiology of ASD is complex, it involves genetic factors with 800 genes recognized, accounting for 4% of
all human genes that are implicated in ASD [21]. Single gene changes, large genomic structural changes
(i.e., deletions or duplications), or smaller CNVs and other polygenic conditions can be inﬂuenced by
the environment and epigenetic factors [70,71]. Genetic testing to pinpoint the underlying cause of
ASD is critical for clinical management and counseling. Further, chromosomal microarray analysis has
demonstrated the highest diagnostic yield for individuals with ASD as compared to other genetic tests
as well as in individuals with ID and/ or behavioral problems, including in developing countries.

High-resolution microarrays now utilize millions of single nucleotide polymorphisms (SNPs)
as probes to test the DNA from patients presenting with neurodevelopmental disorders, intellectual
disabilities, and ASD. These SNP microarrays are used to identify microdeletions (or duplications) with
recognition of dozens of a growing list of deletion or duplication syndromes not previously detected.
For example, a study of custom-made, ultra-high-resolution microarrays reported by Ho et al. [47]
in 2016 were optimized for the detection of neurodevelopmental disorders (Lineagen, Salt Lake City,
Utah) on 10,351 patients presenting for genetic services for neurodevelopmental disorders, ASD, ID,
behavioral problems, or with or without multiple congenital anomalies (MCA) over a period of four
years. Their testing sample had a male:female ratio of 2.5:1 with a mean age of 7 years. Fifty-ﬁve percent
of cases represented patients with a diagnosis of ASD with or without other features. The overall CNV
detection rate of 28.1% was seen in 10,351 consecutive patients and 24.4% in those with ASD along with
33% in those with intellectual disabilities and/or MCA without autism. The rate of pathogenic ﬁndings
was signiﬁcantly lower (4.4%) when the diagnostic indication was ASD only compared to diagnostic
indications of DD/ID/MCA without a reported diagnosis of ASD (i.e., non-ASD cohort) (12.5%).

In the ASD cohort, the overall pathogenic rate was slightly higher for individuals with ASD+
as compared to the overall pathogenic rate for individuals with ASD only. The pathogenic rate in
the ASD+ cohort started at 4.1% in the youngest group and rose to 8.5% in the 5.5–10 years range.
The pathogenic rate in the ASD only cohort rose gradually with age, from 3.4% in the youngest cohort
(0–3.4 years) to a peak at 7.0% in adolescence. In 5694 patients classiﬁed as ASD and 4657 patients
with non-ASD, the most common ﬁndings were 15q11. 2 BP1-BP2 deletions followed by proximal
16p11.2 deletions or duplications, 15q13.3 deletions, and 16p13.1 duplications (see Figure 1). The most
common ﬁnding in the non-ASD cohort was the 22q11.2 deletion causing velo-cardio-facial or DiGeorge
syndrome. This study illustrates the value of CMA testing and its impact on medical management is
now recognized in consensus medical guidelines for the evaluation of children with ASD.

3.3. Next-Generation Sequencing (NGS)

Advances in genomics technology using next-generation sequencing (NGS) have led to discovery
of many disease-causing genes using candidate gene approaches, disease-speciﬁc gene panels, or by
whole-exome sequencing of patients presenting with neurodevelopmental disorders, intellectual
disabilities, or ASD. Applying genomics to the study of neurodevelopment and function has identiﬁed
over 5000 implicated genes using clinical exome sequencing approaches and informatics in aﬀected
individuals. In addition, disease-speciﬁc NGS gene testing panels have been developed and used
in the commercial laboratory setting for testing patients presenting for genetic services, including

Int. J. Mol. Sci. 2020, 21, 4726

7 of 18

approximately 600 genes for intellectual disabilities and over 100 genes available for testing for ASD
(e.g., Fulgent Diagnostics, Irvine, California).

Figure 1. Pie chart showing the top 10 out of 85 genetic ﬁndings from data summarized by Ho et al. [47]
using ultra-high-resolution chromosomal microarrays from over 10,000 consecutive patients presenting
for genetic testing with neurodevelopmental disorders aﬀecting brain function and/or structure of
unknown cause with developmental/intellectual disabilities and/or ASD.

These types of analyses have identiﬁed pathogenic gene variants or mutations which are known
to be disease-causing such as missense or nonsense, but more often variants of unknown clinical
signiﬁcance are found. More testing and information with better interpretations of the genomic change
and impact at the protein level are needed to help determine the role, if any, of the unknown gene
variants in causing the disease under study including for ASD. Hundreds of new causative genes
relating to human diseases and syndromes have been identiﬁed with the use of NGS technology
over the past few years, with expectations of continued success given improvements in genetic
technology, bioinformatics, and expanded genomic databases to search for gene variants and in further
characterizing identiﬁed genes.

Next generation DNA sequencing of the exons (referred to as exome sequencing) or whole-genome
sequencing will continue to ﬁnd new discoveries of disease-causing SNPs, gene regulatory sequences,
or mutations of protein-coding genes for both structural and regulatory proteins. Identifying molecular
signatures of novel or disturbed gene or exon expression, disease-speciﬁc proﬁles and patterns
(i.e., expression heat maps), and recognition of interconnected gene pathways in autism and other
behavioral disorders in the future by using readily available blood elements (e.g., lymphoblasts) should
hold promise for treatments with pharmacological agents by regulating (either increasing or decreasing)
activity of normal (or abnormal) gene function. The study of non-coding RNAs, which control the
amount or quantity of gene expression coding for protein production through micro-RNAs and
the quality of protein production by speciﬁc isoform development by sno-RNAs, will lead to new
areas of research and medical therapies for human diseases. Therefore, this technology should be
considered in the diagnostic evaluation of ASD, either sporadic or with a positive family history of
others similarly aﬀected.

Int. J. Mol. Sci. 2020, 21, x FOR PEER REVIEW 7 of 18  ASD. Figure 1. Pie chart showing the top 10 out of 85 genetic findings from data summarized by Ho et al. [47] using ultra-high-resolution chromosomal microarrays from over 10,000 consecutive patients presenting for genetic testing with neurodevelopmental disorders affecting brain function and/or structure of unknown cause with developmental/intellectual disabilities and/or ASD.  3.3. Next-Generation Sequencing (NGS) Advances in genomics technology using next-generation sequencing (NGS) have led to discovery of many disease-causing genes using candidate gene approaches, disease-specific gene panels, or by whole-exome sequencing of patients presenting with neurodevelopmental disorders, intellectual disabilities, or ASD. Applying genomics to the study of neurodevelopment and function has identified over 5000 implicated genes using clinical exome sequencing approaches and informatics in affected individuals. In addition, disease-specific NGS gene testing panels have been developed and used in the commercial laboratory setting for testing patients presenting for genetic services, including approximately 600 genes for intellectual disabilities and over 100 genes available for testing for ASD (e.g., Fulgent Diagnostics, Irvine, California). These types of analyses have identified pathogenic gene variants or mutations which are known to be disease-causing such as missense or nonsense, but more often variants of unknown clinical significance are found. More testing and information with better interpretations of the genomic change and impact at the protein level are needed to help determine the role, if any, of the unknown gene variants in causing the disease under study including for ASD. Hundreds of new causative genes relating to human diseases and syndromes have been identified with the use of NGS technology over the past few years, with expectations of continued success given improvements in genetic technology, bioinformatics, and expanded genomic databases to search for gene variants and in further characterizing identified genes. Next generation DNA sequencing of the exons (referred to as exome sequencing) or whole-genome sequencing will continue to find new discoveries of disease-causing SNPs, gene regulatory sequences, or mutations of protein-coding genes for both structural and regulatory proteins. Identifying molecular signatures of novel or disturbed gene or exon expression, disease-specific Int. J. Mol. Sci. 2020, 21, 4726

8 of 18

Butler et al. [21] searched the literature and found approximately 800 genes implicated in autism
in the literature as clinically signiﬁcant, relevant, or known to contribute to the risk of ASD. Recent
research revealed that ASD and cancer genes may share common genetic architecture and pathways
with the ﬁrst evidence of the PTEN tumor-suppressor gene playing a role in autism in 2005 [14].
Hence, approximately 800 ASD-related genes and 3500 genes in cancer were examined using the
GeneAnalytics pathways and proﬁling software programs and found shared cell-signaling pathways,
metabolic disturbances, and molecular functions in 138, or 17%, of ASD genes that overlap with cancer
genes [20]. Shared mechanisms may lead to identiﬁcation of common pathology and a better molecular
understanding of causation as well as potential treatment options.

4. Treatment Approaches

4.1. Behavioral Interventions in ASD

4.1.1. For Children and Adolescents with ASD

Weitlauf et al. [72] reviewed 65 studies, comprising 48 randomized trials and 17 nonrandomized
comparative studies, that analyzed the beneﬁt of behavioral interventions. High-intensity applied
behavior analysis (ABA) was associated with improvement in cognitive functioning and language skills
relative to community controls in young children [73]. Early intensive behavioral intervention (EIBI) is
a well-established treatment for young children with ASD and is based on the principles of applied
behavior analysis. Delivered over a period of several years at an average of 20 to 40 hours per week,
it can provide substantial beneﬁt for core ASD symptoms, particularly in terms of communication
skills [74]. Social skills’ interventions including group administered training showed positive eﬀects
on social behaviors for older children [75].

4.1.2. For Adults with ASD

National Institute for Health and Care Excellence (NICE) recommended guidelines for management
and support of children and young people with autism using group or individual social learning
programs to improve social interaction deﬁcits by applying behavioral therapy techniques within a social
learning framework. These include using video modeling, peer feedback, imitation, and reinforcement
to teach conventions of appropriate social interpersonal interaction [76]. There is evidence from
observational studies in adults with ASD that social skills’ groups may be eﬀective at improving
social interaction [77]. CBT can help adults with ASD across a range of domains, particularly in the
context of treating anxiety and OCD, and supporting adults who have diﬃculties related to a history
of victimization [78].

4.2. Medication Treatments in ASD

Psychopharmacological treatment of ASD is challenging due to considerable variability in the
presentation of ASD and commonly occurring comorbidities. Individuals with ASD are typically more
vulnerable to side eﬀects of psychopharmacological agents than their age-matched, neuro-typically
developing peers [79]. Finally, ASD impacts individuals over the course of their lifespan and most of
the literature on psychotropic medications in ASD involves pediatric populations.

A psychopharmacological approach may be beneﬁcial in the treatment of identiﬁed target
symptoms in individuals with ASD. When considering the use of medications, potential beneﬁts and
risks must be weighed on a case-by-case basis. It has been reported that close to half of insured children
with ASD are receiving psychopharmacological interventions, most commonly with stimulants, alpha-2
agonists, antipsychotics, anticonvulsants, and antidepressants [80].

Currently, there are no medications approved for treatment of the core symptoms of ASD
including social communication deﬁcits or repetitive behaviors. Common target symptoms for
which there are eﬀective, evidence-based medication treatment include hyperactivity, inattention,

Int. J. Mol. Sci. 2020, 21, 4726

9 of 18

impulsivity, irritability, aggression, self-injurious behavior, repetitive behaviors (including stereotypies),
and insomnia [81]. For the treatment of irritability associated with ASD, the antipsychotics risperidone
and aripiprazole are licensed and approved by the US Food and Drug Administration [82].

4.2.1. For the Treatment of ADHD Symptoms in ASD

Stimulant medications are considered ﬁrst line agents for attention deﬁcit hyperactivity disorder
(ADHD) in individuals with ASD, given that overall they are most often eﬀective and generally
well tolerated compared to other ADHD medications. The RUPP research team [83] and later
Reichow et al. [84] demonstrated a clear superiority of methylphenidate over placebo in children
with pervasive developmental disorder. However, those with ASD had a greater risk of side eﬀects
with methylphenidate including decreased appetite, insomnia, depressive symptoms, irritability,
higher levels of social withdrawal, and lower treatment response rates compared to youth with ADHD
alone. It should be noted that stimulant medications for ADHD in the amphetamine class are often
used in children with ASD but have not been as rigorously studied.

Regarding non-stimulant medications for ADHD in ASD, both atomoxetine and alpha-2 agonists
have shown beneﬁt. Harfterkamp et al. [85], in a double-blind treatment trial of patients age 6
to 17 years with ADHD and ASD using atomoxetine 1.2 mg/kg/day or placebo for eight weeks,
found that atomoxetine moderately improved ADHD symptoms, but with frequent adverse events
including nausea, decreased appetite, fatigue, and early morning awakening. The alpha-2 agonist
guanfacine has been shown to be eﬀective for ADHD in children with ASD demonstrated by a
double-blind, placebo-controlled trial of guanfacine extended release in which 50% of youth on active
treatment improved on the Clinical Global Impression–Improvement (CGI-I) scale [86], compared to
9.4% on placebo [87], with sedation and transient lowering of blood pressure as the most common
adverse eﬀects.

4.2.2. For the Treatment of Irritability, Aggression, and Self-Injurious Behavior in ASD

Atypical antipsychotics compared to other medications have to date demonstrated the best
evidence for the treatment of irritability in ASD. Risperidone in youth age 5 to 16 years with ASD [88]
in three randomized, placebo-controlled trials showed an over 50% reduction in the irritability score of
the Aberrant Behavior Checklist (ABC-I) irritability scale [89] and the magnitude of the response was
greater when irritability was rated as moderate to severe [90].

Aripiprazole, the second antipsychotic approved by the FDA for the treatment of irritability
associated with autism (in children between the age of 6 and 17 years), demonstrated signiﬁcantly
lower severity scores on the ABC-I and the CGI-I scales for subjects on active medication compared to
placebo in two large-scale, randomized, placebo-controlled studies. Unfortunately, weight gain is a
common side eﬀect of antipsychotics, and increases in body mass index have been shown to be similar
for aripiprazole and risperidone in children with ASD [91].

Anticonvulsant medications divalproex and topiramate have shown some promise for treating
irritability in ASD. Divalproex was beneﬁcial in reducing irritability in a small, randomized,
placebo-controlled trial of children with ASD [92]. However, an earlier trial failed to show separation
from placebo on the ABC-I [93]. Topiramate as monotherapy has no demonstrated beneﬁt in
the treatment of irritability in youth with ASD [94]; however, it reduced the ABC-I score when
co-administered at an average daily dose of 200 mg with risperidone [95]. It is hypothesized that,
as EEG abnormalities are common in children with ASD, symptom reduction with anticonvulsants
may result from treatment of abnormal brain discharges [96].

4.2.3. For the Treatment of Repetitive Behaviors Including Stereotypies in ASD

Fluoxetine has been shown to improve repetitive behaviours in adults with ASD [97]; however,
beneﬁt has not been reliably demonstrated in pediatric populations. In fact, the Cochrane Collaboration
published a systematic review [98], which concluded that for repetitive behaviors in children with ASD

Int. J. Mol. Sci. 2020, 21, 4726

10 of 18

there is not only a lack of available evidence of beneﬁt from treatment with selective serotonin reuptake
inhibitors (SSRIs) including ﬂuoxetine, ﬂuvoxamine, and citalopram, but some evidence for risk of
harm, given a greater incidence of adverse eﬀects, most notably symptoms of behavioral activation.

4.2.4. For the Treatment of Persistent Insomnia in ASD

Exogenous melatonin, (available as an over-the-counter supplement) in both immediate-release
and extended-release formulations, has been shown to be safe and eﬀective in improving sleep patterns
in children with ASD [99]. Some evidence suggests that children with ASD have abnormal melatonin
secretion and circadian rhythm abnormalities compared to non-ASD children [100]. Clonidine
(an alpha-2 agonist) has shown promise in reducing latency of sleep initiation and decreasing nighttime
awakening in ASD [101].

4.3. Pharmacogenetics and Role in Medication Selection and Management

Personalized or precision medicine is emerging in clinical practice based on individual genetic
patterns contributing to pharmacogenetics, particularly in the ﬁeld of psychiatry and treating individuals
with ASD [102–105] with behavior issues including ADHD, irritability, aggression and self-injury,
repetitive behaviors, and persistent insomnia addressed above. Pharmacogenetics is a study of
structural DNA variation that impacts drug metabolism [106,107] and most often based on the
cytochrome P450 enzyme system, primarily active in the liver and coded by genes. Cytochrome P450
enzymes metabolize or break down drugs in the liver with most prescription drugs metabolized by
this enzyme system and, thus, play a signiﬁcant role in the treatment of diseases [107,108]. Variation
in drug response among individuals due to metabolism diﬀerences is now recognized as a major
clinical problem, especially given that the use of several medications per patient is common practice.
Relevant cytochrome P450 gene polymorphisms and diﬀerent racial distributions can identify sources
of variability in drug response by the modulation of metabolism by the cytochrome P450 enzymes
impacting treatment in ASD.

There are over 50 cytochrome P450 hepatic enzymes that are primarily found in the
mitochondria [102–105]. These enzymes metabolize endogenous and xenobiotic substrates including
environmental pollutants and agricultural and plant-based chemicals and are involved in biosynthesis
and metabolism of steroids, vitamins, hormones, lipids, and prostaglandins. About 90% of all drugs are
metabolized by seven diﬀerent cytochrome enzymes including CYP1A2, CYP3A4, CYP3A5, CYPC19,
CYP2D6, CYP2C9 and CYP2B6 [106,109–111]. The most commonly prescribed medications used in
treating patients with psychiatric problems and ASD are broken down by CYP2D6 [102–105]. It should
also be noted that many drugs are also metabolized by more than one cytochrome P450 enzyme and
in addition some drugs (e.g., risperidone) require break down to generate an active metabolite or
functional agent for treatment.

There is growing evidence that cytochrome P450 enzymes may be altered by the environment in
the form of inhibitors or inducers as well as impacting drug–drug interactions. Known inhibitors or
inducers may include common sources such as caﬀeine, grapefruit, broccoli, cabbage, or cauliﬂower
by impacting the individual enzyme activity. For example, if an individual has a reduced form of a
cytochrome P450 enzyme, then an inducer may increase the enzyme response in breaking down the
drug to help that person in metabolizing a speciﬁc medication and, thus, impact response to treatment.
Drug–drug interactions and their concentrations and half-life along with response to inhibitors
and/or inducers can all impact medication levels and treatment in the patient. It should also be noted
that individuals who are either fast or slow metabolizers based on their microsomal P450 enzyme
system genotype patterns may respond diﬀerently to speciﬁc medications and put them at risk for
either failure of drug therapy and/or adverse side eﬀects. Similarly, a better understanding of the
metabolic diﬀerences that occur with age will further impact on drug dosage and selection of speciﬁc
therapeutic agents. Therefore, personalized medicine requires the development of resources for
clinicians including pharmacogenetic dosing guidelines for medications, as 25 to 50% of individuals

Int. J. Mol. Sci. 2020, 21, 4726

11 of 18

do not respond normally to drug dosage or treatment, and this scenario also applies to those with
ASD [102].

Applying this knowledge from pharmacogenomics and identifying genes and polymorphisms
involved in drug metabolism will beneﬁt patients treated for psychiatric and behavioral problems.
The discovery of new classes of drugs and research on existing drugs for new purposes to treat
behavioral problems in patients with ASD are under investigation including clinical trials (e.g., in fragile
X syndrome), holding promise for improved therapy. In addition, the discoveries made in brain
imaging such as functional MRI or PET scans in identifying regions of the brain that are aﬀected in ASD
should allow for new treatment discoveries and applications speciﬁc for the altered regions identiﬁed.

5. Future Directions

Advances and application of genomic testing technology, bioinformatic approaches,
and computational predictions will strengthen genetic testing results and interpretations as more
experience is gained in testing patients presenting for clinical services and diagnosis [112–114].
Increased number of next-generation sequencing (NGS) or whole-exome sequencing (WES) studies in
ASD of individuals from diﬀerent ethnic backgrounds will be required to gain ASD-speciﬁc genomic
information from datasets of both sexes when compared to the normal population. These contributions
should allow a better understanding of the role of genetics, genomics, epigenetics, and speciﬁc
candidate genes and their variants, along with environmental factors playing a role in ASD in
relationship to multifactorial inﬂuences in family studies [38,112,114,115]. Confounding eﬀects
of clinical heterogeneity and diagnostic uncertainty are other complicating issues needing further
characterization and evaluation to gain more experience in clinical assessment, genetics, and treatment
approaches in autism spectrum disorder.

In addition, research with brain and tissue harvested and stored for structural DNA and RNA
expression studies are needed on individuals with ASD having data from cognitive, behavioral, and ASD
assessment tools and neuroimaging results while living. Coding and non-coding expression patterns
and epigenetic (methylation) signals supplemented with WES and CNV data would be beneﬁcial for a
better understanding of the role of genetics in ASD, particularly with larger cohorts of individuals
having similar genetic backgrounds, patterns, and ethnicity to identify large-eﬀect pathogenic variants
for facilitating genotype–phenotype correlations and allow comparisons. The biological processes,
molecular functions with gene-interactions, and pathways that are more autism-speciﬁc may be
identiﬁed through these analytical genetic studies. Currently, there are no molecular pathways
known to be uniquely associated with ASD when disturbed. Some gene variants are more related
to neurodevelopmental disorders and not speciﬁc for autism. Classiﬁcation of gene variants that
speciﬁcally cause ASD alone and not attributable to other neurodevelopmental or psychiatric disorders
are under investigation as rare, large-eﬀect mutations seen in ASD also inﬂuence cognition in a high
proportion of individuals, complicating the degree of impact on the ASD phenotype vs. impact on
cognitive function [42]. Certain neurodevelopmental gene variants may also impact gene function
diﬀerently including neural circuits depending on an individual’s genetic background diﬀerences.

Particular class of variants such as missense or nonsense changes may confer diﬀerent eﬀects at
the protein level. Individual gene variants coding for speciﬁc amino acids may impact more important
protein regions or domains with certain characteristics at speciﬁc amino acid positions, conferring
mild consequences, while other amino acid positions may be more important for protein function.
These areas of gene variant(s)-protein relationships will require more studies in ASD in the future using
improved genetic technology, data collection, and analysis with genotype–phenotype correlations.

Brain tissue regions most often aﬀected in ASD (e.g., hippocampus, cerebellum, etc.) may yield
useful information if studied in those persons with documented autism, particularly with stored clinical
and imaging data with CNVs and gene variants combined with expression patterns and methylation
status in relationship to control subjects who are similarly studied. Mosaicism/tissue-speciﬁc gene
expression should be considered and further studied in view of more males than females aﬀected with

Int. J. Mol. Sci. 2020, 21, 4726

12 of 18

ASD, particularly X-linked genes. Additionally, hormonal-mediated gender inﬂuences or diﬀerential
expression in the brain should be examined for dysregulation in ASD including methylation status of
brain-expressed genes on the X chromosome and interaction with autosomal genes (e.g., X-linked FMR1
gene causing fragile X syndrome [116] and CYFIP1 gene at 15q11.2 involved with coding transporter
for FMR1 protein [117]). These investigations will require more specialized methods with increased
sensitivity such as droplet digital PCR [118].

6. Summary

On behalf of individuals living with ASD and their families and for the beneﬁt of society as a
whole, increased awareness and knowledge regarding autism spectrum disorder and commonly related
neurobehavioral conditions with contribution of genetic diﬀerences are imperative for healthcare
professionals who provide evaluation and treatment services for ASD. Early recognition, diagnosis,
and treatment should increase the likelihood that aﬀected individuals will achieve optimal long-term
outcomes and improved quality of life. Genetic and epigenetic discoveries underlying causes, as well
as factors impacting treatment, such as pharmacogenetic variability, have the potential to improve
the overall health of individuals with ASD. Additional clinical research to improve the evidence base
for various treatment interventions for ASD with related behavioral and psychiatric challenges is
desperately needed.

Author Contributions: A.G. and M.G.B. designed and contributed to the study equally by reviewing the literature,
writing and revising the manuscript and agreeing to publish the article. M.G.B. generated the ﬁgure. All authors
have read and agreed to the published version of the manuscript.

Funding: This research received no external funding.

Acknowledgments: We thank Grace Graham for expert preparation of the manuscript.

Conﬂicts of Interest: The authors declare no conﬂict of interest.

References

1.
2.

Kanner, L. Autistic disturbances of aﬀective contact. Nervous Child. 1943, 32, 217–253.
American Psychiatric Association. Diagnostic and Statistical Manual of Mental Disorders (DSM-5), 5th ed.;
American Psychiatric Association: Washington, DC, USA, 2013.

3. World Health Organization.

ICD-10: International Statistical Classiﬁcation of Diseases and Related Health

Problems: Tenth Revision, 2nd ed.; World Health Organization: Geneva, Switzerland, 2004.

5.

4. Hyman, S.L.; Levy, S.E.; Myers, S.M. AAP Council on Children with Disabilities, Section on Developmental
and Behavioral Pediatrics. Identiﬁcation, Evaluation, and Management of Children with Autism Spectrum
Disorder. Pediatrics 2020, 145, e20193447. [CrossRef] [PubMed]
Rutter, M.; Le Couteur, A.; Lord, C. ADI-R: Autism Diagnostic Interview-Revised (ADI-R); Western Psychological
Services: Los Angeles, CA, USA, 2003.
Lord, C.; DiLavore, P.C.; Gotham, K.; Guthrie, W.; Luyster, R.J.; Risi, S.; Rutter, M. Autism Diagnostic
Observation Schedule: ADOS-2; Western Psychological Services: Los Angeles, CA, USA, 2012.
Tordjman, S.; Cohen, D.; Anderson, G.M.; Botbol, M.; Banitano, R.; Coulon, N.; Roubertous, P.L. Reprint of
“Reframing autism as a behavioral syndrome and not a speciﬁc mental disorder: Implications of genetic and
phenotypic heterogeneity”. Neurosci. Biobehav. Rev. 2018, 89, 132–150.

6.

7.

8. Miles, J.H.; Takahashi, T.N.; Bagby, S.; Sahota, P.K.; Vaslow, D.F.; Wang, C.H.; Hillman, R.E.; Farmer, J.E.
Essential versus complex autism: Deﬁnition of fundamental prognostic subtypes. Am. J. Med. Genet. 2005,
135, 171–180. [CrossRef] [PubMed]
Cohen, D.; Pichard, N.; Tordjman, S.; Baumann, C.; Burglen, L.; Excoﬃer, E.; Lazar, G.; Mazet, P.; Pinquier, C.;
Verloes, A.; et al. Speciﬁc genetic disorders and autism: Clinical contribution towards their identiﬁcation.
J. Autism Dev. Disord. 2005, 35, 103–116. [CrossRef] [PubMed]

9.

10. Rapin, I. Autistic regression and disintegrative disorder: How important the role of epilepsy?

Semin. Pediatr. Neurol. 1995, 2, 278–285. [CrossRef]

Int. J. Mol. Sci. 2020, 21, 4726

13 of 18

11. Miles, J.H.; Hillman, R.E. Value of a clinical morphology examination in autism. Am. J. Med. Genet. 2000, 10,

12.

245–253. [CrossRef]
Fombonne, E.; Roge, B.; Claverie, J.; Courty, S.; Frémolle, J. Microcephaly and macrocephaly in autism.
J. Autism. Dev. Disord. 1999, 29, 113–119. [CrossRef]

13. Miles, J.H. Autism spectrum disorders—A genetics review. Genet Med. 2011, 13, 278–294. [CrossRef]
14. Butler, M.G.; Dasouki, M.J.; Zhou, X.P.; Talebizadeh, Z.; Brown, M.; Takahashi, T.N.; Miles, J.H.; Wang, C.H.;
Stratton, R.; Pilarski, R.; et al. Subset of individuals with autism spectrum disorders and extreme macrocephaly
associated with germline PTEN tumour suppressor gene mutations. J. Med. Genet. 2005, 42, 318–321. [CrossRef]
15. Carper, R.A.; Courchesne, E. Inverse correlation between frontal lobe and cerebellum sizes in children with

autism. Brain 2000, 123, 836–844. [CrossRef] [PubMed]

16. Carper, R.A.; Moses, P.; Tigue, Z.D.; Courchesne, E. Cerebral lobes in autism: Early hyperplasia and abnormal

age eﬀects. Neuroimage 2002, 16, 1038–1051. [CrossRef] [PubMed]

17. Mueller, S.; Keeser, D.; Reiser, M.F.; Teipel, S.; Meindl, T. Functional and structural MR imaging in
neuropsychiatric disorders, part 2: Application in schizophrenia and autism. AJNR Am. J. Neuroradiol. 2012,
33, 2033–2037. [CrossRef] [PubMed]

18. Philip, R.C.; Daubermann, M.R.; Whalley, H.C.; Baynham, K.; Lawrie, S.M.; Stanﬁeld, A.C. A systematic
review and meta-analysis of the fMRI investigation of autism spectrum disorders. Neurosci. Biobehav. Rev.
2012, 36, 901–942. [CrossRef]

19. Kobayashi, A.; Yokota, S.; Takeuchi, H.; Asano, K.; Asano, M.; Sassa, Y.; Taki, Y.; Kawashima, R. Increased
grey matter volume of the right superior temporal gyrus in healthy children with autistic cognitive style:
A VBM study. Brain Cogn. 2020, 139, 105514. [CrossRef]

20. Gabrielli, A.P.; Manzardo, A.M.; Butler, M.G. GeneAnalytics pathways and proﬁling of shared autism and

cancer genes. Int. J. Mol. Sci. 2019, 20, 1166. [CrossRef]

21. Butler, M.G.; Raﬁ, S.K.; Manzardo, A.M. High-resolution chromosome ideogram representation of currently

recognized genes for Autism spectrum disorders. Int. J. Mol Sci. 2015, 16, 6464–6495. [CrossRef]

22. Tordjman, S.; Drapier, D.; Bonnot, O.; Graignic, R.; Fortes, S.; Cohen, D.; Millet, B.; Laurent, C.; Roubertoux, P.L.
Animal models relevant to schizophrenia and autism: Validity and limitations. Behav. Genet. 2007, 37, 61–67.
[CrossRef]

23. Walsh, P.; Elsabbagh, M.; Bolton, P.; Singh, I. In search of biomarkers for autism: Scientiﬁc, social and ethical

challenges. Nat. Rev. Neurosci. 2011, 2, 603–612. [CrossRef]

24. Happé, F.; Ronald, A.; Plomin, R. Time to give up on a single explanation for Autism. Nat. Neurosci. 2006, 9,

25.

26.

1218–1220. [CrossRef]
Schaefer, G.B.; Mendelsohn, N.J.; Professional Practice Guidelines Committee. Clinical genetics evaluation in
identifying the etiology of autism spectrum disorders. Genet. Med. 2008, 10, 301–305. [CrossRef] [PubMed]
Shen, Y.; Dies, K.A.; Holm, I.A.; Bridgemohan, C.; Sobeih, M.M.; Caronna, E.B.; Miller, K.J.; Frazier, J.A.;
Silverstain, I.; Picker, J.; et al. Austism Consortium Clinical Genetics/DNA Diagnostics Collaboration.
Clinical genetic testing for patients with autism spectrum disorders. Pediatrics 2010, 125, 727–735. [CrossRef]
[PubMed]

27. Waye, M.M.Y.; Cheng, H.Y. Genetics and epigenetics of autism: A review. Psychiatry Clin. Neurosci. 2018, 72,

228–244. [CrossRef] [PubMed]

28. Howes, O.D.; Rogdaki, M.; Findon, J.L.; Wichers, R.H.; Charman, T.; King, B.H.; Loth, E.; McAlonan, G.M.;
McCracker, J.T.; Parr, J.R.; et al. Autism spectrum disorder: Consensus guidelines on assessment, treatment
and research from the British Association for Psychopharmacology. J. Psychopharmacol. 2018, 32, 3–29.
[CrossRef] [PubMed]

29. Rosen, T.E.; Mazefsky, C.A.; Vasa, R.A.; Lerner, M.D. Co-occurring psychiatric conditions in autism spectrum

disorder. Int Rev. Psychiatry. 2018, 30, 40–61. [CrossRef] [PubMed]

30. Butler, M.G.; Youngs, E.L.; Roberts, J.L.; Hellings, J.A. Assessment and treatment in autism spectrum

disorders: A focus on genetics and psychiatry. Autism Res. Treat. 2012, 2012, 242537. [CrossRef]

31. Rice, C. Prevalence of Autism spectrum disorders—Autism and developmental disabilities monitoring

network. Morbid Mortal Wkly. Rep. 2006, 58, 1–20.

32. Rose, S.; Niyazov, D.M.; Rossignol, D.A.; Golenthal, M.; Kahler, S.G.; Frye, R.E. Clinical and molecular
characteristics of mitochondrial dysfunction in autism spectrum disorder. Mol. Diagn. Ther. 2018, 22, 571–593.
[CrossRef]

Int. J. Mol. Sci. 2020, 21, 4726

14 of 18

33. Abrahams, B.S.; Geschwind, D.H. Advances in autism genetics: On the threshold of a new neurobiology.

34.

35.

36.

Nat. Rev. Genet. 2008, 9, 341–355. [CrossRef]
Schaefer, G.B.; Mendelsohn, N.J.; Professional Practice and Guidelines Committee. Clinical genetics
evaluation in identifying the etiology of autism spectrum disorders: 2013 guideline revisions. Genet. Med.
2013, 15, 399–407. [CrossRef]
Sebat, J.; Lakshmi, B.; Malhotra, D.; Troge, J.; Lese-Martin, C.; Walsh, T.; Yamrom, B.; Yoon, S.; Krasnitz, A.;
Kendall, J.; et al. Strong association of de novo copy number mutations with autism. Science 2007, 316,
445–449. [CrossRef] [PubMed]
Schaefer, G.B.; Starr, L.; Pickering, D.; Skar, G.; Dehaai, K.; Sanger, W.G. Array comparative genomic
hybridization ﬁndings in a cohort referred for an autism evaluation. J. Child. Neurol. 2010, 25, 1498–1503.
[CrossRef] [PubMed]

37. Wenger, T.L.; Kao, C.; McDonald-McGinn, D.M.; Zackair, E.H.; Bailey, A.; Schultz, R.T.; Morrow, B.E.;
Emanuel, B.S.; Hakonarson, H. The role of mGluR copy number variation in genetic and environmental
forms of syndromic autism spectrum disorder. Sci. Rep. 2016, 6, 19372. [CrossRef] [PubMed]

38. Hallmayer, J.; Cleveland, S.; Torres, A.; Phillips, J.; Cohen, B.; Torigoe, T.; Miller, J.; Fedele, A.; Collins, J.;
Smith, K.; et al. Genetic heritability and shared environmental factors among twin pairs with autism.
Arch. Gen. Psychiatry 2011, 68, 1095–1102. [CrossRef]

39. Tick, B.; Bolton, P.; Happe, F.; Rutther, M.; Rijsdijk, F. Heritability of autism spectrum disorders:

A meta-analysis of twin studies. J. Child Psychol. Psychiatry 2016, 57, 585–595. [CrossRef]

40. Weinder, D.J.; Wigoder, E.M.; Riple, S.; Walters, R.K.; Kosmicki, J.A.; Grove, J.; Samocha, K.E.; Goldstein, J.I.;
Okbay, A.; Bybjerg-Grauhom, J.; et al. Polygenic transmission disequlibirium conﬁrms that common and
rate vairation act additively to create risk for autism spectrum disorders. Nat. Genet. 2017, 49, 978–983.
[CrossRef]

41. Pizzo, L.; Jensen, M.; Polyak, A.; Rosenfeld, J.A.; Mannik, K.; Krishnan, A.; McCready, E.; Pichon, O.;
Le Caignec, C.; Van Dijck, A.; et al. Rare variants in the genetic background modulate cognitive and
developmental phenotypes in individuals carryng disease-associated varients. Genet Med. 2019, 21, 816–825.
[CrossRef]

42. Myers, S.M.; Challman, T.D.; Bernier, R.; Bourgeron, T.; Chung, W.K.; Constantinto, J.N.; Eichler, E.E.;
Jacquemenot, S.; Miller, D.T.; Mitchell, K.J.; et al. Insuﬃcient evidence for “Autism-Speciﬁc” genes. Am. J.
Hum. Genet. 2020, 106, 587–595.

43. Vorstman, J.A.; Parr, J.R.; Moreno-De-Luca, D.; Anney, R.J.L.; Nurnberger, J.I.; Hallmayer, J.F. Autism genetics:

Opportunities and challenges for clinical translation. Nat. Rev. Genet. 2017, 18, 362–376. [CrossRef]
44. Yenkoyan, K.; Griogryan, A.; Fereshetyan, K.; Ypremyan, D. Adances in understanding the pathophysiology

45.

of autism spectrum disorders. Behav. Brain Res. 2017, 331, 92–101. [CrossRef]
Srivastava, S.; Love-Nichols, J.A.; Dies, K.A.; Ledbetter, D.H.; Martin, C.L.; Chung, W.K.; Firth, H.V.;
Frazier, T.; Hansen, R.L.; Prock, L.; et al. NDD exome scoping review work group. Meta-analysis and
multidisciplinary consensus statement: Exome sequencing is a ﬁrst-tier clinical diagnostic test for individuals
with neurodevelopmental disorders. Genet. Med. 2019, 27, 2413–2421. [CrossRef] [PubMed]

46. Dhillon, S.; Hellings, J.A.; Butler, M.G. Genetics and mitochondrial abnormalities in autism spectrum

disorders: A review. Curr. Genom. 2011, 12, 322–332. [CrossRef]

47. Ho, K.S.; Wassman, E.R.; Baxter, A.L.; Hensel, C.H.; Martin, M.M.; Prasad, A.; Twede, H.; Vanzo, R.J.;
Butler, M.G. Chromosomal microarray analysis of consecutive individuals with autism spectrum disorders
using an ultra-high resolution chromosomal microarray optimized for neurodevelopmental disorders. Int. J.
Mol. Sci. 2016, 17, 2070. [CrossRef] [PubMed]

48. Woodbury-Smith, M.; Paterson, A.D.; O’Connor, I.; Zarrei, M.; Yuen, R.K.C.; Howe, J.L.; Thompson, A.;
Parlier, M.; Fernandez, B.; Piven, J.; et al. A genome-wide linkage study of autism spectrum disorder and the
broad autism phenotype in extended pedigrees. J. Neurodev. Disord. 2018, 10, 20. [CrossRef]
Fernandez, B.A.; Roberts, W.; Chung, B.; Weksberg, R.; Meyn, S.; Szatmari, P.; Joseph-George, A.M.; Mackay, S.;
Whitten, K.; Nble, B.; et al. Phenotypic spectrum associated with de novo and inherited deletions and
duplications at 16p11.2 in individuals ascertained for diagnosis of autism spectrum disorder. J. Med. Genet.
2010, 47, 195–203. [CrossRef] [PubMed]

49.

Int. J. Mol. Sci. 2020, 21, 4726

15 of 18

50. Miller, D.T.; Shen, Y.; Weiss, L.A.; Korn, J.; Anselm, I.; Bridgemohan, C.; Cox, G.F.; Dickinson, H.; Gentile, J.;
Harris, D.J.; et al. Microdeletion/duplication at 15q13.2q13.3 among individuals with features of autism and
other neuropsychiatric disorders. J. Med. Genet. 2009, 46, 242–248. [CrossRef]

51. Rossi, M.; El-Khechen, D.; Black, M.H.; Hagmna, K.D.F.; Tang, S.; Powis, Z. Outcomes of diagnostic exome
sequencing in patients with diagnosed or suspected autism spectrum disorders. Pediatr. Neurol. 2017, 70,
34–43. [CrossRef] [PubMed]

52. Tammimies, K.; Marshall, C.R.; Walker, S.; Kaur, G.; Thiruvahindrapuram, B.; Lionel, A.C.; Yuen, R.K.C.;
Uddin, M.; Roberts, W.; Weksberg, R.; et al. Molecular diagnostic yield of chromosomal microarray
analysis and whole-exome sequencing in children with autism spectrum disorder. JAMA 2017, 314, 895–903.
[CrossRef] [PubMed]

53. Wang, K.; Zhang, H.; Ma, D.; Bucan, M.; Glessner, J.T.; Abrahams, B.S.; Salyakina, D.; Imielinski, M.;
Bradﬁeld, J.P.; Sleiman, P.M.A.; et al. Common genetic variants on 5p14.1 associate with autism spectrum
disorders. Nature 2009, 459, 528–533. [CrossRef]

54. Delvin, B.; Scherer, S.W. Genetic architecture in autism spectrum disorder. Curr. Opin. Genet. Dev. 2012,

22, 229–237.

55. Geschwind, D.H.; State, M.W. Gene hunting in autism spectrum disorder: On the path to precision medicine.

Lancet Neurol. 2015, 14, 1109–1120. [CrossRef]

56. Pinto, D.; Delaby, E.; Merico, D.; Barbosa, M.; Merikangas, A.; Klei, L.; Thiruvahindrapuram, B.; Xu, X.;
Ziman, R.; Wang, Z.; et al. Convergence of genes and cellular pathways dysregulated in autism spectrum
disorders. Am. J. Hum. Genet. 2014, 94, 677–694. [CrossRef] [PubMed]

57. Yuen, R.K.C.; Merico, D.; Bookman, M.; Howe, J.L.; Thiruvahindrapuram, B.; Patel, R.V.; Whitney, J.;
Delaux, N.; Bingham, J.; Wang, Z.; et al. Whole genome sequencing resource identiﬁes 18 new candidate
genes for autism spectrum disorder. Nat. Neurosci. 2017, 20, 602–611. [CrossRef] [PubMed]

58. Glessner, J.T.; Wang, K.; Cai, G.; Korvatska, O.; Kim, C.E.; Wood, S.; Zhang, H.; Estes, A.; Brune, C.W.;
Bradﬁeld, J.P.; et al. Autism genome-wide copy number variation reveals ubiquitin and neuronal genes.
Nature 2009, 459, 569–573. [CrossRef]

59. Holt, R.; Monaco, A.P. Links between genetics and pathophysiology in the autism spectrum disorders.

EMBO Molecul. Med. 2011, 3, 438–450. [CrossRef] [PubMed]

60. Khanzada, N.S.; Butler, M.G.; Manzardo, A.M. GeneAnalytics pathway analysis and genetic overlap among
autism spectrum disorder, bipolar disorder and schizophrenia. Int. J. Mol Sci. 2017, 18, 527. [CrossRef]
61. Chatterjee, M.; Schild, D.; Teunissen, C.E. Contactins in the central nervous system: Role in health and

disease. Neural. Regen. Res. 2019, 14, 206–216.

62. Varga, N.A.; Pentelenyi, K.; Balicza, P.; Gezsi, A.; Remenyi, V.; Harsfalvi, V.; Bencsik, R.; Iles, A.; Prekop, C.;
Molnar, M.J. Mitochondrial dysfunction and autism: Comprehensive genetic analyses of children with
autism and mtDNA deletion. Behav Brain Funct. 2018, 14, 4. [CrossRef] [PubMed]

63. Wallace, D.C. Mitochondrial genes and disease. Hosp. Pract. 1986, 21, 77–92.
64. Wallace, D.C. Mitochondrial diseases in mand and mouse. Science 1999, 283, 1482–1488. [CrossRef]
65.

Schon, E.A.; Manfredi, G. Neuronal degeneration and mitochondrial dysfunction. J. Clin. Investig. 2003, 111,
303–312. [CrossRef] [PubMed]

66. DiMauro, S.; Schon, E.A. Mitochondrial respiratory-chain diseases. N. Engl. J. Med. 2003, 348, 2656–2668.

[CrossRef] [PubMed]

67. Pons, R.; Andreu, A.L.; Checcarelli, N.; Vila, M.R.; Engelstad, K.; Sue, C.M.; Shungu, D.; Haggerty, R.; de
Vivo, D.C.; DiMauro, S. Mitochondrial DNA abnormalities and autistic spectrum disorders. J. Pediatr. 2004,
144, 81–85. [CrossRef] [PubMed]
Spelbrink, J.N. Functional organization of mammalian mitochondrial DNA in nucleotides: History, recent
developments, and future challenges. IUBMB 2010, 62, 19–32.

68.

69. Boyle, C.A.; Boulet, S.; Schieve, L.A.; Cohen, R.A.; Blumberg, S.J.; Yeargin-Allsopp, M.; Visser, S.; Kogan, M.D.
Trends in the prevalence of developmental disabilities in the US children, 1997–2008. Pediatrics 2011, 127,
1034–1042. [CrossRef] [PubMed]

70. Heil, K.M.; Schaaf, C.P. The genetics of autism spectrum disorders—A guide for clinicians.

Curr. Psychiatry Rep. 2013, 15, 334. [CrossRef]

Int. J. Mol. Sci. 2020, 21, 4726

16 of 18

71. Roberts, J.L.; Jovanes, K.; Dasouki, M.; Manzardo, A.M.; Butler, M.G. Chromosomal microarray analysis of
consecutive individuals with autism spectrum disorders or learning disability presenting for genetic services.
Gene 2014, 535, 70–78. [CrossRef]

72. Weitlauf, A.S.; McPheeters, M.L.; Peters, B.; Sathe, N.; Travis, R.; Aiello, R.; Williamson, E.;
Veenstra-VanderWeele, J.; Krishnaswami, S.; Jerome, R.; et al. Therapies for Children with Autism Spectrum
Disorder: Behavioral Interventions Update; Comparative Eﬀectiveness Review No. 137. (Prepared by the
Vanderbilt evidence-Based Practice Center Under Contract No. 290-2012-00009-I.) AHRQ Publication No.
14-EHC036-EF.; Agency for Healthcare Research and Quality: Rockville, MD, USA, 2014.

73. Tiura, M.; Kim, J.; Detmers, D.; Baldi, H. Predictors of longitudinal ABA treatment outcomes for children

with autism: A growth curve analysis. Res. Dev. Disabil. 2017, 70, 185–197. [CrossRef]

74. Reichow, B.; Hume, K.; Barton, E.E.; Boyd, B.A. Early intensive behavioral intervention (EIBI) for young
children with autism spectrum disorders (ASD). Cochrane Database Syst. Rev. 2018, 5, CD009260. [CrossRef]
Frankel, F.; Myatt, R.; Sugar, C. A randomized controlled study of parent-assisted children’s friendship
training with children having autism spectrum disorders. J. Autism Dev. Disord. 2010, 40, 827–842. [CrossRef]
76. National Institute for Clinical Excellence. Autism: Recognition, referral, diagnosis and management of

75.

adults on the autism spectrum. Natl. Inst. Health Care Excell. 2012, 142, 18.

77. Hillier, A.; Fish, T.; Cloppert, P. Outcomes of a social and vocational skills support group for adolescents and
young adults on the autism spectrum. Focus Autism Other Dev. Disabl. 2007, 22, 107–115. [CrossRef]
78. Lang, R.; Regester, A.; Lauderdale, S. Treatment of anxiety in autism spectrum disorders using cognitive

behaviour therapy: A systematic review. Dev. Neurorehabil. 2010, 13, 53–63. [CrossRef] [PubMed]

79. Accordino, R.E.; Kidd, C.; Politte, L.C.; Henry, C.A.; McDougle, C.J. Psychopharmacological interventions in

autism spectrum disorder. Expert Opin. Pharmacother. 2016, 17, 937–952.

80. Madden, J.M.; Lakoma, M.D.; Lynch, F.L.; Rusinak, D.; Owen-Smith, A.A.; Coleman, K.J.; Quinn, V.P.;
Yau, V.M.; Qian, Y.X.; Croen, L.A. Psychotropic medication use among insured children with autism spectrum
disorder. J. Autism Dev. Disord. 2017, 47, 144–154. [CrossRef]
Stepanova, E.; Dowling, S.; Phelps, M.; Findling, R.L. Pharmacotherapy of emotional and behavioral
symptoms associated with autism spectrum disorder in children and adolescents. Dialogues Clin. Neurosci.
2017, 19, 395–402.

81.

82. Lamy, M.; Erickson, C.A. Pharmacological management of behavioral disturbances in children and adolescents
with autism spectrum disorders. Curr. Prob. Ped. Adolesc Health Care 2012, 48, 250–264. [CrossRef] [PubMed]
83. Research Units on Pediatric Psychopharmacology (RUPP). Randomized, controlled, crossover trial of
methylphenidate in pervasive developmental disorders with hyperactivity. Arch. Gen. Psychiatry. 2005, 62,
1266–1275. [CrossRef]

84. Reichow, B.; Volkmar, F.R.; Bloch, M.H. Systematic review and meta-analysis of pharmacological treatment of
the symptoms of attention-deﬁcit/hyperactivity disorder in children with pervasive developmental disorders.
J. Autism Dev. Disord. 2013, 43, 2435–4241. [CrossRef]

85. Harfterkamp, M.; van de Loo-Neus, G.; Minderaa, R.B.; van der Gaag, R.-J.; Escobar, R.; Schacht, A.;
Pamulapati, S.; Buietelaar, J.K.; Hoekstra, P.J. A randomized double-blind study of atomoxetine versus
placebo for attention-deﬁcit/hyperactivity disorder symptoms in children with autism spectrum disorders.
J. Am. Acad. Child Adol. Psychiatry 2012, 51, 733–741. [CrossRef]

87.

86. Guy, W. ECDEU Assessment Manual for Psychopharmacology, Revised.; Department of Health, Education,
and Welfare Publication (ADM): National Institute of Mental Health: Rockville, MD, USA, 1976; pp. 76–338.
Scahill, L.; McCracken, J.T.; King, B.H.; Rockhill, C.; Shah, B.; Politte, L.; Sanders, R.; Minjarez, M.; Cowen, J.;
Mullett, J.; et al. Research Unites on Pediatric Psychopharmacology Autism Network. Extended-release
Guanfacine for hyperactivity in children with autism spectrum disorder. Am.
J. Psychiatry 2015, 172,
1197–1206. [CrossRef] [PubMed]

88. Pandina, G.J.; Bossie, C.A.; Youssef, E.; Zhu, Y.; Dunbar, F. Risperidone improves behavioral symptoms in
children with autism in a randomized, double-blind, placebo-controlled trial. J. Autism Dev. Disord. 2007, 37,
367–373. [CrossRef] [PubMed]

89. Aman, M.G.; Singh, N.N. Aberrant Behavior Checklist Manual; Slosson Publications: East Aurora, NY,

USA, 1986.

Int. J. Mol. Sci. 2020, 21, 4726

17 of 18

90. Levine, S.Z.; Kodesh, A.; Goldberg, Y.; Reichenberg, A.; Furukawa, T.A.; Kolevzon, A.; Leucht, S. Initial
severity and eﬃcacy of risperidone in autism: Results from the RUPP trial. Eur. Psychiatry. 2016, 32, 16–20.
[CrossRef]

91. Wink, L.K.; Early, M.; Schaefer, T.; Pottenger, A.; Horn, P.; MDougle, C.J.; Erickson, C.A. Body mass index
change in autism spectrum disorders: Comparison of treatment with risperidone and aripiprazole. J. Child
Adolesc. Psychopharmacol. 2014, 24, 78–82. [CrossRef]

92. Hollander, E.; Chaplin, W.; Soorya, L.; Wasserman, S.; Novotny, S.; Rusoﬀ, J.; Feirsen, N.; Pepa, L.;
Anagnostou, E. Divalproex sodium vs placebo for the treatment of irritability in children and adolescents
with autism spectrum disorders. Neuropsychopharmacol 2010, 35, 990–998. [CrossRef]

93. Hellings, J.A.; Weckbaugh, M.; Nickel, E.J.; Cain, S.E.; Zarcone, J.R.; Reese, R.M.; Hall, S.; Ermer, D.J.;
Tsai, L.Y.; Schroeder, S.R.; et al. A double-blind, placebo-controlled study of valproate for aggression in youth
with pervasive developmental disorders. J. Child Adolesc. Psychopharmacol. 2005, 15, 682–692. [CrossRef]
[PubMed]

94. Mazzone, L.; Ruta, L. Topiramate in children with autistic spectrum disorders. Brain Dev. 2006, 28, 668.

[CrossRef] [PubMed]

95. Rezaei, V.; Mohammadi, M.R.; Ghanizadeh, A.; Sahraian, A.; Tabrizi, M.; Rezazadeh, S.-A.; Akhondzadeh, S.
Double-blind, placebo-controlled trial of risperidone plus topiramate in children with autistic disorder. Prog.
Neuropsychopharmacol. Biol. Psychiatry 2010, 34, 1269–1272. [CrossRef] [PubMed]
Swatzyna, R.J.; Boutros, N.N.; Genovese, A.C.; MacInerney, E.K.; Roark, A.J.; Kozlowski, G.P.
Electroencephalogram (EEG) for children with autism spectrum disorder: Evidential considerations for
routine screening. Eur. Child Adolesc. Psychiatry 2019, 28, 615–624. [CrossRef] [PubMed]

96.

97. Hollander, E.; Soorya, L.; Chaplin, W.; Anagnostou, E.; Taylor, C.P.; Ferretti, C.J.; Wasserman, S.; Swanson, E.;
Settipani, C. A double-blind placebo-controlled trial of ﬂuoxetine for repetitive behaviors and global severity
in adult autism spectrum disorders. Am. J. Psychiatry 2012, 169, 292–299. [CrossRef]

98. Williams, K.; Brignell, A.; Randall, M.; Silove, N.; Hazell, P. Selective serotonin reuptake inhibitors (SSRIs)
for autism spectrum disorders (ASD). Cochrane Database Syst. Rev. 2013, 8, CD004677. [CrossRef] [PubMed]
Souders, M.C.; Zavodny, S.; Eriksen, W.; Sinko, R.; Connell, J.; Kerns, C.; Schaaf, R.; Pinto-Martin, J. Sleep in
children with autism spectrum disorder. Curr. Psychiatry Rep. 2017, 19, 34. [CrossRef] [PubMed]

99.

100. Blackmer, A.B.; Feinstein, J.A. Management of sleep disorders in children with neurodevelopmental disorders:

A review. Pharmacother 2016, 36, 84–98. [CrossRef] [PubMed]

101. Ming, X.; Gordon, E.; Kang, N.; Wagner, G.C. Use of clonidine in children with autism spectrum disorders.

Brain Dev. 2008, 30, 454–460. [CrossRef]

102. Butler, M.G. Pharmacogenetics and psychiatric care: A review and commentary. J. Ment. Health Clin. Psychol.

2018, 2, 17–24. [CrossRef]

103. Weinshilboum, R.M.; Wang, L. Pharmacogenetics and pharmacogenomics: Development, science and

translation. Annu Rev. Genom. Hum. Genet. 2006, 7, 223–245. [CrossRef]

104. Weng, L.; Zhang, L.; Peng, Y.; Huang, R.S. Pharamcogenetics and pharmacogenomics: A bridge to

individualized cancer therapy. Pharmacogenomics 2013, 14, 315–324. [CrossRef]

105. Kirchheiner, J.; Seeringer, A. Clinical implications of pharmacogenetics of cytochrome P450 drug metabolizing

enzymes. Biochim. Biophys. Acta 2007, 1770, 489–494. [CrossRef]

106. Reynolds, K.S. Achieving the promise of personalized medicine. Clin. Pharmacol. Ther. 2012, 92, 401–405.

[CrossRef]

107. Lee, J.W.; Aminken, F.; Bhavsar, A.P.; Shaw, K.; Carleton, B.C.; Hayden, M.R.; Ross, C.J.D. The emerging
era of pharmacogenomics: Current successes, future potential, and challenges. Clin. Genet. 2014, 86, 21–28.
[CrossRef]

108. Kalow, W. Human pharmacogenomics: The development of a science. Hum. Genom. 2004, 1, 375–380.

[CrossRef] [PubMed]

109. Werck-Reichhart, D.; Feyereisen, R. Cytochromes P450: A success story. Genome Biol. 2000, 1, 3003. [CrossRef]

[PubMed]

110. Samer, C.F.; Lorenzini, K.I.; Rollason, V.; Daali, Y.; Desmeues, J.A. Applications of CYP450 testing in the

clinical setting. Mol. Diagn. Ther. 2013, 17, 165–184. [CrossRef]

111. Danielson, P.B. The cytochrome P450 superfamily: Biochemistry, evolution and drug metabolism in humans.

Curr. Drug Metab. 2002, 3, 561–597. [CrossRef] [PubMed]

Int. J. Mol. Sci. 2020, 21, 4726

18 of 18

112. Chen, C.; Chen, Y.; Guan, M.X. A peep into mitochondrial disorder: Multifaceted from mitochondrial DNA

mutations to nuclear gene modulation. Protein Cell 2015, 6, 862–870. [CrossRef] [PubMed]

113. Schaefer, G.B. Clinical genetic aspects of ASD spectrum disorders. Int. J. Mol. Sci. 2016, 17, 180. [CrossRef]
114. Oikonomakis, V.; Kosma, K.; Mitrakos, A.; Sofocleous, C.; Pervanidou, P.; Syrmou, A.; Pampanos, A.; Psoni, S.;
Fryssira, H.; Kanavakis, E.; et al. Recurrent copy number variations as risk factors for autism spectrum
disorders: Analysis of the clinical implications. Clin. Genet. 2016, 89, 708–718. [CrossRef]

115. Hall, L.; Kelley, E. The contribution of epigenetics to understanding genetic factors in autism. Autism 2014,

18, 872–881. [CrossRef]

116. Hagerman, R.J.; Berry-Kravis, E.; Hazlett, H.C.; Bailey, D.B., Jr.; Moine, H.; Kooy, R.F.; Tassone, F.; Gantois, I.;

Sonenberg, N.; Mandel, J.L.; et al. Fragile X Syndrome. Nat. Rev. Dis. Primers 2017, 29, 17065. [CrossRef]

117. Raﬁ, S.K.; Butler, M.G. The 15q11.2 BP1-BP2 Microdeletion (Burnside–Butler) Syndrome: In Silico Analyses
of the Four Coding Genes Reveal Functional Associations with Neurodevelopmental Phenotypes. Int. J.
Med. Sci. 2020, 21, 3296. [CrossRef]

118. Hartin, S.M.; Hossain, W.A.; Butler, M.G.; Francis, D.; Godler, D.E.; Barkataki, S. Analysis of the Prader-Willi
syndrome imprinting center using droplet digitcal PCR and Next-Generation whole-exome sequencing.
Mol. Genet. Genom. Med. 2019, 7, e00575. [CrossRef] [PubMed]

© 2020 by the authors. Licensee MDPI, Basel, Switzerland. This article is an open access
article distributed under the terms and conditions of the Creative Commons Attribution
(CC BY) license (http://creativecommons.org/licenses/by/4.0/).
