# Katrinli_2024_Epigenome-wide association studies identify novel DNA methylation sites associated with PTSD A meta-analysis of 23 military and civilian cohorts._1

medRxiv preprint 

doi: 

https://doi.org/10.1101/2024.07.15.24310422

; 

this version posted July 15, 2024. 

The copyright holder for this preprint 

(which was not certified by peer review)

 is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. 

It is made available under a 

CC-BY-NC-ND 4.0 International license

 . 

Epigenome-wide association studies identify novel DNA methylation sites associated with 

PTSD: A meta-analysis of 23 military and civilian cohorts 

Seyma Katrinli, PhD1, Agaz H Wani, PhD2, Adam X Maihofer, PhD3,4,5, Andrew Ratanatharathorn, 

PhD6,7, Nikolaos P Daskalakis, MD, PhD8,9,10, Janitza Montalvo-Ortiz, PhD11,12,13, Diana L Núñez-

Ríos, PhD11,12,13, Anthony S Zannas, MD, PhD14,15,16,17, Xiang Zhao, MD18, Allison E Aiello, PhD19, 

Allison E Ashley-Koch, PhD20, Diana Avetyan, PhD3, Dewleen G Baker, MD3,4,21, Jean C Beckham, 

PhD22,23,24, Marco P Boks, MD, PhD25, Leslie A Brick, PhD26, Evelyn Bromet, PhD27, Frances A 

Champagne, PhD28, Chia-Yen Chen, MA29, Shareefa Dalvie, PhD30,31, Michelle F Dennis, 

PhD22,23,24, Segun Fatumo, PhD32, Catherine Fortier, PhD9,33, Sandro Galea, MD, PhD34, Melanie E 

Garrett, PhD20, Elbert Geuze, PhD35,36, Gerald Grant, MD37, Michael A Hauser, PhD38, Jasmeet P 

Hayes, PhD39, Sian MJ Hemmings, PhD40,41, Bertrand Russel Huber, MD, PhD42,43, Aarti Jajoo, 

PhD8,9,44, Stefan Jansen, PhD45, Ronald C Kessler, PhD46, Nathan A Kimbrel, PhD22,24,47, Anthony P 

King, PhD48,49, Joel E Kleinman, MD, PhD50,51, Nastassja Koen, PhD52,53,54, Karestan C Koenen, 

PhD7,8,55, Pei-Fen Kuan, PhD56, Israel Liberzon, MD57, Sarah D Linnstaedt, PhD58,59, Adriana Lori, 

PhD60, Benjamin J Luft, MD61, Jurjen J Luykx, MD, PhD62,63,64, Christine E Marx, MD22,65,66, Samuel 

A McLean, MD, PhD67, Divya Mehta, PhD68,69, William Milberg, PhD70, Mark W Miller, PhD71,72, 

Mary S Mufford, PhD73, Clarisse Musanabaganwa, PhD74,75, Jean Mutabaruka, PhD76, Leon 

Mutesa, MD, PhD45,77, Charles B Nemeroff, MD, PhD28,78, Nicole R Nugent, PhD79,80,81, Holly K 

Orcutt, PhD82, Xue-Jun Qin, PhD83, Sheila A M Rauch, PhD84,85, Kerry J Ressler, MD, PhD9,44,60, 

Victoria B Risbrough, PhD3,4,5, Eugène Rutembesa, PhD86, Bart P F Rutten, MD, PhD87, Soraya 

Seedat, MD, PhD40,88, Dan J Stein, MD, PhD52,53,54, Murray B Stein, MD3,21,89, Sylvanus Toikumo, 

PhD40,90, Robert J Ursano, MD91, Annette Uwineza, PhD92, Mieke H Verfaellie, PhD93,94, Eric 

NOTE: This preprint reports new research that has not been certified by peer review and should not be used to guide clinical practice.

1 

 
medRxiv preprint 

doi: 

https://doi.org/10.1101/2024.07.15.24310422

; 

this version posted July 15, 2024. 

The copyright holder for this preprint 

(which was not certified by peer review)

 is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. 

It is made available under a 

CC-BY-NC-ND 4.0 International license

 . 

Vermetten, MD, PhD95,96, Christiaan H Vinkers, MD, PhD97,98,99, Erin B Ware, PhD100, Derek E 

Wildman, PhD101,102, Erika J Wolf, PhD72,103, Ross McD Young, PhD104,105, Ying Zhao, MPH17,58, 

Leigh L van den Heuvel, MD, PhD40,88, PGC-PTSD Epigenetics Workgroup, PsychENCODE PTSD 

Brainomics Project, Traumatic Stress Brain Research Group, Monica Uddin106, Caroline M 

Nievergelt, PhD3,4,5, Alicia K Smith, PhD1,60,107, Mark W Logue, PhD18,71,72 

1Emory University, Department of Gynecology and Obstetrics, Atlanta, GA, US, 2University of 

South Florida, Genomics Program, College of Public Health, Tampa, FL, US, 3University of 

California San Diego, Department of Psychiatry, La Jolla, CA, US, 4Veterans Affairs San Diego 

Healthcare System, Center of Excellence for Stress and Mental Health, San Diego, CA, US, 

5Veterans Affairs San Diego Healthcare System, Research Service, San Diego, CA, US, 6Columbia 

University Mailmain School of Public Health, Department of Epidemiology, New York, NY, US, 

7Harvard T.H. Chan School of Public Health, Department of Epidemiology, Boston, MA, US, 

8Broad Institute of MIT and Harvard, Stanley Center for Psychiatric Research, Cambridge, MA, 

US, 9Harvard Medical School, Department of Psychiatry, Boston, MA, US, 10McLean Hospital, 

Center of Excellence in Depression and Anxiety Disorders, Belmont, MA, US, 11U.S. Department 

of Veterans Affairs National Center of Posttraumatic Stress Disorder, Clinical Neurosciences 

Division, West Haven, CT, US, 12VA Connecticut Healthcare System, West Haven, CT, US, 13Yale 

School of Medicine, Department of Psychiatry, New Haven, CT, US, 14University of North 

Carolina at Chapel Hill, Carolina Stress Initiative, Chapel Hill, NC, US, 15University of North 

Carolina at Chapel Hill, Department of Genetics, Chapel Hill, NC, US, 16University of North 

Carolina at Chapel Hill, Department of Psychiatry, Chapel Hill, NC, US, 17University of North 

2 

 
 
medRxiv preprint 

doi: 

https://doi.org/10.1101/2024.07.15.24310422

; 

this version posted July 15, 2024. 

The copyright holder for this preprint 

(which was not certified by peer review)

 is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. 

It is made available under a 

CC-BY-NC-ND 4.0 International license

 . 

Carolina at Chapel Hill, Institute for Trauma Recovery, Chapel Hill, NC, US, 18Boston University 

School of Public Health, Department of Biostatistics, Boston, MA, US, 19Columbia University, 

Robert N Butler Columbia Aging Center, Department of Epidemiology, New York, NY, US, 20Duke 

University Medical Center, Duke Molecular Physiology Institute, Durham, NC, US, 21Veterans 

Affairs San Diego Healthcare System, Psychiatry Service, San Diego, CA, US, 22Duke University 

School of Medicine, Department of Psychiatry and Behavioral Sciences, Durham, NC, US, 

23Durham VA Health Care System, Research, Durham, NC, US, 24VA Mid-Atlantic Mental Illness 

Research, Education, and Clinical Center (MIRECC), Genetics Research Laboratory, Durham, NC, 

US, 25Brain Center University Medical Center Utrecht, Department of Psychiatry, Utrecht, UT, NL, 

26Alpert Medical School of Brown University, Department of Psychiatry and Human Behavior, 

Providence, RI, US, 27State University of New York at Stony Brook, Epidemiology Research 

Group, Stony Brook, NY, US, 28The University of Texas at Austin, Department of Psychology, 

Austin, TX, US, 29Biogen Inc., Translational Sciences, Cambridge, MA, US, 30University of Cape 

Town, Department of Pathology, Cape Town, Western Province, ZA, 31University of Cape Town, 

Division of Human Genetics, Cape Town, Western Province, ZA, 32MRC/UVRI and London School 

of Hygiene and Tropical Medicine, The African Computational Genomics (TACG) Research Group, 

Entebbe, Wakiso, Uganda, 33VA Boston Healthcare System, TRACTS/GRECC, Boston, MA, US, 

34Boston University School of Public Health, Boston, MA, US, 35Netherlands Ministry of Defence, 

Brain Research and Innovation Centre, Utrecht, UT, NL, 36UMC Utrecht Brain Center Rudolf 

Magnus, Department of Psychiatry, Utrecht, UT, NL, 37Duke University School of Medicine, 

Department of Neurosurgery, Durham, NC, US, 38Duke University School of Medicine, 

Department of Medicine, Durham, NC, US, 39The Ohio State University, Department of 

3 

 
medRxiv preprint 

doi: 

https://doi.org/10.1101/2024.07.15.24310422

; 

this version posted July 15, 2024. 

The copyright holder for this preprint 

(which was not certified by peer review)

 is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. 

It is made available under a 

CC-BY-NC-ND 4.0 International license

 . 

Psychology, Columbus, OH, US, 40Stellenbosch University, Faculty of Medicine and Health 

Sciences, Department of Psychiatry, Cape Town, Western Cape, ZA, 41Stellenbosch University, 

SAMRC Genomics of Brain Disorders Research Unit, Cape Town, Western Cape, ZA, 42Boston 

University School of Medicine, Department of Neurology, Boston, MA, US, 43VA Boston 

Healthcare System, Pathology and Laboratory Medicine, Boston, MA, US, 44McLean Hospital, 

Belmont, MA, US, 45University of Rwanda, College of Medicine and Health Sciences, Kigali, RW, 

46Harvard Medical School, Department of Health Care Policy, Boston, MA, US, 47Durham VA 

Health Care System, Mental Health Service Line, Durham, NC, US, 48The Ohio State University, 

College of Medicine, Institute for Behavioral Medicine Research, Columbus, OH, US, 49The Ohio 

State University, College of Medicine, Psychiatry & Behavioral Health, Columbus, OH, US, 

50Johns Hopkins University School of Medicine, Department of Psychiatry and Behavioral 

Sciences, Baltimore, MD, US, 51Lieber Institute for Brain Development, Baltimore, MD, US, 

52University of Cape Town, Department of Psychiatry & Mental Health, Cape Town, Western 

Province, ZA, 53University of Cape Town, Neuroscience Institute, Cape Town, Western Province, 

ZA, 54University of Cape Town, SA MRC Unit on Risk & Resilience in Mental Disorders, Cape 

Town, Western Province, ZA, 55Massachusetts General Hospital, Psychiatric and 

Neurodevelopmental Genetics Unit (PNGU), Boston, MA, US, 56Stony Brook University, 

Department of Applied Mathematics and Statistics, Stony Brook, NY, US, 57Texas A&M University 

College of Medicine, Department of Psychiatry and Behavioral Sciences, Bryan, TX, US, 

58University of North Carolina at Chapel Hill, Department of Anesthesiology, Chapel Hill, NC, US, 

59University of North Carolina at Chapel Hill, UNC Institute for Trauma Recovery, Chapel Hill, NC, 

US, 60Emory University, Department of Psychiatry and Behavioral Sciences, Atlanta, GA, US, 

4 

 
medRxiv preprint 

doi: 

https://doi.org/10.1101/2024.07.15.24310422

; 

this version posted July 15, 2024. 

The copyright holder for this preprint 

(which was not certified by peer review)

 is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. 

It is made available under a 

CC-BY-NC-ND 4.0 International license

 . 

61Stony Brook University, Department of Medicine, Stony Brook, NY, US, 62Amsterdam University 

Medical Center, Amsterdam Neuroscience Research Institute, Mood, Anxiety, Psychosis, Stress & 

Sleep Program, Amsterdam, NH, NL, 63Amsterdam University Medical Center, Amsterdam Public 

Health Research Institute, Mental Health Program, Amsterdam, NH, NL, 64Amsterdam University 

Medical Center, Department of Psychiatry, Amsterdam, NH, NL, 65Durham VA Health Care 

System, Durham, NC, US, 66VA Mid-Atlantic Mental Illness Research, Education, and Clinical 

Center (MIRECC), Durham, NC, US, 67UNC Institute for Trauma Recovery, Department of 

Psychiatry, Chapel Hill, NC, US, 68Queensland University of Technology, Centre for Genomics and 

Personalised Health, Kelvin Grove, QLD, AU, 69Queensland University of Technology, School of 

Biomedical Sciences, Kelvin Grove, QLD, AU, 70VA Boston Healthcare System, GRECC/TRACTS, 

Boston, MA, US, 71Boston University School of Medicine, Psychiatry, Biomedical Genetics, 

Boston, MA, US, 72VA Boston Healthcare System, National Center for PTSD, Boston, MA, US, 

73University of Cape Town, Department of Psychiatry and Mental Health, Cape Town, Western 

Province, ZA, 74Rwanda Biomedical Center , Research Innovation and Data Science Division, 

Kigali , RW, 75University of Rwanda, Center of Human Genetics, Kigali , RW, 76University of 

Rwanda, Department of Clinical Psychology, Huye, RW, 77University of Rwanda, Center for 

Human Genetics, Kigali, RW, 78The University of Texas at Austin, Department of Psychiatry and 

Behavioral Sciences, Austin, TX, US, 79Alpert Brown Medical School, Department of Emergency 

Medicine, Providence, RI, US, 80Alpert Brown Medical School, Department of Pediatrics, 

Providence, RI, US, 81Alpert Brown Medical School, Department of Psychiatry and Human 

Behavior, Providence, RI, US, 82Northern Illinois University, Department of Psychology, DeKalb, 

IL, US, 83Duke University, Duke Molecular Physiology Institute, Durham, NC, US, 84Emory 

5 

 
medRxiv preprint 

doi: 

https://doi.org/10.1101/2024.07.15.24310422

; 

this version posted July 15, 2024. 

The copyright holder for this preprint 

(which was not certified by peer review)

 is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. 

It is made available under a 

CC-BY-NC-ND 4.0 International license

 . 

University, Department of Psychiatry & Behavioral Sciences, Atlanta, GA, US, 85Joseph Maxwell 

Cleland Atlanta Veterans Affairs Healthcare System, Atlanta, GA, US, 86University of Rwanda, 

Clinical Psychology, Kigali, RW, Rwanda , 87Maastricht Universitair Medisch Centrum, School for 

Mental Health and Neuroscience, Department of Psychiatry and Neuropsychology, Maastricht, 

Limburg, NL, 88Stellenbosch University, SA MRC Extramural Genomics of Brain Disorders 

Research Unit, Cape Town, Western Cape, ZA, 89University of California San Diego, School of 

Public Health, La Jolla, CA, US, 90Stellenbosch University, SA MRC Genomics of Brain Disorders 

Research Unit, Cape Town, Western Cape, ZA, 91Uniformed Services University, Center for the 

Study of Traumatic Stress, Department of Psychiatry, Bethesda, Maryland, US, 92University of 

Rwanda, College of Medicine and Health Sciences, Kigali, Rwanda , 93Boston University School of 

Medicine, Department of Psychiatry, Boston, MA, US, 94VA Boston Healthcare System, Memory 

Disorders Research Center, Boston, MA, US, 95Leiden University Medical Center, Department of 

Psychiatry, Leiden, ZH, NL, 96New York University School of Medicine, Department of Psychiatry, 

New York, NY, US, 97Amsterdam UMC location Vrije Universiteit Amsterdam, Department of 

Anatomy and Neurosciences, Amsterdam, Holland, NL, 98Amsterdam UMC location Vrije 

Universiteit Amsterdam, Department of Psychiatry, Amsterdam, Holland, NL, 99Amsterdam 

University Medical Center, Amsterdam Neuroscience Research Institute, Mood, Anxiety, 

Psychosis, Stress & Sleep Program, Amsterdam, Holland, NL, 100University of Michigan , Survey 

Research Center, Ann Arbor, MI, US, 101University of South Florida, College of Public Health, 

Tampa, FL, US, 102University of South Florida, Genomics Program, Tampa, FL, US, 103Boston 

University Chobanian & Avedisian School of Medicine, Department of Psychiatry, Boston, MA, 

US, 104Queensland University of Technology, School of Clinical Sciences, Kelvin Grove, QLD, AU, 

6 

 
medRxiv preprint 

doi: 

https://doi.org/10.1101/2024.07.15.24310422

; 

this version posted July 15, 2024. 

The copyright holder for this preprint 

(which was not certified by peer review)

 is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. 

It is made available under a 

CC-BY-NC-ND 4.0 International license

 . 

105University of the Sunshine Coast, The Chancellory, Sippy Downs, QLD, AU, 106University of 

South Florida College of Public Health, Genomics Program, Tampa, FL, US, 107Emory University, 

Department of Human Genetics, Atlanta, GA, US 

7 

 
 
 
 
medRxiv preprint 

doi: 

https://doi.org/10.1101/2024.07.15.24310422

; 

this version posted July 15, 2024. 

The copyright holder for this preprint 

(which was not certified by peer review)

 is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. 

It is made available under a 

CC-BY-NC-ND 4.0 International license

 . 

*Corresponding author: 

Mark W. Logue 

National Center for PTSD, Behavioral Science Division at VA Boston Healthcare System, Boston, 

MA 02130, USA 

Department of Psychiatry, Boston University School of Medicine, Boston, MA 02118, USA 

Department of Medicine (Biomedical Genetics), Boston University School of Medicine, Boston, 

MA 02118, USA 

Department of Biostatistics, Boston University School of Public Health, Boston, MA 02118, USA 

Email: loguem@bu.edu 

Phone: 857-364-4079 

Word Count: 3635 

8 

 
 
 
medRxiv preprint 

doi: 

https://doi.org/10.1101/2024.07.15.24310422

; 

this version posted July 15, 2024. 

The copyright holder for this preprint 

(which was not certified by peer review)

 is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. 

It is made available under a 

CC-BY-NC-ND 4.0 International license

 . 

Abstract 

Background: The occurrence of post-traumatic stress disorder (PTSD) following a traumatic 

event is associated with biological differences that can represent the susceptibility to PTSD, the 

impact of trauma, or the sequelae of PTSD itself. These effects include differences in DNA 

methylation (DNAm), an important form of epigenetic gene regulation, at multiple CpG loci 

across the genome. Moreover, these effects can be shared or specific to both central and 

peripheral tissues. Here, we aim to identify blood DNAm differences associated with PTSD and 

characterize the underlying biological mechanisms by examining the extent to which they mirror 

associations across multiple brain regions. 

Methods: As the Psychiatric Genomics Consortium (PGC) PTSD Epigenetics Workgroup, we 

conducted the largest cross-sectional meta-analysis of epigenome-wide association studies 

(EWASs) of PTSD to date, involving 5077 participants (2156 PTSD cases and 2921 trauma-

exposed controls) from 23 civilian and military studies. PTSD diagnosis assessments were 

harmonized following the standardized guidelines established by the PGC-PTSD Workgroup. 

DNAm was assayed from blood using either Illumina HumanMethylation450 or MethylationEPIC 

(850K) BeadChips. A common QC pipeline was applied. Within each cohort, DNA methylation 

was regressed on PTSD, sex (if applicable), age, blood cell proportions, and ancestry. An inverse 

variance-weighted meta-analysis was performed. We conducted replication analyses in tissue 

from multiple brain regions, neuronal nuclei, and a cellular model of prolonged stress. 

Results: We identified 11 CpG sites associated with PTSD in the overall meta-analysis (1.44e-09 

< p < 5.30e-08), as well as 14 associated in analyses of specific strata (military vs civilian cohort, 

sex, and ancestry), including CpGs in AHRR and CDC42BPB. Many of these loci exhibit blood-

9 

 
medRxiv preprint 

doi: 

https://doi.org/10.1101/2024.07.15.24310422

; 

this version posted July 15, 2024. 

The copyright holder for this preprint 

(which was not certified by peer review)

 is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. 

It is made available under a 

CC-BY-NC-ND 4.0 International license

 . 

brain correlation in methylation levels and cross-tissue associations with PTSD in multiple brain 

regions. Methylation at most CpGs correlated with their annotated gene expression levels. 

Conclusions: This study identifies 11 PTSD-associated CpGs, also leverages data from 

postmortem brain samples, GWAS, and genome-wide expression data to interpret the biology 

underlying these associations and prioritize genes whose regulation differs in those with PTSD.   

Keywords: PTSD, trauma, DNA methylation, postmortem brain, GWAS, gene expression 

10 

 
 
 
medRxiv preprint 

doi: 

https://doi.org/10.1101/2024.07.15.24310422

; 

this version posted July 15, 2024. 

The copyright holder for this preprint 

(which was not certified by peer review)

 is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. 

It is made available under a 

CC-BY-NC-ND 4.0 International license

 . 

Introduction 

Posttraumatic stress disorder (PTSD) is a serious psychiatric disorder characterized by 

intrusive memories of the traumatic event(s), avoidance of or numbing to situations that trigger 

those memories, and hyperarousal symptoms that can disturb mental and physical health (1). 

These symptoms are associated with lower levels of self-care, lower compliance with medical 

treatment, and higher rates of substance use (2, 3). Thus, it is not surprising that PTSD increases 

the risk for chronic medical conditions, such as cardiovascular disorders, independent of 

lifestyle factors (e.g., substance use and sleep quality)(4, 5). Although most individuals 

experience at least one traumatic event, only a small fraction develop PTSD(6). Genetic and 

environmental factors contribute to this differential susceptibility in PTSD development upon 

trauma exposure (7, 8). 

Genome-wide association studies (GWAS) of PTSD demonstrated remarkable success at 

identifying relevant genes, many of which are involved in the stress response or immune 

function (see reviews(9, 10)). The recent Psychiatric Genomics Consortium PTSD Workgroup 

(PGC-PTSD) Freeze 3 GWAS identified 95 genomic loci associated with PTSD, implicating genes 

involved in stress, immune, fear, and threat-related processes (11). Nonetheless, genetic 

differences do not fully account for an individual's susceptibility to PTSD. Trauma exposure has 

been shown to alter epigenetic patterns in both animal and human studies, prompting the need 

to conduct epigenetic studies of PTSD in addition to genetic studies (12, 13). Epigenetic 

mechanisms are chemical modifications that can dictate the timing and magnitude of gene 

expression without altering the DNA sequence (14). The most widely studied epigenetic 

mechanism is DNA methylation (DNAm), which is defined as the addition of a methyl group to 

11 

 
medRxiv preprint 

doi: 

https://doi.org/10.1101/2024.07.15.24310422

; 

this version posted July 15, 2024. 

The copyright holder for this preprint 

(which was not certified by peer review)

 is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. 

It is made available under a 

CC-BY-NC-ND 4.0 International license

 . 

cytosine bases, particularly at cytosine-guanine dinucleotides (CpG sites). DNAm patterns 

respond to changes in the environment, are potentially reversible, and can be targeted for 

disease therapies (15, 16). Environmental influences on DNAm are apparent across the life span 

and may provide insight into the biological response to trauma(17). 

Which specific DNAm sites differ across individuals and how they correlate with 

exposures and gene expression can vary across tissues (18). DNAm in human brain tissue, which 

is most relevant to the study of psychiatric disorders, is not easily accessible in living patients 

and hence is not a viable PTSD biomarker for clinical use. However, correlation has been 

observed between peripheral tissues (e.g., blood)  and brain DNAm levels at specific genomic 

loci, and hence blood DNAm can potentially serve as a robust biomarker for implementing early 

intervention and developing improved preventative or therapeutic strategies for PTSD (19, 20). 

Moreover, PTSD symptoms have been linked to the components of the peripheral immune 

system (21, 22) that can be readily assessed in blood DNAm. Multiple peripheral epigenome-

wide association studies (EWASs) of PTSD identified CpGs in genes related to the immune 

system and neurotransmission(23-28). While prior EWASs of PTSD have reported promising 

results, the small sample sizes and variability of analysis methods across studies make it difficult 

to combine and interpret the findings effectively. Recent meta-analyses led by the PGC-PTSD 

Epigenetics Workgroup minimized these issues by increasing sample size, increasing sample 

diversity, and using a common quality control and analysis pipeline(29-33). These meta-analyses 

identified multiple new loci associated with PTSD, including NRG1, AHRR, MAD1L1, and TBXAS1, 

implicating immune dysregulation in those with PTSD (30-33).  

12 

 
medRxiv preprint 

doi: 

https://doi.org/10.1101/2024.07.15.24310422

; 

this version posted July 15, 2024. 

The copyright holder for this preprint 

(which was not certified by peer review)

 is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. 

It is made available under a 

CC-BY-NC-ND 4.0 International license

 . 

Building on the prior work by Smith et al. (31), which conducted an EWAS meta-analysis 

in 1896 participants from 10 cohorts, this study includes 13 additional cohorts with a denser 

and more comprehensive DNAm array, bringing the sample up to 5077 participants from 23 

civilian and military cohorts. Our current investigation replicated the findings of the initial PGC-

PTSD epigenome-wide meta-analysis, reporting lower AHRR methylation in those with PTSD, 

and identified 8 new (11 total) PTSD-associated loci. We also leveraged data from postmortem 

brain samples, a cellular model of prolonged stress, GWAS, and genome-wide gene expression 

studies to interpret the biology underlying these associations and prioritize genes whose 

regulation differs in those with PTSD.   

Methods 

Cohorts and post-traumatic stress disorder assessments 

The study includes 2156 current PTSD cases and 2921 trauma-exposed controls from 9 

civilian cohorts: BEAR, DNHS, DCHS, GTP, NIU, Shared Roots, AURORA, H3A_Rwanda, WTC; and 

9 military cohorts: GMRFQUT, MRS, PRISMO, Army STARRS, PROGrESS, NCPTSD/TRACTS, 

INTRuST, and VA cohorts (VA-M-AA & VA-M-EA). For DNHS, GTP, MRS, PRISMO, and Army 

STARRS, two different datasets were available based on the DNAm array. Two different datasets 

for these five cohorts did not have any overlapping samples and were treated as independent 

studies. Sample characteristics for the 23 studies that participated in the meta-analysis are 

summarized in Table 1.  

The sample is heterogeneous in terms of sex (41% female), ancestry (46% European, 

43% African, and 11% of other ancestries), and cohort type (56% military cohort). Civilian 

13 

 
 
medRxiv preprint 

doi: 

https://doi.org/10.1101/2024.07.15.24310422

; 

this version posted July 15, 2024. 

The copyright holder for this preprint 

(which was not certified by peer review)

 is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. 

It is made available under a 

CC-BY-NC-ND 4.0 International license

 . 

cohorts skew towards being more female (67%) and African (70%), whereas military cohorts are 

predominantly male (80%) and European (64%).  

All participants were exposed to a traumatic event and 42% met the criteria for current 

PTSD. The current PTSD diagnosis was assessed by each study following the standardized 

guidelines established by the PGC-PTSD Workgroup (7). Briefly, current PTSD diagnosis was 

determined based on the specific criteria set by the principal investigator of each study. All 

control subjects had experienced trauma. Subjects without a current PTSD but with a prior 

history of PTSD (i.e., remitted PTSD), were excluded. Detailed descriptions of cohorts and PTSD 

assessments are provided in the Supplement. All participants in these studies gave informed 

consent. The institutional review boards of each respective institution approved these studies. 

DNA methylation  

Whole blood DNAm was measured using the Illumina MethylationEPIC BeadChip in 14 

studies, and the HumanMethylation450 BeadChip in 9 studies (Table 1). All studies used a 

standardized consortium-developed quality control (QC) pipeline that differed somewhat 

depending on which chip was used. The 450K array pipeline (29, 31) is described in eMethods in 

the Supplement. 

The EPIC pipeline (available at https://github.com/PGC-PTSD-EWAS/EPIC_QC) was 

similar to the 450K array pipeline. Samples with probe detection call rates lower than 90% and 

average intensity values that were either less than 50% of the overall sample mean or below 

2000 arbitrary units (AU) were excluded. Probes with detection p-values > 0.01 were considered 

low quality and treated as missing. Probes that were missing in > 10% of the samples within the 

14 

 
 
medRxiv preprint 

doi: 

https://doi.org/10.1101/2024.07.15.24310422

; 

this version posted July 15, 2024. 

The copyright holder for this preprint 

(which was not certified by peer review)

 is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. 

It is made available under a 

CC-BY-NC-ND 4.0 International license

 . 

studies and were cross-hybridizing were removed (34). Data was normalized using single-sample 

Noob (ssNoob) implemented in R package minfi (35). ComBat was used to account for batch 

effects of chip and position while preserving PTSD, age, and sex effects (if applicable) (36). 

Blood-cell composition (i.e., the proportion of CD8+T, CD4+T, natural killer (NK), B cells, 

monocytes, and neutrophils) was estimated using the Robust Partial Correlation (RPC) method 

in Epidish (37) with a reference data specific to EPIC array (38). For studies without genome-

wide genotype data (DNHS, NIU, Shared Roots, AURORA, H3A_Rwanda, GMRFQUT, PROGrESS), 

we estimated ancestry principal components (PCs) from DNAm data, using the method 

developed by Barfield et al. (39), as previously described (33). PCs 2 and 3, which were the 

components that correlate most with self-reported ancestry, were included as covariates (33, 

39). In cohorts with available genome-wide genotype data, PCs 1-3 from GWAS were used to 

adjust for ancestry. We used R package bacon to control inflation, only if doing so results in the 

lambda being closer to 1 (40). To predict smoking status, a DNAm-based smoking score was 

calculated, as previously described (27) for cohorts with EPIC array data. A detailed description 

of DNAm-based ancestry PCs and smoking score calculation is provided in eMethods in the 

Supplement.    

Epigenome-wide association analysis 

The association between PTSD and DNAm was tested using multivariable linear 

regression models within cohorts with balanced plate designs. For studies in which plate layouts 

were not balanced (Shared Roots, H3A_Rwanda, GMRFQUT), we conducted mixed-effect 

regression models, including chip as a random effect term. R package CpGassoc was used to fit 

15 

 
 
medRxiv preprint 

doi: 

https://doi.org/10.1101/2024.07.15.24310422

; 

this version posted July 15, 2024. 

The copyright holder for this preprint 

(which was not certified by peer review)

 is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. 

It is made available under a 

CC-BY-NC-ND 4.0 International license

 . 

the models (41). The models were adjusted for age, sex (if applicable), blood cell composition 

(i.e., CD8T, CD4T, NK, B cell, and monocyte cell proportions), and ancestry PCs. A post-hoc 

sensitivity analysis was performed by including a covariate for smoking: DNAm-based smoking 

score in studies with EPIC data and current smoking status for studies with 450K data. 

Furthermore, we conducted stratified analyses for both sexes, ancestry (European and African 

ancestry), and cohort type (civilian and military cohorts). 

To combine results across studies, we performed inverse-variance weighting (IVW) meta-

analysis in meta (42). Meta-analysis tested 411,786 CpGs common to 450K and EPIC arrays (23 

studies), and 404,794 EPIC array specific CpGs (14 studies). Epigenome-wide significance 

threshold recommended for the MethylationEPIC BeadChip (p < 9.0E-08) was used to determine 

statistical significance (43). Gene Ontology (GO) enrichment analyses were conducted using the 

top 1000 CpGs in missMethyl (44). An FDR threshold of 5% was used to identify significant GO 

terms. 

Cross-tissue association analyses 

Blood – Brain Correlations.  The Blood Brain DNA Methylation Comparison Tool (19) was 

used to assess the correlations between methylation in blood and prefrontal cortex (PFC), 

entorhinal cortex (EC), superior temporal gyrus (STG), and cerebellum. 

Postmortem brain DNAm.  DNAm measured from post-mortem brains was obtained 

from two studies, each of which examined a unique but not necessarily distinct set of brain 

regions and cohorts: the National Center for PTSD Brain Bank cohort (NCPTSD-BB (45)) and the 

PsychENCODE Consortium for PTSD (PEC-PTSD) Brainomics cohort (46) (see eMethods in the 

16 

 
 
medRxiv preprint 

doi: 

https://doi.org/10.1101/2024.07.15.24310422

; 

this version posted July 15, 2024. 

The copyright holder for this preprint 

(which was not certified by peer review)

 is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. 

It is made available under a 

CC-BY-NC-ND 4.0 International license

 . 

Supplement for details), both of which were sourced from the Lieber Institute for Brain 

Development.  

Methylation at PTSD-associated CpGs from the EWAS was tested for association with 

PTSD in DNA extracted from postmortem dorsolateral prefrontal cortex (dlPFC, BA9/46), 

ventromedial prefrontal cortex (vmPFC, BA12/32), amygdala, and dentate gyrus (DG). DNAm in 

the post-mortem tissue was measured using the EPIC array. We examined the associations with 

PTSD in dlPFC and vmPFC of 42 PTSD cases and 30 controls from the NCPTSD-BB. The 

associations between DNAm and PTSD in amygdala and DG were tested in 77 PTSD cases and 77 

controls from the PEC-PTSD. 

Neuronal nuclei.  We examined cross-tissue association from neuronal nuclei isolation 

from the orbitofrontal cortex (OFC) of 25 PTSD cases and 13 healthy controls collected at the 

VA’s NCPTSD-BB (47). Fluorescence-Activated Nuclei Sorting (FANS) protocol was employed to 

isolate NeuN+ cells and the nuclei underwent reduced representation oxidative bisulfite-

sequencing (RRoxBS), as previously described (48). We examined whether there was differential 

methylation and hydroxymethylation within 500 bp of CpGs from the epigenome-wide 

association analyses. Eight CpG sites match between the EPIC array and RRoxBS, two evaluating 

the same position (cg21566642 and cg25691167), and six within 500 bp (cg05575921, 

cg15148933, cg19558029, cg21161138, cg23576855, and cg26599989). 

Cellular model of prolonged stress.  We explored the associations between each of the 

PTSD-associated CpGs resulting from epigenome-wide association analyses and a cellular model 

of prolonged stress in which fibroblasts were subjected to physiological stress hormone 

(cortisol) levels for a prolonged period (51 days) as previously described (49, 50).  

17 

 
medRxiv preprint 

doi: 

https://doi.org/10.1101/2024.07.15.24310422

; 

this version posted July 15, 2024. 

The copyright holder for this preprint 

(which was not certified by peer review)

 is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. 

It is made available under a 

CC-BY-NC-ND 4.0 International license

 . 

Gene Regulation   

Correlations between PTSD-associated CpG sites’ DNAm levels and expression levels of 

the corresponding gene (as determined by the EPIC array annotation) were tested in whole-

blood RNA-sequencing (RNA-seq) data from participants in the BEAR (n=127), AURORA (n=173), 

NCPTSD Merit (n=204), and MRS cohorts (n=128 with multiple visits totaling 357 samples). The 

results were meta-analyzed using the IVW method using a Bonferroni correction for the number 

of CpGs examined. Detailed information about cohort-specific RNA-seq data generation is 

described in eMethods in the Supplement.  

Genetic effects 

To evaluate the effect of nearby (<1 MB) polymorphisms on DNAm levels of CpGs 

associated with PTSD, we used cis-methylation quantitative trait locus (cis-meQTL) data from 

GoDMC(51) and meQTL EPIC(52) databases. For both databases, their default multiple testing 

adjustment was utilized: an FDR threshold of 5% in meQTL EPIC and p < 1e-08 in GoDMC. We 

checked the associations between the identified cis-SNPs and PTSD in the recent Freeze 3 GWAS 

from PGC-PTSD(11). Finally, we evaluated genetic associations between the genes with PTSD-

associated DNAm changes and PTSD, using the gene-based test results from the recent PGC-

PTSD Freeze 3 GWAS(11) using a Bonferroni correction for the 11 genes examined.  

18 

 
 
 
 
 
 
medRxiv preprint 

doi: 

https://doi.org/10.1101/2024.07.15.24310422

; 

this version posted July 15, 2024. 

The copyright holder for this preprint 

(which was not certified by peer review)

 is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. 

It is made available under a 

CC-BY-NC-ND 4.0 International license

 . 

Results 

Epigenome-wide association meta-analysis 

We identified 11 PTSD-associated CpGs that passed the epigenome-wide significance 

threshold (p < 9e-08, Table 2, Figure 1, eFigure 1 in the Supplement). All CpG sites, except one 

site (cg21161138) near AHRR, remained nominally significant (1.52e-03 < p < 2.10e-07) with the 

same direction of association in the sensitivity analysis adjusted for smoking score (eTable 1 and 

eFigure 1 in the Supplement).  

Stratified analyses for sex, ancestry, and cohort type 

The directions of associations of the 11 significant CpGs were consistent at nominal 

significance (p < 0.05) when stratified by both sex, cohort type (civilian and military), and 

European ancestry. However, in the African ancestry stratified analysis, only 8 CpGs remained 

associated (p < 0.05) with PTSD (eTable 2 in the Supplement). We identified 1 epigenome-wide 

significant CpG site associated with PTSD in the female-stratified analysis, 1 in the male-

stratified analysis, 5 in European ancestry-stratified analysis, 1 in the African ancestry-stratified 

analysis, 1 in the analysis for civilian-cohort analysis, and 5 in the military-cohort analysis (Table 

3, eFigures 2-8 in the Supplement). Of note, some CpGs identified in the stratified analyses were 

specific to their stratum and were not associated with PTSD (p > 0.05) in the other stratum. For 

instance, cg25691167 in FERD3L was associated with PTSD in females (p = 4.24E-08), but not in 

males (p = 0.56) or the primarily male military cohorts (p = 0.13).  

19 

 
 
 
 
medRxiv preprint 

doi: 

https://doi.org/10.1101/2024.07.15.24310422

; 

this version posted July 15, 2024. 

The copyright holder for this preprint 

(which was not certified by peer review)

 is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. 

It is made available under a 

CC-BY-NC-ND 4.0 International license

 . 

Gene enrichment analysis 

We identified 3 GO term enrichments in the female-stratified analysis (FDR < 0.05), 

including nervous system development (eTable 3 in the Supplement). We did not identify any 

significant enrichments in the main analysis or other strata. 

Cross-tissue associations 

Of the 19 significant CpGs from the primary and stratified analyses, 3 were not available 

in the Blood Brain DNA Methylation Comparison Tool database. Methylation at 9 of the 

remaining 16 CpGs was correlated between blood and at least one brain region (Figure 2, 

eTable4 in the Supplement). The strongest correlation was observed between the blood and 

PFC (r = 0.91) for cg23576855 (AHRR).  

We next evaluated whether the PTSD-associated CpGs from blood were also associated 

with PTSD in the brain regions of dlPFC, vmPFC, amygdala, and DG, as well as neuronal nuclei. 

Out of 19 CpGs from the primary and stratified analyses in bulk tissue, 9 were nominally 

associated with PTSD in at least one brain region or the neuronal nuclei (p < 0.05, Figure 2, 

eTables 5 and 6 in the Supplement). For instance, PTSD cases had higher cg04987734 

(CDC42BPB) methylation both in the blood (p = 3.26e-8) and the dlPFC (p = 3.9e-3). In addition, 

7 CpGs exhibited nominally significant methylation changes (5 CpGs with the same direction of 

association) in fibroblasts when subjected to physiological stress hormone levels in a  cellular 

model of prolonged stress (0.024 < DNAm < 0.225, p < 0.05, Figure 2, eTable 7 in the 

Supplement)(49). Notably, the CpG sites (cg21161138 [AHRR], cg25320328 [GFI], cg19719391 

20 

 
 
medRxiv preprint 

doi: 

https://doi.org/10.1101/2024.07.15.24310422

; 

this version posted July 15, 2024. 

The copyright holder for this preprint 

(which was not certified by peer review)

 is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. 

It is made available under a 

CC-BY-NC-ND 4.0 International license

 . 

[intergenic], cg25691167 [FERD3L]) that were consistently associated with PTSD in blood and 

postmortem brain also demonstrated a significant change in this cell model. 

Gene Regulation 

Of the 11 PTSD-associated CpGs annotated to a gene, 10 correlated with their gene 

expression in at least one of the 4 RNA-seq cohorts (Figure 2, eTable 8 in the Supplement). 

Meta-analysis across cohorts identified 4 CpGs whose methylation levels were negatively 

correlated with their annotated gene expression after multiple test corrections: cg05575921, 

cg21161138, and cg23576855 with AHRR, and cg27541344 with BCL11B (p < 4.5e-03, -16.69  z 

 -4.86 eTable 8 in the Supplement).  

Genetic regulation of PTSD-associated CpGs  

Of the 19 CpGs from the primary and stratified analyses, 12 CpGs were associated with 

at least one nearby SNP within 1 MB according to GoDMC(51) and meQTL EPIC(53) databases 

(eTable 9 in the Supplement). In total, we identified 41 lead meQTLs, of which 8 were nominally 

associated with PTSD (p < 0.05, albeit in the opposite direction) in the PGC-PTSD Freeze 3 

GWAS(11) (eTable 9 in the Supplement). For instance, lower methylation at the intergenic 

cg14753356 site is associated with PTSD (z = -5.72, p = 1.09E-08, Table 2) and higher 

methylation at cg14753356 is associated with rs28986310 T allele (beta = 0.31, p < 5e-324), 

which increases the risk of PTSD (z = 6.73, p = 1.68e-11, eTable 9 in the Supplement).  

In addition, of the 11 genes harboring PTSD-associated CpGs, AHRR, CDC42BPB, and 

BCL11B were implicated in the recent PGC-PTSD gene-based analysis (3.48e-03 < p < 5.40e-05, 

21 

 
 
 
medRxiv preprint 

doi: 

https://doi.org/10.1101/2024.07.15.24310422

; 

this version posted July 15, 2024. 

The copyright holder for this preprint 

(which was not certified by peer review)

 is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. 

It is made available under a 

CC-BY-NC-ND 4.0 International license

 . 

eTable 10 in the Supplement). All genes are still significant after Bonferroni correction for 11 

tests (p < 4.5e-03). 

Discussion 

In the present epigenome-wide meta-analysis of blood DNAm levels, we identified 11 

CpG sites associated with PTSD. Many of these are also associated with PTSD in multiple brain 

regions and a cellular model of prolonged stress exposure (49). Many of the PTSD-associated 

CpGs were associated with the expression of their respective genes. An examination of the most 

recent PGC-PTSD GWAS (11) indicated that many SNPs near significant CpGs were associated 

with both DNAm levels and PTSD diagnosis. Moreover, many of the EWAS-implicated genes 

were significant in gene-based tests from the GWAS. Thus, the GWAS results represent further 

evidence supporting the role of these genes in PTSD pathogenesis. However, the lower 

significance of the SNPs in association with PTSD relative to the corresponding CpGs (eTable 10 

in the Supplement) indicates that the association between PTSD and the CpGs is likely not 

simply a byproduct of PTSD-associated SNPs. 

Our CDC42BPB (CDC42 binding protein kinase beta) findings are of particular interest. 

CDC42BPB is involved in the regulation of cytoskeletal rearrangement, cell migration, and 

neurodevelopment(54). In the current study, increased CDC42BPB methylation at cg04987734 

was associated with PTSD in blood and the dlPFC. Methylation at this site was also positively 

correlated with CDC42BPB expression in blood. Notably, higher methylation at cg04987734 has 

been associated with depressive symptoms (55) and increased C-reactive protein (CRP) levels 

(56, 57), which is perhaps not surprising given the bi-directional genetic association between 

22 

 
 
medRxiv preprint 

doi: 

https://doi.org/10.1101/2024.07.15.24310422

; 

this version posted July 15, 2024. 

The copyright holder for this preprint 

(which was not certified by peer review)

 is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. 

It is made available under a 

CC-BY-NC-ND 4.0 International license

 . 

PTSD and CRP levels (58). Multiple studies reported increased CRP levels and other 

inflammatory markers in those with PTSD, suggesting inflammation as an important component 

of PTSD (59-61). Future studies are warranted to investigate CDC42BPB methylation as a 

potential mediator of the relationship between PTSD and inflammation. 

We observed 3 CpGs (cg05575921, cg21161138, and cg23576855) in AHRR (aryl-

hydrocarbon receptor repressor) that were identified in the initial PGC-PTSD EWAS (31) and an 

independent study of US veterans (45). The aryl hydrocarbon receptor (AhR) plays a role in 

immunomodulation, including the regulation of T lymphocytes, B cell maturation, and the 

activity of macrophages, dendritic cells, and neutrophils (62), supporting the link between the 

immune system and PTSD. Although methylation at the AHRR CpGs is known to be influenced 

by smoking (63), cg05575921 and cg23576855 remained nominally significant (p = 1.52E-03 and 

p = 7.59E-04, respectively) in our sensitivity analysis adjusting for DNAm-based smoking scores. 

Notably, cg21161138 DNAm was also associated with PTSD in postmortem dlPFC and neuronal 

nuclei and demonstrated a significant change in the cellular model of prolonged stress (49), 

supporting the notion that there is an association between methylation at this locus and PTSD 

which is independent of the effects of smoking.  

The stratified analyses identified DNAm-PTSD associations specific to sex, ancestry, and 

cohort type. The PTSD-associated site cg25691167 (FERD3L) in females (p = 4.24e-08) was not 

associated with PTSD in males (p = 0.57), suggesting that DNAm changes in cg25691167 might 

be sex-specific. Similarly, cg27541344 (BCL11B) was associated with PTSD in the civilian (p = 

7.21e-08), but not the military cohorts (p = 0.38), whereas 3 out of 5 PTSD-associated CpGs in 

the military cohorts were not significant in the civilian cohorts (p > 0.05). We speculate that 

23 

 
medRxiv preprint 

doi: 

https://doi.org/10.1101/2024.07.15.24310422

; 

this version posted July 15, 2024. 

The copyright holder for this preprint 

(which was not certified by peer review)

 is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. 

It is made available under a 

CC-BY-NC-ND 4.0 International license

 . 

some of these DNAm changes specific to sex, ancestry, or cohort type might be due to unique 

characteristics of those strata, such as hormonal factors and environmental exposures.   

Strengths and Limitations 

To our knowledge, this is the largest EWAS of PTSD to date. Our sample is diverse in 

terms of sex, ancestry, and cohort type. We leveraged data from postmortem brain samples, a 

cellular model of prolonged stress, GWAS, and genome-wide expression data to support our 

findings. 

However, the study is not without limitations. First, methylation BeadChips only assess a 

subset of CpG sites in the genome; therefore, we may not capture all PTSD-associated CpG sites. 

Second, this is a cross-sectional study of participants with prior exposure to a traumatic event; 

thus, we were not able to assess whether the differences in DNAm between individuals with 

and without PTSD are a cause or consequence of PTSD or both. Third, our primary meta-analysis 

was performed using measures of blood DNAm. While this strategy provided valuable insights 

for future research on biomarkers of PTSD, it might not accurately represent the DNAm patterns 

within other tissues that are likely the most relevant to PTSD. However, the majority of the 

PTSD-associated CpG sites’ methylation levels were correlated between blood and at least one 

brain region. In addition, most PTSD-associated CpGs in blood were also associated with PTSD in 

one or more brain regions. Fourth, we used bulk tissue and adjusted for cellular heterogeneity, 

which might have obscured some signals, given the alterations in cell composition in those with 

PTSD(64, 65). Additionally, the brain regions examined varied between the online databases 

used to examine the blood-brain correlation of methylation values and the gene expression 

24 

 
 
medRxiv preprint 

doi: 

https://doi.org/10.1101/2024.07.15.24310422

; 

this version posted July 15, 2024. 

The copyright holder for this preprint 

(which was not certified by peer review)

 is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. 

It is made available under a 

CC-BY-NC-ND 4.0 International license

 . 

differences associated with PTSD, which can complicate interpretation. Finally, most cohorts 

that participated in the meta-analysis did not have detailed physical or psychiatric information 

on participants, including detailed information on trauma type and timing, PTSD symptom 

course, and treatment, making it challenging to evaluate and adjust for potential confounders, 

including substance use, comorbidities, or medication use.  

Conclusions 

Taken together, this study replicates our previous findings and identifies novel PTSD-

associated CpGs. Supporting data from multiple sources suggest that epigenetic mechanisms, 

particularly methylation in AHRR and CDC42BPB, may contribute to the complex relationship 

between the immune system and PTSD. 

List of abbreviations 

CpG 

DG 

Cytosine-guanine dinucleotides  

Dentate gyrus  

dlPFC 

Dorsolateral prefrontal cortex  

DNAm   

DNA methylation 

EWAS   

Epigenome-wide association study 

FANS 

Fluorescence-Activated Nuclei Sorting  

GWAS   

Genome-wide association study 

meQTL  

Methylation quantitative trait locus  

NK 

Natural killer  

25 

 
 
 
 
 
 
 
 
medRxiv preprint 

doi: 

https://doi.org/10.1101/2024.07.15.24310422

; 

this version posted July 15, 2024. 

The copyright holder for this preprint 

(which was not certified by peer review)

 is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. 

It is made available under a 

CC-BY-NC-ND 4.0 International license

 . 

OFC 

PC 

Orbitofrontal cortex  

Principal component 

PGC-PTSD 

Psychiatric Genomics Consortium PTSD Workgroup  

PTSD 

Posttraumatic stress disorder  

RPC 

Robust Partial Correlation  

RRoxBS 

Reduced representation oxidative bisulfite-sequencing  

vmPFC  

Ventromedial prefrontal cortex 

Declarations 

Ethics approval and consent to participate 

All participants in these studies gave informed consent. The institutional review boards of each 

respective institution approved these studies. 

Consent for publication 

Not applicable. 

Availability of data and materials 

The main summary statistics data that support the findings of this study will be available within 

Supplementary Data upon publication. Individual-level data from the cohorts or cohort-level 

summary statistics will be made available to researchers following an approved analysis 

proposal through the PGC-PTSD Epigenetics Workgroup with agreement of the cohort PIs. The 

raw data for the GTP cohort is available in the Gene Expression Omnibus database with the 

26 

 
 
 
 
 
 
 
 
medRxiv preprint 

doi: 

https://doi.org/10.1101/2024.07.15.24310422

; 

this version posted July 15, 2024. 

The copyright holder for this preprint 

(which was not certified by peer review)

 is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. 

It is made available under a 

CC-BY-NC-ND 4.0 International license

 . 

accession code GSE132203. Owing to military cohort data sharing restrictions, data from the VA, 

VA MIRECC, MRS, Army STARRS, PRISMO, PROGrESS, and NCPTSD/TRACTS cannot be publicly 

posted. However, such data can be provided in de-identified from a data repository through a 

data use agreement following applicable guidelines on data sharing and privacy protection. For 

additional information on access to these data, including PI contact information for the 

contributing cohorts, please contact the corresponding author. 

Competing interests 

CYC is an employee of Biogen. NPD has served on scientific advisory boards for BioVie Pharma, 

Circular Genomics and Sentio Solutions for unrelated work. NRN serves as an unpaid member of 

the Ilumivu advisory board. SAMR receives support from the Wounded Warrior Project (WWP), 

Department of Veterans Affairs (VA), National Institute of Health (NIH), McCormick Foundation, 

Tonix Pharmaceuticals, Woodruff Foundation, and Department of Defense (DOD). Dr. Rauch 

receives royalties from Oxford University Press and American. KJR serves as a consultant for 

Acer, Bionomics, and Jazz Pharma; SABs for Sage, Boehringer Ingelheim, and Senseye. DJS has 

received consultancy honoraria from Discovery Vitality, Johnson & Johnson, Kanna, L’Oreal, 

Lundbeck, Orion, Sanofi, Servier, Takeda and Vistagen. MBS has in the past 3 years received 

consulting income from Acadia Pharmaceuticals, Aptinyx, atai Life Sciences, BigHealth, Biogen, 

Bionomics, BioXcel Therapeutics, Boehringer Ingelheim, Clexio, Delix Therapeutics, Eisai, 

EmpowerPharm, Engrail Therapeutics, Janssen, Jazz Pharmaceuticals, NeuroTrauma Sciences, 

PureTech Health, Sage Therapeutics, Sumitomo Pharma, and Roche/Genentech. MBS has stock 

options in Oxeia Biopharmaceuticals and EpiVario. MBS has been paid for his editorial work on 

27 

 
 
medRxiv preprint 

doi: 

https://doi.org/10.1101/2024.07.15.24310422

; 

this version posted July 15, 2024. 

The copyright holder for this preprint 

(which was not certified by peer review)

 is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. 

It is made available under a 

CC-BY-NC-ND 4.0 International license

 . 

Depression and Anxiety (Editor-in-Chief), Biological Psychiatry (Deputy Editor), and UpToDate 

(Co-Editor-in-Chief for Psychiatry). MBS has also received research support from NIH, 

Department of Veterans Affairs, and the Department of Defense. MBS is on the scientific 

advisory board for the Brain and Behavior Research Foundation and the Anxiety and Depression 

Association of America. 

Funding 

This work was supported by the National Institute of Mental Health (NIMH; R01MH108826 and 

R01MH106595). This work was also supported by I01BX003477, a Department of Veterans 

Affairs BLR&D grant to MWL; 1R03AG051877, 1R21AG061367-01, RF1AG068121, and 

1I01CX001276-01A2 to EJW; R21MH102834, 5I01CX000431, 5R01MH079806 to MWM; 

R01MD011728 to MU; R01MH105379 to NRN; R01MH117291, R01MH117292, R01MH117293 

to FAC; R01MH108826 to NPD; U01MH115485 to LM, SF, SJ, JM, AU, and DEW; R01MH117291, 

R01MH117292, R01MH117293 to JEK; CDC/NIOSH U01OH012466 to PFK; R01MH117291, 

R01MH117292, R01MH117293 to CBN; R01MH093500 to CMN; 1R15MH099521-01, 

5R21MH085436-02, 1R15HD049907-01A1 to HKO; R01MH108826 to AKS; I01BX002577, 

IK2CX000525, and lK6BX003777 to NAK; U01MH110925 to SAM; R21DA050160 and 

1DP1DA058737 to JMO;  Department of Veterans Affairs B9254-C to WM, B3001-C to CF; 

Department of Defense W81XWH-11-1-0073 and the National Center for Advancing 

Translational Sciences of the NIH UL1TR000433PEC-PTSD to SAMR; BX006186; BX005872 to 

VBR; VIDI award (91718336) from the Netherlands Scientific Organization to BPFR; Dutch 

Research Council (NWO) VIDI grant (09150171910042) to CHV. Brainomics work was supported 

28 

 
 
medRxiv preprint 

doi: 

https://doi.org/10.1101/2024.07.15.24310422

; 

this version posted July 15, 2024. 

The copyright holder for this preprint 

(which was not certified by peer review)

 is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. 

It is made available under a 

CC-BY-NC-ND 4.0 International license

 . 

by R01MH117291, R01MH117292, and R01MH117293. DJS and NK was supported by the South 

African Medical Research Council and the Bill and Melinda Gates Foundation (OPP 1017641). SS 

is supported by the South African Medical Research Council. SK is supported by the Building 

Interdisciplinary Research Careers in Women’s Health of the National Institutes of Health under 

Award Number K12HD085850.  

Author Contributions 

PGC-PTSD writing group: S.K., M.W.L., C.M.N., A.K.S., and M.U. 

Study PI or co-PI: A.E.A., A.E.A.-K., D.G.B., J.C.B., E.B., C.F., S.G., E.G., G.G., M.A.H., S.J., R.C.K., 

N.A.K., J.E.K., K.C.K., P.-F.K., M.W.L., B.L., C.E.M., S.A.M., W.M., M.W.M., L.M., C.B.N., C.M.N., 

N.R.N., H.K.O., S.A.M.R., K.J.R., V.R., S.S., A.K.S., D.J.S., M.B.S., M.U., R.J.U., E.V., D.E.W., E.J.W., 

and R.M.Y 

Obtained funding for studies: J.C.B., M.P.B., S.F., C.F., E.G., M.A.H., R.C.K., K.C.K., M.W.L., J.J.L., 

S.A.M., M.W.M., C.M.N., N.R.N., H.K.O., S.A.M.R., K.J.R., B.P.F.R., A.K.S., M.U., R.J.U., and E.V. 

Clinical: D.G.B., J.C.B., M.F.D., S.F., C.F., E.G., J.P.H., N.A.K., N.K., C.M., J.M., S.A.M.R., K.J.R., E.R., 

A.U., M.H.V., E.V., E.J.W., and L.L.V.D.H. 

Contributed data: A.E.A.-K., D.G.B., J.C.B., F.A.C., N.P.D., M.F.D., C.F., M.A.H., J.P.H., S.M.J.H., 

B.R.H., S.J., S.K., N.A.K., A.P.K., K.C.K., P.-F.K., I.L., A.L, B.L., J.J.L., D.M., W.M., M.W.M., J.M.-O., 

C.M., L.M., C.M.N., N.R.N., S.A.M.R., V.R., S.S., A.K.S., M.B.S., S.T., M.U., A.U., M.H.V., D.E.W., 

E.J.W., A.S.Z., and L.L.V.D.H. 

Statistical analysis: A.E.A.-K., D.A., M.P.B., L.B., C.-Y.C., S.D., N.P.D., S.F., M.E.G., A.J., S.K., A.P.K., 

I.L., M.W.L., A.X.M., D.M., M.S.M., C.M.N., D.L.N.-R., X-J.Q., A.R., B.P.F.R., A.K.S., C.H.V., A.H.W., 

29 

 
 
medRxiv preprint 

doi: 

https://doi.org/10.1101/2024.07.15.24310422

; 

this version posted July 15, 2024. 

The copyright holder for this preprint 

(which was not certified by peer review)

 is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. 

It is made available under a 

CC-BY-NC-ND 4.0 International license

 . 

E.B.W., A.S.Z., X.Z., and Y.Z. 

Bioinformatics: M.P.B., L.B., C.-Y.C., N.P.D., M.E.G., M.A.H., A.J., S.K., S.D.L., M.W.L., A.X.M., A.R., 

B.P.F.R., A.H.W., X.Z., and Y.Z. 

Genomics: M.P.B., M.A.H., S.D.L., A.X.M., B.P.F.R., C.H.V., A.S.Z., and X.Z. 

PI of the EWAS group: M.W.L., C.M.N., A.K.S., and M.U. 

Acknowledgments 

PGC-PTSD Epigenetics Workgroup: Reid S Alisch, Ananda B Amstadter, Don Armstrong, Archana 

Basu, Jean C Beckham, Nicole L Bjorklund, Barbara H Chaiyachati, Judith B M Ensink, Segun 

Fatumo, Leland L Fleming, Sandro Galea, Joel Gelernter, Ryan J Herringa, Sonia Jain, Diana L 

Juvinao-Quintero, Seyma Katrinli, Elizabeth Ketema, José J Martínez-Magaña, Burook Misganaw, 

Shiela Tiemi Nagamatsu, Danny M Nispeling, John Pfeiffer, Christian Schmahl, Gen Shinozaki, 

Clara Snijders, Jennifer A Sumner, Patricia C Swart, Audrey Tyrka, Robert J Ursano, Mirjam van 

Zuiden, Eric Vermetten, Jaqueline S Womersley, Nagy A Youssef, Yuanchao Zheng, Yiwen Zhu, 

Lea Zillich  

PsychENCODE PTSD Brainomics Project: Dhivya Arasappan, Sabina Berretta, Rahul A. Bharadwaj, 

Frances A. Champagne, Leonardo Collado-Torres, Christos Chatzinakos, Nikolaos P. Daskalakis, 

Chris P. DiPietro, Duc M. Duong, Amy Deep-Soboslay, Nick Eagles, Louise Huuki, Thomas Hyde, 

Artemis Iatrou, Aarti Jajoo, Joel E. Kleinman, Charles B. Nemeroff, Geo Pertea, Deanna Ross, 

Nicholas T. Seyfried, Joo Heon Shin, Kerry J.Ressler, Clara Snijders, Ran Tao, Daniel R. 

Weinberger, Stefan Wuchty, Dennis Wylie 

30 

 
 
medRxiv preprint 

doi: 

https://doi.org/10.1101/2024.07.15.24310422

; 

this version posted July 15, 2024. 

The copyright holder for this preprint 

(which was not certified by peer review)

 is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. 

It is made available under a 

CC-BY-NC-ND 4.0 International license

 . 

Traumatic Stress Brain Research Group: Victor E. Alvarez, David Benedek, Alicia Che, Dianne A. 

Cruz, David A. Davis, Matthew J. Girgenti, Ellen Hoffman, Paul E. Holtzheimer, Bertrand R. Huber, 

Alfred Kaye, John H. Krystal, Adam T. Labadorf, Terence M. Keane, Ann McKee, Brian Marx, 

Crystal Noller, Meghan Pierce, William K. Scott, Paula Schnurr, Krista DiSano, Thor Stein,  

Douglas E. Williamson, Keith A. Young 

31 

 
 
medRxiv preprint 

doi: 

https://doi.org/10.1101/2024.07.15.24310422

; 

this version posted July 15, 2024. 

The copyright holder for this preprint 

(which was not certified by peer review)

 is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. 

It is made available under a 

CC-BY-NC-ND 4.0 International license

 . 

Table 1. Overview of the studies 

Cohort 

Array  N 

Cases 
N (%) 

Controls 
N (%) 

Female  
N (%) 

European 
N (%) 

African 
N (%) 

Age 
mean (SD) 

Civilian 
BEAR 
DNHS-1 
DCHS 
GTP-1 
NIU 
Shared Roots 
AURORA 
H3A_Rwanda 
DNHS-2 
GTP-2 
WTC 
Civilian Total 

EPIC  162 
EPIC  423 
EPIC  95 
EPIC  479 
EPIC  140 
EPIC  120 
EPIC  206 
EPIC  73 
450K  100 
450K  265 
450K  180 

126 (78%)  119 (73%)  112 (69%)  3 (2%) 
36 (22%) 
397 (94%)  255 (60%)  23 (5%) 
26 (6%) 
95 (100%)  0 (0%) 
46 (48%) 
49 (52%) 
158 (33%)  321 (67%)  340 (71%)  12 (3%) 
18 (13%) 
61 (51%) 
57 (28%) 
32 (44%) 
40 (40%) 
74 (28%) 
84 (47%) 

15.16 (1.45) 
384 (91%)  54.54 (16.87) 
54 (57%) 
448 (94%)  42.22 (12.25) 
26.01 (1.74) 
120 (100%)  43.15 (10.77) 
131 (64%)  39.24 (14.17) 
73 (100%)  45.54 (7.29) 
53.6 (14.01) 
87 (87%) 
249 (94%)  41.95 (12.37) 
49.72 (8.25) 
2243  632 (28%)  1611 (72%)  1508 (67%)  491 (22%)  1575 (70%)  41.89 (16.27) 

122 (87%)  140 (100%)  110 (79%)  19 (14%) 
85 (71%) 
59 (49%) 
149 (72%)  154 (75%)  67 (33%) 
73 (100%)  0 (0%) 
41 (56%) 
13 (13%) 
60 (60%) 
60 (60%) 
191 (72%)  187 (71%)  16 (6%) 
0 (0%) 
96 (53%) 

138 (77%)  7 (4%) 

26.81 (5.2) 

0 (0%) 

14 (10%) 

63 (50%) 
30 (48%) 
51 (50%) 

EPIC  96 
EPIC  127 
EPIC  89 
EPIC  216 
EPIC  140 

96 (100%)  0 (0%) 
5 (4%) 
88 (69%) 
74 (83%) 
3 (3%) 
149 (69%)  22 (10%) 
40 (29%) 
89 (64%) 

0 (0%) 
48 (50%) 
48 (50%) 
0 (0%) 
64 (50%) 
63 (50%) 
9 (10%) 
24 (27%) 
65 (73%) 
106 (49%)  110 (51%)  0 (0%) 
112 (80%)  28 (20%) 

Military 
68.67 (4.36) 
GMRFQUT 
23.07 (2.18) 
MRS-1 
27.51 (8.63) 
PRISMO-1 
25.13 (4.82) 
Army STARRS-1 
34.77 (8.33) 
PROGrESS 
NCPTSD/TRACTS  EPIC  1028  638 (62%)  390 (38%)  231 (22%)  706 (69%)  123 (12%)  44.06 (13.7) 
22.2 (3.04) 
MRS-2 
27.1 (9.23) 
PRISMO-2 
23.79 (4.25) 
Army STARRS-2 
INTRuST 
34.09 (11.68) 
369 (100%)  38.36 (9.36) 
VA-M-AA 
34.87 (9.89) 
VA-M-EA 
2834  1524 (54%)  1310 (46%)  578 (20%)  1820 (64%)  630 (22%)  37.08 (14.33) 
Military Total 
5077  2156 (42%)  2921 (58%)  2086 (41%)  2311 (46%)  2205 (43%) 39.2 (15.4) 

0 (0%) 
63 (50%) 
0 (0%) 
32 (52%) 
0 (0%) 
51 (50%) 
116 (38%)  187 (62%)  102 (34%)  206 (68%)  58 (19%) 
183 (50%)  186 (50%)  184 (50%)  0 (0%) 
87 (49%) 

450K  126 
450K  62 
450K  102 
450K  303 
450K  369 
450K  176 

72 (57%) 
10 (8%) 
62 (100%)  0 (0%) 
102 (100%)  0 (0%) 

TOTAL 
Participating civilian cohorts: Biomarkers, social, and affective predictors of suicidal thoughts 
and behaviors in adolescents (BEAR), Detroit Neighborhood Health Study (DNHS), Drakenstein 
Child Health Study (DCHS), Grady Trauma Project (GTP), Northern Illinois University Trauma 
Study (NIU), Shared Roots, Advancing Understanding of RecOvery afteR traumA (AURORA), 
Human Heredity and Health in Africa, Rwanda (H3A_Rwanda), World Trade Center 9/11 
Responders (WTC). Participating military cohorts: Gallipoli Medical Research Foundation 
Queensland University of Technology (GMRFQUT), Marine Resiliency Study (MRS), Prospective 
Research in Stress-related Military Operations (PRISMO), Army Study to Assess Risk and 
Resilience in Servicemembers (Army STARRS), PROlonGed ExpoSure and Sertraline Trial 

176 (100%)  0 (0%) 

89 (51%) 

38 (22%) 

32 

 
 
 
 
 
 
 
 
 
 
medRxiv preprint 

doi: 

https://doi.org/10.1101/2024.07.15.24310422

; 

this version posted July 15, 2024. 

The copyright holder for this preprint 

(which was not certified by peer review)

 is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. 

It is made available under a 

CC-BY-NC-ND 4.0 International license

 . 

(PROGrESS), Boston VA - National Center for PTSD/ Translational Research Center for TBI and 
Stress Disorders (NCPTSD/TRACTS), Injury and Traumatic Stress study (INTRuST), and Veterans 
Affairs’ Mental Illness Research, Education and Clinical Centers (VA-M-AA & VA-M-EA). Note: For 
DNHS, GTP, MRS, PRISMO, and Army STARRS cohorts EPIC and 450k datasets represent different 
sets of subjects. 

33 

 
 
 
medRxiv preprint 

doi: 

https://doi.org/10.1101/2024.07.15.24310422

; 

this version posted July 15, 2024. 

The copyright holder for this preprint 

(which was not certified by peer review)

 is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. 

It is made available under a 

CC-BY-NC-ND 4.0 International license

 . 

Table 2. CpG sites associated with current PTSD in the primary meta-analysis 

CpG 
cg05575921 
cg21161138 
cg04583842 
cg14753356 
cg25320328 
cg26599989 
cg19719391 
cg16758086 
cg04987734 
cg23576855 
cg09822192 
Position is based on hg19. 

Position 
chr5:373378 
chr5:399360 
chr16:88103117 
chr6:30720108 
chr1:92953037 
chr11:1297087 
chr4:26789915 
chr1: 6173356 
chr14:103415873 
chr5:373299 
chr14:24801191 

Gene 
AHRR 
AHRR 
BANP 
Intergenic 
GFI1 
TOLLIP 
Intergenic 
CHD5 
CDC42BPB 
AHRR 
ADCY4 

z 
-6.05 
-5.96 
5.81 
-5.72 
-5.65 
5.58 
5.58 
5.55 
5.53 
-5.52 
5.44 

p-value 
1.44E-09 
2.50E-09 
6.29E-09 
1.09E-08 
1.64E-08 
2.35E-08 
2.45E-08 
2.85E-08 
3.26E-08 
3.44E-08 
5.30E-08 

34 

 
 
 
medRxiv preprint 

doi: 

https://doi.org/10.1101/2024.07.15.24310422

; 

this version posted July 15, 2024. 

The copyright holder for this preprint 

(which was not certified by peer review)

 is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. 

It is made available under a 

CC-BY-NC-ND 4.0 International license

 . 

Table 3. CpG sites associated with current PTSD in the stratified analyses 

z 

5.48 

-6.12 

Gene 

FERD3L 

p-value 

Position 

4.24E-08 

9.30E-10* 

-6.85 
5.76 
-5.62 
-5.46 
5.43 

AHRR 
Intergenic 
BANP 

CpG 
Stratified analysis for females 
cg25691167  chr7:19184961 
Stratified analysis for males 
AHRR 
cg05575921  chr5:373378 
Stratified analysis for European ancestry 
cg05575921  chr5:373378 
AHRR 
cg11256214  chr12:110211642  MGC14436 
cg21161138  chr5:399360 
cg15977432  chr19:56709655 
cg04583842  chr16:88103117 
Stratified analysis for African ancestry 
cg02003183  chr14:103415882  CDC42BPB 
Stratified analysis for civilian trauma 
cg27541344  chr14:99650422 
Stratified analysis for military trauma 
cg05575921  chr5:373378 
cg00774777  chr11:76478902 
cg14753356  chr6:30720108 
cg21566642  chr2:233284661 
cg03329539  chr2:233283329 
Position is based on hg19. The sites that were also epigenome-wide significant (p < 9e-08) in the 
primary meta-analysis were indicated by an asterisk (*). 

AHRR 
RP11-21L23.4 
Intergenic 
Intergenic 
Intergenic 

7.24E-12* 
8.17E-09 
1.95E-08* 
4.68E-08 
5.66E-08* 

4.92E-10* 
2.79E-08 
4.53E-08* 
6.36E-08 
8.53E-08 

-6.22 
5.55 
-5.47 
-5.41 
-5.36 

7.2073E-08 

4.26E-08 

BCL11B 

5.39 

5.48 

35 

 
 
 
medRxiv preprint 

doi: 

https://doi.org/10.1101/2024.07.15.24310422

; 

this version posted July 15, 2024. 

The copyright holder for this preprint 

(which was not certified by peer review)

 is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. 

It is made available under a 

CC-BY-NC-ND 4.0 International license

 . 

Figures 

Figure 1. Manhattan Plot of the epigenome-wide association meta-analyses. The x-axis depicts 

chromosomes and the location of each CpG site across the genome. The y-axis depicts the -

log10 of the unadjusted p-value for the association with current PTSD. Each dot represents a 

CpG site. The solid blue line indicates the epigenome-wide statistical significance at p < 9.0e-8. 

36 

 
 
 
 
medRxiv preprint 

doi: 

https://doi.org/10.1101/2024.07.15.24310422

; 

this version posted July 15, 2024. 

The copyright holder for this preprint 

(which was not certified by peer review)

 is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. 

It is made available under a 

CC-BY-NC-ND 4.0 International license

 . 

Fig 2. Summary of all analyses and findings. Figure combines CpGs from the main analysis 

(gold) and stratified analyses for sex, ancestry, and trauma type (light gold); and summarizes the 

results of blood and brain correlations (rose); cross-tissue associations for multiple brain regions 

(light blue), neuronal nuclei (blue), and a fibroblast model of prolonged stress (aqua), gene 

expression (purple); and genetic effects, including methylation quantitative trait loci (meQTL) 

analyses (light green) and genetic associations from the recent PGC-PTSD GWAS (dark green). 

Positive findings (p < 0.05) are indicated with the specific color of the respective category. 

Asterisk (*) indicates epigenome-wide significance (p < 9e-8). Gray represents the CpGs or genes 

were not present in the respective datasets. PFC: prefrontal cortex. EC: entorhinal cortex. STG: 

superior temporal gyrus. CER: cerebellum. dlPFC: dorsolateral prefrontal cortex. vmPFC: 

ventromedial prefrontal cortex. DG: dentate gyrus. 5mC: 5-Methylcytosine. 5hmC: 5-

Hydroxymethylcytosine. GC: glucocorticoid.    

37 

 
 
 
medRxiv preprint 

doi: 

https://doi.org/10.1101/2024.07.15.24310422

; 

this version posted July 15, 2024. 

The copyright holder for this preprint 

(which was not certified by peer review)

 is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. 

It is made available under a 

CC-BY-NC-ND 4.0 International license

 . 

References 

1. 

American Psychiatric Association. Diagnostic and Statistical Manual of Mental Disorders. 

2013. 

2. 

Zen AL, Whooley MA, Zhao S, Cohen BE. Post-traumatic stress disorder is associated with 

poor health behaviors: findings from the heart and soul study. Health Psychol. 2012;31(2):194-

201. 

3. 

Jacobsen LK, Southwick SM, Kosten TR. Substance use disorders in patients with 

posttraumatic stress disorder: a review of the literature. Am J Psychiatry. 2001;158(8):1184-90. 

4. 

O'Donovan A, Cohen BE, Seal KH, Bertenthal D, Margaretten M, Nishimi K, et al. Elevated 

risk for autoimmune disorders in iraq and afghanistan veterans with posttraumatic stress 

disorder. Biol Psychiatry. 2015;77(4):365-74. 

5. 

Ryder AL, Azcarate PM, Cohen BE. PTSD and Physical Health. Curr Psychiatry Rep. 

2018;20(12):116. 

6. 

Benjet C, Bromet E, Karam EG, Kessler RC, McLaughlin KA, Ruscio AM, et al. The 

epidemiology of traumatic event exposure worldwide: results from the World Mental Health 

Survey Consortium. Psychol Med. 2016;46(2):327-43. 

7. 

Logue MW, Amstadter AB, Baker DG, Duncan L, Koenen KC, Liberzon I, et al. The 

Psychiatric Genomics Consortium Posttraumatic Stress Disorder Workgroup: Posttraumatic 

Stress Disorder Enters the Age of Large-Scale Genomic Collaboration. 

Neuropsychopharmacology. 2015;40(10):2287-97. 

38 

 
medRxiv preprint 

doi: 

https://doi.org/10.1101/2024.07.15.24310422

; 

this version posted July 15, 2024. 

The copyright holder for this preprint 

(which was not certified by peer review)

 is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. 

It is made available under a 

CC-BY-NC-ND 4.0 International license

 . 

8. 

Nievergelt CM, Ashley-Koch AE, Dalvie S, Hauser MA, Morey RA, Smith AK, et al. 

Genomic Approaches to Posttraumatic Stress Disorder: The Psychiatric Genomic Consortium 

Initiative. Biol Psychiatry. 2018;83(10):831-9. 

9. 

Polimanti R, Wendt FR. Posttraumatic stress disorder: from gene discovery to disease 

biology. Psychol Med. 2021;51(13):2178-88. 

10. 

Duncan LE, Cooper BN, Shen H. Robust Findings From 25 Years of PTSD Genetics 

Research. Curr Psychiatry Rep. 2018;20(12):115. 

11. 

Nievergelt CM, Maihofer AX, Atkinson EG, Chen C-Y, Choi KW, Coleman JR, et al. 

Discovery of 95 PTSD loci provides insight into genetic architecture and neurobiology of trauma 

and stress-related disorders. medRxiv. 2023:2023.08.31.23294915. 

12. 

Daskalakis NP, Rijal CM, King C, Huckins LM, Ressler KJ. Recent Genetics and Epigenetics 

Approaches to PTSD. Curr Psychiatry Rep. 2018;20(5):30. 

13. 

Zannas AS, Provencal N, Binder EB. Epigenetics of Posttraumatic Stress Disorder: Current 

Evidence, Challenges, and Future Directions. Biol Psychiatry. 2015;78(5):327-35. 

14. 

Bonasio R, Tu S, Reinberg D. Molecular signals of epigenetic states. Science. 

2010;330(6004):612-6. 

15. 

Baker-Andresen D, Ratnu VS, Bredy TW. Dynamic DNA methylation: a prime candidate 

for genomic metaplasticity and behavioral adaptation. Trends Neurosci. 2013;36(1):3-13. 

16. 

Rasmi Y, Shokati A, Hassan A, Aziz SG, Bastani S, Jalali L, et al. The role of DNA 

methylation in progression of neurological disorders and neurodegenerative diseases as well as 

the prospect of using DNA methylation inhibitors as therapeutic agents for such disorders. IBRO 

Neurosci Rep. 2023;14:28-37. 

39 

 
medRxiv preprint 

doi: 

https://doi.org/10.1101/2024.07.15.24310422

; 

this version posted July 15, 2024. 

The copyright holder for this preprint 

(which was not certified by peer review)

 is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. 

It is made available under a 

CC-BY-NC-ND 4.0 International license

 . 

17.  Murgatroyd C, Wu Y, Bockmuhl Y, Spengler D. Genes learn from stress: how infantile 

trauma programs us for depression. Epigenetics. 2010;5(3):194-9. 

18. 

Lokk K, Modhukur V, Rajashekar B, Martens K, Magi R, Kolde R, et al. DNA methylome 

profiling of human tissues identifies global and tissue-specific methylation patterns. Genome 

Biol. 2014;15(4):r54. 

19. 

Hannon E, Lunnon K, Schalkwyk L, Mill J. Interindividual methylomic variation across 

blood, cortex, and cerebellum: implications for epigenetic studies of neurological and 

neuropsychiatric phenotypes. Epigenetics. 2015;10(11):1024-32. 

20. 

Daskalakis NP, Cohen H, Nievergelt CM, Baker DG, Buxbaum JD, Russo SJ, et al. New 

translational perspectives for blood-based biomarkers of PTSD: From glucocorticoid to immune 

mediators of stress susceptibility. Exp Neurol. 2016;284(Pt B):133-40. 

21.  Morrison FG, Miller MW, Logue MW, Assef M, Wolf EJ. DNA methylation correlates of 

PTSD: Recent findings and technical challenges. Prog Neuropsychopharmacol Biol Psychiatry. 

2019;90:223-34. 

22. 

Katrinli S, Oliveira NCS, Felger JC, Michopoulos V, Smith AK. The role of the immune 

system in posttraumatic stress disorder. Transl Psychiatry. 2022;12(1):313. 

23. 

Uddin M, Aiello AE, Wildman DE, Koenen KC, Pawelec G, de Los Santos R, et al. 

Epigenetic and immune function profiles associated with posttraumatic stress disorder. Proc 

Natl Acad Sci U S A. 2010;107(20):9470-5. 

24. 

Smith AK, Conneely KN, Kilaru V, Mercer KB, Weiss TE, Bradley B, et al. Differential 

immune system DNA methylation and cytokine regulation in post-traumatic stress disorder. Am 

J Med Genet B Neuropsychiatr Genet. 2011;156B(6):700-8. 

40 

 
medRxiv preprint 

doi: 

https://doi.org/10.1101/2024.07.15.24310422

; 

this version posted July 15, 2024. 

The copyright holder for this preprint 

(which was not certified by peer review)

 is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. 

It is made available under a 

CC-BY-NC-ND 4.0 International license

 . 

25. 

Kuan PF, Waszczuk MA, Kotov R, Marsit CJ, Guffanti G, Gonzalez A, et al. An epigenome-

wide DNA methylation study of PTSD and depression in World Trade Center responders. Transl 

Psychiatry. 2017;7(6):e1158. 

26. 

Rutten BPF, Vermetten E, Vinkers CH, Ursini G, Daskalakis NP, Pishva E, et al. Longitudinal 

analyses of the DNA methylome in deployed military servicemen identify susceptibility loci for 

post-traumatic stress disorder. Mol Psychiatry. 2018;23(5):1145-56. 

27. 

Logue MW, Miller MW, Wolf EJ, Huber BR, Morrison FG, Zhou Z, et al. An epigenome-

wide association study of posttraumatic stress disorder in US veterans implicates several new 

DNA methylation loci. Clin Epigenetics. 2020;12(1):46. 

28. 

Hammamieh R, Chakraborty N, Gautam A, Muhie S, Yang R, Donohue D, et al. Whole-

genome DNA methylation status associated with clinical PTSD measures of OIF/OEF veterans. 

Transl Psychiatry. 2017;7(7):e1169. 

29. 

Ratanatharathorn A, Boks MP, Maihofer AX, Aiello AE, Amstadter AB, Ashley-Koch AE, et 

al. Epigenome-wide association of PTSD from heterogeneous cohorts with a common multi-site 

analysis pipeline. Am J Med Genet B Neuropsychiatr Genet. 2017;174(6):619-30. 

30. 

Uddin M, Ratanatharathorn A, Armstrong D, Kuan PF, Aiello AE, Bromet EJ, et al. 

Epigenetic meta-analysis across three civilian cohorts identifies NRG1 and HGS as blood-based 

biomarkers for post-traumatic stress disorder. Epigenomics. 2018;10(12):1585-601. 

31. 

Smith AK, Ratanatharathorn A, Maihofer AX, Naviaux RK, Aiello AE, Amstadter AB, et al. 

Epigenome-wide meta-analysis of PTSD across 10 military and civilian cohorts identifies 

methylation changes in AHRR. Nat Commun. 2020;11(1):5965. 

41 

 
medRxiv preprint 

doi: 

https://doi.org/10.1101/2024.07.15.24310422

; 

this version posted July 15, 2024. 

The copyright holder for this preprint 

(which was not certified by peer review)

 is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. 

It is made available under a 

CC-BY-NC-ND 4.0 International license

 . 

32. 

Snijders C, Maihofer AX, Ratanatharathorn A, Baker DG, Boks MP, Geuze E, et al. 

Longitudinal epigenome-wide association studies of three male military cohorts reveal multiple 

CpG sites associated with post-traumatic stress disorder. Clin Epigenetics. 2020;12(1):11. 

33. 

Katrinli S, Maihofer AX, Wani AH, Pfeiffer JR, Ketema E, Ratanatharathorn A, et al. 

Epigenome-wide meta-analysis of PTSD symptom severity in three military cohorts implicates 

DNA methylation changes in genes involved in immune system and oxidative stress. Mol 

Psychiatry. 2022;27(3):1720-8. 

34.  McCartney DL, Walker RM, Morris SW, McIntosh AM, Porteous DJ, Evans KL. 

Identification of polymorphic and off-target probe binding sites on the Illumina Infinium 

MethylationEPIC BeadChip. Genom Data. 2016;9:22-4. 

35. 

Fortin JP, Triche TJ, Jr., Hansen KD. Preprocessing, normalization and integration of the 

Illumina HumanMethylationEPIC array with minfi. Bioinformatics. 2017;33(4):558-60. 

36. 

Leek JT, Johnson WE, Parker HS, Jaffe AE, Storey JD. The sva package for removing batch 

effects and other unwanted variation in high-throughput experiments. Bioinformatics. 

2012;28(6):882-3. 

37. 

Teschendorff AE, Breeze CE, Zheng SC, Beck S. A comparison of reference-based 

algorithms for correcting cell-type heterogeneity in Epigenome-Wide Association Studies. BMC 

Bioinformatics. 2017;18(1):105. 

38. 

Salas LA, Koestler DC, Butler RA, Hansen HM, Wiencke JK, Kelsey KT, et al. An optimized 

library for reference-based deconvolution of whole-blood biospecimens assayed using the 

Illumina HumanMethylationEPIC BeadArray. Genome Biol. 2018;19(1):64. 

42 

 
medRxiv preprint 

doi: 

https://doi.org/10.1101/2024.07.15.24310422

; 

this version posted July 15, 2024. 

The copyright holder for this preprint 

(which was not certified by peer review)

 is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. 

It is made available under a 

CC-BY-NC-ND 4.0 International license

 . 

39. 

Barfield RT, Almli LM, Kilaru V, Smith AK, Mercer KB, Duncan R, et al. Accounting for 

population stratification in DNA methylation studies. Genet Epidemiol. 2014;38(3):231-41. 

40. 

van Iterson M, van Zwet EW, Consortium B, Heijmans BT. Controlling bias and inflation in 

epigenome- and transcriptome-wide association studies using the empirical null distribution. 

Genome Biol. 2017;18(1):19. 

41. 

Barfield RT, Kilaru V, Smith AK, Conneely KN. CpGassoc: an R function for analysis of DNA 

methylation microarray data. Bioinformatics. 2012;28(9):1280-1. 

42. 

Balduzzi S, Rucker G, Schwarzer G. How to perform a meta-analysis with R: a practical 

tutorial. Evid Based Ment Health. 2019;22(4):153-60. 

43.  Mansell G, Gorrie-Stone TJ, Bao Y, Kumari M, Schalkwyk LS, Mill J, et al. Guidance for 

DNA methylation studies: statistical insights from the Illumina EPIC array. BMC Genomics. 

2019;20(1):366. 

44. 

Phipson B, Maksimovic J, Oshlack A. missMethyl: an R package for analyzing data from 

Illumina's HumanMethylation450 platform. Bioinformatics. 2016;32(2):286-8. 

45. 

Logue MW, Miller MW, Wolf EJ, Huber BR, Morrison FG, Zhou Z, et al. An epigenome-

wide association study of posttraumatic stress disorder in US veterans implicates several new 

DNA methylation loci. Clinical Epigenetics. 2020; IN PRESS. 

46. 

Daskalakis NP, Iatrou A, Chatzinakos C, Jajoo A, Snijders C, DiPietro CP, et al. Systems 

Biology Dissection of PTSD and MDD across brain regions, cell-types and blood. Science. In 

Revision. 

43 

 
medRxiv preprint 

doi: 

https://doi.org/10.1101/2024.07.15.24310422

; 

this version posted July 15, 2024. 

The copyright holder for this preprint 

(which was not certified by peer review)

 is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. 

It is made available under a 

CC-BY-NC-ND 4.0 International license

 . 

47. 

Friedman MJ, Huber BR, Brady CB, Ursano RJ, Benedek DM, Kowall NW, et al. VA's 

National PTSD Brain Bank: a National Resource for Research. Curr Psychiatry Rep. 

2017;19(10):73. 

48. 

Rompala G, Nagamatsu ST, Martinez-Magana JJ, Nunez-Rios DL, Wang J, Girgenti MJ, et 

al. Profiling neuronal methylome and hydroxymethylome of opioid use disorder in the human 

orbitofrontal cortex. Nat Commun. 2023;14(1):4544. 

49. 

Zannas AS. Naturalistic Stress Hormone Levels Drive Cumulative Epigenomic Changes 

along the Cellular Lifespan. Int J Mol Sci. 2021;22(16). 

50. 

Leung CS, Kosyk O, Welter EM, Dietrich N, Archer TK, Zannas AS. Chronic stress-driven 

glucocorticoid receptor activation programs key cell phenotypes and functional epigenomic 

patterns in human fibroblasts. iScience. 2022;25(9):104960. 

51.  Min JL, Hemani G, Hannon E, Dekkers KF, Castillo-Fernandez J, Luijk R, et al. Genomic and 

phenotypic insights from an atlas of genetic effects on DNA methylation. Nat Genet. 

2021;53(9):1311-21. 

52. 

Villicana S, Castillo-Fernandez J, Hannon E, Christiansen C, Tsai PC, Maddock J, et al. 

Genetic impacts on DNA methylation help elucidate regulatory genomic processes. Genome 

Biol. 2023;24(1):176. 

53. 

Villicaña S, Castillo-Fernandez J, Hannon E, Christiansen C, Tsai P-C, Maddock J, et al. 

Genetic impacts on DNA methylation help elucidate regulatory genomic processes. bioRxiv. 

2023:2023.03.31.535045. 

44 

 
medRxiv preprint 

doi: 

https://doi.org/10.1101/2024.07.15.24310422

; 

this version posted July 15, 2024. 

The copyright holder for this preprint 

(which was not certified by peer review)

 is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. 

It is made available under a 

CC-BY-NC-ND 4.0 International license

 . 

54. 

Chen XQ, Tan I, Leung T, Lim L. The myotonic dystrophy kinase-related Cdc42-binding 

kinase is involved in the regulation of neurite outgrowth in PC12 cells. J Biol Chem. 

1999;274(28):19901-5. 

55. 

Story Jovanova O, Nedeljkovic I, Spieler D, Walker RM, Liu C, Luciano M, et al. DNA 

Methylation Signatures of Depressive Symptoms in Middle-aged and Elderly Persons: Meta-

analysis of Multiethnic Epigenome-wide Studies. JAMA Psychiatry. 2018;75(9):949-59. 

56. 

Ligthart S, Marzi C, Aslibekyan S, Mendelson MM, Conneely KN, Tanaka T, et al. DNA 

methylation signatures of chronic low-grade inflammation are associated with complex 

diseases. Genome Biol. 2016;17(1):255. 

57.  Wielscher M, Mandaviya PR, Kuehnel B, Joehanes R, Mustafa R, Robinson O, et al. DNA 

methylation signature of chronic low-grade inflammation and its role in cardio-respiratory 

diseases. Nat Commun. 2022;13(1):2408. 

58.  Muniz Carvalho C, Wendt FR, Maihofer AX, Stein DJ, Stein MB, Sumner JA, et al. 

Dissecting the genetic association of C-reactive protein with PTSD, traumatic events, and social 

support. Neuropsychopharmacology. 2021;46(6):1071-7. 

59. 

Eraly SA, Nievergelt CM, Maihofer AX, Barkauskas DA, Biswas N, Agorastos A, et al. 

Assessment of plasma C-reactive protein as a biomarker of posttraumatic stress disorder risk. 

JAMA Psychiatry. 2014;71(4):423-31. 

60. 

Yang JJ, Jiang W. Immune biomarkers alterations in post-traumatic stress disorder: A 

systematic review and meta-analysis. J Affect Disord. 2020;268:39-46. 

45 

 
medRxiv preprint 

doi: 

https://doi.org/10.1101/2024.07.15.24310422

; 

this version posted July 15, 2024. 

The copyright holder for this preprint 

(which was not certified by peer review)

 is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. 

It is made available under a 

CC-BY-NC-ND 4.0 International license

 . 

61. 

Passos IC, Vasconcelos-Moreno MP, Costa LG, Kunz M, Brietzke E, Quevedo J, et al. 

Inflammatory markers in post-traumatic stress disorder: a systematic review, meta-analysis, and 

meta-regression. Lancet Psychiatry. 2015;2(11):1002-12. 

62. 

Hao N, Whitelaw ML. The emerging roles of AhR in physiology and immunity. Biochem 

Pharmacol. 2013;86(5):561-70. 

63.  Wilson R, Wahl S, Pfeiffer L, Ward-Caviness CK, Kunze S, Kretschmer A, et al. The 

dynamics of smoking-related disturbed methylation: a two time-point study of methylation 

change in smokers, non-smokers and former smokers. BMC Genomics. 2017;18(1):805. 

64. 

Katrinli S, Smith AK. Immune system regulation and role of the human leukocyte antigen 

in posttraumatic stress disorder. Neurobiol Stress. 2021;15:100366. 

65. 

Kim GS, Smith AK, Xue F, Michopoulos V, Lori A, Armstrong DL, et al. Methylomic profiles 

reveal sex-specific differences in leukocyte composition associated with post-traumatic stress 

disorder. Brain Behav Immun. 2019;81:280-91. 

46
