# Legaki_2024_Altered Expression of Neuroplasticity-Related Genes in Alcohol Addiction and Treatment.

Article
Altered Expression of Neuroplasticity-Related Genes in Alcohol
Addiction and Treatment

Evangelia Legaki 1,†, Nikolas Dovrolis 1,†, Nikoletta Moscholiou 1, Ilias Koutromanos 2,3, Efthimios Vassilopoulos 2,
Antonios Dakanalis 4

and Elias Tzavellas 2,*

, Maria Gazouli 1,*

1

2

Laboratory of Biology, Department of Basic Biological Science, School of Medicine, National and Kapodistrian
University of Athens, 11527 Athens, Greece; evaglegaki@med.uoa.gr (E.L.); ndovroli@med.uoa.gr (N.D.);
nikoletta.moscholiou@gmail.com (N.M.)
First Department of Psychiatry, “Aiginition” Hospital, School of Medicine, National and Kapodistrian
University of Athens, 11528 Athens, Greece; koutro.ilias.23@gmail.com (I.K.); efvasilop@med.uoa.gr (E.V.)
3 Department of Psychiatry and Psychotherapy, Psychiatric Services Aargou AG, 5210 Brugg-Windisch, Switzerland
4 Department of Medicine and Surgery, University of Milano Bicocca, Via Cadore 38, 20900 Monza, Italy;

antonios.dakanalis@unimib.it

* Correspondence: mgazouli@med.uoa.gr (M.G.); etzavell@med.uoa.gr (E.T.); Tel.: +30-2107462231 (M.G.)
†

These authors contributed equally to this work.

Abstract: Alcohol use disorder’s complexity arises from genetic and environmental factors, with
alcohol metabolism genes and neurotransmitter pathways being critical. This study aims to analyze
synaptic plasticity gene expression changes in individuals with AUD in order to study their contri-
bution to AUD development and to identify potential biomarkers of treatment response. RNA was
extracted from whole peripheral blood (20 patients, 10 healthy controls), before and after treatment
(Qiagen AllPrep RNA/DNA Mini Kit), and the gene expression of 84 genes related to neuroplasticity
was studied using the RT2 Profiler for Human Synaptic Plasticity RT-PCR Array (PAHS-126ZA,
Qiagen), comparing AUD patients to control and responders to non-responders. The potential
prognostic/predictive biomarkers were searched using machine learning models. A total of 35 dys-
regulated genes were found in AUD patients. EPHB2, EGR, and AKT1 were increased, while TIMP1,
NCAM1, and GRM2 were decreased. Responders showed distinct gene expression profiles at baseline.
After treatment, the expression of 57 genes was normalized, while NCAM1, GRM2, and BDNF showed
the most significant recovery. EGR4, INHBA, and NCAM1 emerged as potential biomarkers to predict
treatment success. These results indicate that gene profiles in peripheral blood can serve as prognostic
markers for the prognosis and treatment of AUD, although further validation is required.

Keywords: alcohol use disorder (AUD); synaptic plasticity; neuroplasticity; gene expression; biomarker

1. Introduction

Alcohol use disorder (AUD) is a significant global public health challenge affecting
millions of people. Defined by excessive drinking and persistent alcohol-seeking behavior,
AUD encompasses a spectrum of problematic use leading to substantial impairment,
as outlined in the Diagnostic and Statistical Manual of Mental Disorders (DSM-V) [1].
The disorder affects nearly 6% of individuals annually, contributing to profound socio-
economic and public health burdens worldwide [2]. According to WHO data from 2016,
alcohol abuse rates among males in Greece were reported at 9.4%, with females at 2.9%.
Meanwhile, alcohol dependence affected 4.2% of males and 1.3% of females, suggesting
lower prevalence rates compared to Northern Europe [3]. Over the past two decades,
national trends indicate a reduction in overall alcohol consumption in Greece; however,
alcohol consumption among Greek teenagers remains among the highest in Europe [4].

While the physical health implications of AUD are well documented, its profound
impact on mental health remains a critical concern. The disorder’s pathophysiology is

Citation: Legaki, E.; Dovrolis, N.;

Moscholiou, N.; Koutromanos, I.;

Vassilopoulos, E.; Dakanalis, A.;

Gazouli, M.; Tzavellas, E. Altered

Expression of Neuroplasticity-Related

Genes in Alcohol Addiction and

Treatment. Int. J. Mol. Sci. 2024, 25,

11349. https://doi.org/10.3390/

ijms252111349

Academic Editors: María Candelaria

Martín-González and Emilio

González-Reimers

Received: 26 September 2024

Revised: 16 October 2024

Accepted: 17 October 2024

Published: 22 October 2024

Copyright: © 2024 by the authors.

Licensee MDPI, Basel, Switzerland.

This article is an open access article

distributed under the terms and

conditions of the Creative Commons

Attribution (CC BY) license (https://

creativecommons.org/licenses/by/

4.0/).

Int. J. Mol. Sci. 2024, 25, 11349. https://doi.org/10.3390/ijms252111349

https://www.mdpi.com/journal/ijms

 International Journal of Molecular SciencesInt. J. Mol. Sci. 2024, 25, 11349

2 of 19

complex, involving genetic predisposition, environmental factors, and cumulative effects of
alcohol consumption, underscoring the need for effective treatment strategies [5]. Genome-
wide association studies (GWAS) have revealed significant insights into how specific genes
contribute to susceptibility and severity of AUD. Key genes involved in the breakdown of
alcohol, such as alcohol dehydrogenases (ADH) and aldehyde dehydrogenase (ALDH), play
essential roles. In addition to alcohol metabolism, genes related to neurotransmitter systems
are also implicated. Serotonin and dopamine pathways, crucial for regulating mood and
processing rewards, have been associated with susceptibility to alcohol use disorder (AUD).
Variations in genes affecting these systems can impact neurotransmitter levels, potentially
influencing an individual’s inclination to seek out and respond to alcohol. Furthermore, genes
within the Dynorphin Kappa Opioid Receptor System, the Endocannabinoid system, and
pathways involved in synaptic plasticity also contribute to AUD. These systems regulate brain
functions such as stress response, reward mechanisms, and neuronal communication, all of
which are critical in the onset and progression of AUD [6].

Synaptic plasticity is the property of synapses to strengthen or weaken in response
to changes in both the amplitude and the temporal dynamics of neuronal activity. Sen-
sory inputs and intrinsic brain activity can affect long-term changes in synaptic efficacy
and eventually increase or decrease neuronal connectivity by modulating the number of
synapses [7,8]. Synaptic plasticity, essential for brain development and function, enables
adaptability, learning, and overall mental well-being [9]. This critical brain function also
plays a key role in both short- and long-term memory, and the mechanisms underlying
these changes have been linked to the pathophysiology and treatment of multiple neurobi-
ological disorders, including depression [10]. Synaptic plasticity is also crucial in the early
development of neural circuitry, and evidence is accumulating that impairments in synaptic
plasticity mechanisms contribute to several prominent neuropsychiatric disorders [7].

Environmental factors such as stress, trauma, and substance use can profoundly affect
synaptic plasticity, contributing to a range of psychiatric disorders. Unlike focal brain
injuries, these disorders involve widespread changes across brain regions crucial for cogni-
tion, emotion regulation, and motivation. They are influenced by complex genetic factors
shaped by both neurodevelopment and environmental experiences, leading to chronic,
recurrent conditions with slow recovery and high relapse rates. Genetic investigations
suggest the involvement of genes like brain-derived neurotrophic factor (BDNF) and the
NMDA receptor in these disorders, influencing synaptic connectivity [9].

Likewise, repeated substance use results in neurobiological changes, which alters the
brain’s reward circuitry and increases synaptic plasticity in brain areas such as the prefrontal
cortex, amygdala, and striatum. These changes heighten motivation for the substance,
leading to tolerance, dependence, and withdrawal symptoms. By understanding these
dysregulations in synaptic plasticity, we gain valuable insights into developing targeted
therapies for psychiatric disorders [9].

Concerning the AUD therapeutic approaches, individuals who are struggling with
alcohol dependence can receive assistance from the special inpatient alcohol detoxification
units of the hospitals by taking part in any one of a comprehensive range of evidence-based
treatment options.

Taken the above into consideration, in this study, we aimed to profile gene expression
changes associated with alcohol abuse and to identify specific baseline differences between
individuals who will and will not respond to therapeutic interventions. By comparing the
transcriptomic profiles of these individuals before and after the intervention, we aim to
quantify the effects of therapeutic intervention on transcription levels. This investigation
will facilitate the differentiation of patient groups based on established gene expression pro-
files, thereby advancing both clinical and research applications in the context of therapeutic
interventions for alcohol use disorder.

Int. J. Mol. Sci. 2024, 25, 11349

3 of 19

2. Results
2.1. Effect of Alcohol Abuse on Synaptic Plasticity

As previously described, we combined the samples of alcoholics at baseline, regardless
of their response to treatment later and compared them with healthy controls. Statistically
significant changes (p < 0.05 and fold regulation of ±2) highlight, in total, 35 dysregulated
genes. EPHB2 showed the highest positive fold regulation of 19.63, indicating substantial
upregulation. Three EGR family members, EGR4, EGR1, and EGR2 exhibited a notable
upregulation with fold changes of 3.34, 3.29, and 2.79, respectively. AKT1 was upregulated
with a fold change of 3.2, and CREB1 had an upregulation of 2.74-fold. GRM5 was upregu-
lated with a fold change of 2.7, while CAMK2G and RELA both showed a fold regulation
of 2.58. MMP9 exhibited an upregulation of 2.4 fold, NGFR of 2.37, and HOMER1 of
2.36-fold. Finally, DLG4 was upregulated with a fold change of 2.31, PICK1 of 2.14, and
PRKG1 2.11 fold. On the other end, the genes YWHAQ, GRM8, GRIN1, GRM3, GABRA5,
and GNAI1 were moderately downregulated with a fold regulation of −2.02, −2.08, −2.15,
−2.42, −2.5, −2.58, respectively. PLAT had a downregulation of −2.88, BDNF −2.93, and
ADCY1 of −3.27 fold. CAMK2A exhibited a −3.34 fold downregulation while NFKBIB
demonstrated a fold regulation of −3.48 fold. JUNB was downregulated −3.55 fold, GRIA1
−3.75 fold, GRM2 −4.04 fold, GRM1 −4.27 fold, ADCY8 −4.5 fold, NOS1 −4.64 fold,
PPP3CA −4.82 fold, and NCAM1 showed a fold regulation of −4.99 fold. TIMP1 had the
highest negative fold regulation of −7.6, indicating a substantial downregulation (Figure 1).

Figure 1. Differential expression of synaptic plasticity-related genes in alcohol use disorder (AUD)
versus healthy controls. Box plots representing the fold regulation of 35 synaptic plasticity-related
genes in AUD patients compared to healthy controls. Gene expression was transformed to log2(1/∆Ct)
values for visualization.

Int. J. Mol. Sci. 2024, 25, x FOR PEER REVIEW 4 of 21    Figure 1. Differential expression of synaptic plasticity‐related genes in alcohol use disorder (AUD) versus healthy controls. Box plots representing the fold regulation of 35 synaptic plasticity‐related genes in AUD patients compared to healthy controls. Gene expression was transformed to log2(1/ΔCt) values for visualization. Studying the correlations between these dysregulated genes to identify co‐expression patterns in both the alcoholic and control groups allows us to discern specific patterns indicating synergistic roles or the presence of common, invisible for this analysis, transcriptional factors. To highlight the strongest gene pairwise correlations, as described in our methodology, we focused on Spearman correlation coefficients higher than 0.8 for strong positive and lower than −0.8 for strong negative correlations (Figure 2). Int. J. Mol. Sci. 2024, 25, 11349

4 of 19

Studying the correlations between these dysregulated genes to identify co-expression
patterns in both the alcoholic and control groups allows us to discern specific patterns
indicating synergistic roles or the presence of common, invisible for this analysis, transcrip-
tional factors. To highlight the strongest gene pairwise correlations, as described in our
methodology, we focused on Spearman correlation coefficients higher than 0.8 for strong
positive and lower than −0.8 for strong negative correlations (Figure 2).

Figure 2. Co-expression analysis of synaptic plasticity-related genes in healthy controls and alcoholics.
Spearman correlation heatmaps for synaptic plasticity-related genes in healthy controls (left) and
alcoholics (right). Results filtered for rho less than −0.8 and greater than 0.8.

The healthy control group showed a remarkably greater presence of such correlations
indicative of high synaptic activity and plasticity. For example, in the healthy control
group EGR4, MMP9, ADCY1, AKT1, and PICK1 paired with each other with correlations
of 1.00 and 0.99, revealing a strongly co-regulated cluster. The same pattern is shown for
HOMER1, GRM5, DLG4, NCAM1, and PICK1, with the same rho values. Interestingly
GRIA1 shows strong negative correlations with multiple of these genes, such as EGR4,
GRM5, MMP9, HOMER1, DLG4, PRKG1, ADCY1, NOS1, NCAM1, and PICK1. The results
of the alcoholic group are significantly less showing of no inverse correlation between these
genes, showing no pairwise correlation of rho 0.90–1.00 and only highlighting the pairs of
GRM3 and EGR1 with the highest correlation with a coefficient of 0.86, EGR1 and GNAI1
with a coefficient of 0.82, PPP3CA and GRIA1 with a coefficient of 0.82, and finally PPP3CA
and GRIN1, which share a correlation coefficient of 0.82. Interestingly, even though the
genes GRM3 and GNAI1 or GRIA1 and GRIN1 share correlations with the genes EGR1 and
PPP3CA, respectively, they could not surpass our rho > 0.8 threshold as pairs.

2.2. Prediction of Response to Clinical Therapeutic Intervention

Prediction of response to therapeutic intervention is a cornerstone of precision medicine.
To identify with a simple molecular technique if a patient will be more susceptible to
a specific intervention before starting is crucial in deciding the next steps in therapy. For
that reason, as described in our methodology, we decided to compare the transcriptomic
profiles of patients who will or will not respond to our intervention versus each other.
This comparison allows, in a clinical and a research setting, us to differentiate the different
groups based on already established profiles.

In total, 15 genes showed significant differences when comparing responders to
non-responders to intervention at baseline. EGR4 exhibits the highest upregulation with

Int. J. Mol. Sci. 2024, 25, x FOR PEER REVIEW 5 of 21    Figure 2. Co‐expression analysis of synaptic plasticity‐related genes in healthy controls and alcoholics. Spearman correlation heatmaps for synaptic plasticity‐related genes in healthy controls (left) and alcoholics (right). Results filtered for rho less than −0.8 and greater than 0.8. The healthy control group showed a remarkably greater presence of such correlations indicative of high synaptic activity and plasticity. For example, in the healthy control group EGR4, MMP9, ADCY1, AKT1, and PICK1 paired with each other with correlations of 1.00 and 0.99, revealing a strongly co‐regulated cluster. The same pattern is shown for HOMER1, GRM5, DLG4, NCAM1, and PICK1, with the same rho values. Interestingly GRIA1 shows strong negative correlations with multiple of these genes, such as EGR4, GRM5, MMP9, HOMER1, DLG4, PRKG1, ADCY1, NOS1, NCAM1, and PICK1. The results of the alcoholic group are significantly less showing of no inverse correlation between these genes, showing no pairwise correlation of rho 0.90–1.00 and only highlighting the pairs of GRM3 and EGR1 with the highest correlation with a coefficient of 0.86, EGR1 and GNAI1 with a coefficient of 0.82, PPP3CA and GRIA1 with a coefficient of 0.82, and finally PPP3CA and GRIN1, which share a correlation coefficient of 0.82. Interestingly, even though the genes GRM3 and GNAI1 or GRIA1 and GRIN1 share correlations with the genes EGR1 and PPP3CA, respectively, they could not surpass our rho > 0.8 threshold as pairs. 2.2. Prediction of Response to Clinical Therapeutic Intervention Prediction of response to therapeutic intervention is a cornerstone of precision medicine. To identify with a simple molecular technique if a patient will be more susceptible to a specific intervention before starting is crucial in deciding the next steps in therapy. For that reason, as described in our methodology, we decided to compare the transcriptomic profiles of patients who will or will not respond to our intervention versus each other. This comparison allows, in a clinical and a research setting, us to differentiate the different groups based on already established profiles. In total, 15 genes showed significant differences when comparing responders to non‐responders to intervention at baseline. EGR4 exhibits the highest upregulation with a fold regulation of 4.38, indicating a strong biomarker candidate. This result is followed by GABRA5 and NR4A1, both with a fold regulation of 2.41. EGR3 is upregulated with a fold regulation of 2.4, while GNAI1 shows an increase at 2.36, GRIA1 is upregulated by 2.32‐fold, and both GRM7 and NFKBIB exhibit a fold regulation of 2.21. On the other hand, some genes are downregulated, with INHBA showing the most substantial decrease at −4.72, followed by RELA at −4.15, while ARC has a fold regulation of −3.37, and TNF is Int. J. Mol. Sci. 2024, 25, 11349

5 of 19

a fold regulation of 4.38, indicating a strong biomarker candidate. This result is followed
by GABRA5 and NR4A1, both with a fold regulation of 2.41. EGR3 is upregulated with
a fold regulation of 2.4, while GNAI1 shows an increase at 2.36, GRIA1 is upregulated by
2.32-fold, and both GRM7 and NFKBIB exhibit a fold regulation of 2.21. On the other hand,
some genes are downregulated, with INHBA showing the most substantial decrease at
−4.72, followed by RELA at −4.15, while ARC has a fold regulation of −3.37, and TNF
is downregulated by −2.26 fold. CEBPB and CEBPD are downregulated by −2.19 and
−2-fold, respectively. Finally, GRIN1 shows the least downregulation at −2.02 (Figure 3).

Figure 3. Baseline gene expression differences between responders and non-responders to alcohol
detoxification therapy. Box plots representing the fold regulation of 15 synaptic plasticity-related
genes. Gene expression was transformed to log2(1/∆Ct) values for visualization. These gene expres-
sion profiles suggest potential biomarkers that could predict therapeutic response in AUD patients.

Examining the co-expression of these genes in our two groups individually, we see
that, in the responder group, only the pairs of GRM7-GNAI1 and INHBA-ARC are strongly
correlated, both with a rho of 0.84, while, in non-responders, the correlation network is
much more expansive. Notably, in non-responders, the pairs of GNAI1-NR4A1, NFKBIB-
EGR3, and GRIA1-GRIN1 exhibit an absolute correlation coefficient of 1.00, while GRM7-
EGR4 and TNF-GRM7 show a rho of 0.94, among others (Figure 4).

Int. J. Mol. Sci. 2024, 25, x FOR PEER REVIEW 6 of 21   downregulated by −2.26 fold. CEBPB and CEBPD are downregulated by −2.19 and −2‐fold, respectively. Finally, GRIN1 shows the least downregulation at −2.02 (Figure 3).  Figure 3. Baseline gene expression differences between responders and non‐responders to alcohol detoxification therapy. Box plots representing the fold regulation of 15 synaptic plasticity‐related genes. Gene expression was transformed to log2(1/ΔCt) values for visualization. These gene expression profiles suggest potential biomarkers that could predict therapeutic response in AUD patients. Examining the co‐expression of these genes in our two groups individually, we see that, in the responder group, only the pairs of GRM7‐GNAI1 and INHBA‐ARC are strongly correlated, both with a rho of 0.84, while, in non‐responders, the correlation network is much more expansive. Notably, in non‐responders, the pairs of GNAI1‐NR4A1, NFKBIB‐EGR3, and GRIA1‐GRIN1 exhibit an absolute correlation coefficient of 1.00, while GRM7‐EGR4 and TNF‐GRM7 show a rho of 0.94, among others (Figure 4). Int. J. Mol. Sci. 2024, 25, 11349

6 of 19

Figure 4. Correlation matrix of synaptic plasticity-related genes in responders versus non-responders
to alcohol detoxification therapy. Spearman correlation heatmaps for synaptic plasticity-related genes
in responders (left) and non-responders (right) to therapy. Responders show limited co-expression
between gene pairs, while non-responders exhibit a more extensive and complex gene interaction
network, suggesting that non-responders may have more widespread dysregulation in synaptic
plasticity-related pathways. Results filtered for rho less than −0.8 and greater than 0.8.

2.3. Effect of Therapeutic Intervention on Transcriptomic Profile

Quantifying response to a treatment requires a combination of clinical observations
and molecular markers. To quantify the effects of our therapeutic intervention, we are
investigating the effects on transcription levels before and after the therapeutic intervention.
DGEA highlights a significant reversal of the effects of alcohol on gene expression, pre-
senting most of the genes previously dysregulated, striving to match healthy control levels
with a particular emphasis on downregulating gene expression previously exacerbated by
alcohol. In total, 57 genes showed differential expression after the intervention, 18 of whom
were previously found dysregulated due to alcohol (Figure 5).

In particular, NCAM1 exhibited a significant upregulation by 7.9-fold (previously
downregulated in alcoholics versus healthy controls by −5.53 fold), ADCY8 showed
an upregulation with a fold change of 2.08 (previously −3.04 fold), GRM2 showed
a 2.06-fold increase (previously −3.57 fold), and BDNF was upregulated by 2.94-fold
(previously downregulated in alcoholics versus healthy controls by −3.32 fold). Similarly,
several genes demonstrated downregulation after treatment on the responders. Specifically,
MMP9 showed a fold change of −2.43 (previously upregulated in alcoholics versus healthy
controls by 2.03-fold), AKT1 demonstrated a fold regulation of −2.45 (previously upregu-
lated by 3.25-fold), EGR2 was downregulated by −2.06-fold (previously upregulated by
2.49-fold), and DLG4 showed a fold change of −3.39 (previously upregulated by 2.16-fold).
NGFR demonstrated a fold regulation of −6.47-fold (previously upregulated in alcoholics
versus healthy controls by 2.33-fold), RELN exhibited a fold change of −4.9 (previously
upregulated by 2.03-fold), EGR1 demonstrated a fold change of −5.68 (previously upregu-
lated by 3.68-fold), while EGR4 exhibited a fold change of −5.44 (previously upregulated
in alcoholics versus healthy controls by 5.2-fold). The most substantial downregulations
were observed in HOMER1 with a fold change of −16.77 and EPHB2 with a fold change
of −20.47, both previously upregulated in alcoholics versus healthy controls by 2.78 and
20.81-fold, respectively. Interestingly, JUNB exhibited a downregulation of −2.31 fold,
NOS1 of −3.41, YWHAQ −6.75-fold, and PLAT −6.87-fold, further downregulating genes
whose expression was negatively impacted by alcohol versus healthy controls.

Int. J. Mol. Sci. 2024, 25, x FOR PEER REVIEW 7 of 21    Figure 4. Correlation matrix of synaptic plasticity‐related genes in responders versus non‐responders to alcohol detoxification therapy. Spearman correlation heatmaps for synaptic plasticity‐related genes in responders (left) and non‐responders (right) to therapy. Responders show limited co‐expression between gene pairs, while non‐responders exhibit a more extensive and complex gene interaction network, suggesting that non‐responders may have more widespread dysregulation in synaptic plasticity‐related pathways. Results filtered for rho less than −0.8 and greater than 0.8. 2.3. Effect of Therapeutic Intervention on Transcriptomic Profile Quantifying response to a treatment requires a combination of clinical observations and molecular markers. To quantify the effects of our therapeutic intervention, we are investigating the effects on transcription levels before and after the therapeutic intervention. DGEA highlights a significant reversal of the effects of alcohol on gene expression, presenting most of the genes previously dysregulated, striving to match healthy control levels with a particular emphasis on downregulating gene expression previously exacerbated by alcohol. In total, 57 genes showed differential expression after the intervention, 18 of whom were previously found dysregulated due to alcohol (Figure 5). Int. J. Mol. Sci. 2024, 25, 11349

7 of 19

Figure 5. Gene expression changes before and after alcohol detoxification therapy in responders. Box
plots representing the fold regulation of 57 synaptic plasticity-related genes. Gene expression was
transformed to log2(1/∆Ct) values for visualization. Previously upregulated genes such as EGR4, EPHB2,
and HOMER1 are downregulated after therapy, suggesting a reversal of alcohol-induced dysregulation.

Int. J. Mol. Sci. 2024, 25, x FOR PEER REVIEW 8 of 21    Figure 5. Gene expression changes before and after alcohol detoxification therapy in responders. Box plots representing the fold regulation of 57 synaptic plasticity‐related genes. Gene expression was transformed to log2(1/ΔCt) values for visualization. Previously upregulated genes such as EGR4, EPHB2, and HOMER1 are downregulated after therapy, suggesting a reversal of alcohol‐induced dysregulation. Int. J. Mol. Sci. 2024, 25, 11349

8 of 19

Regarding co-expression of the 57 perturbed genes in which we observed high activity
before and after intervention but with significant changes in their correlation profiles, after
therapy, we had almost half the correlations achieving a rho less than −0.8 and greater than
0.8 (196 after vs. 355 before), reminiscent of the lower activity of the healthy controls (Figure 6).

Figure 6. Correlation matrix of synaptic plasticity-related genes before and after alcohol detoxification
therapy in responders. Spearman correlation heatmaps for synaptic plasticity-related genes in
responders before (left) and after (right) therapy. The therapy appears to restore the balance in
synaptic plasticity-related gene expression, as evidenced by the reduction in co-expression patterns.
Results filtered for rho less than −0.8 and greater than 0.8.

In addition, we noticed no significant inverse correlation in alcoholics before treatment
for these genes, but after treatment, the pair of GRM2-EGR3 had a rho of −0.93, the pair
GRM2-CEBPB of −0.83, the pair GRM2-PICK1 of −0.81, and the pair GRM2-NTF4 of −0.81.

3. Discussion

This study focuses on how alcohol addiction directly influences the expression levels
of blood cell neuroplasticity-related genes in humans. In addition, it allows us to identify
how synaptic plasticity can be targeted by therapeutic intervention restoring the “normal”
transcriptomic profile. As previously suggested, the blood cell and CNS correlation may
seem logical if one considers that blood cells regularly travel through many different body
regions and may be exposed to the same environment as CNS tissue; having relatively
direct access to the blood, they detect and respond to its changes [11]. It is well accepted
that blood-based biomarkers are increasingly recognized for their utility and relevance in
psychiatric disorders, as they provide quantifiable insights into the biological underpin-
nings of mental health conditions. By integrating these biomarkers into the biopsychosocial
model, researchers can better understand the complex interactions between biological,
psychological, and social factors that contribute to psychiatric disorders. This knowledge
enhances diagnostic accuracy, facilitates early intervention, and enables personalized treat-
ment strategies tailored to individual patient needs. Furthermore, blood biomarkers have
the potential to illuminate the pathophysiology of mental health conditions, ultimately
improving patient outcomes and reducing the global burden of these disorders [12–14].

Our study reveals 35 dysregulated genes that can effectively differentiate between
alcoholics and healthy controls. Among them, most notably, EPHB2 was found to be signifi-
cantly upregulated to alcoholics. It is well known that EPHB2 deficiency has been related to
depression-like conditions and memory impairments in animal studies [15]. Interestingly,

Int. J. Mol. Sci. 2024, 25, x FOR PEER REVIEW 9 of 21   In particular, NCAM1 exhibited a significant upregulation by 7.9‐fold (previously downregulated in alcoholics versus healthy controls by −5.53 fold), ADCY8 showed an upregulation with a fold change of 2.08 (previously −3.04 fold), GRM2 showed a 2.06‐fold increase (previously −3.57 fold), and BDNF was upregulated by 2.94‐fold (previously downregulated in alcoholics versus healthy controls by −3.32 fold). Similarly, several genes demonstrated downregulation after treatment on the responders. Specifically, MMP9 showed a fold change of −2.43 (previously upregulated in alcoholics versus healthy controls by 2.03‐fold), AKT1 demonstrated a fold regulation of −2.45 (previously upregulated by 3.25‐fold), EGR2 was downregulated by −2.06‐fold (previously upregulated by 2.49‐fold), and DLG4 showed a fold change of −3.39 (previously upregulated by 2.16‐fold). NGFR demonstrated a fold regulation of −6.47‐fold (previously upregulated in alcoholics versus healthy controls by 2.33‐fold), RELN exhibited a fold change of −4.9 (previously upregulated by 2.03‐fold), EGR1 demonstrated a fold change of −5.68 (previously upregulated by 3.68‐fold), while EGR4 exhibited a fold change of −5.44 (previously upregulated in alcoholics versus healthy controls by 5.2‐fold). The most substantial downregulations were observed in HOMER1 with a fold change of −16.77 and EPHB2 with a fold change of −20.47, both previously upregulated in alcoholics versus healthy controls by 2.78 and 20.81‐fold, respectively. Interestingly, JUNB exhibited a downregulation of −2.31 fold, NOS1 of −3.41, YWHAQ −6.75‐fold, and PLAT −6.87‐fold, further downregulating genes whose expression was negatively impacted by alcohol versus healthy controls. Regarding co‐expression of the 57 perturbed genes in which we observed high activity before and after intervention but with significant changes in their correlation profiles, after therapy, we had almost half the correlations achieving a rho less than −0.8 and greater than 0.8 (196 after vs. 355 before), reminiscent of the lower activity of the healthy controls (Figure 6).  Figure 6. Correlation matrix of synaptic plasticity‐related genes before and after alcohol detoxification therapy in responders. Spearman correlation heatmaps for synaptic plasticity‐related genes in responders before (left) and after (right) therapy. The therapy appears to restore the balance in synaptic plasticity‐related gene expression, as evidenced by the reduction in co‐expression patterns. Results filtered for rho less than −0.8 and greater than 0.8. In addition, we noticed no significant inverse correlation in alcoholics before treatment for these genes, but after treatment, the pair of GRM2‐EGR3 had a rho of −0.93, Int. J. Mol. Sci. 2024, 25, 11349

9 of 19

in accordance with our findings, Sanz-Martos et al. [16] observed increased expression of
EPHB2 in ethanol-exposed rats. The role of the EphB2 protein is the signal transmission in
hippocampal development, and it also plays a basic role in synapse formation and synaptic
stabilization [17,18]. Several studies supported that, since EPHB2 is implicated in synaptic
plasticity [19] and considering the interaction with alcohol [20], it can be an attractive target
for future studies.

The EGR family genes, including EGR1, EGR2, and EGR4, have shown significant
upregulation in the alcoholic group of the present study. The EGR genes encode a family of
immediate early gene transcription factors that mediate the transcription of various genes
related to neuronal development and plasticity, cognition, circadian rhythm, and social
behaviors [21]. This family of genes has been referred to be implicated in various psychiatric
conditions. For instance, EGR1 has been closely associated with major depressive disorder
(MDD). Stress-related fear memory has been linked to increased EGR1 expression, and
knocking down EGR1 expression effectively blocks the fear-related response [22]. Reversely,
in depression and schizophrenia, most studies revealed a decreased expression of EGR
genes. Covington et al. (2010) observed a decreased expression of EGR1 in the medial
prefrontal cortex of depressed patients who were refractory to treatment, as well as in non-
medicated subjects. Notably, this brain region was consistently reported as affected in both
human patients and animal models of depression [23]. Moreover, the downregulation of
EGR1, EGR2, and EGR3 transcripts was reported in the postmortem brains of patients with
schizophrenia [24]. Among EGR genes, EGR3 has demonstrated a strong and consistent
association with schizophrenia in both family-based and case–control association studies
within Japan [24] and Korea [25]. Additionally, a genetic link between EGR2 and EGR3
and bipolar disorder was recently identified [26]. However, our results did not reveal
a significant differential expression of EGR3, similar to a study conducted in a Chinese
population that reported no association [27].

Our study also highlighted the upregulation of AKT1; however, to the best of our
knowledge, there are scarce data about the association of AKT1 with alcohol disorder
or other psychiatric conditions. Excessive alcohol intake has been shown to result in a
sustained activation of the AKT pathway but not the ERK1/2 pathway in the nucleus ac-
cumbens (NAc) [28]. This activation is brain region-specific and is believed to engage the
AKT/mTORC1 pathway, contributing to mechanisms in the NAc shell and orbitofrontal
cortex (OFC) that play a key role in the development and persistence of alcohol-drinking be-
haviors. In contrast, decreased AKT1 expression was observed in the dorsolateral prefrontal
cortex of individuals with schizophrenia and bipolar disorder, with a significant reduction
in schizophrenic patients and a moderate effect size in bipolar patients [29]. These findings
underline the distinct involvement of the AKT pathway in both alcohol-related behavior
and psychiatric disorders.

Furthermore, our results identified the upregulation of the gene CREB1 in individ-
uals with alcohol dependence compared to healthy controls. CREB1 encodes the cyclic
AMP-responsive element-binding protein–1 (CREB1), a transcription factor crucial for
neuronal activity, synaptic plasticity, learning, and memory processes, including semantic
memory, episodic memory, and executive functions [30–33]. CREB signaling, which is
regulated through CREB phosphorylation, plays a significant role in neural plasticity and
is disrupted in various pathological states, such as addiction and psychiatric disorders,
including depression and schizophrenia [34,35]. The phosphorylation of CREB leads to its
nuclear localization and recruitment of transcriptional coactivators, such as the CREB bind-
ing protein (CBP) and p300. These coactivators possess intrinsic histone acetyltransferase
(HAT) activity, catalyzing histone acetylation—a key epigenetic modification associated
with increased gene transcription [36]. The dysregulation of CREB signaling, particularly
in the context of addiction, is thought to alter synaptic plasticity through mechanisms
like long-term potentiation (LTP) and long-term depression (LTD), which are essential
for the development of compulsive behaviors and addiction progression [37,38]. Several
studies explored the role of CREB1 in psychiatric conditions, notably schizophrenia. On a

Int. J. Mol. Sci. 2024, 25, 11349

10 of 19

genetic level, CREB1 polymorphisms have been linked to schizophrenia, bipolar disorder,
and major depressive disorder [39–41]. However, transcriptomic analyses produced con-
flicting results regarding CREB1 expression in schizophrenia. While some studies report
downregulation in the prefrontal cortex (PFC) [41], others, including a meta-analysis by
Ohayon et al. (2020), have found upregulation in Brodmann Area 10 (BA10) [42] and the
dorsolateral prefrontal cortex (DLPFC) [43]. This discrepancy might be attributed to differ-
ences in brain regions studied or methodological variations. Interestingly, animal models
suggest that elevated CREB expression could lead to reduced sensitivity to emotional
stimuli, anhedonia, and depressive-like behaviors [44–48], further emphasizing CREB1’s
complex role in mental health disorders. Moreover, CREB1 regulates several genes involved
in synaptic plasticity, including BDNF together with several glutamatergic genes (GRM5,
GRM7, GRID1, and GRIN2A) and protein kinase C genes (PRKCA and PRKCB,) underscor-
ing its pivotal role in modulating the neural circuits that underlie learning, memory, and
potentially addiction [49].

Recent research into the role of metabotropic glutamate receptors (mGluRs) in alcohol
use disorder (AUD) has uncovered significant insights into how these receptors influence
alcohol-related behaviors. The metabotropic glutamate (mGlu) receptor family, consisting
of eight subtypes (mGlu1 to mGlu8), is divided into three groups based on their genetic,
pharmacological, and functional characteristics. Group I receptors (mGlu1 and mGlu5) are
primarily located at postsynaptic sites, where they signal through Gq proteins, leading
to the activation of pathways involved in synaptic plasticity and neuronal excitability. In
contrast, Group II (mGlu2 and mGlu3) and Group III (mGlu4, mGlu6–mGlu8) receptors are
mainly found at presynaptic locations and couple with Gi proteins to inhibit neurotrans-
mitter release by reducing cyclic AMP (cAMP) levels [50]. Our data revealed an increased
expression of GRM5 and a decreased expression of GRM1 and GRM2 in blood samples from
individuals with AUD when compared to healthy controls. Our findings of increased GRM5
expression in AUD patients is consistent with the existing literature that suggests GRM5
plays a critical role in the modulation of alcohol-related behaviors [51–53]. Studies have
shown that mGluR5 is involved in various aspects of mood disorders and addiction [54],
including the reinforcement of alcohol consumption and the regulation of negative affect
associated with AUD [55]. For instance, Kasten et al. (2019) highlighted that modulation of
mGluR5 could reduce ethanol intake and alleviate alcohol-induced anxiety-like behavior,
emphasizing its potential as a therapeutic target. This receptor’s upregulation may serve as
a compensatory response to prolonged alcohol exposure [55]. This response aligns with
rodent studies, where inhibiting or genetically targeting GRM5 was shown to significantly
reduce alcohol self-administration and relapse-like behavior [56]. However, this receptor’s
role is complex, with some studies noting variability in its expression depending on the
brain region, the type of alcohol exposure, and the time since the last exposure [51]. This
complexity might explain the increased GRM5 expression observed in our study, reflecting
its dynamic regulation in response to chronic alcohol use.

Conversely, our data showed a decrease in GRM1 and GRM2 expression in blood
samples from individuals with AUD, which diverges from some aspects of the existing
literature. While the role of GRM1 in AUD is well documented, with studies indicating
that mGlu1, along with mGlu5, is involved in the glutamatergic system’s regulation of
alcohol-related behaviors, the literature often reports an upregulation of mGluR1 in specific
brain regions of animal models (nucleus accumbens, central amygdala) in response to
alcohol exposure [57–62]. However, other studies highlighted that chronic alcohol exposure
was shown to decrease mGlu1 mRNA expression in brain regions such as the hippocampus
and cerebellum [63]. The reduced expression of GRM1 in our study might reflect similar
mechanisms in peripheral tissues with some brain regions, potentially serving as a marker
for the neuroadaptive changes occurring centrally or indicating a compensatory response
to chronic alcohol exposure. This reduction is associated with increased sensitivity to the
neurochemical effects of alcohol, supporting the notion that diminished mGlu1 function
may exacerbate alcohol-related behaviors [64–66].

Int. J. Mol. Sci. 2024, 25, 11349

11 of 19

The decrease in GRM2 expression is particularly noteworthy given that GRM2 encodes
mGluR2, a receptor critical for synaptic transmission and the regulation of the alcohol-
related reward system. Previous research has demonstrated that loss or reduction in
mGluR2 function leads to insufficient control of glutamate release, contributing to excessive
alcohol consumption [67]. Our findings are consistent with these observations, suggesting
that reduced GRM2 expression may be a key factor in the dysregulation of glutamatergic
signaling in AUD, particularly in key brain regions like the nucleus accumbens and the
ventral tegmental area [50].

The HOMER1 gene, identified as upregulated in our study, plays a crucial role in the
glutamatergic pathway, primarily through its interactions with Group I metabotropic gluta-
mate receptors (mGluRs), specifically mGlu1 and mGlu5. Homer proteins, including those
encoded by HOMER1, are essential for regulating behavioral sensitivity to alcohol and
other drugs of abuse. Among the postsynaptic scaffolding proteins, the Homer family has
been well characterized, particularly as a key element influencing synaptic plasticity [68].
Given that GRM5 (encoding mGluR5) was also upregulated in our study, it is plausible that
increased HOMER1 expression similarly facilitates mGluR5 activity, thereby contributing to
alcohol’s reinforcing effects. A prior study examined the association between genetic vari-
ants of HOMER1 and HOMER2, located on chromosomes 5q14.2 and 15q24.3, respectively,
with cocaine dependence. This study reported a significant link between the HOMER1 SNP
rs6871510 genotypes and cocaine dependence [69].

While there are limited data on HOMER1 expression in alcohol use disorder (AUD),
most findings focus on HOMER2. Research, both preclinical and clinical, has underscored
the significance of HOMER2 isoforms as active contributors to alcohol-induced behavioral
and cellular plasticity [70,71]. Overexpression of HOMER2 is linked to an increase in alco-
hol’s potency and efficacy in eliciting reward, through its enhancement of both appetitive
and consummatory behaviors related to alcohol. This upregulation likely enhances alco-
hol’s rewarding effects by promoting mGlu1/5 interactions. In contrast, HOMER2 deletion,
knockdown, or blockade results in an alcohol-avoiding phenotype [58,60,71–73].

Interestingly, our data also revealed an increased expression of CAMK2G and a de-
creased expression of GRIA1. The current literature lacks enough data for the association of
CAMK2G and alcohol disorder. CaMKII, particularly its α-subunit (αCaMKII), plays a key
role in learning, memory, and addiction. Most data have shown CAMK2A involvement in
drug addiction in both humans and animal models. CaMKII levels in the ventral tegmental
area (VTA) and nucleus accumbens (NAcc) are critical for psychostimulant sensitization
and self-administration. Though less is known about its role in alcohol addiction, alcohol
has been found to affect brain CaMKII levels and increase phosphorylation at Thr286,
influencing cellular signaling, behavior, and neurotoxicity [74]. Conversely, the downreg-
ulation of GRIA1, which encodes the AMPA receptor subunit 1, is particularly important.
GRIA1 is crucial for fast excitatory synaptic transmission, and its reduced expression
may impair AMPA receptor-mediated signaling, potentially disrupting synaptic plasticity
necessary for learning and memory. This aligns with studies showing decreased GRIA1
and GRIA2 transcript levels in the medial prefrontal cortex (mPFC) of alcohol-dependent
mice, suggesting a role for AMPAR plasticity in the glutamatergic dysfunction associated
with ethanol withdrawal. A negative correlation between GRIA1 mRNA levels and total
ethanol intake further suggests that reduced GRIA1 expression may be linked to higher
alcohol consumption, reinforcing the idea that ethanol exposure disrupts AMPA receptor
function and contributes to addiction-related neural changes [75]. Notably, our findings
are further supported by prior research that used a pathway-based approach to examine
genetic variants within this system. Variants in genes such as GRM1, GRM5, HOMER1,
HOMER2, EIF4E, EEF2, GRIA1, GRIA4, MTOR, and CAMK2A have been shown to be
associated with increased alcohol consumption, specifically the number of drinking days
per month, when considered collectively. Furthermore, hypermethylation at a CpG site
in the 3′-untranslated region of EEF2 was observed in individuals who consumed alcohol

Int. J. Mol. Sci. 2024, 25, 11349

12 of 19

more frequently, indicating potential epigenetic regulation of this pathway in relation to
alcohol use [66].

Furthermore, TIMP-1 was downregulated while MMP9 levels were elevated in subjects
with alcohol intoxication in our study. Acute alcohol intoxication has been associated with
increased cytokine production, oxidative stress, and liver apoptosis [76]. Recent research
suggests that even a single binge drinking episode can disturb the balance of the fibrosis
process [77]. Zdanowicz et al. suggest that even one episode of alcohol intoxication
in adolescents can disrupt fibrosis markers, with MMP9 and TIMP-1 emerging as more
sensitive indicators of liver damage than ALT and AST. While MMP9 and TIMP-1 are
expressed in various organs, the elevated levels observed in this study likely reflect the
direct impact of ethanol on the liver [78]. Additionally, MMP9 polymorphisms, such as
the C/T variant, have been linked to alcohol dependence and elevated serum MMP9
levels [79]. In MMP9 knockout mice, chronic alcohol exposure impaired synaptic plasticity
and reduced alcohol-seeking behavior, both of which were restored by reintroducing
MMP9 [80]. This same polymorphism has also been associated with increased motivation
to drink in alcoholic patients.

Limitations of our study include the relatively small number of samples, which,
combined with our targeted approach, cannot produce prediction models capable of gener-
alization. In addition, confounding factors, like smoking [81], can influence gene expression
profiles in blood samples and should be considered. In our study, however, due to the high
accuracy provided by strong differential gene expression in our limited samples, those
factors only add complexity to the models without having any real impact on their accu-
racy. Regardless, the results of this study further strengthen the value of expression data
in predicting response to alcohol detoxification intervention and its effects. At baseline,
15 genes were identified with significant differences in expression, helping classify the
samples and positioning genes such as EGR4, GABRA5, RELA, and INHBA as a strong
candidate biomarker for predicting response to the detoxification approach. The reversal
of alcohol-induced dysregulation in genes related to synaptic plasticity and the shift in
gene–gene interactions suggest that the therapy facilitates recovery at the molecular level.
Several genes that were upregulated or downregulated due to alcohol exposure showed a
significant reversal towards the expression levels seen in healthy controls. This outcome
suggests that the intervention effectively alleviates some of the molecular disruptions
caused by alcohol use. Additionally, the reduction in the number of strong correlations
(with rho values > 0.8 or < −0.8) from 355 before treatment to 196 after treatment reflects a
shift towards gene expression patterns more similar to those of healthy controls. This shift
suggests that the gene expression network may be returning to a more regulated state, less influ-
enced by the chaotic expression patterns induced by alcohol consumption. These observations
highlighted in our results are compatible with the current literature that shows an early effect of
alcohol withdrawal towards the improvement of neuropsychological impairments [82].

4. Materials and Methods
4.1. Participants

A prospective, observational study of adult patients follow-up at First Department of
Psychiatry of Aeginition University Hospital, in Greece, was performed. Twenty patients
(17 men and 3 women, mean age 52 ± 6.93 years) fulfilled the DSM-V diagnostic crite-
ria for alcohol abuse/dependence [83] and were admitted at this specialized department
for alcohol detoxification on an inpatient basis. Subjects included in this study had to
fulfill the following criteria: (a) age between 18 and 70 years, (b) absence of serious physi-
cal illness (as assessed through physical examination and routine laboratory screening),
(c) absence of another pre- or co-existing major psychiatric disorder on the DSM-V, (d) ab-
sence of another drug abuse, and (e) the mere presence of affective symptoms or symptoms
of anxiety was not considered to be an exclusion criterion. Ten healthy individuals (7 men
and 3 women, mean age 48 ± 5.23 years) were included as controls. Participation in this
project was on a voluntary basis. This study was approved by the hospitals’ institutional

Int. J. Mol. Sci. 2024, 25, 11349

13 of 19

review board (346/19-4-22), and all patients provided their informed consent to participate.
Subjects were treated on an inpatient basis and had to abstain from alcohol. Before the
start of therapy and after 12 months, peripheral blood samples were obtained and stored at
−80 ◦C until use.

4.2. Participant Evaluation and Response Characterization

A general clinical and biochemical medical screen was performed to exclude severely
impaired individuals. Participants were diagnosed by the Schedules for Clinical Assess-
ment in Neuropsychiatry and assessed through the Composite International Diagnostic
Interview [84] (section on alcohol consumption) for their pattern of alcohol abuse, potential
major life problems related to alcohol consumption, and the occurrence of withdrawal
symptoms in the past; a structured questionnaire similar to the one proposed by the World
Health Organization [85] was also used to assess the pattern of alcohol use. This ques-
tionnaire included items related to lifetime, past year, and past month frequency and
quantity of alcohol use. Furthermore, sociodemographic data (age, socioeconomic status,
marital status, level of education) and previous psychiatric history (pre-existent diagnosis,
medication, hospitalizations) were recorded.

The WAIS-IV (Wechsler Adult Intelligence Scale-Fourth Edition) and the MMPI (Min-
nesota Multiphasic Personality Inventory) are two examples of neuropsychological exams
that are used to generate tailored treatment plans. This action helps to ensure that underly-
ing psychological problems are addressed in a complete manner [86,87].

A standard detoxification protocol was applied, comprising a short-term psychother-
apy of cognitive–behavioral orientation. Psychotherapeutic treatment consisted both
of individual sessions (PSAI) and family interventions. Vitamin replacement was ini-
tiated (Vitamin C, E, Vitamins of the B complex) and oral diazepam was administered
(30–60 mg per day) upon intake. Diazepam was gradually tapered off, with a reduction
rate of 20% every 2 days and was stopped after 7–10 days. No other medication was used
over this period. Additionally, a meta-cognitive stress management technique that aims to
reduce stress and improve physical and mental health, the Pythagorean Self Awareness
Intervention (PSAI), was also used. It involves a cognitive restructuring based on the
principles of Pythagoras’ philosophy, which activates the process of a deep spiritual intro-
spection and allows the trainee to evaluate, reward, and criticize his actions independently
of the toxic effects of the stressful event and the negative emotion that distorts thinking
and life choices. The specific program treats individuals holistically; individuals come
into full contact with themselves, evaluating their actions through self-observation. Via
this path, they detect any automatic thoughts, rationalize them, and finally replace them
with more functional ones. This metacognitive technique influences the cognitive triad
of thought–emotion–behavior, since, through recall and self-observation, compunctions
and ruminations are reduced situations that foster the development of psychopathology.
PSAI seems to activate the circuit of introspection (“inner consciousness”), a neurobiologi-
cal process that relates with Default Mode Network (DMN). The DMN is a neurological
connection system regarding cognitions that emanates from self-relevant procedures and
thereby contributes to corrective behaviors of the individual based on higher cognitive
functions. Apart from the cognitive level, PSAI changes and affects the whole lifestyle and
the daily routine of the participant, a result with great importance in addiction issues [88].
In research on alcohol use disorder (AUD), definitions of relapse are notably incon-
sistent and contentious. Defining relapse is essential for comprehending AUD recovery
trajectories and the impact of specific events, such as a return to drinking, on clinical
outcomes, including microbiome alterations. Definitions of relapse in the literature range
from “any alcohol consumption” to “episodes of heavy drinking” [89]. Incorporating both
remission and the biopsychosocial improvements into the definition of relapse seems cru-
cial [90]. The DSM-5 defines AUD remission as a period in which no symptoms (excluding
cravings) are present [83]. Relapse can be regarded as either a conclusion (complete return

Int. J. Mol. Sci. 2024, 25, 11349

14 of 19

to excessive drinking) or a component of a dynamic transformation process, wherein lapses
frequently occur but do not invariably result in a full relapse [91,92].

It is essential for our study to choose a relapse definition that harmonizes clinical
significance with feasibility. We recommend employing the DSM-5 AUD diagnostic criteria
during the follow-up or the threshold of 4/5 drinks for women/men after 4 days of absti-
nence, as endorsed by NIAAA (2005) guidelines [93] and utilized in treatment trials [91].
This term corresponds with the notion of “heavy drinking” as an indicator of relapse,
which has been linked to adverse psychosocial consequences and diminished recovery
pathways [92–94]. Furthermore, characterizing relapse as the intake of 4 to 5 alcoholic
beverages enables the assessment of significant alterations in alcohol consumption that are
likely to influence the microbiota, so offering a more refined comprehension of relapse is
beneficial in relation to both biological and clinical consequences. This method resolves
the ambiguity in relapse definitions by concentrating on a quantifiable outcome that indi-
cates substantial alterations in alcohol use and the related physiological and psychological
hazards [89,90]. This definition also facilitates the monitoring of microbiota alterations
after substantial drinking episodes, yielding crucial insights into the relationship between
relapse and gut health over time. With these criteria, 14 patients were characterized as
responders and 6 as non-responders.

4.3. Cohort Stratification

For the purposes of our analyses, our samples were grouped in the following categories
and were then pairwise analyzed: group 1 are alcoholics who responded to our intervention
at baseline, group 2 are alcoholics who did not respond to our intervention at baseline,
group 3 are the same subjects of group 1 after therapy, group 4 are the same subjects of
group 2 after therapy, and, finally, we have the group of our non-alcoholic control subjects.
We compared all samples of alcoholics at baseline with our healthy controls to profile gene
expression changes associated with alcohol abuse. Next, we compared responders’ to non-
responders’ expression profiles at baseline to identify transcriptional differences, which
could serve as predictive factors between individuals who would and would not respond
to therapy, given that the outcome is known. Finally, we compared the transcriptomic
differences imposed by our intervention to assess its effects upon success.

4.4. Differential Gene Expression Analysis (DGEA)

Total RNA from peripheral blood was extracted using the Qiagen AllPrep RNA/DNA
Mini Kit (Qiagen, Hilden, Germany). cDNA synthesis was performed with the RT2 First
Strand Kit (Qiagen) following the manufacturer’s guidelines. Gene expression was quan-
tified using the RT2 Profiler for Human Synaptic Plasticity RT-PCR Array (PAHS-126ZA,
Qiagen) with the RT2 qPCR SYBR Green Master Mix (Qiagen), testing a total of 84 genes.
Differential gene expression analysis (DGEA) was performed using the RT2 Pro-
filer PCR Array Data Analysis version 3.5 software from Qiagen on the following group
pairs: (1) group 1 and group 2 samples together versus controls to determine gene ex-
pression changes profiling alcohol abuse, (2) group 1 versus group 2 samples to identify
specific differences at baseline between those individuals who will and will not respond to
therapeutic intervention, and, finally, (3) group 3 versus group 1 to identify the effects of a
successful therapeutic intervention. Unfortunately, group 4 did not have enough samples
to allow us to perform DGEA. All samples satisfied the quality criteria for PCR Array
reproducibility, RT efficiency, and the absence of genomic DNA contamination.

Using five housekeeping genes (ACTB, B2M, GAPDH, HPRT1, and RPLP0) for within-
sample normalization, we calculated the 2−∆Ct values for all our genes. Fold change
differences were determined by the 2−∆∆Ct method and were reported in the form of fold
regulation in the results (genes exhibiting under-expression are denoted as the negative
reciprocal of the fold change, while overexpressed genes are presented as the fold change).
Statistical significance was assessed using Student’s t-tests conducted on the 2−∆Ct values
for each gene in all group comparisons. All results were deemed as statistically significant

Int. J. Mol. Sci. 2024, 25, 11349

15 of 19

based on their p < 0.05 values and their fold regulation thresholds of less than −2 or greater
than 2. Plots were created in R v. 4.3.0 [95], and log2(1/∆Ct) values were used to normalize
and visualize expression differences using the ggplot2 [96] package.

4.5. Participant Evaluation and Response Characterization

Correlation analysis is important for deciphering complex biological systems and
identifying potential relationships between genes. By investigating what are essentially
co-expression patterns, we can uncover the regulatory mechanisms underlying various
biological processes and highlight genes that show coordinated expression changes across
different conditions. To complement our DGEA and explore how the dysregulated genes
might interact with each other, we employed an R once again. We calculated Spearman’s
correlation coefficient (rho) for each gene pair and constructed a correlation matrix using the
cor() function for each group used in our DGEA. We chose Spearman’s non-parametric test,
as the Shapiro–Wilk normality test indicated that gene expressions were not all normally
distributed. The correlation matrix was then filtered to include only gene pairs with rho
values less than −0.8 (inverse correlation, when one gene is upregulated, the other is
downregulated and vice versa) or greater than 0.8, thereby capturing only the strongest
relationships across all the samples under study.

5. Conclusions

In conclusion, our study combined blood expression profiles that, when detected
before therapy, may predict response to alcohol detoxification approaches. Collectively, our
results set the background for both accomplishing significant insights into synaptic path-
ways in AUD and also developing predictors of detoxification therapy failure. Additionally,
our study supports that the expression pattern of genes related to synaptic plasticity in
peripheral blood can be a novel tool for clinical diagnosis, prognostication, and monitoring
therapeutic responses by a minimally invasive approach. Validation of our findings in
larger patient cohorts will be essential before the further development of such biomarkers
is proposed.

Author Contributions: Conceptualization, E.T. and M.G.; methodology, A.D.; software, N.D.; valida-
tion, N.D. and E.V.; formal analysis, I.K.; investigation, E.L. and N.M.; resources, A.D. and E.T.; data
curation, N.D.; writing—original draft preparation, E.L. and N.D; writing—review and editing, E.L.,
I.K. and M.G.; visualization, E.V.; supervision, M.G.; project administration, M.G. All authors have
read and agreed to the published version of the manuscript.

Funding: This research received no external funding.

Institutional Review Board Statement: This study was conducted in accordance with the Declaration
of Helsinki and approved by the Ethics Committee of Medical School, National and Kapodistrian
University of Athens for studies involving humans (No. 346/19-4-22).

Informed Consent Statement: Informed consent was obtained from all subjects involved in this study.

Data Availability Statement: The data generated and/or analyzed during the current study are
available from the corresponding author upon reasonable request.

Conflicts of Interest: The authors declare no conflicts of interest.

References

Carvalho, A.F.; Heilig, M.; Perez, A.; Probst, C.; Rehm, J. Alcohol use disorders. Lancet 2019, 394, 781–792. [CrossRef]
1.
Deak, J.D.; Miller, A.P.; Gizer, I.R. Genetics of alcohol use disorder: A review. Curr. Opin. Psychol. 2018, 27, 56–61. [CrossRef]
2.
3. World Health Organization (WHO). Alcohol Consumption Greece. ALCOHOL CONSUMPTION: LEVELS AND PATTERNS.
2016. Available online: https://cdn.who.int/media/docs/default-source/country-profiles/substances-abuse/grc.pdf?sfvrsn=
8622444a_3 (accessed on 10 September 2024).
Tsoumakas, K.; Tanaka, M.; Petsios, K.; Fildisis, G.; Gkoutzivelakis, A.; Pavlopoulou, I. Alcohol Drinking Habits and Negative
Experiences among Adolescents in Greece. Open J. Pediatr. 2014, 04, 222–230. [CrossRef]
Jones, J.D.; Comer, S.D.; Kranzler, H.R. The Pharmacogenetics of Alcohol Use Disorder. Alcohol. Clin. Exp. Res. 2015, 39, 391–402. [CrossRef]

4.

5.

Int. J. Mol. Sci. 2024, 25, 11349

16 of 19

6. Martin, C.R.; Preedy, V.R.; Patel, V.B.; Rajendram, R. (Eds.) Handbook of the Biology and Pathology of Mental Disorders; Springer

7.
8.

9.

Nature: Dordrecht, The Netherlands, 2025; ISBN 978-3-031-32035-4.
Citri, A.; Malenka, R.C. Synaptic Plasticity: Multiple Forms, Functions, and Mechanisms. Neuropsychopharmacology 2008, 33, 18–41. [CrossRef]
Bassi, M.S.; Iezzi, E.; Gilio, L.; Centonze, D.; Buttari, F. Synaptic Plasticity Shapes Brain Connectivity: Implications for Network
Topology. Int. J. Mol. Sci. 2019, 20, 6193. [CrossRef]
Appelbaum, L.G.; Shenasa, M.A.; Stolz, L.; Daskalakis, Z. Synaptic plasticity and mental health: Methods, challenges and
opportunities. Neuropsychopharmacology 2022, 48, 113–120. [CrossRef]

10. Duman, R.S.; Aghajanian, G.K.; Sanacora, G.; Krystal, J.H. Synaptic plasticity and depression: New insights from stress and

rapid-acting antidepressants. Nat. Med. 2016, 22, 238–249. [CrossRef]

11. Guelfi, G.; Casano, A.B.; Menchetti, L.; Bellicci, M.; Suvieri, C.; Moscati, L.; Carotenuto, P.; Santoro, M.M.; Diverio, S. A cross-talk
between blood-cell neuroplasticity-related genes and environmental enrichment in working dogs. Sci. Rep. 2019, 9, 6910.
[CrossRef] [PubMed]

12. The Lancet Psychiatry. Blood biomarkers in psychiatry. Lancet Psychiatry 2016, 3, 693. [CrossRef] [PubMed]
13. Lill, M.; Kõks, S.; Soomets, U.; Schalkwyk, L.C.; Fernandes, C.; Lutsar, I.; Taba, P. Peripheral blood RNA gene expression profiling

in patients with bacterial meningitis. Front. Neurosci. 2013, 7, 33. [CrossRef]

14. Glannon, W. Biomarkers in Psychiatric Disorders. Camb. Q. Healthc. Ethics 2022, 31, 444–452. [CrossRef]
15. Zhen, L.; Shao, T.; Luria, V.; Li, G.; Li, Z.; Xu, Y.; Zhao, X. EphB2 Deficiency Induces Depression-Like Behaviors and Memory

16.

Impairment: Involvement of NMDA 2B Receptor Dependent Signaling. Front. Pharmacol. 2018, 9, 862. [CrossRef]
Sanz-Martos, A.B.; Fuentes-Verdugo, E.; Merino, B.; Morales, L.; Pérez, V.; Capellán, R.; Pellón, R.; Miguéns, M.; del Olmo,
N. Schedule-induced alcohol intake during adolescence sex dependently impairs hippocampal synaptic plasticity and spatial
memory. Behav. Brain Res. 2023, 452, 114576. [CrossRef]

17. Robichaux, M.A.; Chenaux, G.; Ho, H.-Y.H.; Soskis, M.J.; Dravis, C.; Kwan, K.Y.; Šestan, N.; Greenberg, M.E.; Henkemeyer, M.;
Cowan, C.W. EphB receptor forward signaling regulates area-specific reciprocal thalamic and cortical axon pathfinding. Proc.
Natl. Acad. Sci. USA 2014, 111, 2188–2193. [CrossRef] [PubMed]

18. Henderson, N.T.; Dalva, M.B. EphBs and ephrin-Bs: Trans-synaptic organizers of synapse development and function. Mol. Cell.

Neurosci. 2018, 91, 108–121. [CrossRef] [PubMed]
Sloniowski, S.; Ethell, I.M. Looking forward to EphB signaling in synapses. Semin. Cell Dev. Biol. 2012, 23, 75–82. [CrossRef]
19.
20. Dou, X.; Lee, J.Y.; Charness, M.E. Neuroprotective Peptide NAPVSIPQ Antagonizes Ethanol Inhibition of L1 Adhesion by

Promoting the Dissociation of L1 and Ankyrin-G. Biol. Psychiatry 2020, 87, 656–665. [CrossRef]

21. Kim, S.H.; Song, J.Y.; Joo, E.J.; Lee, K.Y.; Shin, S.Y.; Lee, Y.H.; Ahn, Y.M.; Kim, Y.S. Genetic association of the EGR2 gene with

bipolar disorder in Korea. Exp. Mol. Med. 2012, 44, 121–129. [CrossRef]

22. Gallo, F.T.; Katche, C.; Morici, J.F.; Medina, J.H.; Weisstaub, N.V. Immediate Early Genes, Memory and Psychiatric Disorders: Focus

on c-Fos, Egr1 and Arc. Front. Behav. Neurosci. 2018, 12, 79. [CrossRef]

23. Covington, H.E., III; Lobo, M.K.; Maze, I.; Vialou, V.; Hyman, J.M.; Zaman, S.; LaPlant, Q.; Mouzon, E.; Ghose, S.; Tamminga, C.A.; et al.
Antidepressant Effect of Optogenetic Stimulation of the Medial Prefrontal Cortex. J. Neurosci. 2010, 30, 16082–16090. [CrossRef]
24. Yamada, K.; Gerber, D.J.; Iwayama, Y.; Ohnishi, T.; Ohba, H.; Toyota, T.; Aruga, J.; Minabe, Y.; Tonegawa, S.; Yoshikawa, T. Genetic
analysis of the calcineurin pathway identifies members of the EGR gene family, specifically EGR3, as potential susceptibility
candidates in schizophrenia. Proc. Natl. Acad. Sci. USA 2007, 104, 2815–2820. [CrossRef]

25. Kim, S.H.; Song, J.Y.; Joo, E.; Lee, K.Y.; Ahn, Y.M.; Kim, Y.S. EGR3 as a potential susceptibility gene for schizophrenia in Korea.

Am. J. Med. Genet. Part B Neuropsychiatr. Genet. 2010, 153B, 1355–1360. [CrossRef]

26. A Mansour, H.; E Talkowski, M.; Wood, J.; Chowdari, K.V.; McClain, L.; Prasad, K.; Montrose, D.; Fagiolini, A.; Friedman, E.S.;
Allen, M.H.; et al. Association study of 21 circadian genes with bipolar I disorder, schizoaffective disorder, and schizophrenia.
Bipolar Disord. 2009, 11, 701–710. [CrossRef]

27. Liu, B.-C.; Zhang, J.; Wang, L.; Li, X.-W.; Wang, Y.; Ji, J.; Yang, F.-P.; Wan, C.-L.; Gao, L.-H.; Xu, Y.-F.; et al. No association between
EGR gene family polymorphisms and schizophrenia in the Chinese population. Prog. Neuro-Psychopharmacol. Biol. Psychiatry
2010, 34, 506–509. [CrossRef] [PubMed]

28. Neasta, J.; Ben Hamida, S.; Yowell, Q.V.; Carnicella, S.; Ron, D. AKT Signaling Pathway in the Nucleus Accumbens Mediates

Excessive Alcohol Drinking Behaviors. Biol. Psychiatry 2011, 70, 575–582. [CrossRef]

29. Thiselton, D.L.; Vladimirov, V.I.; Kuo, P.-H.; McClay, J.; Wormley, B.; Fanous, A.; O’neill, F.A.; Walsh, D.; Oord, E.J.V.D.; Kendler,
K.S.; et al. AKT1 Is Associated with Schizophrenia Across Multiple Symptom Dimensions in the Irish Study of High Density
Schizophrenia Families. Biol. Psychiatry 2007, 63, 449–457. [CrossRef]

30. Avgan, N.; Sutherland, H.G.; Lea, R.A.; Spriggens, L.K.; Haupt, L.M.; Shum, D.H.K.; Griffiths, L.R. A CREB1 Gene Polymorphism

(rs2253206) Is Associated with Prospective Memory in a Healthy Cohort. Front. Behav. Neurosci. 2017, 11, 86. [CrossRef]

31. Kandel, E.R. The molecular biology of memory: cAMP, PKA, CRE, CREB-1, CREB-2, and CPEB. Mol. Brain 2012, 5, 14. [CrossRef]
32. Wolf, C.; An, Y.; Tanaka, T.; Bilgel, M.; Gonzalez, C.; Triolo, M.K.; Resnick, S. Cross-Sectional and Longitudinal Effects of CREB1
Genotypes on Individual Differences in Memory and Executive Function: Findings from the BLSA. Front. Aging Neurosci. 2017,
9, 142. [CrossRef] [PubMed]

33. Oliveira, A.M.; Wood, M.A.; McDonough, C.B.; Abel, T. Transgenic mice expressing an inhibitory truncated form of p300 exhibit

long-term memory deficits. Learn. Mem. 2007, 14, 564–572. [CrossRef]

Int. J. Mol. Sci. 2024, 25, 11349

17 of 19

34. Hettema, J.M.; An, S.; Oord, E.J.v.D.; Neale, M.C.; Kendler, K.S.; Chen, X. Association study of CREB1 with Major Depressive

Disorder and related phenotypes. Am. J. Med. Genet. Part B Neuropsychiatr. Genet. 2009, 150B, 1128–1132. [CrossRef]

35. McPherson, C.S.; Lawrence, A.J. The Nuclear Transcription Factor CREB: Involvement in Addiction, Deletion Models and Looking

Forward. Curr. Neuropharmacol. 2007, 5, 202–212. [CrossRef]

36. Zhang, H.; Kyzar, E.J.; Bohnsack, J.P.; Kokare, D.M.; Teppen, T.; Pandey, S.C. Adolescent alcohol exposure epigenetically regulates

CREB signaling in the adult amygdala. Sci. Rep. 2018, 8, 10376. [CrossRef]

37. Malenka, R.C.; Bear, M.F. LTP and LTD: An Embarrassment of Riches. Neuron 2004, 44, 5–21. [CrossRef]
38. Kauer, J.A.; Malenka, R.C. Synaptic plasticity and addiction. Nat. Rev. Neurosci. 2007, 8, 844–858. [CrossRef]
39. Li, M.; Luo, X.-J.; Rietschel, M.; Lewis, C.M.; Mattheisen, M.; Müller-Myhsok, B.; Jamain, S.; Leboyer, M.; Landén, M.; Thompson,
P.M.; et al. Allelic differences between Europeans and Chinese for CREB1 SNPs and their implications in gene expression
regulation, hippocampal structure and function, and bipolar disorder susceptibility. Mol. Psychiatry 2013, 19, 452–461. [CrossRef]
40. Ma, L.; Wu, D.-D.; Ma, S.L.; Tan, L.; Chen, X.; Tang, N.L.; Yao, Y.-G. Molecular evolution in the CREB1 signal pathway and a rare

haplotype in CREB1 with genetic predisposition to schizophrenia. J. Psychiatr. Res. 2014, 57, 84–89. [CrossRef]

41. Xiao, X.; Zhang, C.; Grigoroiu-Serbanescu, M.; Wang, L.; Li, L.; Zhou, D.; Yuan, T.-F.; Wang, C.; Chang, H.; Wu, Y.; et al. The cAMP
responsive element-binding (CREB)-1 gene increases risk of major psychiatric disorders. Mol. Psychiatry 2017, 23, 1957–1967.
[CrossRef] [PubMed]

42. Ohayon, S.; Yitzhaky, A.; Hertzberg, L. Gene expression meta-analysis reveals the up-regulation of CREB1 and CREBBP in

43.

Brodmann Area 10 of patients with schizophrenia. Psychiatry Res. 2020, 292, 113311. [CrossRef]
Fromer, M.; Roussos, P.; Sieberts, S.K.; Johnson, J.S.; Kavanagh, D.H.; Perumal, T.M.; Ruderfer, D.M.; Oh, E.C.; Topol, A.; Shah,
H.R.; et al. Gene expression elucidates functional impact of polygenic risk for schizophrenia. Nat. Neurosci. 2016, 19, 1442–1453.
[CrossRef] [PubMed]

44. Barrot, M.; Olivier, J.D.A.; Perrotti, L.I.; DiLeone, R.J.; Berton, O.; Eisch, A.J.; Impey, S.; Storm, D.R.; Neve, R.L.; Yin, J.C.; et al.
CREB activity in the nucleus accumbens shell controls gating of behavioral responses to emotional stimuli. Proc. Natl. Acad. Sci.
USA 2002, 99, 11435–11440. [CrossRef] [PubMed]

45. Muschamp, J.W.; Veer, A.V.; Parsegian, A.; Gallo, M.S.; Chen, M.; Neve, R.L.; Meloni, E.G.; Carlezon, W.A. Activation of CREB in
the Nucleus Accumbens Shell Produces Anhedonia and Resistance to Extinction of Fear in Rats. J. Neurosci. 2011, 31, 3095–3103.
[CrossRef] [PubMed]

46. Green, T.A.; Alibhai, I.N.; Roybal, C.N.; Winstanley, C.A.; Theobald, D.E.; Birnbaum, S.G.; Graham, A.R.; Unterberg, S.; Graham, D.L.;
Vialou, V.; et al. Environmental Enrichment Produces a Behavioral Phenotype Mediated by Low Cyclic Adenosine Monophosphate
Response Element Binding (CREB) Activity in the Nucleus Accumbens. Biol. Psychiatry 2010, 67, 28–35. [CrossRef]

47. Wang, H.; Xu, J.; Lazarovici, P.; Quirion, R.; Zheng, W. cAMP Response Element-Binding Protein (CREB): A Possible Signaling

Molecule Link in the Pathophysiology of Schizophrenia. Front. Mol. Neurosci. 2018, 11, 255. [CrossRef]

48. Pliakas, A.M.; Carlson, R.R.; Neve, R.L.; Konradi, C.; Nestler, E.J.; Carlezon, W.A. Altered Responsiveness to Cocaine and
Increased Immobility in the Forced Swim Test Associated with Elevated cAMP Response Element-Binding Protein Expression in
Nucleus Accumbens. J. Neurosci. 2001, 21, 7397–7403. [CrossRef] [PubMed]

49. Bramham, C.R.; Messaoudi, E. BDNF function in adult synaptic plasticity: The synaptic consolidation hypothesis. Prog. Neurobiol.

50.

51.

2005, 76, 99–125. [CrossRef]
Joffe, M.E.; Centanni, S.W.; Jaramillo, A.A.; Winder, D.G.; Conn, P.J. Metabotropic Glutamate Receptors in Alcohol Use Disor-
der: Physiology, Plasticity, and Promising Pharmacotherapies. ACS Chem. Neurosci. 2018, 9, 2188–2204. [CrossRef]
Johnson, K.A.; Lovinger, D.M. Allosteric Modulation of Metabotropic Glutamate Receptors in Alcohol Use Disorder: Insights
from Preclinical Investigations. Adv. Pharmacol. 2020, 88, 193–232. [CrossRef]

52. Olive, M.F.; Mcgeehan, A.J.; Kinder, J.R.; McMahon, T.; Hodge, C.W.; Janak, P.H.; Messing, R.O. The mGluR5 Antagonist
6-Methyl-2-(phenylethynyl)pyridine Decreases Ethanol Consumption via a Protein Kinase Cϵ-Dependent Mechanism. Mol.
Pharmacol. 2004, 67, 349–355. [CrossRef]

53. Goodwani, S.; Saternos, H.; Alasmari, F.; Sari, Y. Metabotropic and ionotropic glutamate receptors as potential targets for the

treatment of alcohol use disorder. Neurosci. Biobehav. Rev. 2017, 77, 14–31. [CrossRef] [PubMed]

54. Terbeck, S.; Akkus, F.; Chesterman, L.P.; Hasler, G. The role of metabotropic glutamate receptor 5 in the pathogenesis of mood
disorders and addiction: Combining preclinical evidence with human Positron Emission Tomography (PET) studies. Front.
Neurosci. 2015, 9, 86. [CrossRef]

55. Kasten, C.R.; Holmgren, E.B.; Wills, T.A. Metabotropic Glutamate Receptor Subtype 5 in Alcohol-Induced Negative Affect. Brain

Sci. 2019, 9, 183. [CrossRef]

56. Bäckström, P.; Bachteler, D.; Koch, S.; Hyytiä, P.; Spanagel, R. mGluR5 Antagonist MPEP Reduces Ethanol-Seeking and Relapse

Behavior. Neuropsychopharmacology 2003, 29, 921–928. [CrossRef]

57. Alasmari, F.; Bell, R.L.; Rao, P.; Hammad, A.M.; Sari, Y. Peri-adolescent drinking of ethanol and/or nicotine modulates astroglial
glutamate transporters and metabotropic glutamate receptor-1 in female alcohol-preferring rats. Pharmacol. Biochem. Behav. 2018,
170, 44–55. [CrossRef] [PubMed]

58. Cozzoli, D.K.; Goulding, S.P.; Zhang, P.W.; Xiao, B.; Hu, J.-H.; Ary, A.W.; Obara, I.; Rahn, A.; Abou-Ziab, H.; Tyrrel, B.; et al. Binge
Drinking Upregulates Accumbens mGluR5-Homer2-PI3K Signaling: Functional Implications for Alcoholism. J. Neurosci. 2009,
29, 8655–8668. [CrossRef] [PubMed]

Int. J. Mol. Sci. 2024, 25, 11349

18 of 19

59. Cozzoli, D.K.; Courson, J.; Wroten, M.G.; Greentree, D.I.; Lum, E.N.; Campbell, R.R.; Thompson, A.B.; Maliniak, D.; Worley, P.F.;
Jonquieres, G.; et al. Binge Alcohol Drinking by Mice Requires Intact Group 1 Metabotropic Glutamate Receptor Signaling within
the Central Nucleus of the Amygdala. Neuropsychopharmacology 2014, 39, 435–444. [CrossRef]
Szumlinski, K.K.; Ary, A.W.; Lominac, K.D.; Klugmann, M.; Kippin, T.E. Accumbens Homer2 Overexpression Facilitates
Alcohol-Induced Neuroplasticity in C57BL/6J Mice. Neuropsychopharmacology 2007, 33, 1365–1378. [CrossRef]

60.

61. Lee, K.M.; Coelho, M.A.; McGregor, H.A.; Solton, N.R.; Cohen, M.; Szumlinski, K.K. Adolescent Mice Are Resilient to Alcohol
Withdrawal-Induced Anxiety and Changes in Indices of Glutamate Function within the Nucleus Accumbens. Front. Cell. Neurosci.
2016, 10, 265. [CrossRef]

63.

62. Obara, I.; Bell, R.L.; Goulding, S.P.; Reyes, C.M.; Larson, L.A.; Ary, A.W.; Truitt, W.A.; Szumlinski, K.K. Differential Effects of
Chronic Ethanol Consumption and Withdrawal on Homer/Glutamate Receptor Expression in Subregions of the Accumbens and
Amygdala of P Rats. Alcohol. Clin. Exp. Res. 2009, 33, 1924–1934. [CrossRef]
Simonyi, A.; Christian, M.R.; Sun, A.Y.; Sun, G.Y. Chronic Ethanol-Induced Subtype- and Subregion-Specific Decrease in the
mRNA Expression of Metabotropic Glutamate Receptors in Rat Hippocampus. Alcohol. Clin. Exp. Res. 2004, 28, 1419–1423.
[CrossRef] [PubMed]
Szumlinski, K.K.; Dehoff, M.H.; Kang, S.H.; Frys, K.A.; Lominac, K.D.; Klugmann, M.; Rohrer, J.; Griffin, W., III; Toda, S.;
Champtiaux, N.P.; et al. Homer Proteins Regulate Sensitivity to Cocaine. Neuron 2004, 43, 401–413. [CrossRef] [PubMed]
65. Lominac, K.D.; Kapasova, Z.; Hannun, R.A.; Patterson, C.; Middaugh, L.D.; Szumlinski, K.K. Behavioral and neurochemical
interactions between Group 1 mGluR antagonists and ethanol: Potential insight into their anti-addictive properties. Drug Alcohol
Depend. 2006, 85, 142–156. [CrossRef] [PubMed]

64.

66. Meyers, J.L.; Salling, M.C.; Almli, L.M.; Ratanatharathorn, A.; Uddin, M.; Galea, S.; Wildman, D.E.; Aiello, A.E.; Bradley, B.;
Ressler, K.; et al. Frequency of alcohol consumption in humans; the role of metabotropic glutamate receptors and downstream
signaling pathways. Transl. Psychiatry 2015, 5, e586. [CrossRef] [PubMed]

67. Lovinger, D.M.; Kash, T.L. Mechanisms of Neuroplasticity and Ethanol’s Effects on Plasticity in the Striatum and Bed Nucleus of

the Stria Terminalis. Alcohol Res. 2015, 37, 109–124. [PubMed]

68. Castelli, V.; Brancato, A.; Cavallaro, A.; Lavanco, G.; Cannizzaro, C. Homer2 and Alcohol: A Mutual Interaction. Front. Psychiatry

2017, 8, 268. [CrossRef]

69. Dahl, J.P.; Kampman, K.M.; Oslin, D.W.; Weller, A.E.; Lohoff, F.W.; Ferraro, T.N.; O’Brien, C.; Berrettini, W.H. Association of
a polymorphism in the Homer1 gene with cocaine dependence in an African American population. Psychiatr. Genet. 2005,
15, 277–283. [CrossRef]

70. Lum, E.N.; Campbell, R.R.; Rostock, C.; Szumlinski, K.K. mGluR1 within the nucleus accumbens regulates alcohol intake in mice

71.

under limited-access conditions. Neuropharmacology 2014, 79, 679–687. [CrossRef]
Szumlinski, K.K.; Lominac, K.D.; Oleson, E.B.; Walker, J.K.; Mason, A.; Dehoff, M.H.; Klugman, M.; Cagle, S.; Welt, K.; During,
M.; et al. Homer2 Is Necessary for EtOH-Induced Neuroplasticity. J. Neurosci. 2005, 30, 7054–7061. [CrossRef]

72. Haider, A.; Woodward, N.C.; Lominac, K.D.; Sacramento, A.D.; Klugmann, M.; Bell, R.L.; Szumlinski, K.K. Homer2 within the
nucleus accumbens core bidirectionally regulates alcohol intake by both P and Wistar rats. Alcohol 2015, 49, 533–542. [CrossRef]
73. Cozzoli, D.K.; Courson, J.; Caruana, A.L.; Miller, B.W.; Greentree, D.I.; Thompson, A.B.; Wroten, M.G.; Zhang, P.; Xiao, B.; Hu,
J.; et al. Nucleus Accumbens mGluR5-Associated Signaling Regulates Binge Alcohol Drinking Under Drinking-in-the-Dark
Procedures. Alcohol. Clin. Exp. Res. 2012, 36, 1623–1633. [CrossRef] [PubMed]

74. Easton, A.C.; The GESGA Consortium; Lucchesi, W.; Lourdusamy, A.; Lenz, B.; Solati, J.; Golub, Y.; Lewczuk, P.; Fernandes, C.;
Desrivieres, S.; et al. αCaMKII Autophosphorylation Controls the Establishment of Alcohol Drinking Behavior. Neuropsychophar-
macology 2013, 38, 1636–1647. [CrossRef] [PubMed]
Siddiqi, M.T.; Podder, D.; Pahng, A.R.; Athanason, A.C.; Nadav, T.; Cates-Gatto, C.; Kreifeldt, M.; Contet, C.; Roberts, A.J.;
Edwards, S.; et al. Prefrontal cortex glutamatergic adaptations in a mouse model of alcohol use disorder. Addict. Neurosci. 2023,
9, 100137. [CrossRef]

75.

76. Ohashi, K.; Pimienta, M.; Seki, E. Alcoholic liver disease: A current molecular and clinical perspective. Liver Res. 2018, 2, 161–172.

[CrossRef] [PubMed]

77. Torp, N.; Israelsen, M.; Nielsen, M.J.; Åstrand, C.P.; Juhl, P.; Johansen, S.; Hansen, C.D.; Madsen, B.; Villesen, I.F.; Leeming, D.J.; et al.
Binge drinking induces an acute burst of markers of hepatic fibrogenesis (PRO-C3). Liver Int. 2021, 42, 92–101. [CrossRef] [PubMed]
78. Zdanowicz, K.; Kowalczuk-Kryston, M.; Olanski, W.; Werpachowska, I.; Mielech, W.; Lebensztejn, D.M. Increase in Serum MMP9 and
TIMP-1 Concentrations during Alcohol Intoxication in Adolescents—A Preliminary Study. Biomolecules 2022, 12, 710. [CrossRef]
79. Badami, V.M.; Rice, C.D.; Lois, J.H.; Madrecha, J.; Yates, B.J. Distribution of hypothalamic neurons with orexin (hypocretin) or
melanin concentrating hormone (MCH) immunoreactivity and multisynaptic connections with diaphragm motoneurons. Brain
Res. 2010, 1323, 119–126. [CrossRef]

80. Carroll, J.E.; Irwin, M.R.; Levine, M.; Seeman, T.E.; Absher, D.; Assimes, T.; Horvath, S. Epigenetic Aging and Immune Senescence in
Women With Insomnia Symptoms: Findings From the Women’s Health Initiative Study. Biol. Psychiatry 2016, 81, 136–144. [CrossRef]
81. Kõks, G.; Uudelepp, M.-L.; Limbach, M.; Peterson, P.; Reimann, E.; Kõks, S. Smoking-Induced Expression of the GPR15 Gene

Indicates Its Potential Role in Chronic Inflammatory Pathologies. Am. J. Pathol. 2015, 185, 2898–2906. [CrossRef]

82. Angerville, B.; Ritz, L.; Pitel, A.-L.; Beaunieux, H.; Houchi, H.; Martinetti, M.P.; Naassila, M.; Dervaux, A. Early Improvement of
Neuropsychological Impairments During Detoxification in Patients with Alcohol Use Disorder. Alcohol Alcohol. 2022, 58, 46–53. [CrossRef]

Int. J. Mol. Sci. 2024, 25, 11349

19 of 19

83. American Psychiatric Association. Diagnostic and Statistical Manual of Mental Disorders, 5th ed.; American Psychiatric Associa-

tion: Arlington, VA, USA, 2013.

84. Wittchen, H.U.; Nelson, C.B. The Composite International Diagnostic Interview: An Instrument for Measuring Mental Health Outcome? In
Mental Health Outcome Measures; Thornicroft, G., Tansella, M., Eds.; Springer: Berlin/Heidelberg, Germany, 1996; pp. 179–187. [CrossRef]
85. Hughes, P.H.; Venulet, J.; Khant, U.; Mora, M.E.M.; Navaratnam, V.; Poshyachinda, V.; Rootman, I.; Salan, R.; Wadud, K.A. Core
Data for Epidemiological Studies of Nonmedical Drug Use; World Health Organization: Geneva, Switzerland, 1980; pp. 1–100.
86. Valmas, M.M.; Ruiz, S.M.; Gansler, D.A.; Sawyer, K.S.; Oscar-Berman, M. Social Cognition Deficits and Associations with Drinking

History in Alcoholic Men and Women. Alcohol. Clin. Exp. Res. 2014, 38, 2998–3007. [CrossRef] [PubMed]

87. Merz, Z.C.; Van Patten, R.; Hurless, N.; Grant, A.; McGrath, A.B. Furthering the Understanding of Wechsler Adult Intelligence

88.

89.

Scale-Fourth Edition Factor Structure in a Clinical Sample. Appl. Neuropsychol. Adult 2021, 28, 12–23. [CrossRef]
Shapira-Lichter, I.; Oren, N.; Jacob, Y.; Gruberger, M.; Hendler, T. Portraying the unique contribution of the default mode network
to internally driven mnemonic processes. Proc. Natl. Acad. Sci. USA 2013, 110, 4950–4955. [CrossRef] [PubMed]
Sliedrecht, W.; Roozen, H.; de Waart, R.; Dom, G.; Witkiewitz, K. Variety in Alcohol Use Disorder Relapse Definitions: Should the
Term “Relapse” Be Abandoned? J. Stud. Alcohol Drugs 2022, 83, 248–259. [CrossRef] [PubMed]

90. Hagman, B.T.; Falk, D.; Litten, R.; Koob, G.F. Defining Recovery From Alcohol Use Disorder: Development of an NIAAA Research

Definition. Am. J. Psychiatry 2022, 179, 807–813. [CrossRef]

91. Maisto, S.A.; Roos, C.R.; Hallgren, K.A.; Moskal, D.; Wilson, A.D.; Witkiewitz, K. Do Alcohol Relapse Episodes During Treatment
Predict Long-Term Outcomes? Investigating the Validity of Existing Definitions of Alcohol Use Disorder Relapse. Alcohol. Clin.
Exp. Res. 2016, 40, 2180–2189. [CrossRef]

92. Witkiewitz, K.; Marlatt, G.A. Relapse Prevention for Alcohol and Drug Problems: That Was Zen, This Is Tao. Am. Psychol. 2004,

59, 224–235. [CrossRef] [PubMed]

93. National Institute on Alcohol Abuse and Alcoholism (NIAAA). Helping Patients Who Drink Too Much: A Clinician’s Guide; National

Institutes of Health: Bethesda, MD, USA, 2005.

94. Pearson, M.R.; Kirouac, M.; Witkiewitz, K. Questioning the Validity of the 4+/5+ Binge or Heavy Drinking Criterion in College

and Clinical Populations. Addiction 2016, 112, 215–222. [CrossRef]
Ihaka, R.; Gentleman, R. R: A Language for Data Analysis and Graphics. J. Comput. Graph. Stat. 1996, 5, 299–314. [CrossRef]

95.
96. Wickham, H. ggplot2. Wiley Interdiscip. Rev. Comput. Stat. 2011, 3, 180–185. [CrossRef]

Disclaimer/Publisher’s Note: The statements, opinions and data contained in all publications are solely those of the individual
author(s) and contributor(s) and not of MDPI and/or the editor(s). MDPI and/or the editor(s) disclaim responsibility for any injury to
people or property resulting from any ideas, methods, instructions or products referred to in the content.
