# Masini_2020_An Overview of the Main Genetic, Epigenetic and Environmental Factors Involved in Autism Spectrum Disorder Focusing on Synaptic Activity.

Review
An Overview of the Main Genetic, Epigenetic and
Environmental Factors Involved in Autism Spectrum
Disorder Focusing on Synaptic Activity

Elena Masini 1,†
Giuseppe Doneddu 3, Roberta Fadda 4

, Eleonora Loi 1,†

, Ana Florencia Vega-Benedetti 1
and Patrizia Zavattari 1,*

, Marinella Carta 2,

1 Department of Biomedical Sciences, Unit of Biology and Genetics, University of Cagliari,
09042 Cagliari, Italy; elena.masini97@gmail.com (E.M.); eleonora.loi@unica.it (E.L.);
anaf.vegab@unica.it (A.F.V.-B.)

2 Center for Pervasive Developmental Disorders, Azienda Ospedaliera Brotzu, 09121 Cagliari, Italy;

marinellacarta@aob.it

3 Centro per l’Autismo e Disturbi correlati (CADc), Nuovo Centro Fisioterapico Sardo, 09131 Cagliari, Italy;

io.donez@gmail.com

4 Department of Pedagogy, Psychology, Philosophy, University of Cagliari, 09123 Cagliari, Italy;

robfadda@unica.it

* Correspondence: pzavattari@unica.it
† The authors contributed equally to this work.

Received: 30 September 2020; Accepted: 30 October 2020; Published: 5 November 2020

Abstract: Autism spectrum disorder (ASD) is a neurodevelopmental disorder that aﬀects social
interaction and communication, with restricted interests, activity and behaviors. ASD is highly
familial, indicating that genetic background strongly contributes to the development of this condition.
However, only a fraction of the total number of genes thought to be associated with the condition
have been discovered. Moreover, other factors may play an important role in ASD onset. In fact,
it has been shown that parental conditions and in utero and perinatal factors may contribute to ASD
etiology. More recently, epigenetic changes, including DNA methylation and micro RNA alterations,
have been associated with ASD and proposed as potential biomarkers. This review aims to provide a
summary of the literature regarding ASD candidate genes, mainly focusing on synapse formation
and functionality and relevant epigenetic and environmental aspects acting in concert to determine
ASD onset.

Keywords: autism spectrum disorder; ASD; genetic factors; epigenetic factors; environmental factors;
pervasive developmental disorder; post-synaptic density; CNV; SNP; gene fusion

1. Introduction

Autism is a complex syndrome characterized by a range of conditions and symptoms that frame
it as a spectrum of disorders (autism spectrum disorder, ASD), including relevant physiological and
biochemical ones, whose core symptoms includes social deﬁcits and restrictive/repetitive behaviors.
In the 1960s, thanks to the studies of Bernard Rimland, it was understood that ASD is a psychiatric
disorder which might be grounded on a combination of genetic and environmental factors. To cope
with ASD, it is necessary to achieve positive results through multidisciplinary, biomedical and
behavioral therapies. Early diagnosis and intensive therapeutic interventions greatly improve the
disease outcomes. In this context, the discovery of new diagnostic methods to detect ASD-related
genetic alterations and biomarkers becomes fundamental in order to make an early diagnosis of
the disorder.

Int. J. Mol. Sci. 2020, 21, 8290; doi:10.3390/ijms21218290

www.mdpi.com/journal/ijms

 International Journal of Molecular Sciences(cid:1)(cid:2)(cid:3)(cid:1)(cid:4)(cid:5)(cid:6)(cid:7)(cid:8)(cid:1)(cid:1)(cid:2)(cid:3)(cid:4)(cid:5)(cid:6)(cid:7)Int. J. Mol. Sci. 2020, 21, 8290

2 of 22

In this review, we provide an overview of the genetic, epigenetic and environmental factors

contributing to ASD pathogenesis.

2. Autism

2.1. Clinical Characteristics of ASD

ASD can be considered as a group of early-onset neuroevolutionary disorders which seem to be
at the basis of alterations in brain connectivity, with cascading eﬀects on many neuropsychological
functions [1,2]. The The Diagnostic and Statistical Manual of Mental Disorders (DSM-5) (The American
Psychiatric Association, APA, Philadelphia, PA, USA, 2013) gives the condition of autism the attribute
of “spectrum” and uses criteria derived from diagnostic research assessment tools.

As indicated in the DSM-5 (APA, Philadelphia, PA, USA, 2013), individuals with ASD are
characterized by persistent deﬁcits in social communication and social interaction across multiple
contexts and by restricted, repetitive patterns of behaviour, interests or activities. Deﬁciency in social
communication and social interaction might appear in the form of deﬁcits in social-emotional
reciprocity, in nonverbal communicative behaviours used for social interaction and deﬁcit in
developing, maintaining and understanding age-appropriate relationships. As reported in the DSM-5,
symptoms could be masked during early development and fully manifest only when social demands
exceed limited capacities or may be hidden by learned strategies in later life. The impairments
should cause clinically signiﬁcant damage in social, occupational or other important areas of
current functioning.

ASD symptoms should not be better explained by a diagnosis of intellectual disability (ID) or

global developmental delay. Intellectual disability and ASD might co-occur.

The DSM-5 proposes diﬀerentiations based on commorbidity with intellectual impairment,
language impairment, another neurodevelopmental, mental, or behavioral disorder, genetic or medical
condition or environmental factors. Furthermore, it is possible to diﬀerentiate between diﬀerent levels
of severity according to the level of support required to function in daily contexts. According to this
description, it is clear that diﬀerent clinical variants of ASD exist and should be taken into account
for diagnosis and intervention (APA, 2013). A distinction is made between a congenital form of
ASD, representing a small percentage of cases in which the symptoms occur shortly after birth and in
which the genetic ﬁngerprint is prevalent, and a regressive or acquired form, in which the disorder
appears after a period of typical development and it is not characterised by typical and constant genetic
abnormalities, although several single nucleotide polymorphisms (SNPs) have been associated with the
disease [3,4]. SNPs constitute variations of a single nucleotide in certain DNA traits. SNPs associated
with ASD have been identiﬁed in genes encoding for proteins involved in diﬀerent processes, including:
cellular detoxiﬁcation, some neuronal receptors and metabolism of several neurotransmitters and
metabolites, in particular those of the metabolic circuits of methylation and transulfuration [5,6].

2.2. Epidemiology

Autism has been considered relatively rare for many years, with a prevalence of less than 1 in
1000 children, while today, the estimated rate is 1 in 160, and it seems likely to increase in the coming
years (World Health Organization, Geneva, Switzerland, 2019). In the last decade, the study of ASD
genetics has proved to be crucial not only to interpret and explain its phenotypic heterogeneity but
also to discover new diagnostic procedures and therapies. It is estimated to-date that hundreds of
genes are involved in ASD, resulting in a uniﬁed spectrum of diﬀerent phenotypes, including diﬀerent
language and social deﬁcits with various associated sub-phenotypes [7].

ASD show an unequal distribution based on gender: males have a four times higher risk of
developing the disorder than females. A number of hypotheses have been made to interpret this unequal
prevalence: in females, a higher dose of genetic “defect” is required than in males, consistent with the
hypothesis of the contribution of protective genetic factors in females, because in males, there is only

Int. J. Mol. Sci. 2020, 21, 8290

3 of 22

one X chromosome, so when alterations appear, it cannot be compensated by the normality of a second
X chromosome. An association between testosterone levels and ASD risk has also been described:
males have more frequent inﬂammatory reactions and use the brain in a more focal way and therefore
suﬀer more from alterations in neuronal development that aﬀect the connection systems between the
diﬀerent areas [8].

Genetic, environmental and developmental factors play a key role in the onset of autism spectrum
disorders, as highlighted from many epidemiological studies [9,10]. It is unlikely that a single condition
or event plays a major role in the causality of ASD; based on research to date, rather, none of
the risk factors identiﬁed is a necessary and suﬃcient condition for ASD. Even for syndromic or
secondary autism, which refers to autism with a single deﬁned cause, such as fragile X syndrome and
tuberous sclerosis, none of these etiologies are speciﬁc to autism because each of them encompasses a
variable proportion of individuals with and without autism [11]. At present, ASD appears to have a
multifactorial etiology to which developmental (in utero and early childhood), environmental and
genetic aspects contribute, in as-yet unknown and diﬀerent ways. Emerging methodologies in genomics
and epigenomics research could be the key to elucidate the mysteries underlying the epidemiology of
autism spectrum disorder.

3. Genetic and Epigenetic Factors

The involvement of genetic etiology in ASD was ﬁrst suggested by a study on twins reported in
the 1970s. The genetic heritability of a trait can be estimated by comparing the phenotypic concordance
between monozygotic twins (MZ), which have 100% genetic similarity, and dizygotic twins (DZ),
who have approximately 50% genetic similarity. The greater the diﬀerence between the concordance of
MZ twins and DZ twins, the higher the genetic heritability and the contribution of genetics to that
trait. The genetic ﬁngerprint was conﬁrmed by the high concordance of autism in monozygotic twins
(60–90%) compared to dizygotic twins (5–40%) [12,13].

One of the largest studies to date, involving more than two million children born between 1982
and 2006 in Sweden, concluded that ASD has an inheritability of 45–56% [14]. A study of all twins
born in the UK between 1994 and 1996 estimated an inheritability of more than 56% using concordance
in MZ (0.77–0.99) compared to DZ (0.22–0.65) twins [13].

Considering all lines of evidence, the genetic heritability of ASD is estimated to play a fundamental
role in ASD onset, together with environmental and epigenetic factors. However, despite most of
the studies aimed at understanding the etiological basis of ASD having been focused on the genetic
component, it has been possible to associate genetic variants to only a relatively small fraction of ASD
patients. The problem, known as the “missing heritability issue”, is common to most complex genetic
diseases. Several hypotheses have been put forward to justify the missing heritability, such as the
existence of poorly characterised variants, genotype/genotype interactions, incomplete penetrance,
epigenetic factors and genotype/environment interactions [15–17].

In the 1990s, most of the research consisted of candidate gene studies, focusing on a particular
gene that might be involved in ASD. From 2005, technologies such as whole-exome sequencing (WES),
but also microarrays, have allowed genome-wide studies, leading to the identiﬁcation of diﬀerent
variations in copy number (CNVs), DNA segments larger than 1 kilobase present in a variable copy
number compared to a reference genome and single nucleotide variations (SNVs) in autistic patients,
suggesting a highly heterogeneous genetic architecture. CNVs would contribute at about 15% and
SNVs at 7% to the causes of ASD. Only a few of these genetic alterations have such complete penetrance
that they are associated with ASD in almost every person who carries that variant [18]. On the contrary,
genetic alterations with incomplete penetrance, variable expressivity or both are more frequently
observed. However, the cause in most ASDs (>75%) remains elusive [18]. Genome-wide association
studies (GWAS) have identiﬁed several SNPs associated with ASD. The ﬁrst GWAS study allowed the
identiﬁcation of six polymorphisms, including some localized in CDH10 and CDH9, genes encoding
for cadherins, proteins that are important in cell adhesion, as common genetic variants in ASD [19].

Int. J. Mol. Sci. 2020, 21, 8290

4 of 22

However, the fact that several GWASs failed to identify some relevant loci despite the use of genetic
data from more than 1000 families aﬀected by autism suggests that the eﬀect of individual common
variants is relatively small [20].

On the other hand, WES analysis of aﬀected and unaﬀected individuals has proven to be a
powerful approach that oﬀers new opportunities of sporadic cases studies and has the ability to
detect mutations and de novo variants with incomplete penetrance [21]. WES has already led to the
identiﬁcation of over 150 new candidate genes for ASD.

In addition to SNPs, evidence is accumulating that CNVs play an important role in human
neuropsychiatric diseases. ASD patients have three to ﬁve times more de novo CNVs than other family
members and unaﬀected controls [22,23].

CNVs can inﬂuence gene expression, thus contributing to the pathogenesis of the disease through
various mechanisms, including gene dosage, gene interruption, position eﬀects, gene fusion and
unmasking of recessive alleles or polymorphisms [24].

Screening for CNVs has proven to be a method of choice for identifying genes associated with
ASD susceptibility [25]. Although CNVs associated with the disease are usually unique and show a low
frequency in the population, they are identiﬁed in 8–21% of individuals with ASD and are most likely
related to a severe clinical picture [26,27]. In addition, previous studies have indicated that individuals
with syndromic ASD and intellectual disability have more pathogenic CNVs than individuals with
non-syndromic ASD or ID [26,28].

CNVs may also lead to the generation of chimeric genes. Several studies have investigated
whether fusion transcripts may lead to an increased ASD susceptibility. Holt and colleagues
identiﬁed a fusion transcript involving MAPKAPK5 and ACAD10 genes in two ASD probands.
However, the fusion transcript was detected at similar rates in both ASD patients and controls and
had a premature stop codon, suggesting that it may be degraded by nonsense-mediated decay [29].
Similarly, Pagnamenta et al. identiﬁed a DOCK4-IMMP2L fusion transcript, likely to be subjected to
nonsense-mediated decay in ASD individuals and their unaﬀected family members [30]. A study
conducted on a multiplex family identiﬁed a BST1-CD38 fusion transcript in one ASD proband with
asthma, suggesting that it may be related to the more severe phenotype of this patient compared to the
other ASD sibling [31]. qRT-PCR analysis showed that the fusion transcript was less expressed compared
to the wild-type BST1 transcript in the lymphoblastoid cell line derived from the proband, while the
aberrant protein was not detected in a preliminary Western blot analysis [31]. Recently, our research
group identiﬁed a microdeletion leading to the formation of an ELMOD3-SH2D6 chimeric transcript
in a multiplex ASD family [32]. SH2D6 is expressed at extremely low levels in blood cells. On the
other hand, the fusion transcript was highly expressed in PMBCs from the two ASD siblings and their
unaﬀected mother carrying the deletion, suggesting that it was not subjected to nonsense-mediated
decay. Bioinformatic analysis has shown that the fusion transcript would encode for a chimeric
protein with an interrupted domain of ELMOD3 and would not contain the canonical SH2D6 sequence,
suggesting an impaired function of the protein [32]. These results suggest a possible contribution of
fusion transcripts in the complex ASD phenotype. Therefore, in case of copy number loss, the possible
transcript fusion and chimeric protein product should be deeply investigated.

Recent studies have shown that epigenetic factors, including DNA methylation, hystone modifications

and microRNAs (miRNAs), could play an important role in predisposition to autism.

We herein provide an overview of the main candidate genes (extensively reviewed in [33]) and
epigenetic mechanisms involved in ASD etiology. Figure 1 summarizes some ASD candidate genes
and epigenetic factors, belonging to the pathways mainly associated with ASD described in this review.

Int. J. Mol. Sci. 2020, 21, 8290

5 of 22

Figure 1. Candidate genes and epigenetic factors representative of the main processes involved in autism
spectrum disorder (ASD) development. The illustration shows a synapse between neurons (presynaptic
cell in violet and postsynaptic cell in green). On the bottom-left, a cell body of a neuron including
diﬀerent nuclear and cytoplasmic mechanisms involved in ASD. In the nucleus, several processes
implicated in gene expression regulation are shown: (1) chromatin packaging and factors involved in
chromatin remodeling; (2) gene transcription regulated by transcription factors; (3) DNA methylation
at promoter region associated with transcription inhibition of target genes; (4) alternative splicing and
mRNA export to the cytoplasm. In the cytoplasm, the following mechanisms are shown: (5) regulation of
protein translation by the CYFIP1-EIF4E-FMR1 complex; (6) post-transcriptional regulation by miRNA;
(7) protein ubiquitination and degradation by proteasome. On the right, the synapse architecture
and functionality mechanisms associated with ASD. In the presynaptic cell, (8) TSC proteins and
(9) The neurexin/neuroligin transsynaptic complex and (10) the voltage-gated ion
co-chaperons.
channels are represented. In the postsynaptic cell, (11) actin ﬁlaments, capping proteins and scaﬀold
proteins; (12) some members of PI3K/AKT pathway, RAS signal transduction pathway and MET
receptor tyrosine kinase pathway. Chromatin remodelers are indicated in beige, transcription factors
in pink, proteins involved in RNA binding and export in light blue, protein ubiquitination in purple,
scaﬀold proteins in red, cell growth and proliferation proteins in green and their related pathway
members in grey. A more comprehensive list of ASD candidate genes can be found in Table 1 and along
the text. Figure created using BioRender.com images.

3.1. Relevant Candidate Genes

Case-control studies on population and animal models have pointed out more than 800 genes
associated with autism. The most affected genes in ASD encode for proteins involved in chromatin
remodeling and transcriptional regulation, cell proliferation and mostly synaptic architecture and
functionality. In this review, we will focus on this last category since our recent studies have also pointed
out alterations in these genes fundamental for a proper synaptic function. Table 1 provides a summary
of several genes clearly implicated in ASD, included in the SFARI Gene database as high confidence ASD
genes (release 26 October 2020, gene.sfari.org) belonging to the other categories. Most of these genes
were indicated in the largest exome sequencing study of ASD to date [34], as well as in the list narrowing
down the number of amygdala-expressed genes associated to the social pathophysiology of ASD [35].

1     Int. J. Mol. Sci. 2020, 21, 8290

6 of 22

Table 1. Several relevant ASD candidate genes.

Category

Gene Symbol

Gene Name

Alterations

Associated Syndromes

ARID1B

ASXL3

ATRX

AUTS2

CHD2

CHD7

Chromatin regulators

CHD8

Transcription
factors/regulators

mRNA binding and
traﬃcking regulator

FOXP1

FOXP2

MED13L

POGZ

RAI1

TBR1

TCF4

ZBTB20

FMR1
and its
pathways

ANKRD11

Ankyrin repeat domain 11

Mutations; copy number loss

KBG syndrome; Cornelia de
Lange syndrome

Coﬃn–Siris syndrome

Mutations

Bainbridge-Ropers syndrome

AT-rich interaction
domain 1B

ASXL Transcriptional
Regulator 3

Mutations; copy number loss;
copy number gain;
translocation

ATRX Chromatin Remodeler Mutations; copy number loss
Mutations; copy number loss;
copy number gain; inversion;
translocation

Autism susceptibility
candidate 2

Mutations; copy number loss

Tourette syndrome

CREBBP

CREB-binding protein

Mutations; copy number loss

Chromodomain helicase
DNA binding protein 2
Chromodomain helicase
DNA binding protein 7

Chromodomain helicase
DNA binding protein 8

EHMT1

MBD5

MECP2

Euchromatic histone-lysine
N-methyltransferase 1

Methyl-CpG binding
domain protein 5

Methyl CpG binding
protein 2

SETD5

SET domain containing 5

ADNP

Activity-dependent
neuroprotector homeobox

FOXG1

Forkhead box G1

Forkhead box P1

Forkhead box P2

Mediator complex subunit
13-like
Pogo transposable element
with ZNF domain

Retinoic Acid Induced 1

T-box, brain, 1

Transcription factor 4

Mutations; copy number loss;
translocation
Mutations; copy number loss;
copy number gain;
translocation

Mutations; copy number loss;
copy number gain;
translocation
Mutations; copy number loss;
copy number gain; inversion;
translocation
Mutations; copy number loss;
copy number gain; promoter
methylation
Mutations; copy number loss

Mutations; copy number loss

Mutations; copy number loss;
copy number gain;
translocation
Mutations; copy number loss;
inversion; translocation
Mutations; copy number loss;
translocation
Mutations; copy number loss;
copy number gain
Mutations; copy number loss;
copy number gain
Mutations; copy number loss;
copy number gain
Mutations; copy number loss

Mutations; copy number loss;
translocation

Zinc ﬁnger and BTB domain
containing 20

Mutations; copy number loss;
translocation

Fragile X mental
retardation 1

Mutations; copy number loss

CHARGE syndrome

Rubinstein–Taybi syndrome,
Menke-Hennekam syndrome
1, Tourette syndrome

Kleefstra syndrome

2q23.1 microdeletion
syndrome, Kleefstra syndrome

Rett syndrome, X-linked
intellectual disability, MECP2
duplication syndrome

Helsmoortel-Van der Aa
syndrome

Rett syndrome, FOXG1
syndrome, West syndrome,

FOXP2-related speech and
language disorder

White–Sutton syndrome

Smith–Magenis syndrome,
Potocki–Lupski syndrome

Pitt–Hopkins syndrome

3q13.31 microdeletion
syndrome, Primrose
syndrome,

Fragile X syndrome, Fragile
X-associated tremor/ataxia
syndrome

Protein degradation

UBE3A

Ubiquitin protein ligase E3A Mutations; copy number gain

Angelman syndrome

Cell
growth/proliferation

DYRK1A

NF1

PTEN
and its
pathways

SYNGAP1
TSC1/TSC2

Dual-speciﬁcity
tyrosine-(Y)-phosphorylation
regulated kinase 1A
Neuroﬁbromin 1

Mutations; copy number loss;
inversion; translocation

Mutations; copy number loss

Phosphatase and tensin
homolog

Mutations; copy number loss

Synaptic Ras GTPase
activating protein 1
Tuberous sclerosis 1/2

Mutations; copy number loss;
translocation
Mutations

Cowden syndrome,
Macrocephaly/autism
syndrome, PTEN hamartoma
tumor syndrome

Protein modiﬁcation

CDKL5

Cyclin-dependent
kinase-like 5

Mutations; copy number loss;
copy number gain
translocation

Rett syndrome, Angelman
syndrome

Prepared by the authors with data from gene.sfari.org (release 26 October 2020).

Int. J. Mol. Sci. 2020, 21, 8290

7 of 22

Synaptic Architecture and Functionality

It is not surprising that many candidate ASD genes are involved in synaptic architecture and
function, which allows the transmission of information between neurons and between neurons and
other cells, such as muscle, sensory and other cells. Many ASD candidate genes are involved in
dendritic spine formation. Dendritic spines are small actin-rich protrusions that form the postsynaptic
part of most excitatory synapses. Remodeling of actin cytoskeleton is responsible for the changes in the
shape and size of dendritic spines and, consequently, to the synaptic functions [36]. Actin regulation
mechanisms regulate the formation, maturation and plasticity of dendritic spines and of neuronal
processes, such as learning and memory [36]. Abnormalities in the number and shape of dendritic
spines have been observed in several neurological disorders, including autism, and contribute to brain
dysfunction [37].

Post-synaptic density proteins (PSD), including cell adhesion molecules, scaffold proteins, receptors and
cytoskeleton proteins, are fundamental for synaptic transmission and plasticity. Alterations of these proteins
have been associated with many neurological disorders, including ASD [38].

•

Cell adhesion molecules

Neurexins (NRXN) and neuroligins (NLGN) are transmembrane synaptic proteins that form
the neurexin/neuroligin transsynaptic complex, crucial for synaptic function [39]. NLGNs bind to
SHANK3 through PSD-95 and other synaptic proteins.

Loss of function NRX1 variants in ASD individuals have been identified in multiple studies [23,40,41].
Studies conducted in animal models knockout (KO) for NLGN and NRXN family members showed that
mice develop ASD-like symptoms and have confirmed their role in synaptic function [42–44].

CNTNAP2, also known as CASPR2, encodes for a member of the NRXN family that serves as an
adhesion protein, primarily between neuronal and glial cells. CNVs encompassing CNTNAP2 and
resulting in its decreased expression have been described in subjects with ASD [45–47]. The suppression
of CNTNAP2 in murine models causes autistic behaviors, such as repetitive behaviors and reduced
socialization and communication [48].

•

Scaﬀold proteins

SHANK gene family, including SHANK1, SHANK2 and SHANK3, has been suggested as a strong
candidate for ASD. SHANK proteins are multi-domain post synaptic density scaﬀold proteins that
connect neurotransmitter receptors, ion channels and other membrane proteins to cytoskeleton actin
and signaling proteins. These proteins are important for synapse formation and dendritic spine
maturation [49]. Rare deletions of SHANK2 and de novo variants causing loss of protein function have
been identiﬁed in individuals with ASD [50]. A microdeletion encompassing SHANK3 determines
Phelan–McDermid syndrome, characterized by intellectual disability, ASD, severe speech disorders and
epilepsy [51]. A meta-analysis of SHANK mutations found low frequency of SHANK1 and SHANK2
deleterious mutations in contrast to the high frequency of loss of function SHANK3 mutations in cases
with ASD [52].

Our studies led to the identiﬁcation of another promising ASD candidate gene, CAPG. This gene
encodes for a member of the gelsolin family of actin-regulatory proteins, important for the remodeling
of actin architecture. A microdeletion encompassing the entire CAPG gene has been recently described
in three completely independent families in the heterozygous [53,54] and homozygous state [55].
Importantly, a reduced CAPG expression, both at transcriptional and protein levels, has been detected
in the Sardinian family members carrying the deletion, and reduced CAPG mRNA levels have been
also observed in an independent cohort of 13 non-Sardinian ASD cases compared to age-matched
healthy controls [54].

Several studies have demonstrated the importance of CAPG for the formation of functional
synapses. In fact, experiments conducted on cultured hippocampal neurons have demonstrated that
capping proteins are present at the branched actin ﬁlament network of dendritic spine heads and they

Int. J. Mol. Sci. 2020, 21, 8290

8 of 22

are fundamental for dendritic spine development [56,57]. In fact, CAPG knock-down led to a decline
in spine density and to an increased number of ﬁlopodia-like protrusions [56].

•

Voltage-gated ion channels
The role of genetic defects of diﬀerent ion channels in the pathogenesis of ASD is well established.
In fact, GWAS, WES and WGS have identiﬁed several polymorphisms and rare variants in calcium,
sodium and potassium channels in ASD subjects (reviewed in [58]).

Point mutations in CACNA1C gene, which encode for l-type voltage-gated Ca2+

channel Cav1.2,
lead to Timothy syndrome (TS), a disorder aﬀecting multiple organs and characterized by an autistic
phenotype [59,60]. l-type channels are mainly expressed in neuronal dendrites and cell bodies and
are crucial for the activation of Ca2+
-signaling pathways and for neuronal excitability [61]. Defects in
CACNA1C prevent the inactivation of the channel and lead to its prolonged opening and consequent
increase in Ca2+

ﬂux [60,62].

Mutations in other genes encoding for l-type and T-type Ca2+

channels, such as CACNA1D,
CACNA1E, CACNA1F and CACNA1H, have been described in ASD [63–67]. Moreover, mutations of
CACNB2, the gene encoding for the regulatory β2 subunit of CACNA1C, have been found in ASD
families [68].

Genetic defects in genes encoding for sodium channels, such as SCN1A, SCN2A, SCN3A,
SCN7A and SNC8A, have been also associated with ASD [69–73]. Voltage-dependent sodium channels
are mainly expressed in neurons and glial cells and are fundamental for the initiation and propagation
of action potentials. Mutations of SCN1A cause Dravet syndrome, characterized by seizures and
frequently manifesting, also, autistic symptoms. A study has shown that Scn1a+/− heterozygous KO
mice display stereotypical and anxious behaviors other than seizures [74].

Several studies have also shown that mutations in genes encoding for K

channels, including
KCNMA1, KCND2, KCNJ10, KCNQ3 and KCNQ5, may play a central role in ASD etiology [75–78]. It has
+
been shown that KO of Fmr1 in mice results in K
channel dysregulation and consequent dysregulation
of synaptic transmission [79].

+

Some evidence has shown that voltage-dependent anion channel (VDAC) genes, a class of
postsynaptic density genes highly expressed in several brain regions, could be implicated in ASD.
In fact, autoantibodies against VDAC proteins have been detected in autistic individuals, suggesting
a possible causal role in ASD pathogenesis [80]. Moreover, the beneﬁcial eﬀects observed in ASD
patients treated with coenzyme Q, or other agents inﬂuencing the transport of electrons, have been
attributed to the control of such molecules on the porin channels [81]. Recently, a study conducted by
our research group identiﬁed a 2-bp frameshift deletion of VDAC3 in an ASD family [54].

3.2. Epigenetic Factors

There is increasing evidence supporting the possible role of epigenetic aberrations, including DNA

methylation alterations and microRNAs, in ASD etiology.

•

DNA methylation

Several studies have conducted global methylation analyses in peripheral tissues as well as

post-mortem brain tissues from ASD subjects and controls.

Studies conducted on lymphoblastoid cell lines and whole-blood DNA from monozygotic twins
discordant for ASD diagnosis and controls identiﬁed several diﬀerentially methylated regions (DMRs)
between discordant MZ twins and between ASD patients and control samples [82,83].

Zhu et al. identiﬁed 400 DMRs, enriched at promoters of genes involved in neuronal development,
between placentas from children later diagnosed with ASD and those from typically developing
controls. Methylation levels of two DMRs, mapping on CYP2E1 and IRS2, were respectively associated
with genotype within the DMR and prenatal vitamin use [84].

Conversely, a large epigenome-wide association study, performed on blood-DNA from 796 ASD cases
and 858 controls, did not detect any differentially methylated CpG site after correction for multiple testing [85].

Int. J. Mol. Sci. 2020, 21, 8290

9 of 22

Similarly, Siu et al. did not detect any DNA methylation patterns clearly distinguishing
heterogenous ASD cases from controls. However, they identiﬁed unique DNA methylation signatures
for ASD individuals with 16p11.2 deletions or pathogenic variants of CHD8 [86].

On the other hand, Kimura et al. identiﬁed a potential biomarker for adult ASD. The identiﬁed
CpG site was hypermethylated in whole-blood DNA from ASD patients compared to controls and
mapped on the PPP2R2C gene, which resulted down-regulated in ASD subjects [87].

Alterations in Alu methylation patterns have been observed in ASD cases sub-grouped based on

Autism Diagnostic Interview-Revised scores compared with matched controls [88].

More recently, a genome-wide methylation study was performed on post-mortem tissue samples
from diﬀerent brain regions dissected from ASD subjects and controls. Wide-spread methylation
diﬀerences, with more pronounced eﬀects in cortical regions compared to cerebellum, were detected
between idiopathic ASD cases and controls and in individuals carrying 15q11–13 duplication [89].

Another global methylation study has recently reported that diﬀerentially methylated CpG
sites identiﬁed in ASD cases compared to controls are enriched in pathways converging on
mitochondrial metabolism and protein ubiquitination, suggesting a possible role of DNA methylation
and mitochondrial dysfunction in ASD [90].

Other studies measuring methylation levels of candidate genes using targeted approaches detected
hypermethylation of genes including APOE [91] and HTR2A [92] and hypomethylation of genes such
as HTR4 [93] in ASD subjects compared to controls.

The functional impact of locus-speciﬁc Mecp2 methylation on ASD onset has been recently
demonstrated in vitro and in vivo. MECP2 is known to be hypermethylated and down-regulated in
ASD subjects. The authors of this study employed the CRISPR-dCas9 methylation editing system
to induce methylation of the Mecp2 transcription start site in Neuro-2a cells and in mouse models,
resulting in Mecp2 down-regulation and the acquisition of behavioral changes attributable to an ASD
phenotype in mice [94].

• miRNA

Several studies have found that miRNA expression proﬁles are dysregulated in diﬀerent matrices,

including saliva, blood and brain tissues, from ASD individuals [95].

A study conducted on saliva samples from individuals with and without ASD has identiﬁed a
panel of four microRNAs diﬀerentially expressed between ASD patients and controls [96]. A panel of
ﬁve salivary miRNAs has shown an accuracy of about 90% in the detection of developmental disorders,
including ASD [97].

Down-regulation of miR-6126 has been detected in peripheral blood samples from adult individuals
with ASD. The predicted targets of this miRNA belong to neuronal and oxytocin pathways [98]. Similarly,
Ozkul et al. identiﬁed a consistent decrease and a slight reduction in six microRNAs (miR-19a-3p,
miR-361-5p, miR-3613-3p, miR-150-5p, miR-126-3p and miR-499a-5p) in serum samples from ASD
children and their unaﬀected family members compared to healthy controls. This result was replicated
in the blood, hypothalamus and sperm of two ASD mouse models [99]. Another study conducted on
serum samples identiﬁed a panel of three miRNAs (miR-130-3p, miR-181b-5p and miR-320a) showing
an area under the curve >0.85 in distinguishing ASD subjects from controls [100].

Studies conducted on knockout (KO) mouse models for some ASD candidate genes, including Fmr1,
Mecp2 and Ube3A, have observed dysregulation of diﬀerent miRNAs and evaluated their regulatory
role in neuronal context [101–103].

4. Environmental Factors

As mentioned above, several studies investigated the possible role of environmental factors in
the etiology of ASD. According to recent studies, up to 40–50% of variance in ASD liability could
be determined by environmental factors, such as drugs, toxic exposures, parental age, nutrition,
fetal environment and many others [104–106]. However, while for some potential risk factors, there is

Int. J. Mol. Sci. 2020, 21, 8290

10 of 22

strong evidence, supported by association studies but also by in vitro and in vivo studies, only weak
associations have been described for many others.

Below is reported an overview of the most-studied environmental factors that have been found to

potentially contribute to cause ASD (recently reviewed in [107]) (Figure 2).

Figure 2. Environmental factors associated with ASD. The illustration indicates the putative impact
of environmental factors on embryonic and fetal development with a particular focus on neuronal
development and synaptic function. Figure created using BioRender.com images.

Parental age is one of the most established environmental ASD risk factors. In fact, much evidence
has correlated advanced paternal age (APA) with the development of bipolar disorder, schizophrenia,
ADHD and ASD [108]. A meta-analysis of 27 studies on the association between advanced parental
age and ASD showed that a 10-year increase in maternal and paternal age is associated with a 20%
higher risk of ASD in children [109]. A study has shown that age-related methylation changes observed
in sperm could be related to an increased ASD risk in the oﬀspring [110]. APA has been associated
with reduced cortical thickness of the right posterior ventral cingulate cortex in ASD oﬀspring [111].
Experiments conducted in mouse models have conﬁrmed that APA is associated with the development
of autism-like symptoms in the oﬀspring [112,113] and with altered cortical morphology in male APA
mice [112]. Moreover, behaviors related to ASD have also been observed in the second generation of
mice with older grandfathers, suggesting that genetic and epigenetic alterations associated with APA
are heritable [113].

Perinatal risk factors are also among the most-studied ASD risk factors and among the most diﬃcult
to determine and predict in advance. Two comprehensive meta-analyses examined 60 obstetric factors
and found statistically signiﬁcant associations between ASD risk and umbilical cord complications,
injury or trauma at birth, multiple births, maternal hemorrhage, low birth weight, neonatal anemia,
genital malformation, ABO or Rh blood group incompatibility and hyperbilirubinemia [114,115].
Another study has described an association between increased risk of autism and diﬀerent factors,
including caesarean section delivery, induced labor, management age less than 36 weeks and fetal
distress [116].

Fetal exposure to sex steroids represents a potential risk factor for ASD. In fact, the fetal testosterone
theory has been proposed to explain the higher ASD prevalence in males [8]. However, this hypothesis

 2  Int. J. Mol. Sci. 2020, 21, 8290

11 of 22

is controversial. In fact, while higher testosterone levels have been reported in ASD women, ASD males
display testosterone levels similar to controls [117]. Baron-Cohen and colleagues have supposed that
testosterone has eﬀects on brain development during the prenatal masculinization window. The authors
detected higher levels of sex steroids and cortisol in the amniotic ﬂuid samples from male autistic
patients compared with matched typically developing controls [118]. Recently, the same authors
reported an association between fetal estrogen levels, important in synaptogenesis and corticogenesis,
and autism risk [119]. Similarly, a study conducted on post-mortem brains detected reduced levels of
estrogen receptor beta, aromatase and estrogen coactivators in the frontal gyrus of subjects with ASD
compared to controls [120]. Moreover, several SNPs in genes encoding for proteins involved in sex
steroid synthesis/transport have been associated with autistic traits [121].

The health condition of the mother has a major impact on the risk of ASD. It appears that
maternal nutrition in pregnancy is of fundamental importance as it determines the nutrients available
to support fetal growth [122]. Therefore, diets lacking speciﬁc nutrients can have adverse eﬀects on
fetal development. It has been shown that even short intervals between pregnancies can be harmful,
since the body needs time, up to one year after childbirth, to recover acceptable levels of several
essential substances [123].

Deﬁciencies of micronutrients, including vitamins and trace elements, have been associated with
an increased risk of ASD. For instance, a Swedish study found that maternal vitamin D deﬁciency is
associated with the risk of ASD with ID in the oﬀspring [124]. Unbalanced levels of vitamins have also
been detected in ASD children. Moreover, beneﬁcial eﬀects of vitamin supplementation have been
observed in ASD patients [125]. Similarly, altered hair and/or blood concentrations of several trace
elements, including chromium, magnesium and zinc, have been found in ASD patients compared with
controls [126]. Association between maternal deﬁciency of microelements and ASD risk has also been
reported. For example, iron deﬁciency, common in pregnant women, was associated with a ﬁve-fold
increased risk of ASD, especially in the presence of other risk factors [127].

The involvement of altered trace elements concentrations on ASD phenotype is also supported by
in vitro and animal studies. The effects on unbalanced metal levels on synapse formation and functionality
have been evaluated on hippocampal cultured cells from rat brains, finding that the metal profile of
autistic children led to down-regulation of crucial synaptic components, including Shank proteins and
NMDA receptor subunits, and reduction of synaptic density. Interestingly, it was observed that zinc
supplementation was able to revert the observed alterations [128]. The importance of zinc in ASD is
supported by multiple pieces of evidence, including a strong association between low levels of this metal
and ASD risk as well as a causative role of zinc deficiency in neuronal defects and development of
ASD-related symptoms and “therapeutic” effects of zinc supplementation. In fact, low hair and serum
levels of zinc and/or altered Zn/Cu ratio have been detected in children with ASD [129–131]. The effects
of zinc deficiency on ASD have been observed in animal models. It has been shown that prenatal zinc
deficiency alters social behavior in mice [132]. In vitro and in vivo studies have shown that zinc deficiency
at the synaptic level leads to a decrease in ProSAP/Shank family members, a reduction in synaptic density
and the development of ASD-related symptoms in mice [133]. In fact, other studies have shown that Zn2+
ions, highly abundant at PSD level, influence the recruitment of ProSAP1/Shank2 or ProSAP2/Shank3
for a correct formation and maturation of synapses [134]. The mechanism regulating the abundance of
Zn at PSD levels has been hypothesized: Zn is released from pre-synaptic terminals and can translocate
into post-synaptic neurons through Zn-permeable channels, including NMDA, and voltage-gated Ca2+
and AMPA channels [135]. Notably, a recent study has shown maternal zinc supplementation can
prevent ASD-associated deficits in Shank3 KO mouse models [136]. Recently, Shih et al. have elegantly
demonstrated the impact of the crosstalk between genetic and environmental factors in the development of
ASD. The authors have shown that KO of Cttnbp2, an actin cytoskeleton regulator, leads to a reduction in Zn
concentration and expression levels of different synaptic proteins, consequently affecting dendritic spine
formation and leading to the development of autism-like behaviors rescued by zinc supplementation [137].

Int. J. Mol. Sci. 2020, 21, 8290

12 of 22

The beneficial effects of zinc supplementation have been also observed in pregnant women, where an
increase in zinc intake has been shown to reduce the risk of neural tube defects in the offspring [138].

The association between maternal obesity and risk of ASD in oﬀspring is controversial. A Swedish
study described a relationship between maternal BMI and ASD at population level; however,
sibling analysis did not reveal any association between elevated maternal BMI and ASD risk [139].
On the other hand, a meta-analysis has shown a 28% and 36% increased risk of ASD in oﬀspring
born from overweight and obese mothers, respectively [140], even though there were relatively small
numbers of ASD cases within the category of maternal underweight. However, it has also been shown
that children born from underweight mothers are also at high risk of ASD [141]. Another study
has shown that the combination of maternal obesity and maternal diabetes was associated with an
increased risk of ASD and ID [142].

Maternal consumption of substances such as smoke, alcohol and medicines during pregnancy
might be a potential risk factor as well. However, the association between smoking and alcohol-use
with ASD is rather weak; in fact, two meta-analyses showed no evidence that smoking is a risk factor
for ASD [143,144]. Moreover, cohort studies or case-control studies have examined the risk of ASD
due to maternal alcohol consumption, indicating that mild to moderate consumption does not pose
any risk [145–148].

The safety of medicines during pregnancy is very diﬃcult to establish. In the ASD literature,
antidepressant and anticonvulsant drugs have emerged as drugs of potential interest. An example is
valproic acid, a drug that has been used to treat epilepsy or as a mood stabilizer in bipolar disorder.
Its use during pregnancy leads to congenital malformations, developmental delay and cognitive
malfunction [149,150]. Maternal use of selective serotonin uptake inhibitors has been associated with a
50% increase in ASD risk, although maternal psychiatric condition is a confounding factor [151].

An association between maternal diseases and ASD risk has been shown. A meta-analysis limited
to case-control studies identiﬁed a 62% increased ASD risk among diabetic mothers compared to
non-diabetic mothers [152], while a second study found a 74% increase in ASD risk for pregestational
diabetes and 43% for gestational diabetes [153].

It has also been shown that maternal viral and bacterial infections are associated with ASD
risk [154–156]. Two meta-analyses found that maternal autoimmune diseases are associated with an
increased risk of ASD in oﬀspring [157,158]. However, it is not the presence of viruses and bacteria
per se to be associated with ASD development, but the immune response they cause, a conclusion
supported by research identifying elevated inﬂammatory markers and antibodies in pregnant women
with autistic children [159,160]. This hypothesis is supported by animal studies where maternal
immune activation, induced by diﬀerent immunogens, has been shown to induce post-natal brain
dysfunction observable in a phenotype characteristic of ASD and other neurological disorders [161].
It is interesting to note that, although from diﬀerent perspectives, much evidence points out the
importance of maternal immune system condition during fetal development. Several studies have
also reported an association between family history of autoimmune diseases and ASD [158,162,163].
It could be speculated that this link could be supported by maternal levels of zinc. In fact, our recent
meta-analysis has shown that Zn concentrations in both serum and plasma levels of patients with
It is known that Zn plays an
autoimmunity are signiﬁcantly lower compared to controls [164].
important role in the regulation of the immune system and, according to the studies provided above,
it plays also an important role in neuronal tube formation.

Finally, exposure to toxic xenobiotics could represent another potential environmental risk factor.
Substances such as brominated ﬂame retardants could cause mitochondrial toxicity through a variety
of mechanisms, leading to an altered energy balance in the brain [165], an important association
with autism since mitochondrial dysfunction has been documented in patients with ASD [166].
Heavy metals can have a negative impact on many body functions by inducing neurological and
behavioral damage. A meta-analysis of three case-control studies found a 60% increase in the risk of
ASD due to exposure to high levels of inorganic mercury [167]. A recent case-control study found

Int. J. Mol. Sci. 2020, 21, 8290

13 of 22

that exposure to organophosphates during pregnancy is associated with a 60% increase in the risk
of ASD [168]. This category includes non-persistent organic pollutants, including phthalates and
bisphenol A, and persistent organic pollutants, including DDT, PCB and PBDE. A research group
found that three out of ﬁve studies on phthalate exposure showed a signiﬁcant association between
phthalate exposure and ASD risk [169]. Exposure to PCBs and PBDEs appears to alter calcium-related
signal pathways, leading to alterations in dendritic growth and consequent abnormalities in neuronal
connectivity, a key feature of ASD [170,171].

Table 2 summarizes several environmental factors associated with ASD described above.

Table 2. Several environmental factors associated with ASD.

Factor

Evidence

Parental age

Perinatal factors
Sex steroids

Maternal nutrition

Fetal exposure to drugs,
smoke, alcohol
Maternal diseases

Maternal infections

Fetal exposure to toxic
xenobiotics

Association studies; meta-analyses;
animal studies
Meta-analyses
Association studies
Association studies; meta-analyses;
in vitro studies, animal studies

Association studies; meta-analyses

Meta-analyses
Association studies; meta-analyses;
animal studies
Association studies; meta-analyses;
in vitro studies; animal studies

References

[108–113]

[114–116]
[117–121]

[123–134,136–142]

[143–151]

[152,153]

[154–161]

[166–171]

5. Conclusions

Given the complexity of the etiology of autism and the increasing prevalence of new conﬁrmed
cases of ASD worldwide, there is an urgent need to ﬁnd eﬀective diagnostic methods and study as
many risk factors as possible—not only genetic but also epigenetic and environmental ones—without
neglecting also genetic/environmental interactions, where risk factors inﬂuence each other.

Author Contributions: Conceptualization, E.L. and P.Z.; bibliographic search, E.M. and E.L.; writing—original
draft preparation, E.M., E.L., R.F. and P.Z.; writing—review and editing, all authors; visualization, A.F.V.-B.;
supervision, E.L. and P.Z.; funding acquisition, P.Z., M.C., G.D. All authors have read and agreed to the published
version of the manuscript.
Funding: This research was funded by the Fondazione di Sardegna, grant number BSPROG21/2017.

Conﬂicts of Interest: The authors declare no conﬂict of interest.

References

1.

2.

3.

4.

5.

Kana, R.K.; Uddin, L.Q.; Ekenet, T.; Echugani, D.; Müller, R.-A. Brain connectivity in autism. Front. Hum. Neurosci.
2014, 8, 349. [CrossRef] [PubMed]
Narzisi, A.; Muratori, F.; Calderoni, S.; Fabbro, F.; Urgesi, C. Neuropsychological Proﬁle in High Functioning
Autism Spectrum Disorders. J. Autism Dev. Disord. 2012, 43, 1895–1909. [CrossRef] [PubMed]
Goin-Kochel, R.P.; Myers, B.J. Congenital Versus Regressive Onset of Autism Spectrum Disorders.
Focus Autism Other Dev. Disabil. 2005, 20, 169–179. [CrossRef]
Tamouza, R.; Fernell, E.; Eriksson, M.A.; Anderlid, B.; Manier, C.; Mariaselvam, C.M.; Boukouaci, W.;
Leboyer, M.; Gillberg, C. HLA Polymorphism in Regressive and Non-Regressive Autism: A Preliminary
Study. Autism Res. 2019, 13, 182–186. [CrossRef]
Cohen, J.; Eyu, S.; Efu, Y.; Eli, X. Synaptic proteins and receptors defects in autism spectrum disorders.
Front. Cell. Neurosci. 2014, 8, 276. [CrossRef]

Int. J. Mol. Sci. 2020, 21, 8290

14 of 22

6.

7.

James, S.J.; Melnyk, S.; Jernigan, S.; Cleves, M.A.; Halsted, C.H.; Wong, D.H.; Cutler, P.; Bock, K.; Boris, M.;
Bradstreet, J.J.; et al. Metabolic endophenotype and related genotypes are associated with oxidative stress
in children with autism. Am. J. Med. Genet. Part B Neuropsychiatr. Genet. 2006, 141, 947–956. [CrossRef]
[PubMed]
Iossifov, I.; O’Roak, B.J.; Sanders, S.J.; Ronemus, M.; Krumm, N.; Levy, D.; Stessman, H.A.; Witherspoon, K.T.;
Vives, L.; Patterson, K.E.; et al. The contribution of de novo coding mutations to autism spectrum disorder.
Nature 2014, 515, 216–221. [CrossRef] [PubMed]

8. Werling, D.M. The role of sex-diﬀerential biology in risk for autism spectrum disorder. Biol. Sex Diﬀer. 2016,

7, 1–18. [CrossRef]

9. Newschaﬀer, C.J.; Croen, L.A.; Daniels, J.; Giarelli, E.; Grether, J.K.; Levy, S.E.; Mandell, D.S.; Miller, L.A.;
Pinto-Martin, J.; Reaven, J.; et al. The Epidemiology of Autism Spectrum Disorders. Annu. Rev. Public Health
2007, 28, 235–258. [CrossRef] [PubMed]

10. Elsabbagh, M.; Mercure, E.; Hudry, K.; Chandler, S.; Pasco, G.; Charman, T.; Pickles, A.; Baron-Cohen, S.;
Bolton, P.; Johnson, M.H. Infant Neural Sensitivity to Dynamic Eye Gaze Is Associated with Later Emerging
Autism. Curr. Biol. 2012, 22, 338–342. [CrossRef] [PubMed]

11. Benvenuto, A.; Moavero, R.; Alessandrelli, R.; Manzi, B.; Curatolo, P. Syndromic autism: Causes and

pathogenetic pathways. World J. Pediatr. 2009, 5, 169–176. [CrossRef] [PubMed]

12. Hu, V.W.; Devlin, C.A.; Debski, J.J. ASD Phenotype—Genotype Associations in Concordant and Discordant
Monozygotic and Dizygotic Twins Stratiﬁed by Severity of Autistic Traits. Int. J. Mol. Sci. 2019, 20, 3804.
[CrossRef]

13. Colvert, E.; Tick, B.; McEwen, F.; Stewart, C.; Curran, S.R.; Woodhouse, E.; Gillan, N.; Hallett, V.; Lietz, S.;
Garnett, T.; et al. Heritability of Autism Spectrum Disorder in a UK Population-Based Twin Sample.
JAMA Psychiatry 2015, 72, 415–423. [CrossRef] [PubMed]
Sandin, S.; Lichtenstein, P.; Kuja-Halkola, R.; Larsson, H.; Hultman, C.M.; Reichenberg, A. The Familial Risk
of Autism. JAMA 2014, 311, 1770–1777. [CrossRef]

14.

15. Eichler, E.E.; Flint, J.; Gibson, G.; Kong, A.; Leal, S.M.; Moore, J.H.; Nadeau, J.H. Missing heritability and
strategies for ﬁnding the underlying causes of complex disease. Nat. Rev. Genet. 2010, 11, 446–450. [CrossRef]
16. Génin, E. Missing heritability of complex diseases: Case solved? Hum. Genet. 2020, 139, 103–113. [CrossRef]
17. Wi´sniowiecka-Kowalnik, B.; Nowakowska, B.A. Genetics and epigenetics of autism spectrum disorder—Current

evidence in the field. J. Appl. Genet. 2019, 60, 37–47. [CrossRef] [PubMed]
Fatemi, S.H. The Molecular Basis of Autism; Springer: New York, NY, USA, 2015; ISBN 9781493921904.
18.
19. Wang, K.; Zhang, H.; Ma, D.; Bucan, M.; Glessner, J.T.; Abrahams, B.S.; Salyakina, D.; Imielinski, M.;
Bradﬁeld, J.P.; Sleiman, P.M.A.; et al. Common genetic variants on 5p14.1 associate with autism spectrum
disorders. Nature 2009, 459, 528–533. [CrossRef]

20. Anney, R.; Klei, L.; Pinto, D.; Almeida, J.; Bacchelli, E.; Baird, G.; Bolshakova, N.; Bölte, S.; Bolton, P.F.;
Bourgeron, T.; et al. Individual common variants exert weak eﬀects on the risk for autism spectrum disorders.
Hum. Mol. Genet. 2012, 21, 4781–4792. [CrossRef]
Sener, E.F.; Canatan, H.; Ozkul, Y. Recent Advances in Autism Spectrum Disorders: Applications of Whole
Exome Sequencing Technology. Psychiatry Investig. 2016, 13, 255–264. [CrossRef]

21.

22. Pinto, D.; Pagnamenta, A.T.; Klei, L.; Anney, R.; Merico, D.; Regan, R.; Conroy, J.; Magalhaes, T.R.; Correia, C.;
Abrahams, B.S.; et al. Functional impact of global rare copy number variation in autism spectrum disorders.
Nature 2010, 466, 368–372. [CrossRef]

23. Marshall, C.R.; Noor, A.; Vincent, J.B.; Lionel, A.C.; Feuk, L.; Skaug, J.; Shago, M.; Moessner, R.; Pinto, D.;
Ren, Y.; et al. Structural Variation of Chromosomes in Autism Spectrum Disorder. Am. J. Hum. Genet. 2008,
82, 477–488. [CrossRef]

24. Lupski, J.R.; Stankiewicz, P. Genomic Disorders: Molecular Mechanisms for Rearrangements and Conveyed

Phenotypes. PLoS Genet. 2005, 1, e49. [CrossRef] [PubMed]

25. Devlin, B.; Scherer, S.W. Genetic architecture in autism spectrum disorder. Curr. Opin. Genet. Dev. 2012, 22,

229–237. [CrossRef]

26. Napoli, E.; Russo, S.; Casula, L.; Alesi, V.; Amendola, F.A.; Angioni, A.; Novelli, A.; Valeri, G.; Menghini, D.;
Vicari, S. Array-CGH Analysis in a Cohort of Phenotypically Well-Characterized Individuals with “Essential”
Autism Spectrum Disorders. J. Autism Dev. Disord. 2018, 48, 442–449. [CrossRef]

Int. J. Mol. Sci. 2020, 21, 8290

15 of 22

27. Beaudet, A.L. The utility of chromosomal microarray analysis in developmental and behavioral pediatrics.

28.

Child Dev. 2013, 84, 121–132. [CrossRef] [PubMed]
Jacquemont, M.-L.; Sanlaville, D.; Redon, R.; Raoul, O.; Cormier-Daire, V.; Lyonnet, S.; Amiel, J.; Le Merrer, M.;
Heron, D.; De Blois, M.-C.; et al. Array-based comparative genomic hybridisation identiﬁes high frequency of
cryptic chromosomal rearrangements in patients with syndromic autism spectrum disorders. J. Med. Genet.
2006, 43, 843–849. [CrossRef]

29. Holt, R.; Sykes, N.H.; Conceição, I.; Cazier, J.-B.; Anney, R.J.; Oliveira, G.G.; Gallagher, L.; Vicente, A.M.;
Monaco, A.P.; Pagnamenta, A.T. CNVs leading to fusion transcripts in individuals with autism spectrum
disorder. Eur. J. Hum. Genet. 2012, 20, 1141–1147. [CrossRef]

30. Pagnamenta, A.T.; Bacchelli, E.; De Jonge, M.V.; Mirza, G.; Scerri, T.S.; Minopoli, F.; Chiocchetti, A.;
Ludwig, K.U.; Hoﬀmann, P.; Paracchini, S.; et al. Characterization of a Family with Rare Deletions in
CNTNAP5 and DOCK4 Suggests Novel Risk Loci for Autism and Dyslexia. Biol. Psychiatry 2010, 68, 320–328.
[CrossRef]

31. Ceroni, F.; Sagar, A.; Simpson, N.H.; Gawthrope, A.J.; Newbury, D.F.; Pinto, D.; Francis, S.M.; Tessman, D.C.;
Cook, E.H.; Monaco, A.P.; et al. A Deletion Involving CD 38 and BST 1 Results in a Fusion Transcript in a
Patient With Autism and Asthma. Autism Res. 2014, 7, 254–263. [CrossRef]

32. Loi, E.; Moi, L.; Blois, S.; Bacchelli, E.; Benedetti, A.F.V.; Cameli, C.; Fadda, R.; Maestrini, E.; Carta, M.;
Doneddu, G.; et al. ELMOD3—SH2D6 gene fusion as a possible co-star actor in autism spectrum disorder
scenario. J. Cell. Mol. Med. 2019, 24, 2064–2069. [CrossRef] [PubMed]

33. Yin, J.; Schaaf, C.P. Autism genetics—An overview. Prenat. Diagn. 2016, 37, 14–30. [CrossRef]
34.

Satterstrom, F.K.; Kosmicki, J.A.; Wang, J.; Breen, M.S.; De Rubeis, S.; An, J.-Y.; Peng, M.; Collins, R.; Grove, J.;
Klei, L.; et al. Large-Scale Exome Sequencing Study Implicates Both Developmental and Functional Changes
in the Neurobiology of Autism. Cell 2020, 180, 568–584.e23. [CrossRef]

35. Herrero, M.J.; Velmeshev, D.; Hernandez-Pineda, D.; Sethi, S.; Sorrells, S.; Banerjee, P.; Sullivan, C.; Gupta, A.R.;
Kriegstein, A.R.; Corbin, J.G. Identiﬁcation of amygdala-expressed genes associated with autism spectrum
disorder. Mol. Autism 2020, 11, 1–14. [CrossRef] [PubMed]

36. Hotulainen, P.; Hoogenraad, C. Actin in dendritic spines: Connecting dynamics to function. J. Cell Biol. 2010,

189, 619–629. [CrossRef]

37. Van Spronsen, M.; Hoogenraad, C.C. Synapse Pathology in Psychiatric and Neurologic Disease. Curr. Neurol.

Neurosci. Rep. 2010, 10, 207–214. [CrossRef]

38. Kaizuka, T.; Takumi, T. Postsynaptic density proteins and their involvement in neurodevelopmental disorders.

39.

J. Biochem. 2018, 163, 447–455. [CrossRef]
Südhof, T.C. Neuroligins and neurexins link synaptic function to cognitive disease. Nature 2008, 455, 903–911.
[CrossRef] [PubMed]

40. Kim, H.-G.; Kishikawa, S.; Higgins, A.W.; Seong, I.-S.; Donovan, D.J.; Shen, Y.; Lally, E.; Weiss, L.A.; Najm, J.;
Kutsche, K.; et al. Disruption of Neurexin 1 Associated with Autism Spectrum Disorder. Am. J. Hum. Genet.
2008, 82, 199–207. [CrossRef]
Schaaf, C.P.; Boone, P.M.; Sampath, S.; Williams, C.; Bader, P.I.; Mueller, J.M.; Shchelochkov, O.A.; Brown, C.W.;
Crawford, H.P.; Phalen, J.A.; et al. Phenotypic spectrum and genotype–phenotype correlations of NRXN1
exon deletions. Eur. J. Hum. Genet. 2012, 20, 1240–1247. [CrossRef]

41.

42. Radyushkin, K.; Hammerschmidt, K.; Boretius, S.; Varoqueaux, F.; El-Kordi, A.; Ronnenberg, A.; Winter, D.;
Frahm, J.; Fischer, J.; Brose, N.; et al. Neuroligin-3-deﬁcient mice: Model of a monogenic heritable form of
autism with an olfactory deﬁcit. Genes Brain Behav. 2009, 8, 416–425. [CrossRef]

43. Liang, J.; Xu, W.; Hsu, Y.-T.; Yee, A.X.; Chen, L.; Südhof, T.C. Conditional neuroligin-2 knockout in adult
medial prefrontal cortex links chronic changes in synaptic inhibition to cognitive impairments. Mol. Psychiatry
2015, 20, 850–859. [CrossRef] [PubMed]

44. Pak, C.; Danko, T.; Zhang, Y.; Aoto, J.; Anderson, G.R.; Maxeiner, S.; Yi, F.; Wernig, M.; Südhof, T.C.
Human Neuropsychiatric Disease Modeling using Conditional Deletion Reveals Synaptic Transmission
Defects Caused by Heterozygous Mutations in NRXN1. Cell Stem Cell 2015, 17, 316–328. [CrossRef]
45. Arking, D.E.; Cutler, D.J.; Brune, C.W.; Teslovich, T.M.; West, K.; Ikeda, M.; Rea, A.; Guy, M.; Lin, S.;
Cook, E.H.; et al. A Common Genetic Variant in the Neurexin Superfamily Member CNTNAP2 Increases
Familial Risk of Autism. Am. J. Hum. Genet. 2008, 82, 160–164. [CrossRef] [PubMed]

Int. J. Mol. Sci. 2020, 21, 8290

16 of 22

46. Alarcón, M.; Abrahams, B.S.; Stone, J.L.; Duvall, J.A.; Perederiy, J.V.; Bomar, J.M.; Sebat, J.; Wigler, M.;
Martin, C.L.; Ledbetter, D.H.; et al. Linkage, Association, and Gene-Expression Analyses Identify CNTNAP2
as an Autism-Susceptibility Gene. Am. J. Hum. Genet. 2008, 82, 150–159. [CrossRef] [PubMed]

47. Nord, A.S.; Network, S.P.; Roeb, W.; Dickel, D.E.; Walsh, T.; Kusenda, M.; O’Connor, K.L.; Malhotra, D.;
McCarthy, S.E.; Stray, S.M.; et al. Reduced transcript expression of genes aﬀected by inherited and de novo
CNVs in autism. Eur. J. Hum. Genet. 2011, 19, 727–731. [CrossRef]

48. Peñagarikano, O.; Abrahams, B.S.; Herman, E.I.; Winden, K.D.; Gdalyahu, A.; Dong, H.; Sonnenblick, L.I.;
Gruver, R.; Almajano, J.; Bragin, A.; et al. Absence of CNTNAP2 Leads to Epilepsy, Neuronal Migration
Abnormalities, and Core Autism-Related Deﬁcits. Cell 2011, 147, 235–246. [CrossRef]

49. Kreienkamp, H.-J. Scaﬀolding Proteins at the Postsynaptic Density: Shank as the Architectural Framework.
In Protein-Protein Interactions as New Drug Targets; Springer: Berlin, Germany, 2008; Volume 186, pp. 365–380.
50. Berkel, S.; Marshall, C.; Weiss, B.; Howe, J.; Roeth, R.; Moog, U.; Endris, V.; Roberts, W.; Szatmari, P.;
Pinto, D.; et al. Mutations in the SHANK2 synaptic scaﬀolding gene in autism spectrum disorder and mental
retardation. Nat. Genet. 2010, 42, 489–491. [CrossRef]
Jiang, Y.-H.; Ehlers, M.D. Modeling Autism by SHANK Gene Mutations in Mice. Neuron 2013, 78, 8–27.
[CrossRef]

51.

52. Leblond, C.S.; Nava, C.; Polge, A.; Gauthier, J.; Huguet, G.; Lumbroso, S.; Giuliano, F.; Stordeur, C.;
Depienne, C.; Mouzat, K.; et al. Meta-analysis of SHANK Mutations in Autism Spectrum Disorders:
A Gradient of Severity in Cognitive Impairments. PLoS Genet. 2014, 10, e1004580. [CrossRef]

53. Pinto, D.; Delaby, E.; Merico, D.; Barbosa, M.; Merikangas, A.; Klei, L.; Thiruvahindrapuram, B.; Xu, X.;
Ziman, R.; Wang, Z.; et al. Convergence of Genes and Cellular Pathways Dysregulated in Autism Spectrum
Disorders. Am. J. Hum. Genet. 2014, 94, 677–694. [CrossRef]

54. Bacchelli, E.; Loi, E.; Cameli, C.; Moi, L.; Vega-Benedetti, A.F.; Blois, S.; Fadda, A.; Bonora, E.; Mattu, S.;
Fadda, R.; et al. Analysis of a Sardinian Multiplex Family with Autism Spectrum Disorder Points to
Post-Synaptic Density Gene Variants and Identiﬁes CAPG as a Functionally Relevant Candidate Gene.
J. Clin. Med. 2019, 8, 212. [CrossRef]

55. Lahbib, S.; Leblond, C.S.; Hamza, M.; Regnault, B.; Lemée, L.; Mathieu, A.; Jaouadi, H.; Mkaouar, R.;
Ben Youssef-Turki, I.; Belhadj, A.; et al. Homozygous 2p11.2 deletion supports the implication of ELMOD3
in hearing loss and reveals the potential association of CAPG with ASD/ID etiology. J. Appl. Genet. 2018, 60,
49–56. [CrossRef]
Fan, Y.; Tang, X.; Vitriol, E.A.; Chen, G.; Zheng, J.Q. Actin capping protein is required for dendritic spine
development and synapse formation. J. Neurosci. 2011, 31, 10228–11033. [CrossRef] [PubMed]

56.

57. Korobova, F.; Svitkina, T.M. Molecular Architecture of Synaptic Actin Cytoskeleton in Hippocampal Neurons
Reveals a Mechanism of Dendritic Spine Morphogenesis. Mol. Biol. Cell 2010, 21, 165–176. [CrossRef]
Schmunk, G.; Gargus, J.J. Channelopathy pathogenesis in autism spectrum disorders. Front. Genet. 2013,
4, 222. [CrossRef]

58.

59. Despang, P.; Salamon, S.; Breitenkamp, A.F.; Kuzmenkina, E.; Herzig, S.; Matthes, J. Autism-associated
mutations in the CaVβ2 calcium-channel subunit increase Ba2+-currents and lead to diﬀerential modulation
by the RGK-protein Gem. Neurobiol. Dis. 2020, 136, 104721. [CrossRef]
Splawski, I.; Timothy, K.W.; Sharpe, L.M.; Decher, N.; Kumar, P.; Bloise, R.; Napolitano, C.; Schwartz, P.J.;
Joseph, R.M.; Condouris, K.; et al. CaV1.2 Calcium Channel Dysfunction Causes a Multisystem Disorder
Including Arrhythmia and Autism. Cell 2004, 119, 19–31. [CrossRef]

60.

61. Catterall, W.A.; Perez-Reyes, E.; Snutch, T.P.; Striessnig, J. International Union of Pharmacology. XLVIII.
Nomenclature and Structure-Function Relationships of Voltage-Gated Calcium Channels. Pharmacol. Rev.
2005, 57, 411–425. [CrossRef]

62. Barrett, C.F.; Tsien, R.W. The Timothy syndrome mutation diﬀerentially aﬀects voltage- and
calcium-dependent inactivation of CaV1.2 L-type calcium channels. Proc. Natl. Acad. Sci. USA 2008, 105,
2157–2162. [CrossRef]

63. Hemara-Wahanui, A.; Berjukow, S.; Hope, C.I.; Dearden, P.K.; Wu, S.-B.; Wilson-Wheeler, J.; Sharp, D.M.;
Lundon-Treweek, P.; Clover, G.M.; Hoda, J.-C.; et al. A CACNA1F mutation identiﬁed in an X-linked retinal
disorder shifts the voltage dependence of Cav1.4 channel activation. Proc. Natl. Acad. Sci. USA 2005, 102,
7553–7558. [CrossRef]

Int. J. Mol. Sci. 2020, 21, 8290

17 of 22

64. Pinggera, A.; Lieb, A.; Benedetti, B.; Lampert, M.; Monteleone, S.; Liedl, K.R.; Tuluc, P.; Striessnig, J.
CACNA1D De Novo Mutations in Autism Spectrum Disorders Activate Cav1.3 L-Type Calcium Channels.
Biol. Psychiatry 2015, 77, 816–822. [CrossRef]

65. Pinggera, A.; Mackenroth, L.; Rump, A.; Schallner, J.; Beleggia, F.; Wollnik, B.; Striessnig, J.
New gain-of-function mutation shows CACNA1D as recurrently mutated gene in autism spectrum disorders
and epilepsy. Hum. Mol. Genet. 2017, 26, 2923–2932. [CrossRef]

66. Boczek, N.J.; Miller, E.M.; Ye, D.; Nesterenko, V.V.; Tester, D.J.; Antzelevitch, C.; Czosek, R.J.; Ackerman, M.J.;
Ware, S.M. Novel Timothy syndrome mutation leading to increase in CACNA1C window current.
Heart Rhythm 2015, 12, 211–219. [CrossRef]
Splawski, I.; Yoo, D.S.; Stotz, S.C.; Cherry, A.; Clapham, D.E.; Keating, M.T. CACNA1HMutations in Autism
Spectrum Disorders. J. Biol. Chem. 2006, 281, 22085–22091. [CrossRef]

67.

68. Breitenkamp, A.F.S.; Matthes, J.; Nass, R.D.; Sinzig, J.; Lehmkuhl, G.; Nürnberg, P.; Herzig, S. Rare Mutations of
CACNB2 Found in Autism Spectrum Disease-Aﬀected Families Alter Calcium Channel Function. PLoS ONE
2014, 9, e95579. [CrossRef]

69. Morrow, E.M.; Yoo, S.-Y.; Flavell, S.W.; Kim, T.-K.; Lin, Y.; Hill, R.S.; Mukaddes, N.M.; Balkhy, S.; Gascon, G.;
Hashmi, A.; et al. Identifying Autism Loci and Genes by Tracing Recent Shared Ancestry. Science 2008, 321,
218–223. [CrossRef]

70. Veeramah, K.R.; O’Brien, J.E.; Meisler, M.H.; Cheng, X.; Dib-Hajj, S.D.; Waxman, S.G.; Talwar, D.; Girirajan, S.;
Eichler, E.E.; Restifo, L.L.; et al. De Novo Pathogenic SCN8A Mutation Identiﬁed by Whole-Genome
Sequencing of a Family Quartet Aﬀected by Infantile Epileptic Encephalopathy and SUDEP. Am.
J.
Hum. Genet. 2012, 90, 502–510. [CrossRef]

71. Chen, C.-P.; Lin, S.-P.; Chern, S.-R.; Chen, Y.-J.; Tsai, F.-J.; Wu, P.-C.; Wang, W. Array-CGH detection of a
de novo 2.8 Mb deletion in 2q24.2→q24.3 in a girl with autistic features and developmental delay. Eur. J.
Med. Genet. 2010, 53, 217–220. [CrossRef]
Sanders, S.J.; Murtha, M.T.; Gupta, A.R.; Murdoch, J.D.; Raubeson, M.J.; Willsey, A.J.; Ercan-Sencicek, A.G.;
DiLullo, N.M.; Parikshak, N.N.; Stein, J.L.; et al. De novo mutations revealed by whole-exome sequencing
are strongly associated with autism. Nature 2012, 485, 237–241. [CrossRef] [PubMed]

72.

73. Weiss, L.A.; Escayg, A.; Kearney, J.A.; Trudeau, M.; Macdonald, B.T.; Mori, M.; Reichert, J.; Buxbaum, J.D.;
Meisler, M.H. Sodium channels SCN1A, SCN2A and SCN3A in familial autism. Mol. Psychiatry 2003, 8,
186–194. [CrossRef]

74. Han, S.; Tai, C.; Westenbroek, R.E.; Yu, F.H.; Cheah, C.S.; Potter, G.B.; Rubenstein, J.L.; Scheuer, T.;
De La Iglesia, H.O.; Catterall, W.A. Autistic-like behaviour in Scn1a+/− mice and rescue by enhanced
GABA-mediated neurotransmission. Nature 2012, 489, 385–390. [CrossRef]

75. Gilling, M.; Rasmussen, H.B.; Calloe, K.; Sequeira, A.F.; Baretto, M.; Oliveira, G.; Almeida, J.; Lauritsen, M.B.;
Ullmann, R.; Boonen, S.E.; et al. Dysfunction of the Heteromeric KV7.3/KV7.5 Potassium Channel is
Associated with Autism Spectrum Disorders. Front. Genet. 2013, 4, 54. [CrossRef]

77.

76. Gross, C.; Yao, X.; Pong, D.L.; Jeromin, A.; Bassell, G.J. Fragile X Mental Retardation Protein Regulates
Protein Expression and mRNA Translation of the Potassium Channel Kv4.2. J. Neurosci. 2011, 31, 5693–5698.
[CrossRef]
Sicca, F.; Imbrici, P.; D’Adamo, M.C.; Moro, F.; Bonatti, F.; Brovedani, P.; Grottesi, A.; Guerrini, R.;
Masi, G.; Santorelli, F.M.; et al. Autism with Seizures and Intellectual Disability: Possible Causative Role of
Gain-of-function of the Inwardly-Rectifying K+ Channel Kir4.1. Neurobiol. Dis. 2011, 43, 239–247. [CrossRef]
78. Laumonnier, F.; Eroger, S.; Guérin, P.; Molinari, F.; M’Rad, R.; Cahard, D.; Belhadj, A.; Halayem, M.;
Persico, A.M.; Elia, M.; et al. Association of a Functional Deﬁcit of the BKCa Channel, a Synaptic Regulator
of Neuronal Excitability, With Autism and Mental Retardation. Am. J. Psychiatry 2006, 163, 1622. [CrossRef]
79. Deng, P.Y.; Rotman, Z.; Blundon, J.A.; Cho, Y.; Cui, J.; Cavalli, V.; Zakharenko, S.S.; Klyachko, V.A.
FMRP Regulates Neurotransmitter Release and Synaptic Information Transmission by Modulating Action
Potential Duration via BK Channels. Neuron 2013, 77, 696–711. [CrossRef]

80. Gonzalez–Gronow, M.; Cuchacovich, M.; Francos, R.; Cuchacovich, S.; Fernandez, M.D.P.; Blanco, A.;
Bowers, E.V.; Kaczowka, S.; Pizzo, S.V. Antibodies against the voltage-dependent anion channel (VDAC) and
its protective ligand hexokinase-I in children with autism. J. Neuroimmunol. 2010, 227, 153–161. [CrossRef]
81. Gvozdjáková, A.; Löw, H.; Sun, I.; Navas, P.; Crane, F.L. Plasma membrane coenzyme Q: Evidence for a role

in autism. Biol. Targets Ther. 2014, 8, 199. [CrossRef]

Int. J. Mol. Sci. 2020, 21, 8290

18 of 22

82. Wong, C.C.Y.; Meaburn, E.L.; Ronald, A.; Price, T.S.; Jeﬀries, A.R.; Schalkwyk, L.C.; Plomin, R.; Mill, J.
Methylomic analysis of monozygotic twins discordant for autism spectrum disorder and related behavioural
traits. Mol. Psychiatry 2013, 19, 495–503. [CrossRef]

83. Nguyen, A.; Rauch, T.A.; Pfeifer, G.P.; Hu, V.W. Global methylation proﬁling of lymphoblastoid cell lines
reveals epigenetic contributions to autism spectrum disorders and a novel autism candidate gene, RORA,
whose protein product is reduced in autistic brain. FASEB J. 2010, 24, 3036–3051. [CrossRef]

86.

84. Zhu, Y.; Mordaunt, C.E.; Yasui, D.H.; Marathe, R.; Coulson, R.L.; Dunaway, K.; Jianu, J.M.; Walker, C.K.;
Ozonoﬀ, S.; Hertz-Picciotto, I.; et al. Placental DNA methylation levels at CYP2E1 and IRS2 are associated
with child outcome in a prospective autism study. Hum. Mol. Genet. 2019, 28, 2659–2674. [CrossRef]
85. Andrews, S.V.; Sheppard, B.; Windham, G.C.; Schieve, L.A.; Schendel, D.E.; Croen, L.E.; Chopra, P.;
Alisch, R.S.; Newschaﬀer, C.J.; Warren, S.T.; et al. Case-control meta-analysis of blood DNA methylation and
autism spectrum disorder. Mol. Autism 2018, 9, 40. [CrossRef]
Siu, M.T.; Butcher, D.T.; Turinsky, A.L.; Cytrynbaum, C.; Stavropoulos, D.J.; Walker, S.; Caluseriu, O.;
Carter, M.; Lou, Y.; Nicolson, R.; et al. Functional DNA methylation signatures for autism spectrum disorder
genomic risk loci: 16p11.2 deletions and CHD8 variants. Clin. Epigenetics 2019, 11, 1–19. [CrossRef]
87. Kimura, R.; Nakata, M.; Funabiki, Y.; Suzuki, S.; Awaya, T.; Murai, T.; Hagiwara, M. An epigenetic biomarker
for adult high-functioning autism spectrum disorder. Sci. Rep. 2019, 9, 13662–13667. [CrossRef] [PubMed]
Saeliw, T.; Tangsuwansri, C.; Thongkorn, S.; Chonchaiya, W.; Suphapeetiporn, K.; Mutirangura, A.;
Tencomnao, T.; Hu, V.W.; Sarachana, T. Integrated genome-wide Alu methylation and transcriptome proﬁling
analyses reveal novel epigenetic regulatory networks associated with autism spectrum disorder. Mol. Autism
2018, 9, 27. [CrossRef]

88.

89. Wong, C.C.Y.; Smith, R.G.; Hannon, E.; Ramaswami, G.; Parikshak, N.N.; Assary, E.; Troakes, C.;
Poschmann, J.; Schalkwyk, L.C.; Sun, W.; et al. Genome-wide DNA methylation profiling identifies convergent
molecular signatures associated with idiopathic and syndromic autism in post-mortem human brain tissue.
Hum. Mol. Genet. 2019, 28, 2201–2211. [CrossRef]
Stathopoulos, S.; Gaujoux, R.; Lindeque, Z.; Mahony, C.; Van Der Colﬀ, R.; Van Der Westhuizen, F.; O’Ryan, C.
DNA Methylation Associated with Mitochondrial Dysfunction in a South African Autism Spectrum Disorder
Cohort. Autism Res. 2020, 13, 1079–1093. [CrossRef]

90.

91. Hu, Z.; Yang, Y.; Zhao, Y.; Yu, H.; Ying, X.; Zhou, D.; Zhong, J.; Zheng, Z.; Liu, J.; Pan, R.; et al.
APOE hypermethylation is associated with autism spectrum disorder in a Chinese population. Exp. Ther. Med.
2018, 15, 4749–4754. [CrossRef]

92. Hranilovi´c, D.; Blaževi´c, S.; Stefulj, J.; Zill, P. DNA Methylation Analysis of HTR2A Regulatory Region in

Leukocytes of Autistic Subjects. Autism Res. 2015, 9, 204–209. [CrossRef]

93. Hu, Z.; Ying, X.; Huang, L.; Zhao, Y.; Zhou, D.; Liu, J.; Zhong, J.; Huang, T.; Zhang, W.; Cheng, F.; et al.
Association of human serotonin receptor 4 promoter methylation with autism spectrum disorder. Medicine
2020, 99, e18838. [CrossRef]

94. Lu, Z.; Liu, Z.; Mao, W.; Wang, X.; Zheng, X.; Chen, S.; Cao, B.; Huang, S.; Zhang, X.; Zhou, T.; et al.
Locus-speciﬁc DNA methylation of Mecp2 promoter leads to autism-like phenotypes in mice. Cell Death Dis.
2020, 11, 85. [CrossRef]

95. Hicks, S.D.; Middleton, F.A. A Comparative Review of microRNA Expression Patterns in Autism Spectrum

Disorder. Front. Psychiatry 2016, 7, 176. [CrossRef]

96. Hicks, S.D.; Carpenter, R.L.; Wagner, K.E.; Pauley, R.; Barros, M.; Tierney-Aves, C.; Barns, S.; Greene, C.D.;
Middleton, F.A. Saliva MicroRNA Diﬀerentiates Children with Autism from Peers with Typical and Atypical
Development. J. Am. Acad. Child. Adolesc. Psychiatry 2020, 59, 296–308. [CrossRef]
Sehovic, E.; Spahic, L.; Smajlovic-Skenderagic, L.; Pistoljevic, N.; Dzanko, E.; Hajdarpasic, A. Identiﬁcation of
developmental disorders including autism spectrum disorder using salivary miRNAs in children from
Bosnia and Herzegovina. PLoS ONE 2020, 15, e0232351. [CrossRef]

97.

98. Hagiwara, M.; Kimura, R.; Funabiki, Y.; Awaya, T.; Murai, T.; Hagiwara, M. MicroRNA proﬁling in adults

with high-functioning autism spectrum disorder. Mol. Brain 2019, 12, 82–84. [CrossRef]

99. Ozkul, Y.; Taheri, S.; Bayram, K.K.; Sener, E.F.; Mehmetbeyoglu, E.; Öztop, D.B.; Aybuga, F.; Tufan, E.;
Bayram, A.; Dolu, N.; et al. A heritable proﬁle of six miRNAs in autistic patients and mouse models. Sci. Rep.
2020, 10, 1–14. [CrossRef]

Int. J. Mol. Sci. 2020, 21, 8290

19 of 22

100. Vasu, M.M.; Anitha, A.; Thanseem, I.; Suzuki, K.; Yamada, K.; Takahashi, T.; Wakuda, T.; Iwata, K.; Tsujii, M.;
Sugiyama, T.; et al. Serum microRNA proﬁles in children with autism. Mol. Autism 2014, 5, 40. [CrossRef]
101. Vatsa, N.; Kumar, V.; Singh, B.K.; Kumar, S.S.; Sharma, A.; Jana, N.R. Down-Regulation of miRNA-708
Promotes Aberrant Calcium Signaling by Targeting Neuronatin in a Mouse Model of Angelman Syndrome.
Front. Mol. Neurosci. 2019, 12, 35. [CrossRef]

102. Urdinguio, R.G.; Fernández, A.F.; López-Nieva, P.; Rossi, S.; Huertas, D.; Kulis, M.; Liu, C.-G.; Croce, C.M.;
Calin, G.A.; Esteller, M. Disrupted microRNA expression caused by Mecp2 loss in a mouse model of Rett
syndrome. Epigenetics 2010, 5, 656–663. [CrossRef]

103. Edbauer, D.; Neilson, J.R.; Foster, K.A.; Wang, C.-F.; Seeburg, D.P.; Batterton, M.N.; Tada, T.; Dolan, B.M.;
Sharp, P.A.; Sheng, M. Regulation of Synaptic Structure and Function by FMRP-Associated MicroRNAs
miR-125b and miR-132. Neuron 2010, 65, 373–384. [CrossRef]

104. Gaugler, T.; Klei, L.; Sanders, S.J.; Bodea, C.A.; Goldberg, A.P.; Lee, A.B.; Mahajan, M.C.; Manaa, D.;
Pawitan, Y.; Reichert, J.G.; et al. Most genetic risk for autism resides with common variation. Nat. Genet.
2014, 46, 881–885. [CrossRef]

105. Deng, W.; Zou, X.; Deng, H.; Li, J.; Tang, C.; Wang, X.; Guo, X. The Relationship Among Genetic Heritability,
Environmental Eﬀects, and Autism Spectrum Disorders. J. Child Neurol. 2015, 30, 1794–1799. [CrossRef]
106. Edelson, L.R.; Saudino, K.J. Genetic and environmental inﬂuences on autistic-like behaviors in 2-year-old

twins. Behav. Genet. 2009, 39, 255–264. [CrossRef]

107. Bölte, S.; Girdler, S.; Marschik, P.B. The contribution of environmental exposure to the etiology of autism

spectrum disorder. Cell. Mol. Life Sci. 2019, 76, 1275–1297. [CrossRef]

108. Janecka, M.; Mill, J.; Basson, M.A.; Goriely, A.; Spiers, H.; Reichenberg, A.; Schalkwyk, L.; Fernandes, C.
Advanced paternal age eﬀects in neurodevelopmental disorders—Review of potential underlying
mechanisms. Transl. Psychiatry 2017, 7, e1019. [CrossRef]

109. Wu, S.; Wu, F.; Ding, Y.; Hou, J.; Bi, J.; Zhang, Z. Advanced parental age and autism risk in children:

A systematic review and meta-analysis. Acta Psychiatr. Scand. 2016, 135, 29–41. [CrossRef]

110. Atsem, S.; Reichenbach, J.; Potabattula, R.; Dittrich, M.; Nava, C.; Depienne, C.; Böhm, L.; Rost, S.; Hahn, T.;
Schorsch, M.; et al. Paternal age eﬀects on spermFOXK1andKCNA7methylation and transmission into the
next generation. Hum. Mol. Genet. 2016, 25, 4996–5005. [CrossRef]

111. Kojima, M.; Yassin, W.; Owada, K.; Aoki, Y.; Kuwabara, H.; Natsubori, T.; Iwashiro, N.; Gonoi, W.; Takao, H.;
Kasai, K.; et al. Neuroanatomical Correlates of Advanced Paternal and Maternal Age at Birth in Autism
Spectrum Disorder. Cereb. Cortex 2018, 29, 2524–2532. [CrossRef]

112. Foldi, C.J.; Eyles, D.; McGrath, J.J.; Burne, T.H.J. Advanced paternal age is associated with alterations in
discrete behavioural domains and cortical neuroanatomy of C57BL/6J mice. Eur. J. Neurosci. 2010, 31, 556–564.
[CrossRef] [PubMed]

113. Sampino, S.; Juszczak, G.R.; Zacchini, F.; Swiergiel, A.H.; Modlinski, J.A.; Loi, P.; Ptak, G. Grand-paternal age
and the development of autism-like symptoms in mice progeny. Transl. Psychiatry 2014, 4, e386. [CrossRef]
114. Gardener, H.; Spiegelman, D.; Buka, S.L. Prenatal risk factors for autism: Comprehensive meta-analysis.

Br. J. Psychiatry 2009, 195, 7–14. [CrossRef]

115. Gardener, H.; Spiegelman, D.; Buka, S.L. Perinatal and Neonatal Risk Factors for Autism: A Comprehensive

Meta-analysis. Pediatrics 2011, 128, 344–355. [CrossRef]

116. Wang, C.; Geng, H.; Liu, W.; Zhang, G. Prenatal, perinatal, and postnatal factors associated with autism.

Medicine 2017, 96, e6696. [CrossRef]

117. Bejerot, S.; Eriksson, J.M.; Bonde, S.; Carlström, K.; Humble, M.B.; Eriksson, E. The extreme male brain
revisited: Gender coherence in adults with autism spectrum disorder. Br. J. Psychiatry 2012, 201, 116–123.
[CrossRef]

118. Baron-Cohen, S.; Auyeung, B.; Nørgaard-Pedersen, B.; Hougaard, D.M.; Abdallah, M.W.; Melgaard, L.;
Cohen, A.S.; Chakrabarti, B.; Ruta, L.; Lombardo, M.V. Elevated fetal steroidogenic activity in autism.
Mol. Psychiatry 2015, 20, 369–376. [CrossRef]

119. Baron-Cohen, S.; Tsompanidis, A.; Auyeung, B.; Nørgaard-Pedersen, B.; Hougaard, D.M.; Abdallah, M.;
Cohen, A.; Pohl, A. Foetal oestrogens and autism. Mol. Psychiatry 2020, 25, 2970–2978. [CrossRef] [PubMed]
120. Crider, A.; Thakkar, R.; Ahmed, A.O.; Pillai, A. Dysregulation of estrogen receptor beta (ERβ),
aromatase (CYP19A1), and ER co-activators in the middle frontal gyrus of autism spectrum disorder
subjects. Mol. Autism 2014, 5, 46. [CrossRef]

Int. J. Mol. Sci. 2020, 21, 8290

20 of 22

121. Chakrabarti, B.; Dudbridge, F.; Kent, L.; Wheelwright, S.; Hill-Cawthorne, G.; Allison, C.; Banerjee-Basu, S.;
Baron-Cohen, S. Genes related to sex steroids, neural growth, and social-emotional behavior are associated
with autistic traits, empathy, and Asperger syndrome. Autism Res. 2009, 2, 157–177. [CrossRef]

122. Jackson, A.A.; Robinson, S.M. Dietary guidelines for pregnancy: A review of current evidence. Public Health Nutr.

2001, 4, 625–630. [CrossRef]

123. Van Eijsden, M.; Smits, L.J.M.; Van Der Wal, M.F.; Bonsel, G.J. Association between short interpregnancy
intervals and term birth weight: The role of folate depletion. Am. J. Clin. Nutr. 2008, 88, 147–153. [CrossRef]
[PubMed]

124. Magnusson, C.; Kosidou, K.; Dalman, C.; Lundberg, M.; Lee, B.K.; Rai, D.; Karlsson, H.; Gardner, R.;
Arver, S. Maternal vitamin D deﬁciency and the risk of autism spectrum disorders: Population-based study.
BJPsych Open 2016, 2, 170–172. [CrossRef]

125. Robea, M.; Luca, A.-C.; Ciobica, A. Relationship between Vitamin Deﬁciencies and Co-Occurring Symptoms

in Autism Spectrum Disorder. Medicine 2020, 56, 245. [CrossRef]

126. Saghazadeh, A.; Ahangari, N.; Hendi, K.; Saleh, F.; Rezaei, N. Status of essential elements in autism spectrum

disorder: Systematic review and meta-analysis. Rev. Neurosci. 2017, 28, 783–809. [CrossRef]

127. Schmidt, R.J.; Tancredi, D.J.; Krakowiak, P.; Hansen, R.L.; Ozonoﬀ, S. Maternal Intake of Supplemental Iron

and Risk of Autism Spectrum Disorder. Am. J. Epidemiol. 2014, 180, 890–900. [CrossRef]

128. Hagmeyer, S.; Mangus, K.; Boeckers, T.M.; Grabrucker, A.M. Eﬀects of Trace Metal Proﬁles Characteristic for

Autism on Synapses in Cultured Neurons. Neural Plast. 2015, 2015, 1–16. [CrossRef]

129. Goyal, D.K.; Neil, J.R.; Simmons, S.D.; Mansab, F.; Benjamin, S.; Pitﬁeld, V.; Boulet, S.; Miyan, J.A.

Zinc Deﬁciency in Autism: A Controlled Study. Insights Biomed. 2019, 4, 4. [CrossRef]

130. Yasuda, H.; Yoshida, K.; Yasuda, Y.; Tsutsui, T. Infantile zinc deﬁciency: Association with autism spectrum

disorders. Sci. Rep. 2011, 1, 129. [CrossRef]

131. Li, S.-O.; Wang, J.-L.; Bjørklund, G.; Zhao, W.-N.; Yin, C.-H. Serum copper and zinc levels in individuals with

autism spectrum disorders. NeuroReport 2014, 25, 1216–1220. [CrossRef]

132. Grabrucker, S.; Boeckers, T.M.; Grabrucker, A.M. Gender Dependent Evaluation of Autism like Behavior in

Mice Exposed to Prenatal Zinc Deﬁciency. Front. Behav. Neurosci. 2016, 10, 37. [CrossRef]

133. Grabrucker, S.; Jannetti, L.; Eckert, M.; Gaub, S.; Chhabra, R.; Pfaender, S.; Mangus, K.; Reddy, P.P.;
Rankovic, V.; Schmeisser, M.J.; et al. Zinc deﬁciency dysregulates the synaptic ProSAP/Shank scaﬀold and
might contribute to autism spectrum disorders. Brain 2013, 137, 137–152. [CrossRef] [PubMed]

134. Grabrucker, A.M.; Knight, M.J.; Proepper, C.; Bockmann, J.; Joubert, M.; Rowan, M.; Nienhaus, G.U.;
Garner, C.C.; Bowie, J.U.; Kreutz, M.R.; et al. Concerted action of zinc and ProSAP/Shank in synaptogenesis
and synapse maturation. EMBO J. 2011, 30, 569–581. [CrossRef]

135. Frederickson, C.J.; Koh, J.-Y.; Bush, A.I. The neurobiology of zinc in health and disease. Nat. Rev. Neurosci.

2005, 6, 449–462. [CrossRef] [PubMed]

136. Vyas, Y.; Lee, K.; Jung, Y.; Montgomery, J.M. Inﬂuence of maternal zinc supplementation on the development
of autism-associated behavioural and synaptic deﬁcits in oﬀspring Shank3-knockout mice. Mol. Brain 2020,
13, 1–18. [CrossRef]

137. Shih, P.-Y.; Hsieh, B.-Y.; Lin, M.-H.; Huang, T.-N.; Tsai, C.-Y.; Pong, W.-L.; Lee, S.-P.; Hsueh, Y.-P.
CTTNBP2 Controls Synaptic Expression of Zinc-Related Autism-Associated Proteins and Regulates Synapse
Formation and Autism-like Behaviors. Cell Rep. 2020, 31, 107700. [CrossRef]

138. Velie, E.M.; Block, G.; Shaw, G.M.; Samuels, S.J.; Schaﬀer, D.M.; Kulldorﬀ, M. Maternal supplemental and
dietary zinc intake and the occurrence of neural tube defects in California. Am. J. Epidemiol. 1999, 150,
605–616. [CrossRef]

139. Gardner, R.M.; Lee, B.K.; Magnusson, C.; Rai, D.; Frisell, T.; Karlsson, H.; Idring, S.; Dalman, C. Maternal body
mass index during early pregnancy, gestational weight gain, and risk of autism spectrum disorders:
Results from a Swedish total population and discordant sibling study. Int. J. Epidemiol. 2015, 44, 870–883.
[CrossRef]

140. Wang, Y.; Tang, S.; Xu, S.; Weng, S.; Liu, Z. Maternal Body Mass Index and Risk of Autism Spectrum Disorders

in Oﬀspring: A Meta-analysis. Sci. Rep. 2016, 6, 34248. [CrossRef] [PubMed]

141. Getz, K.D.; Anderka, M.T.; Werler, M.M.; Jick, S.S. Maternal Pre-pregnancy Body Mass Index and Autism
Spectrum Disorder among Oﬀspring: A Population-Based Case-Control Study. Paediatr. Périnat. Epidemiol.
2016, 30, 479–487. [CrossRef]

Int. J. Mol. Sci. 2020, 21, 8290

21 of 22

142. Li, M.; Fallin, M.D.; Riley, A.; Landa, R.; Walker, S.O.; Silverstein, M.; Caruso, D.; Pearson, C.; Kiang, S.;
Dahm, J.L.; et al. The Association of Maternal Obesity and Diabetes with Autism and Other Developmental
Disabilities. Pediatrics 2016, 137, e20152206. [CrossRef]

143. Tang, S.; Wang, Y.; Gong, X.; Wang, G.-H. A Meta-Analysis of Maternal Smoking during Pregnancy and
Autism Spectrum Disorder Risk in Oﬀspring. Int. J. Environ. Res. Public Health 2015, 12, 10418–10431.
[CrossRef]

144. Rosen, B.N.; Lee, B.K.; Lee, N.L.; Yang, Y.; Burstyn, I. Maternal Smoking and Autism Spectrum Disorder:

A Meta-analysis. J. Autism Dev. Disord. 2014, 45, 1689–1698. [CrossRef]

145. Visser, J.C.; Rommelse, N.; Vink, L.; Schrieken, M.; Oosterling, I.J.; Van Der Gaag, R.J.; Buitelaar, J.K. Narrowly
Versus Broadly Deﬁned Autism Spectrum Disorders: Diﬀerences in Pre- and Perinatal Risk Factors. J. Autism
Dev. Disord. 2012, 43, 1505–1516. [CrossRef]

146. Perrone-McGovern, K.; Simon-Dack, S.; Niccolai, L. Prenatal and Perinatal Factors Related to Autism, IQ,

and Adaptive Functioning. J. Genet. Psychol. 2015, 176, 1–10. [CrossRef]

147. Eliasen, M.H.; Tolstrup, J.S.; Andersen, A.-M.N.; Gronbaek, M.; Olsen, J.; Strandberg-Larsen, K.; Grønbæk, M.
Prenatal alcohol exposure and autistic spectrum disorders—A population-based prospective study of 80 552
children and their mothers. Int. J. Epidemiol. 2010, 39, 1074–1081. [CrossRef] [PubMed]

148. Gallagher, C.; McCarthy, F.P.; Ryan, R.M.; Khashan, A.S. Maternal Alcohol Consumption during Pregnancy
and the Risk of Autism Spectrum Disorders in Oﬀspring: A Retrospective Analysis of the Millennium Cohort
Study. J. Autism Dev. Disord. 2018, 48, 3773–3782. [CrossRef]

149. Meador, K.J. Eﬀects of in Utero Antiepileptic Drug Exposure. Epilepsy Curr. 2008, 8, 143–147. [CrossRef]

[PubMed]

150. Veroniki, A.A.; Cogo, E.; Rios, P.; Straus, S.E.; Finkelstein, Y.; Kealey, R.; Reynen, E.; Soobiah, C.; Thavorn, K.;
Hutton, B.; et al. Comparative safety of anti-epileptic drugs during pregnancy: A systematic review and
network meta-analysis of congenital malformations and prenatal outcomes. BMC Med. 2017, 15, 1–20.
[CrossRef]

151. Kobayashi, T.; Matsuyama, T.; Takeuchi, M.; Ito, S. Autism spectrum disorder and prenatal exposure to
selective serotonin reuptake inhibitors: A systematic review and meta-analysis. Reprod. Toxicol. 2016, 65,
170–178. [CrossRef]

152. Wan, H.; Zhang, C.; Li, H.; Luan, S.; Liu, C. Association of maternal diabetes with autism spectrum disorders

in oﬀspring. Medicine 2018, 97, e9438. [CrossRef] [PubMed]

153. Xu, G.; Jing, J.; Bowers, K.; Liu, B.; Bao, W. Maternal diabetes and the risk of autism spectrum disorders in
the oﬀspring: A systematic review and meta-analysis. J. Autism Dev. Disord. 2014, 44, 766–775. [CrossRef]
154. Jiang, H.-Y.; Xu, L.-L.; Shao, L.; Xia, R.-M.; Yu, Z.-H.; Ling, Z.; Yang, F.; Deng, M.; Ruan, B. Maternal infection
during pregnancy and risk of autism spectrum disorders: A systematic review and meta-analysis.
Brain Behav. Immun. 2016, 58, 165–172. [CrossRef]

155. Zerbo, O.; Qian, Y.; Yoshida, C.; Grether, J.K.; Van De Water, J.; Croen, L.A. Maternal Infection during

Pregnancy and Autism Spectrum Disorders. J. Autism Dev. Disord. 2015, 45, 4015–4025. [CrossRef]

156. Atladóttir, H.Ó.; Thorsen, P.; Østergaard, L.; Schendel, D.E.; Lemcke, S.; Abdallah, M.; Parner, E.T.
Maternal Infection Requiring Hospitalization During Pregnancy and Autism Spectrum Disorders. J. Autism
Dev. Disord. 2010, 40, 1423–1430. [CrossRef]

157. Chen, S.-W.; Zhong, X.-S.; Jiang, L.-N.; Zheng, X.-Y.; Xiong, Y.-Q.; Ma, S.-J.; Qiu, M.; Huo, S.-T.; Ge, J.; Chen, Q.
Maternal autoimmune diseases and the risk of autism spectrum disorders in oﬀspring: A systematic review
and meta-analysis. Behav. Brain Res. 2016, 296, 61–69. [CrossRef]

158. Wu, S.; Ding, Y.; Wu, F.; Li, R.; Xie, G.; Hou, J.; Mao, P. Family history of autoimmune diseases is associated
with an increased risk of autism in children: A systematic review and meta-analysis. Neurosci. Biobehav. Rev.
2015, 55, 322–332. [CrossRef]

159. Brown, A.S.; Sourander, A.; Hinkka-Yli-Salomäki, S.; McKeague, I.W.; Sundvall, J.; Surcel, H.-M. Elevated maternal

C-reactive protein and autism in a national birth cohort. Mol. Psychiatry 2013, 19, 259–264. [CrossRef]

160. Brown, A.S.; Surcel, H.-M.; Hinkka-Yli-Salomäki, S.; Cheslack-Postava, K.; Bao, Y.; Sourander, A. Maternal thyroid
autoantibody and elevated risk of autism in a national birth cohort. Prog. Neuro-Psychopharmacol. Biol. Psychiatry
2015, 57, 86–92. [CrossRef]

Int. J. Mol. Sci. 2020, 21, 8290

22 of 22

161. Meyer, U.; Feldon, J.; Fatemi, S.H. In-vivo rodent models for the experimental investigation of prenatal
immune activation eﬀects in neurodevelopmental brain disorders. Neurosci. Biobehav. Rev. 2009, 33,
1061–1079. [CrossRef]

162. Comi, A.M.; Zimmerman, A.W.; Frye, V.H.; Law, P.A.; Peeden, J.N. Familial Clustering of Autoimmune
Disorders and Evaluation of Medical Risk Factors in Autism. J. Child Neurol. 1999, 14, 388–394. [CrossRef]
163. Atladóttir, H.O.; Pedersen, M.G.; Thorsen, P.; Mortensen, P.B.; Deleuran, B.; Eaton, W.W.; Parner, E.T.;
Sutton, R.M.; Niles, D.; Nysaether, J.; et al. Association of Family History of Autoimmune Diseases and
Autism Spectrum Disorders. Pediatrics 2009, 124, 687–694. [CrossRef]

164. Sanna, A.; Firinu, D.; Zavattari, P.; Valera, P. Zinc Status and Autoimmunity: A Systematic Review and

Meta-Analysis. Nutrients 2018, 10, 68. [CrossRef] [PubMed]

165. Napoli, E.; Hung, C.; Wong, S.; Giulivi, C. Toxicity of the Flame-Retardant BDE-49 on Brain Mitochondria
and Neuronal Progenitor Striatal Cells Enhanced by a PTEN-Deﬁcient Background. Toxicol. Sci. 2013, 132,
196–210. [CrossRef]

166. Rossignol, D.A.; Frye, R.E. Mitochondrial dysfunction in autism spectrum disorders: A systematic review

and meta-analysis. Mol. Psychiatry 2012, 17, 290–314. [CrossRef]

167. Yoshimasu, K.; Kiyohara, C.; Takemura, S.; Nakai, K. A meta-analysis of the evidence on the impact of
prenatal and early infancy exposures to mercury on autism and attention deﬁcit/hyperactivity disorder in
the childhood. NeuroToxicology 2014, 44, 121–131. [CrossRef]

168. Shelton, J.F.; Geraghty, E.M.; Tancredi, D.; Delwiche, L.D.; Schmidt, R.J.; Ritz, B.; Hansen, R.L.;
Hertz-Picciotto, I. Neurodevelopmental Disorders and Prenatal Residential Proximity to Agricultural
Pesticides: The CHARGE Study. Environ. Health Perspect. 2014, 122, 1103–1109. [CrossRef]

169. Jeddi, M.Z.; Janani, L.; Memari, A.H.; Akhondzadeh, S.; Yunesian, M. The role of phthalate esters in autism

development: A systematic review. Environ. Res. 2016, 151, 493–504. [CrossRef]

170. Dingemans, M.M.; Berg, M.V.D.; Westerink, R.H.S. Neurotoxicity of Brominated Flame Retardants: (In)direct
Eﬀects of Parent and Hydroxylated Polybrominated Diphenyl Ethers on the (Developing) Nervous System.
Environ. Health Perspect. 2011, 119, 900–907. [CrossRef]

171. Wayman, G.A.; Bose, D.D.; Yang, N.; Lesiak, A.; Bruun, N.; Impey, S.; LeDoux, V.; Pessah, I.N.; Lein, P.J.
PCB-95 Modulates the Calcium-Dependent Signaling Pathway Responsible for Activity-Dependent Dendritic
Growth. Environ. Health Perspect. 2012, 120, 1003–1009. [CrossRef]

Publisher’s Note: MDPI stays neutral with regard to jurisdictional claims in published maps and institutional
aﬃliations.

© 2020 by the authors. Licensee MDPI, Basel, Switzerland. This article is an open access
article distributed under the terms and conditions of the Creative Commons Attribution
(CC BY) license (http://creativecommons.org/licenses/by/4.0/).
