# Nievergelt_2024_Genome-wide association analyses identify 95 risk loci and provide insights into the neurobiology of post-traumatic stress disorder.

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

HHS Public Access
Author manuscript
Nat Genet. Author manuscript; available in PMC 2024 September 13.

Published in final edited form as:

Nat Genet. 2024 May ; 56(5): 792–808. doi:10.1038/s41588-024-01707-9.

Genome-wide association analyses identify 95 risk loci and 
provide insights into the neurobiology of posttraumatic stress 
disorder

A full list of authors and affiliations appears at the end of the article.

Abstract

Posttraumatic stress disorder (PTSD) genetics are characterized by lower discoverability than most 
other psychiatric disorders. The contribution to biological understanding from previous genetic 
studies has thus been limited. We performed a multi-ancestry meta-analysis of genome-wide 
association studies across 1,222,882 individuals of European ancestry (137,136 cases) and 58,051 
admixed individuals with African and Native American ancestry (13,624 cases). We identified 
95 genome-wide significant loci (80 novel). Convergent multi-omic approaches identified 43 

Corresponding author: Caroline Nievergelt, cnievergelt@ucsd.edu.
*Contributed equally
Author Contributions
PGC-PTSD writing group: E.G.A., S.-A.B., C.-Y.C., K.W.C., J.R.I.C., N.P.D., L.E.D., K.C.K., A.X.M., R.A.M., C.M.N., R.P., K.J.R., 
and M.B.S.
Study PI or co-PI: A.B.A., S.B. Andersen, P.A.A., A.E.A.-K., S.B. Austin, E.A., D.B., D.G.B., J.C.B., S. Belangero, C. Benjet, J.M.B., 
L.J.B., J.I.B., G.B., R.B., A.D.B., J.R.C., C.S.C., L.K.B., J.D., D.L.D., T.d.-C., K.D., G.D., A.D.-K., N.F., L.A.F., A.F., N.C.F., B.G., 
J.G., E.G., C.F.G., A.G.U., M.A.H., A.C.H., V.H., I.B.H., D.M.H., K. Hveem, M. Jakovljevic, A.J., I.J., T.J., K.-I.K., M.L.K., R.C.K., 
N.A.K., K.C.K., R.K., H.R.K., W.S.K., B.R.L., K.L., I.L., B.L., C.M., N.G.M., K.A.M., S.A.M., S.E.M., D.M., W.P.M., M.W.M., 
C.P.M., O.M., P.B.M, E.C.N., C.M.N., M.N., S.B.N., N.R.N., P.M.P., A.L.P., R.H.P., M.A.P., B.P., A.P., K.J.R., V.R., P.R.B., K.R., 
H.R., G.S., S. Seedat, J.S. Seng, A.K.S., S.R.S., D.J.S., M.B.S., R.J.U., U.V., S.J.H.v.R., E.V., J.V., Z.W., M.W., H.W., T.W., M.A.W., 
D.E.W., C.W., R.M.Y., H.Z., L.A.Z., and J.-A.Z.
Obtained funding for studies: A.B.A., P.A.A., A.E.A.-K., S.B. Austin, J.C.B., S. Belangero, C. Benjet, J.M.B., L.J.B., G.B., A.D.B., 
C.S.C., J.D., T.d.-C., A.F., N.C.F., J.D.F., C.E.F., E.G., C.F.G., M.H., M.A.H., A.C.H., V.H., I.B.H., D.M.H., K. Hveem, T.J., N.A.K., 
K.C.K., R.K., W.S.K., B.R.L., B.L., C.M., N.G.M., K.A.M., S.A.M., S.E.M., J.M., W.P.M., M.W.M., C.P.M., O.M., P.B.M, E.C.N., 
C.M.N., M.N., N.R.N., H.K.O., M.A.P., B.P., K.J.R., B.O.R., G.S., M.S., A.K.S., S.R.S., M.H.T., R.J.U., U.V., E.V., J.V., Z.W., M.W., 
T.W., M.A.W., D.E.W., R.Y., R.M.Y., and L.A.Z.
Clinical: C.A., P.A.A., E.A., D.B., D.G.B., J.C.B., L.B., L.J.B., E.A.B., R.B., A.C.B., A.D.B., S. Børte, L.C., J.R.C., K.W.C., L.K.B., 
M.F.D., T.d.-C., S.G.D., G.D., A.D.-K., N.F., N.C.F., J.D.F., C.E.F., S.G., E.G., A.G.U., S.B.G., L.G., C.G., V.H., D.M.H., M. 
Jakovljevic, A.J., G.D.J., M.L.K., A.K., N.A.K., N.K., R.K., W.S.K., B.R.L., L.A.M.L., K.L., C.E.L., B.L., J.L.M.-K., S.A.M., P.B.M., 
H.K.O., P.M.P., M.S.P., E.S.P., A.L.P., M.P., R.H.P., M.A.P., B.P., A.P., B.O.R., A.O.R., G.S., L.S., J.S. Seng, C.M.S., S. Stensland, 
M.H.T., W.K.T., E.T., M.U., U.V., L.L.v.d.H., E.V., Z.W., Y.W., T.W., D.E.W., B.S.W., S.W., E.J.W., R.Y., K.A.Y., and L.A.Z.
Contributed data: O.A.A., P.A.A., S.B. Austin, D.G.B., S. Belangero, L.J.B., R.B., R.A.B., A.D.B., J.R.C., J.M.C.-d.-A., S.Y.C., 
S.A.P.C., A.M.D., L.K.B., D.L.D., A.E., N.C.F., D.F., C.E.F., S.G., B.G., S.M.J.H., D.M.H., L.M.H., K. Hveem, A.J., I.J., M.L.K., 
J.L.K., R.C.K., A.P.K., R.K., W.S.K., L.A.M.L., K.L., D.F.L., C.E.L., I.L., B.L., M.K.L., S.M., G.A.M., K.M., A.M., K.A.M., S.E.M., 
J.M., L.M., O.M., P.B.M., M.N., S.B.N., N.R.N., M.O., P.M.P., M.S.P., E.S.P., A.L.P., M.P., R.H.P., M.A.P., K.J.R., V.R., P.R.B., A. 
Rung, G.S., L.S., S.E.S., M.S., C.S., S. Seedat, J.S. Seng, D. Silove, J.W.S., S.R.S., M.B.S., A.K.T., E.T., U.V., L.L.v.d.H., M.V.H., 
M.W., T.W., D.E.W., S.W., K.A.Y., C.C.Z., G.C.Z., L.A.Z., and J.-A.Z.
Statistical analysis: A.E.A.-K., A. Batzler, C. Bergner, A. Brandolino, S. Børte, C.C., C.-Y.C., S.A.P.C., J.R.I.C., L.C.-C., B.J.C., S.D., 
S.G.D., A.D., L.E.D., C.F., M.E.G., B.G., S.B.G., S.D.G., C.G., S.H., E.M.H., K. Hogan, H.H., G.D.J., K.K., P.-F.K., D.F.L., M.W.L., 
A.L, Y.L., A.X.M., S.M., C.M., D.M., J.M., V.M., E.A.M., M.S.M., C.M.N., G.A.P., M.P., X.-J.Q., A.R., A.L.R., S.S.d.V., C.S., A.S., 
C.M.S., S. Stensland, J.S.S., J.A.S., F.R.W., B.S.W., Y. Xia, Y. Xiong, and C.C.Z.
Bioinformatics: A.E.A.-K., A. Batzler, M.P.B., S. Børte, C.C., C.-Y.C., J.R.I.C., N.P.D., C.D.P., S.G.D., A.D., H.E., M.E.G., K. Hogan, 
H.H., K.K., P.-F.K., D.F.L., S.D.L., A.L, A.X.M., G.A.M., D.M., J.M., V.M., E.A.M., G.A.P., A.R., A.S., J.S.S., F.R.W., B.S.W., C.W., 
Y. Xia, Y. Xiong, and C.C.Z.
Genomics: M.P.B., J.B.-G., M.B.-H., N.P.D., T.d.-C., F.D., A.D., K.D., H.E., L.G., M.A.H., J.J., P.-F.K., S.D.L., J.J.L., I.K., J.M., 
L.M., K.J.R., B.P.F.R., S.S.d.V., A.S., C.H.V., and D.E.W.
PGC-PTSD management group: M.H. and M.Z.

Code availability
Analysis code is made available at GitHub (https://github.com/nievergeltlab/freeze3_gwas) and Zenodo118.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Nievergelt et al.

Page 2

potential causal genes, broadly classified as neurotransmitter and ion channel synaptic modulators 
(e.g., GRIA1, GRM8, CACNA1E), developmental, axon guidance, and transcription factors (e.g., 
FOXP2, EFNA5, DCC), synaptic structure and function genes (e.g., PCLO, NCAM1, PDE4B), 
and endocrine or immune regulators (e.g., ESR1, TRAF3, TANK). Additional top genes influence 
stress, immune, fear, and threat-related processes, previously hypothesized to underlie PTSD 
neurobiology. These findings strengthen our understanding of neurobiological systems relevant to 
PTSD pathophysiology, while also opening new areas for investigation.

Introduction

Posttraumatic stress disorder (PTSD) is characterized by intrusive thoughts, hyperarousal, 
avoidance, and negative alterations in cognitions and mood that can become persistent for 
some individuals after traumatic event exposure. Approximately 5.6% of trauma-exposed 
adults world-wide have PTSD during their lifetimes, and rates are higher in those with high 
levels and certain types of trauma exposure such as combat survivors and assault victims1. 
PTSD is a chronic condition for many, posing a substantial quality-of-life and economic 
burden to individuals and society2.

Substantial advances are being made in the understanding of PTSD biology through 
preclinical studies3, many of which are focused on fear systems in the brain, and some 
of which are being translated to human studies of PTSD4. Human neuroimaging studies 
highlight probable dysfunction in brain fear circuitry that includes deficits in top-down 
modulation of the amygdala by regulatory regions such as the anterior cingulate and 
ventromedial prefrontal cortex5,6. Neuroendocrine studies have identified abnormalities 
in the HPA axis and glucocorticoid-induced gene expression in the development and 
maintenance of PTSD7,8. However, many questions remain about the pathophysiology of 
PTSD, and new targets are needed for prevention and treatment.

While twin and genetic studies demonstrated that risk of developing PTSD conditional on 
trauma exposure is partly driven by genetic factors9,10, the specific characterization of the 
genetic architecture of PTSD is just emerging as very large meta-analyses of genome-wide 
association studies (GWAS) become available. Recent research by our workgroup – the 
Psychiatric Genomic Consortium for PTSD (PGC-PTSD)11,12 and the VA Million Veteran 
Program (MVP)13 – contributed to an increased appreciation for the genetic complexity of 
PTSD as a highly polygenic disorder. Despite sample sizes of over 200,000 individuals, 
these studies identified at most 16 PTSD risk loci, which were not consistent across datasets, 
indicating the necessity of still larger sample sizes. In addition, these studies did not examine 
the X chromosome, which comprises 5% of the human genome, and may be particularly 
important given sex differences in PTSD prevalence.

Furthermore, GWAS to date have had limited power to identify credible treatment 
candidates. PTSD is also known frequently to be comorbid and genetically correlated 
with other mental (e.g., major depressive disorder [MDD]; attention deficit hyperactivity 
disorder)14 and physical health conditions (e.g., cardiovascular disease; obesity)15–17, but 
studies to date are limited in their ability to parse shared and disorder-specific loci and link 
them to underlying biological systems. Importantly, prior GWAS are severely limited in 

Nat Genet. Author manuscript; available in PMC 2024 September 13.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Nievergelt et al.

Page 3

generalizing their findings to non-European ancestries. Recent work on polygenic risk scores 
(PRS) in PTSD shows potential utility of these measures in research16–18, but also, vexingly, 
limited cross-population transferability. Without expansion to other ancestries, there is a risk 
that recent advances in PTSD genetics will result in the widening of research and treatment 
disparities. This inequity is particularly troubling in the US given the disproportionately high 
burden of trauma and PTSD faced by populations of African, Native, and Latin American 
origin19,20.

In the present analysis, we synthesize data from 88 studies to perform a multi-ancestry meta-
analysis of GWAS data from European ancestry (EA) (n = 137,136 cases and 1,085,746 
controls), African ancestry (AA) (n = 11,560 cases and 39,474 controls), and Native 
American ancestry (LAT) (n = 2,064 cases and 4,953 controls) samples, including analyses 
of the X chromosome. We follow-up on GWAS findings to examine global and local 
heritability, infer involvement of brain regions and neuronal systems using transcriptomic 
data, describe shared genetic effects with comorbid conditions, and use multi-omic data to 
prioritize a set of 43 putatively causal genes (Fig. 1). Lastly, we use this information to 
identify potential candidate pathways for future PTSD treatment studies. Together, these 
findings mark significant progress towards discovering the pathophysiology of trauma and 
stress-related disorders and inform future intervention approaches for PTSD and related 
conditions.

Results

Data collection and GWAS

The PGC-PTSD21 Freeze 3 data collection includes 1,307,247 individuals from 88 studies 
(Supplementary Table 1). Data in this freeze were assembled from three primary sources 
(Fig. 1a): PTSD studies based on clinician administered or self-reported instruments 
(Freeze 2.511,12 plus subsequently collected studies), MVP release 3 GWASs utilizing the 
Posttraumatic Stress Disorder Checklist (PCL for DSM-IV)13, and 10 biobank studies with 
electronic health record (EHR)-derived PTSD status. We included 95 GWASs, including EA 
(n = 1,222,882; effective sample size (neff) = 641,533), AA (n = 51,034; neff = 42,804) and 
LAT (n = 7,017; neff = 6,530) participants (Supplementary Table 2).

European ancestry PTSD GWAS

Population, screening, and case ascertainment differences between datasets led to the 
assumption that there would be substantial cross-dataset variation in PTSD genetic 
signal. We investigated this possibility using the software MiXeR22,23. Overall, we 
found no evidence for subset-specific genetic causal variation (see Supplementary Note, 
Supplementary Tables 3 and 4 and Extended Data Fig. 1 for further details). Given the 
similarities of the PTSD subsets, we performed a sample-size weighted fixed-effects meta-
analysis of GWAS. For the EA meta-analysis (137,136 cases and 1,085,746 controls), the 
GC lambda was 1.55, the LDSC24 intercept was 1.0524 (s.e. = 0.0097) (Supplementary 
Table 5), and the attenuation ratio was 0.0729 (s.e. = 0.0134), indicating that 92.7% of the 
observed inflation in test-statistics was due to polygenic signal; thus, artifacts produced only 
minimal inflation.

Nat Genet. Author manuscript; available in PMC 2024 September 13.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Nievergelt et al.

Page 4

The EA meta-analysis identified 81 independent genome-wide significant (GWS) loci, 
including 5 GWS loci on the X chromosome (Extended Data Fig. 2, Supplementary Figs. 
1 and 2, Supplementary Table 6, regional association plots in Supplementary Data 1, forest 
plots in Supplementary Data 2, and Supplementary Note). Relative to recent prior PTSD 
GWAS, 67 loci are novel11–13 (Supplementary Table 7). No region exhibited significant 
effect size heterogeneity (Supplementary Fig. 3).

We next sought to gain insights into whether loci harbor multiple independent variants. 
While FUMA25 annotations reported independent lead SNPs within risk loci based on 
pair-wise LD (Supplementary Table 8), COJO26 analysis of each locus conditional on the 
leading variants suggested that only one locus carried a conditionally independent GWS 
SNP (rs3132388 on chromosome 6, P = 2.86 × 10−9). This locus however, is in the MHC 
region, whose complicated linkage disequilibrium (LD) structure27 may not be accurately 
captured by reference panels.

African and Native American ancestry PTSD GWAS meta-analyses

The AA meta-analysis included 51,034 predominantly admixed individuals (n = 11,560 
cases and 39,474 controls). There was minimal inflation of test statistics, with GC lambda 
= 1.031. No GWS loci were identified (Supplementary Fig. 4). The LAT meta-analysis 
was performed in 7,017 individuals (n = 2,064 cases and 4,953 controls). There was 
minimal inflation of test statistics (GC lambda = 0.993) and no GWS loci were identified 
(Supplementary Fig. 5).

Multi-ancestry GWAS meta-analysis

A multi-ancestry fixed-effects meta-analysis of EA, AA, and LAT GWAS (150,793 cases, 
1,130,197 controls) identified 85 GWS loci. Compared to the EA meta-analysis, 10 loci lost 
GWS, while 14 previously suggestive loci (P < 5 × 10−7) became GWS (Fig. 2). In total, 
the present study identified 95 unique GWS PTSD loci between the EA and multi-ancestry 
meta-analyses (Table 1). Due to the complex local ancestry structure in AA and LAT 
individuals, which complicates LD modeling, we focused subsequent fine-mapping analyses 
(Fig. 1b) on data from the EA GWAS.

Gene-mapping

To link GWS SNPs to relevant protein coding genes, we applied three gene mapping 
approaches implemented in FUMA: positional mapping, expression quantitative trait loci 
(eQTL), and chromatin interaction mapping (Supplementary Table 9). GWS SNPs within 
the 81 EA loci mapped to 415 protein coding genes under at least one mapping strategy. A 
total of 230 genes (55%) were mapped by two or more strategies, and 85 (20%) genes were 
mapped by all three strategies (Supplementary Fig. 6). Notably, some genes were implicated 
across independent risk loci by chromatin interactions/eQTL mapping, including EFNA5, 
GRIA1, FOXP2, MDFIC, WSB2, VSIG10, PEBP1, and C17orf58. Chromatin interaction 
plots are shown in Supplementary Data 3.

Nat Genet. Author manuscript; available in PMC 2024 September 13.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Nievergelt et al.

Page 5

Functional annotation and fine-mapping of risk loci

Functional annotations were used to gain insights into the functional role of SNPs within the 
81 risk loci (Supplementary Table 10): 72 loci contained at least one SNP with Combined 
Annotation Dependent Depletion (CADD)28 scores suggestive of deleteriousness to gene 
function (≥12.37), 43 loci contained GWS SNPs with Regulome DB29 scores likely to affect 
binding, and 23 loci contained at least one SNP in the exon region of a gene.

To narrow the credible window of risk loci and identify potentially causal SNPs, we fine-
mapped loci using PolyFun+SUSIE30, which identified a credible set for 67 loci. Credible 
set window lengths were on average 62% of the original set lengths (Supplementary Table 
11) and contained a median of 23 credible SNPs (range 1–252). Only one contained a 
SNP with posterior inclusion probability > 0.95, a missense SNP in the exon of ANAPC4 
(rs34811474, R[CGA]>Q[CAA]; Supplementary Table 12).

Gene-based, gene-set, and gene-tissue analyses

As an alternative approach to SNP-based association analysis, we tested the joint association 
of markers within genes using a gene-based association analysis in MAGMA31, which is a 
2-stage method that first maps SNPs to genes and then tests whether a gene is significantly 
associated with PTSD. The gene-based analysis identified 175 GWS genes (Supplementary 
Table 13 and Supplementary Fig. 7). Of these, 52 were distinct from the genes implicated 
by the gene-mapping of individual SNPs within GWS loci. These notably include DRD2, 
which has been thoroughly investigated in the context of psychiatric disorders and is a 
significant GWAS locus for multiple psychiatric disorders including schizophrenia32 (see 
Supplementary Note and Supplementary Table 14 for further investigation of conditionally 
independent SNPs within these 52 genes).

MAGMA gene-set analysis of 15,483 pathways and gene ontology (GO) terms from 
MSigDB33 identified 12 significant GO terms. Significant terms were related to the 
development and differentiation of neurons (e.g. go_central_nervous_system_development, 
P = 2.0 × 10−7), the synaptic membrane (e.g. go_postsynaptic_membrane, P = 6.9 × 10−7), 
gene regulation (go_positive_regulation_of_gene_expression, P = 1.0 × 10−6), and nucleic 
acid binding (P = 1.52 × 10−6) (Extended Data Fig. 3 and Supplementary Table 15).

MAGMA gene-tissue analysis of 54 tissue types showed PTSD gene enrichment in the brain 
(most notably in cerebellum, but also cortex, hypothalamus, hippocampus and amygdala) 
and in the pituitary, with enrichment found across all 13 examined brain regions (Extended 
Data Fig. 4). Cell type analysis conducted in midbrain tissue data34 identified GABAergic 
neurons, GABA neuroblasts, and mediolateral neuroblast type 5 cell types as having 
enriched associations above other brain cell types tested (P < 0.05/268) (Extended Data 
Fig. 5). GABAergic neurons remained significant (P = 4.4 × 10−5) after stepwise conditional 
analysis of other significant cell types.

Multi-omic investigation of PTSD

To gain insights into which particular genes in enriched brain tissues were contributing to 
PTSD, we conducted a combination of a transcriptome-wide association study (TWAS)35 

Nat Genet. Author manuscript; available in PMC 2024 September 13.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Nievergelt et al.

Page 6

and summary based Mendelian randomization (SMR) analyses36 using GTEx brain tissue 
data based on the EA GWAS summary data. TWAS identified 25 genes within 9 loci with 
Bonferroni-significantly different genetically regulated expression levels between PTSD 
cases and controls (P < 0.05/14,935 unique genes tested) (Fig. 3a, Supplementary Fig. 8, and 
Supplementary Table 16). SMR identified 26 genes within 4 loci whose expression levels 
were putatively causally associated with PTSD (P < 0.05/9,003 unique genes tested) (Fig. 
3b and Supplementary Table 17). Many of these genes have been previously implicated 
in PTSD37 and other psychiatric disorders (e.g., CACNA1E, CRHR1, FOXP2, MAPT, 
WNT3). Notably, the 3p21.31 (including RBM6, RNF123, MST1R, GMPPB, INKA1), 
6p22.1 (including ZCAN9 and HCG17) and 17q21.31 (including ARHGAP27, ARL17A, 
CRHR1, MAPT, FAM215B, LRRC37A2, PLEKHM1, and SPPL2C) regions contained >10 
putative causal genes each.

Among the GTEx tissues with the most TWAS and SMR signals was the dorsolateral 
prefrontal cortex (dlPFC). To gain insight into cell type resolution, we conducted 
MAGMA for cell-type-specific markers of dlPFC and cell-type-specific SMR. MAGMA 
showed a significant enrichment of dlPFC inhibitory and excitatory neurons, but also of 
oligodendrocytes and oligodendrocyte precursor cells (Supplementary Table 18), while 
the SMR analyses identified cell-type-specific signals for 8 genes (KANSL1, ARL17B, 
LINC02210-CRHR1, LRRC37A2, ENSG00000262633, MAPT, ENSG00000273919, 
PLEKHM1) over 3 loci (6 out of 8 from 17q21.31) and all cell types (P < 0.05/1,885 
unique genes tested) whose expression levels were potentially causally associated with 
PTSD (Supplementary Table 19). The top gene, KANSL1, was significant in all cell types.

Given previously reported associations between blood-based protein levels and PTSD38,39, 
we performed protein quantitative trait loci (pQTL) SMR36 analysis for PTSD using data 
from the UK Biobank Pharma Proteomics Project40 (n = 54,306 samples and n = 1,209 
proteins). We identified 16 genes within 9 loci whose protein levels were significantly 
associated with PTSD (P < 0.05/1,209 and PHEIDI > 0.05) (Fig. 3c and Supplementary Table 
20), including members of the TNF superfamily (e.g., CD40, TNFRSF13C), implicating 
TNF-related immune activation in PTSD.

Gene prioritization

One research objective was to identify the genes with the greatest evidence of being 
responsible for the associations observed at each identified PTSD locus. Following recent 
research methods41, we prioritized genes based on weighted sum of evidence scores taken 
across the functional annotation and post-GWAS analyses (Fig. 1b). Based on the absolute 
and relative scores of genes within risk loci, we ranked genes into Tier 1 (greater likelihood 
of being the causal risk gene) and Tier 2 (prioritized over other GWAS-implicated genes, 
but lower likelihood than Tier 1 of being the causal gene). 75% of loci contained prioritized 
genes (Tier 1 or Tier 2); the remaining loci did not contain any genes over the minimum 
threshold of evidence (score ≥ 4) to suggest prioritization. The prioritized genes for the top 
20% of loci (ranked by locus P-value) are shown in Figure 4. A complete list of scores and 
rankings for all 415 protein coding genes mapped to risk loci is available in Supplementary 
Data 4.

Nat Genet. Author manuscript; available in PMC 2024 September 13.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Nievergelt et al.

Page 7

We performed pathway enrichment analysis of the Tier 1 genes in SynGO. From Tier 1, 11 
genes mapped to the set of SynGO annotated genes (CACNA1E, DCC, EFNA5, GRIA1, 
GRM8, LRFN5, MDGA2, NCAM1, OLFM1, PCLO, and SORCS3). Relative to other 
brain-expressed genes, Tier 1 genes were significantly overrepresented in the synapse (P = 
0.0009, qFDR = 0.003), pre- and post-synapse (P = 0.0086, qFDR = 0.0086 and P = 0.003, 
qFDR = 0.004, respectively), and four subcategories (Extended Data Fig. 6). By contrast, 
there was no significant overrepresentation of genes when we applied this test to the entire 
set of 415 protein coding genes. Other notable Tier 1 genes included PDE4B related to 
synaptic function and TNF-related immune-regulatory genes, including TANK and TRAF3.

Genetic architecture of PTSD

SNP) estimated by LDSC was 0.053 (s.e. = 0.002, P = 6.8 

SNP-based heritability (h2
× 10−156). Whereas previous reports suggested sex-specific differences in PTSD11, no 
significant differences were found (P = 0.13), and rg between male and female subsets 
was high (rg = 0.98, s.e. = 0.05, P = 1.2 × 10−98; Supplementary Table 5). MiXeR estimated 
10,863 (s.e. = 377) influential variants and a discoverability of 7.4 × 10−6 (s.e. = 2.2 × 10−7) 
(Supplementary Table 3), indicating a genetic architecture comparable to other psychiatric 
disorders42.

Partitioned heritability across 28 functional categories identified enrichment in histone 
markers (H3K9ac peaks: 6.3-fold enrichment, s.e. = 1.12, P = 3.11 × 10−6; H3K4me1: 1.5-
fold enrichment, s.e. = 0.14, P = 3.3 × 10−4; Supplementary Table 21), and in evolutionary 
constrained regions across 29 Eutherians (18.37-fold enrichment, s.e. = 1.18, P = 1.29 × 
10−17). This is consistent with findings for multiple other psychiatric disorders, but has not 
been previously identified in PTSD42.

Contextualization of PTSD among psychiatric disorders

We measured the genetic overlap between PTSD and other psychiatric disorders using the 
most recent available datasets32,43–52. We observed moderate to high positive rg between 
PTSD and other psychiatric disorders (Extended Data Fig. 7a). To gain further insights into 
this overlap, we used MiXeR to quantify the genetic overlap in causal variation between 
PTSD and bipolar disorder (BPD), MDD, and schizophrenia (SCZ) (Extended Data Fig. 
7b). The strong majority (79–99%) of the variation influencing PTSD risk also influenced 
these disorders (Extended Data Fig. 7b and Supplementary Tables 22 and 23). Similar to rg, 
PTSD had the highest fraction of concordant effect directions with MDD (among the shared 
variation) (87% concordant, s.e. = 2%), significantly higher than the directional concordance 
with BPD (67%, s.e. = 1%) and SCZ (65%, s.e. = 0.5%).

While our results indicate an overall strong rg between PTSD and MDD (rg = 0.85, s.e. = 
0.008, P < 2 × 10−16), the correlation between PTSD and MDD varied significantly across 
PTSD subsets, with the most homogeneously assessed subset, MVP, showing the lowest 
correlation, and the biobank subset being most strongly associated (Supplementary Table 
24). Further, to evaluate if specific genetic regions differ substantially from genome-wide 
estimates, we used LAVA53 to estimate the local h2
SNP and rg of PTSD and MDD across 
the genome, as partitioned into 2,495 approximately independent regions (Supplementary 

Nat Genet. Author manuscript; available in PMC 2024 September 13.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Nievergelt et al.

Page 8

Table 25). Local h2
SNP was significant (P < 0.05/2,495) for both PTSD and MDD in 141 
regions. Of these, local rg was significant (P < 0.05/141) in 40 regions, all in the positive 
effect direction, where the mean local rg was 0.57 (s.d. = 0.24). In addition, we assessed 
the local rg between PTSD and MDD specifically for the 76 autosomal GWS EA loci 
(Supplementary Table 26). While LAVA identified 20 significantly correlated loci (rg < 6.58 
× 10−4), there was also evidence for PTSD loci lacking evidence for correlation with MDD 
(Supplementary Figs. 9 and 10 showcase 6 selected loci with low and high rg).

Contextualization of PTSD across other phenotype domains

Considering all 1,114 traits with SNP-based heritability z > 6 available from the Pan-UKB54 
analysis, we observed Bonferroni-significant rg of PTSD with 73% of them (Supplementary 
Table 27). Examining the extremes of estimates observed, the top positive rg was with 
sertraline prescription (rg = 0.88, P = 3.25 × 10−20), a medication frequently prescribed for 
PTSD and other internalizing disorders55. Other leading associations included medication 
poisonings (e.g. “Poisoning by psychotropic agents” rg = 0.88, P = 3.92 × 10−20), which 
could support a link with accidental poisonings or self-harm behaviors56,57. Converging 
with epidemiologic studies, there were correlations with gastrointestinal symptoms58 (e.g., 
“Nausea and vomiting” rg = 0.80, P = 2.39 × 10−16), mental health comorbidities59 (e.g., 
“Probable Recurrent major depression (severe)” rg = 0.87, P = 1.18 × 10−18; “Recent 
restlessness” rg = 0.86, P = 4.21 × 10−54), chronic pain60 (multi-site chronic pain rg = 0.63, 
P = 7.5 × 10−301) and reduced longevity61–63 (“Mother’s age at death” rg = −0.51, P = 7.6 × 
10−27).

Drug target and class analysis

We extended MAGMA gene-set analysis to investigate 1,530 gene sets comprising known 
drug targets (Supplementary Table 28). We identified one drug (stanozolol, an anabolic 
steroid) significantly enriched for targets associated with PTSD (P = 1.62 × 10−5). However, 
stanozolol has only two target genes in our analyses (ESR1, JUN), and likely reflects the 
strong association of ESR1 with PTSD in gene-level analyses (P = 8.94 × 10−12).

We further examined whether high-ranking drug targets were enriched for 159 drug classes 
defined by Anatomical Therapeutic Chemical (ATC) codes. We identified two broad 
classes where drugs were significantly enriched for association in drug target analyses 
(Supplementary Table 29). These were opioid drugs (ATC code N02A, P = 2.75 × 10−4), 
and psycholeptics (ATC code N05, P = 3.62 × 10−5), particularly antipsychotics (ATC code 
N05A, P = 3.55 × 10−7). However, sensitivity analyses limited to drugs with 10 or more 
targets identified no significant drug target sets nor drug classes.

Polygenic predictive scoring

We evaluated the predictive accuracy of PRS based on PTSD Freeze 3 in a set of MVP 
holdout samples (Fig. 5). In EA holdouts, risk was significantly different across the range 
of PTSD PRS. For example, individuals in the highest quintile of PTSD PRS had 2.4 times 
the relative risk of PTSD (log relative risk s.e. = 0.032; 95%CI = [2.25, 2.56]; P = 1.16 
× 10−167) than individuals in the lowest quintile. PRS explained 6.6% of the phenotypic 
variation in PTSD (Nagelkerke’s R2 transformed to the liability scale at 15% population 

Nat Genet. Author manuscript; available in PMC 2024 September 13.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Nievergelt et al.

Page 9

and sample prevalence), representing a major improvement over PRS based on Freeze 2. 
In contrast, among AA holdout samples, PRS explained only 0.9% (liability scale) of the 
variation in PTSD, consistent with previous work suggesting that AA PRS based on EA data 
lag behind in prediction64.

Discussion

In the largest PTSD GWAS to date, we analyzed data from over one million individuals 
and identified a total of 95 independent risk loci across analyses, a five-fold increase over 
the most recent PTSD GWAS11–13. Compared to previous PTSD GWAS, we confirmed 14 
out of 24 loci and identified 80 novel PTSD loci. Variant discovery in psychiatric GWAS 
follows a sigmoid curve, rapidly increasing once sample size passes a given threshold. This 
analysis passes that inflection point in PTSD65, thus representing a major milestone in PTSD 
genetics. Moreover, by leveraging complementary research methodologies, our findings 
provide new functional insights and a deeper characterization of the genetic architecture of 
PTSD.

Tissue and cell-type enrichments revealed involvement of cerebellum, in addition to 
other traditionally PTSD-associated brain regions, and interneurons in PTSD risk. 
Structural alterations in the cerebellum are associated with PTSD66, and large postmortem 
transcriptomic studies of PTSD consistently reveal differential expression of interneuron 
markers in prefrontal cortical tissue and amygdala nuclei67–69. We used a combination 
of TWAS and SMR to probe the causal genes operating within the enriched tissues and 
cell types with brain transcriptomic data. The identified signals were concentrated in some 
GWAS loci like 17q21.31 whose inversion region is associated with a range of psychiatric 
phenotypes and linked to changes in brain structure and function. KANSL1, ARL17B, 
LINC02210-CRHR1 (encoding a fusion protein with CRHR1) and LRRC37A2 were the 
top causal genes in both neuronal and non-neuronal cell-types. KANSL1 plays a critical 
role in brain development. Furthermore, the first single cell transcriptomic study of PTSD 
confirmed neuronal, excitatory and inhibitory, alterations in 17q21.31 with top alterations in 
ARL17B, LINC02210-CRHR1 and LRRC37A2, while also emphasizing the involvement of 
immune and glucocorticoid response in neurons70.

Notably, although PTSD risk in epidemiological studies is higher in women than men71, 
here we found no sex differences in heritability. Five loci on the X chromosome associated 
with the disorder. Our finding that the estrogen receptor (ESR1) gene was identified in 
GWAS, as well as observations of differential effects of estrogen levels on a variety PTSD 
symptoms,72,73 suggests the importance of further analyses of ESR1 as a potential mediator 
of observed sex differences.

Our analyses prioritized 43 genes as Tier 1 (likely causal) based on weighted sum 
of evidence scores taken across the functional annotation and post-GWAS analyses. 
These genes can broadly be classified as neurotransmitter and ion channel synaptic 
plasticity modulators (e.g., GRIA1, GRM8, CACNA1E), developmental, axon guidance and 
transcription factors (e.g., FOXP2, EFNA5, DCC), synaptic structure and function genes 
(e.g., PCLO, NCAM1, PDE4B), and endocrine and immune regulators (e.g., ESR1, TRAF3, 

Nat Genet. Author manuscript; available in PMC 2024 September 13.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Nievergelt et al.

Page 10

TANK). Furthermore, many additional genes with known function in related pathways were 
genome-wide significant and met Tier 2 prioritization criteria (e.g., GABBR1, CACNA2D2, 
SLC12A5, CAMKV, SEMA3F, CTNND1, and CD40). Together, these top genes show 
a remarkable convergence with neural network, synaptic plasticity and immune processes 
implicated in psychiatric disease. Furthermore, CRHR170,74, WNT375,76, and FOXP277,78, 
among other genes, are implicated in preclinical and clinical work related to stress, fear 
and threat-processing brain regions thought to underlie the neurobiology of PTSD. These 
findings largely support existing mechanistic hypotheses, and it will be important to examine 
how these genes and pathways function in already identified stress-related neural circuits 
and biological systems. Furthermore, while some of the prioritized genes are largely within 
pathways currently indicated in PTSD, many of the specific genes and encoded proteins 
were not previously established and warrant further investigation. Additionally, many genes 
and noncoding RNAs were not previously identified in any psychiatric or stress-related 
disorder, and offer an important road map for determining next steps in understanding 
new mechanisms of vulnerability for posttraumatic psychopathology. Future mechanistic 
research in preclinical models should examine whether targeting combinations of these 
genes, for example via polygenic targeting, epigenetic, or knockdown approaches, would 
have increased power in regulating stress, fear, cognitive dysfunction or other symptoms and 
behaviors seen in PTSD.

We observed highly shared polygenicity between PTSD and other psychiatric disorders, 
albeit with effect discordance across the shared variation. In particular, in some cases we 
found that the genetic correlation of PTSD with MDD is as high or higher than genetic 
correlations between different cohorts, with different measures, of PTSD. Thus, our findings 
corroborate the hypothesis that psychiatric disorders share a substantial amount of risk 
variation but are differentiated by disorder-specific effect sizes43. Across the disorders we 
assessed, the correlation between PTSD and MDD was highest, in agreement with existing 
genetic multi-factor models of psychopathology that consistently cluster these disorders 
together42,79 and concordant with their epidemiologic co-morbidity80. Evaluation of local 
patterns of heritability and genetic correlation however indicates disorder-specific risk 
variation, which will serve as targets for follow-up in cross-disorder investigations. We 
note that as GWAS of psychiatric traits grow in size and power, the field is seeing relatively 
strong genetic correlations among these traits, as well as with other behavioral and medical 
traits. This likely reflects, in part, the reality that there is substantial shared genetic variance 
among these traits, while not excluding the consistent observations that: (1) these traits 
do vary considerably in the magnitude of their genetic correlations, and (2) local genetic 
correlations reveal even greater genetic heterogeneity among these traits than global genetic 
correlations alone would lead us to believe. Finally, while PTSD is the most well-understood 
psychiatric outcome of trauma exposure, it is well documented that trauma is a risk factor 
for many different psychiatric disorders, with perhaps depression as the highest risk. Thus, 
these shared areas of overlap may represent general trauma vulnerability as well.

Despite the high level of overall correlation between PTSD and depression, we also note 
certain areas of clear distinction. When we examined local genetic correlations between 
PTSD and depression within all significant loci from the EA PTSD GWAS, we found that 
there were some regions with significant local heritability for PTSD but not depression, 

Nat Genet. Author manuscript; available in PMC 2024 September 13.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Nievergelt et al.

Page 11

suggestive of PTSD-specific signals. In contrast, we also find other regions with clear shared 
signals showing local correlation across depression and PTSD, indicating that we have the 
power to detect shared and distinct local heritability. Together these findings suggest several 
PTSD-specific loci worthy of further investigation.

Further identification of PTSD genetic loci will provide therapeutic insights81. We explored 
whether genes targeted by specific drugs (and drug classes) were enriched for GWAS 
signal. These analyses provided tentative support for antipsychotics and opioid drugs – 
known psychiatric drug classes – and were driven by gene-wise associations with DRD2 
(antipsychotics) and CYP2D6 (opioids). Atypical antipsychotics may have efficacy in 
treating severe PTSD, but otherwise their use is not supported82. Similarly, whereas some 
observational studies find that chronic opioid use worsens PTSD outcomes83, there is 
preclinical work motivating the further study of opioid subtype-specific targeting (e.g., 
partial MOR1 agonism, κ-type opioid receptor [KOR1] antagonism) in the treatment of 
comorbid PTSD and opioid use disorders84. Analyses in better-powered datasets may 
identify drug repositioning opportunities and could use the predicted effect of associated 
variants on gene expression to indicate whether drug candidates would be beneficial or 
contraindicated in people with PTSD.

In summary, we report 81 loci associated with PTSD in EA meta-analysis and 85 loci when 
expanding to cross-ancestry analyses. While these results represent a milestone in PTSD 
genetics and point to exciting potential target genes, further investment into data collection 
from underrepresented populations of diverse ancestries is needed for identification of 
additional risk variants and to generate equitable and more robust PRS.

Methods

Participants and studies

EHR studies

PTSD assessment and DNA collection for GWAS analysis were performed by each 
study following their protocols. A description of the studies included and the phenotypic 
and genotyping methods for each study are provided in the Supplementary Note and 
Supplementary Table 1. We complied with relevant ethical regulations for human research. 
All participants provided written informed consent, and studies were approved by the 
relevant institutional review boards and the UCSD IRB (protocol #16097×).

A total of 10 EHR-based cohorts (not including the MVP, which also contributed data) 
provided GWAS summary statistics. These cohorts consisted of four US-based sites 
(Vanderbilt University Medical Center’s BioVu, the Mass General Brigham Biobank, Mount 
Sinai’s BioMe, and Mayo Clinic’s MayoGC) and six non-US sites (iPSYCH from Denmark, 
FinnGen, HUNT Study from Norway, STR-STAGE from Sweden, UK Biobank, and Estonia 
Biobank). More details on procedures at each site are provided in the Supplementary Note. 
At each site, a broad definition of PTSD cases was defined based on patients having at least 
1 PTSD or other stress disorder code (see Supplementary Note for the list of corresponding 
ICD-9 and 10 codes). All other patients without such a code were defined as controls. From 

Nat Genet. Author manuscript; available in PMC 2024 September 13.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Nievergelt et al.

Page 12

a total of 817,181 participants across all cohorts, this case definition resulted in 78,687 cases 
based on the broad definition (9.6%).

Data assimilation

Subjects were genotyped on Illumina (n = 84 studies) or Affymetrix genotyping arrays (n = 
5 studies) (Supplementary Table 1). Studies that provided direct access to pre-quality control 
genotype data (n = 64 studies) were deposited on the LISA server for central processing and 
analysis by the PGC-PTSD analyst. Studies with data sharing restrictions (n = 24 studies) 
were processed and analyzed following their own site-specific protocols (Supplementary 
Table 28) and shared GWAS summary statistics for inclusion in meta-analysis.

Genotype quality control and imputation

Genotype data was processed separately by study. For genotype data processed by the PGC-
PTSD analyst, quality control was performed using a uniform set of criteria, as implemented 
in the RICOPILI85 pipeline version 2019_Oct_15.001. Modifications were made to the 
pipeline to allow for ancestrally diverse data and are noted where applicable. Quality 
control: using SNPs with call rates >95%, samples were excluded with call rates <98%, 
deviation from expected inbreeding coefficient (fhet < −0.2 or >0.2), or a sex discrepancy 
between reported and estimated sex based on inbreeding coefficients calculated from SNPs 
on X chromosomes. SNPs were excluded for call rates <98%, a > 2% difference in missing 
genotypes between cases and controls, or being monomorphic. Hardy-Weinberg equilibrium 
was calculated within only in the largest homogenous ancestry group found in the data. 
SNPs with a Hardy-Weinberg equilibrium P-value < 1 × 10−6 in controls were excluded.

After quality control, datasets were lifted over to the GRCh37/hg19 human genome 
reference build. SNP name inconsistencies were corrected, and genotypes were aligned 
to the strand of the imputation reference panel. Markers with non-matching allele codes 
or with excessive MAF difference (> 0.15) with the selected corresponding population 
in the reference data were removed. The pipeline was modified so that only the largest 
homogenous ancestry group in the data was used for the calculation of MAF. For ambiguous 
markers, strand was matched by comparing allele frequencies: if a strand flip resulted in 
a lower MAF difference between the study and the reference data, the strand was flipped. 
Ambiguous markers with high MAF (> 0.4) were removed. The genome was broken into 
132 approximately equally sized chunks. For each chunk, genotypes were phased using 
Eagle v2.3.5 and phased genotypes were imputed into the Haplotype Reference Consortium 
panel86 using minimac3. Imputed datasets were deposited with the PGC DAC and are 
available for approved requests.

Studies with data sharing restrictions followed similar criteria for quality control, as detailed 
in Supplementary Table 28 and in the references in the supplemental material. Studies 
were imputed to either the 1000G phase 3, HRC, SISu panel, or a composite panel. 
GWAS summary data were lifted to the GRCh37 reference build where required. As 
differences in the imputation panels and genome reference build can result in SNP-level 
discrepancies between datasets, each set of summary data was examined for correspondence 
to the centrally imputed data. Multi-allelic SNPs and SNPs with non-matching allele 

Nat Genet. Author manuscript; available in PMC 2024 September 13.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Nievergelt et al.

Page 13

codes were excluded. Strand ambiguous SNPs with high MAF difference (>20%) from the 
average frequency calculated in the PGC-PTSD data were flagged and examined for strand 
correspondence.

Ancestry determination

GWAS

For studies where the PGC analyst had genotype data access, ancestry was determined 
using a global reference panel11 using SNPweights87. The ancestry pipeline was shared 
with external sites to be utilized where possible. Participants were placed into three large 
groupings: European and European Americans (EA; individuals with ≥90% European 
ancestry), African and African-Americans (AA; individuals with ≥5% African ancestry, 
<90% European ancestry, <5% East Asian, Native American, Oceanian, and Central-South 
Asian ancestry; and individuals with ≥50% African ancestry, <5% Native American, 
Oceanian, and <1% Asian ancestry), and Latin Americans (LAT; individuals with ≥5% 
Native American ancestry, <90% European, <5% African, East Asian, Oceanian, and 
Central-South Asian ancestry). Native Americans (individuals with ≥60% Native American 
ancestry, <20% East Asian, <15% Central-South Asian, and <5% African and Oceanian 
ancestry) were grouped together with LAT. All other individuals were excluded from the 
current analyses. For the MVP cohort, ancestry was determined using standard principal 
components analysis approach where MVP samples were projected onto a PC space made 
from 1000 Genomes Phase 3 (KGP3) samples with known population origins (EUR, AFR, 
EAS, SAS, and AMR populations). EHR cohorts followed their own site-specific ancestry 
classification protocols.

GWAS was performed with stratification by ancestry group and study. Strata were 
only analyzed if they had a minimum of 50 cases and 50 controls, or alternatively 
200 participants total. Where noted (Supplementary Table 2), small studies of similar 
composition were jointly genotyped so they could be analyzed together as a single unit. For 
GWAS, the association between each SNP and PTSD was tested under an additive genetic 
model, using a regression model appropriate to the data structure. The statistical model, 
covariates, and analysis software used to analyze each study is detailed in Supplementary 
Table 30. In brief, studies of unrelated individuals with continuous (case/control) measures 
of PTSD were analyzed using PLINK 1.988 using a linear (logistic) regression model 
that included 5 PCs as covariates. For studies that retained related individuals, analyses 
were performed using methods that account for relatedness. QIMR was analyzed using 
GEMMA89 v0.96, including the first five PCs as covariates. RCOG was analyzed using 
the generalized disequilibrium test90. UKBB was analyzed using BOLT-LMM91 including 
6 PCs, and batch and center indicator variables as covariates. VETS was analyzed using 
BOLT-LMM including 5 PCs as covariates. EHR-based studies that included related 
individuals were analyzed using saddle point approximation methods to account for case/
control imbalances. AGDS and QIM2 were analyzed using SAIGE92 including 4 PCs 
and study-specific covariates. BIOV was analyzed using SAIGE including 10 PCs and 
age of record. ESBB, FING, HUNT, and SWED were analyzed using SAIGE including 
5 PCs. UKB2 was analyzed using REGENIE93 including 6 PCs, assessment center, and 

Nat Genet. Author manuscript; available in PMC 2024 September 13.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Nievergelt et al.

Page 14

genotyping batch covariates. GWAS was additionally performed stratified by sex. For the X 
chromosome analysis, sex was added as a covariate.

Meta-analysis

Sample-size weighted fixed-effects meta-analysis was performed with METAL94. Within 
each dataset and ancestry group, summary statistics were filtered to MAF ≥ 1% and 
imputation information score ≥ 0.6. Meta-analyses were performed within the EA, AA, and 
LAT ancestry groups. A multi-ancestry meta-analysis was performed as the meta-analysis 
of the three meta-analyses. Genome-wide significance was declared at P < 5 × 10−8. 
Heterogeneity between datasets was tested with the Cochran test. Markers with summary 
statistics in less than 80% of the total effective sample size were removed from meta-
analyses. LDSC24 intercept was used to estimate inflation of test statistics related to artifacts 
rather than genetic signal. The proportion of inflation of test statistics due to the actual 
polygenic signal (rather than other causes such as population stratification) was estimated as 
1 − (LDSC intercept − 1)/(mean observed Chi-square − 1).

Regional association plots

Regional association plots were generated using LocusZoom95 with 1.5-Mb windows 
around the index variant (unless the locus region was wider than 1.5 Mb, in which case it 
was the locus region plotted plus an additional buffer to include data up to the recombination 
region). The LD patterns plotted were based on the 1000 Genomes Phase 3 reference data96, 
where a sample ancestry appropriate subpopulation (EUR, AFR, or AMR) was used.

Conditional analysis of significant loci

To determine if there were independent significant SNPs within risk loci, GCTA Conditional 
and Joint Analysis26 was performed. Stepwise selection was performed using the --cojo-slct 
option and default parameters, where UKBB European genotype data was used to model LD 
structure.

SNP heritability

h2
SNP of PTSD was estimated using LDSC. LD scores calculated within KGP3 European 
populations (https://data.broadinstitute.org/alkesgroup/LDSCORE/) were used for the input. 
Analyses were limited to HapMap 3 SNPs, with the MHC region excluded (chr6: 26–
34 million base pairs). SNP-based heritability was also calculated as partitioned across 
28 functional annotation categories (https://data.broadinstitute.org/alkesgroup/LDSCORE/) 
using stratified LDSC97.

Comparisons of genetic architecture

We used univariate MiXeR (version 1.3)22,23 to contrast the genetic architecture of 
phenotypes. MiXeR estimates SNP-based heritability and two components that are 
proportional to heritability: the proportion of non-null SNPs (polygenicity), and the variance 
of effect sizes of non-null SNPs (discoverability). MiXeR was applied to GWAS summary 
statistics under the default settings with the supplied European ancestry LD reference panel. 
The results reported for the number of influential variants reflects the number of SNPs 

Nat Genet. Author manuscript; available in PMC 2024 September 13.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Nievergelt et al.

Page 15

necessary to explain 90% of SNP-based heritability. Bivariate MiXeR was used to estimate 
phenotype-specific polygenicity and the shared polygenicity between phenotypes. Goodness 
of fit of the MiXeR model relative to simpler models of polygenic overlap was assessed 
using AIC values. Heritability, polygenicity and discoverability estimates were contrasted 
between datasets using the z-test.

Local genetic correlation analyses

SNP and rg between PTSD and MDD50 were estimated using LAVA53. KGP3 

Local h2
European data were used as the LD reference. Local h2
the genome, as partitioned into 2,495 approximately equally sized LD blocks. Local rg was 
only evaluated for loci where local heritability was significant (P < 0.05/2,495) in both 
phenotypes. Significance of local rg was based on Bonferroni adjustment for the number of 
rg evaluated.

SNP and rg were evaluated across 

Polygenic risk scores (PRS)

PRS were calculated in ancestry-stratified MVP holdout samples, based on the EA Freeze 
3 PTSD GWAS. GWAS summary statistics were filtered to common (MAF > 1%), well-
imputed variants (INFO > 0.8). Indels and ambiguous SNPs were removed. PRS-CS98 was 
used to infer posterior effect sizes of SNPs, using the KGP3 EUR based LD reference 
panel supplied with the program, with the global shrinkage parameter set to 0.01, 1,000 
MCMC iterations with 500 burn-in iterations, and the Markov chain thinning factor set to 5. 
PRS were calculated using the --score option in PLINK 1.9, using the best-guess genotype 
data of target samples, where for each SNP the risk score was estimated as the posterior 
effect size multiplied by the number of copies of the risk allele. PRS was estimated as the 
sum of risk scores over all SNPs. PRS were used to predict PTSD status under logistic 
regression, adjusting for 5 PCs. The proportion of variance explained by PRS for each study 
was estimated as the difference in Nagelkerke’s R2 between a model containing PRS plus 
covariates and a model with only covariates.

Functional mapping and annotation

We used the SNP2GENE module in FUMA25 v1.4.1 (https://fuma.ctglab.nl) to annotate 
and visualize GWAS results. The complete set of parameters used for FUMA analysis are 
shown in the Supplementary Note. Independent genomic risk loci were identified (r2 < 0.6, 
calculated using ancestry-appropriate KGP3 reference genotypes). SNPs within risk loci 
were mapped to protein coding genes using positional mapping (10-kb window), eQTL 
mapping (GTEx v8 brain tissue99, BRAINEAC100, and CommonMind101 data sources), 
and chromatin interaction mapping (PsychENCODE102 and HiC103,104 of brain tissue 
types) methods. Chromatin interactions and eQTLs were plotted in circos plots. SNPs 
were annotated to functional annotation databases including ANNOVAR105, CADD28, and 
RegulomeDB29.

Nat Genet. Author manuscript; available in PMC 2024 September 13.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Nievergelt et al.

Novelty of risk loci

Page 16

The start and stop positions of independent risk loci were assessed for positional overlap 
with existing PTSD loci11–13. Loci were declared novel if their boundaries did not overlap 
with a variant reported significant in prior GWAS.

MAGMA gene-based and gene-set analyses

Gene-based association analyses were conducted using MAGMA31 v1.08. SNPs were 
positionally mapped (0-kb window) to 19,106 protein-coding genes. The SNP-wide mean 
model was used to derive gene-level P-values, with an ancestry appropriate KGP3 reference 
panel was used to model LD. Significance was declared based on Bonferroni adjustment 
for the number of genes tested. Gene-based association statistics were used in MAGMA 
for gene-set and gene-property analyses. Gene-set analysis used the MsigDB33 version 
7.0 including 15,483 curated gene-sets and gene-ontology (GO) terms. Gene-property 
analysis of tissues and tissue subtypes was performed using GTEx v8 expression data, 
with adjustment for the average expression of all tissues in the dataset. To evaluate cell type 
specific enrichment, the FUMA cell type module was used, selecting 12 datasets related 
to the brain (full list in Supplementary Note). Finally, MAGMA was used to estimate the 
enrichment of dlPFC cell types in PTSD risk based on the DER21 marker gene list from 
PsychEncode Consortium Phase 1 resource release102.

GWAS fine-mapping

Polygenic functionally informed fine-mapping (PolyFun)30 software was used to annotate 
our results data with per-SNP heritabilities, as derived from a meta-analysis of 15 UK 
Biobank traits. PTSD risk loci were fine-mapped using SUSIE106, with these per SNP 
heritabilities used as priors, pre-computed UKB-based summary LD information used as the 
LD reference, and locus start and end positions as determined by FUMA. The SUSIE model 
assumed a maximum of two causal variants.

Expression quantitative trait loci (eQTL) and blood protein quantitative trait loci (pQTL) 
analyses

To test for a joint association between GWAS summary statistics SNPs and eQTL, the 
SMR method36, a Mendelian randomization approach, was used. SMR software (version 
1.03) was run using the default settings. The European samples of the 1000G were 
used as a reference panel. Bonferroni multiple-testing correction was applied on SMR 
P-value (PSMR). Moreover, a post-filtering step was applied by conducting heterogeneity 
in dependent instruments (HEIDI) test. The HEIDI test distinguishes the causality and 
pleiotropy models from the linkage model by considering the pattern of associations using 
all SNPs significantly associated with gene expression in the cis-eQTL region. The null 
hypothesis is that a single variant is associated with both trait and gene expression, while 
the alternative hypothesis is that trait and gene expression are associated with two distinct 
variants. Finally, gene-trait associations based on SMR-HEIDΙ were defined as the ones 
for which PSMR met the Bonferroni significance threshold and had PHEIDI > 0.05. We 
conducted a combination of SMR and HEIDI based on GTEx project latest (version 8) 
multi-tissue cis-eQTL databases99 from 13 brain regions and pituitary tissue that showed 

Nat Genet. Author manuscript; available in PMC 2024 September 13.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Nievergelt et al.

Page 17

significant enrichment in MAGMA/FUMA analyses (see above). We also used cell-type-
specific eQTLs in dlPFC for SMR analyses107. Finally, we used a blood UK Biobank 
pQTLs database of 1,463 plasma proteins40 relying on a very large population (54,306) for 
SMR/HEIDI analysis to evaluate biomarker potential.

Brain focused TWAS

JEPEGMIX2-P108 software with default settings was used to conduct TWAS on 13 brain 
regions and pituitary tissue that showed significant enrichment in MAGMA/FUMA analyses 
using our PEC-DLPFC GReX model. JEPEGMIX2-P was applied on GWAS summary 
statistics to estimate gene-trait associations. This method was preferable since it relied on 
a covariance matrix based on 33k samples compared to other TWAS methods which use 
less than 3k samples109. To determine significance, a Bonferroni correction threshold for 
the unique number of genes tested was applied (P < 0.05/14,935). As a less conservative 
approach, we also applied FDR at a q-value threshold of 0.05.

Gene prioritization

Genes within risk loci were prioritized following the general approach previously 
described41. Genes were given prioritization scores based on the weighted sum of evidence 
across all evidence categories: FUMA positional, eQTL, and CI mapping, variant and gene 
annotation scores (CADD, predicted loss of impact [pLI], and RDB scores), positional 
overlap in fine-mapping, significance in gene-based analyses, brain tissue TWAS, eQTL 
SMR, and pQTL SMR. Weights for each evidence category are provided in Supplementary 
Table 31. Within a given locus, the evidence scores were compared across genes to identify 
the most likely causal gene. Genes with scores ≥ 4 were ranked as either Tier 1 (greater 
likelihood of being the causal risk gene) or Tier 2 (lower likelihood of being the causal risk 
gene) and genes with scores < 4 were left unranked. The ranking algorithm is as follows. For 
a given locus, if there was a gene whose evidence score ≥ 4 and this gene’s score was > 20% 
higher than all other genes in the locus, it was ranked as a Tier 1 gene (greater likelihood 
of being the causal risk gene). Within a locus with a Tier 1 gene, other genes with scores 
between 20% and 50% lower than the Tier 1 gene were labeled as Tier 2. For loci without a 
Tier 1 gene, all genes with scores ≥ 4 that were within 50% of the leading gene were ranked 
as Tier 2.

SynGO

PTSD-related genes were tested for overrepresentation among genes related to synaptic 
terms in the SynGO110 web interface (https://www.syngoportal.org/). Brain expressed genes 
were selected as the background list for the overrepresentation tests. SynGO terms with FDR 
q < 0.05 were considered as being overrepresented.

Drug targeting analyses

Following a previously described approach111, we analyzed the enrichment of gene-level 
associations with PTSD in genes targeted by individual drugs. We then examined the 
enrichment of specific drug classes among these drug target associations. We obtained 
gene-level associations using MAGMA31 v1.08. Variant-level associations were converted 

Nat Genet. Author manuscript; available in PMC 2024 September 13.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Nievergelt et al.

Page 18

to gene-level associations using the “multi=snp-wise” model, which aggregates Z scores 
derived from the lowest and the mean variant-level P value within the gene boundary. We 
set gene boundaries 35 kb upstream and 10 kb downstream of the transcribed regions from 
build 37 reference data (National Center for Biotechnology Information, available at https://
ctg.cncr.nl/software/magma).

We performed drug target analysis using competitive gene-set tests implemented in 
MAGMA. Drug target sets were defined as the targets of each drug from: the Drug–Gene 
Interaction database DGIdb v.4.2.0112, the Psychoactive Drug Screening Database Ki DB113, 
ChEMBL v27114, the Target Central Resource Database v6.7.0115, and DSigDB v1.0116, all 
downloaded in October 2020. We additionally used the drug target sets to identify targets of 
drugs of interest from gene-based analyses.

We grouped drugs according to the Anatomical Therapeutic Chemical class of the drug111. 
Results from the drug target analysis were ranked, and the enrichment of each class in the 
drug target analysis was assessed with enrichment curves. We calculated the area under 
the enrichment curve and compared the ranks of drugs within the class to those outside 
the class using the Wilcoxon Mann-Whitney test. Multiple testing was controlled using a 
Bonferroni-corrected significance threshold of P < 3.27 × 10−5 for drug target analysis and 
P < 4.42 × 10−4 for drug class analysis, accounting for 1,530 drug sets and 113 drug classes 
tested.

We initially limited drug target analyses to drugs with two or more targets. However, 
results suggested this low limit may lead to false positive findings. As a sensitivity analysis, 
we further limited these analyses to drugs with 10 or more targets. Multiple testing was 
controlled using a Bonferroni-corrected significance threshold of P < 5.42 × 10−5 for drug 
target analysis and P < 7.94 × 10−4 for drug class analysis, accounting for 923 drug sets and 
63 drug classes tested.

Genetic correlations and causal associations with other phenotypes

Using LDSC, we assessed the rg of PTSD derived from the PGC meta-analysis conducted 
in EUR cohorts with traits available from the Pan-UKB analysis conducted in EUR samples. 
Details regarding the Pan-UKB analysis are available at https://pan.ukbb.broadinstitute.org/. 
Briefly, Pan-UKB genome-wide association statistics were generated using the SAIGE 
and including a kinship matrix as a random effect and covariates as fixed effects. The 
covariates included age, sex, age × sex, age2, age2 × sex, and the top-10 within-ancestry 
principal components. We limited our analysis to data derived from UKB participants of 
European descent (n = 420,531) because of the limited sample size available in the other 
ancestry groups. Initially, we calculated SNP-based heritability of phenotypes available from 
Pan-UKB, retaining only those with SNP-based heritability z > 6 (Supplementary Table 25) 
as recommended by the developers of LDSC117. To define traits genetically correlated with 
PTSD, we applied a Bonferroni correction accounting for the number of tests performed.

Nat Genet. Author manuscript; available in PMC 2024 September 13.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Nievergelt et al.

Extended Data

Page 19

Extended data Figure 1: Comparison of the genetic architecture of PTSD in the three main data 
sources.
Quantification of polygenicity and polygenic overlap in the three main data subsets based 
on (1) symptom scores in clinical studies and cohorts assessed on a variety of instruments 
in Freeze 2.5 (yellow; 26,080 cases and 192,966 controls), (2) PCL (for DSM-IV) based 
symptom scores in the MVP (red; 32,372 cases and 154,317 controls), and (3) ICD9/10 
codes in EHR studies (blue; 78,684 cases and 738,463 controls) indicate a similar genetic 
architecture. The circles on the top half of the plot depict univariate MiXeR estimates of the 
total polygenicity for each data subset. Numbers within circles indicate polygenicity values, 
expressed as the number of variants (in thousands, with SE in parenthesis) necessary to 
explain 90% of SNP based heritability (h2
SNP). h2
SNP estimates are written in the boxes at 
the bottom of the circles. The Euler diagrams on the bottom half of the plot depict bivariate 
MiXeR estimates of the polygenic overlap between data subsets. Values in the overlapping 
part of the Euler diagrams denote shared polygenicity and values on the non-overlapping 
parts note dataset-specific polygenicity. Genetic correlations (rg) between dataset pairs are 
noted in the boxes below the Euler diagrams. Arrowed lines are drawn between univariate 
and bivariate results to indicate which dataset pairs are being evaluated. Abbreviations: Neff, 
effective sample size.

Nat Genet. Author manuscript; available in PMC 2024 September 13.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Nievergelt et al.

Page 20

Extended data Figure 2: Manhattan plot of the PTSD GWAS meta-analysis in individuals of 
European ancestry (EA).
Results of the EA GWAS meta-analysis (137,136 PTSD cases, 1,085,746 controls) 
identifying 81 genome-wide significant PTSD loci. The y axis refers to the −log10 p-value 
from a meta-analysis using a sample size weighted fixed-effects model. Circle colors 
alternate between chromosomes: even chromosomes are colored blue and odd chromosomes 
are colored black. The horizontal red bar indicates genome-wide significant associations (p 
< 5×10−8).

Extended Data Figure 3: Significant PTSD gene-sets.

Nat Genet. Author manuscript; available in PMC 2024 September 13.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Nievergelt et al.

Page 21

MAGMA gene-set analysis using the Molecular Signatures database (MSigDB) identifies 
11 significant gene-sets. The dotted line indicates significance adjusted for the number 
of comparisons (p < 0.05/15,483 gene-sets). Bars depict -log10 p-values. Corresponding 
gene-set names are indicated to the left of bars. Terms are clustered and colored according to 
their Gene Ontology term category (biological processes, yellow; molecular function, blue; 
cellular component, red).

Extended Data Figure 4: MAGMA tissue enrichment analysis.
MAGMA gene-property analysis in 53 specific tissue types from GTEx v8 shows 
enrichment of PTSD-related genes in 13 brain tissue types and in the pituitary. Bars depict 
-log10 p-values. Corresponding tissue names are indicated below bars. The dotted horizontal 
line indicates statistical significance adjusted for the number of comparisons (p < 0.05/53). 
Significant tissues are colored red.

Nat Genet. Author manuscript; available in PMC 2024 September 13.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Nievergelt et al.

Page 22

Extended Data Figure 5: MAGMA cell-type enrichment analysis in midbrain.
MAGMA gene-property analysis of 25 midbrain cell types (GSE76381) indicate enrichment 
of GABAergic neurons, GABAergic neuroblasts, and mediolateral neuroblasts. Vertical bars 
depict -log10 p-values. Significant cell types are colored blue and grey if not. The dotted 
horizontal line indicates statistical significance adjusted for the number of comparisons 
(p < 0.05/25). The asterisk (*) indicates that GABAergic neurons remained significant 
in stepwise conditional analysis of the other significant cell types. Abbreviations: Gaba - 
GABAergic neurons; NbGaba - neuroblast gabaergic; NbML1–5, mediolateral neuroblasts; 
DA0–2 - dopaminergicneurons; Sert, serotonergic; RN, red nucleus; Rgl 1–3, radial glia-like 
cells; NbM, medial neuroblast; OPC, oligodendrocyte precursor cells. ProgFPL - progenitor 
lateral floorplate; OMTN - oculomotor and trochlear nucleus; Endo, endothelial cells; 
ProgM, progenitor midline;NProg, neuronal progenitor; ProgBP, progenitorbasal plate; Mgl, 
microglia; ProgFPM, progenitor medial floorplate; Peric – pericytes.

Nat Genet. Author manuscript; available in PMC 2024 September 13.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Nievergelt et al.

Page 23

Extended Data Figure 6: PTSD genes in SynGO.
Sunburst plots show enrichment of PTSD-related genes in SynGO cellular components. The 
synapse is at the center ring, pre- and post-synaptic locations are at the first rings, and child 
terms are in subsequent outer rings. a, enrichment test results for all 415 genes mapped 
to PTSD GWAS loci by FUMA from one of three gene-mapping strategies (positional, 
expression quantitative trait loci, and chromatin interaction mapping). b, enrichment test 
results for 43 genes prioritized into Tier 1 using a gene prioritization strategy. Plots are 
colored by -log10 Q-value (see color code in the bar at left) from enrichment of PTSD genes 
relative to a brain expressed background set.

Nat Genet. Author manuscript; available in PMC 2024 September 13.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Nievergelt et al.

Page 24

SNP) 

Extended Data Figure 7: Genetic correlations and polygenic overlap between PTSD and other 
psychiatric disorders.
a, Genetic correlations (rg) between PTSD and other psychiatric disorders are indicated by 
circles that are drawn along the x axis. Red dots indicate SNP based heritability (h2
z-score > 6 in the psychiatric disorder GWAS and colored grey to indicate z-score < 6 
(rg estimates may be unreliable). The first author and publication year of source summary 
data is noted in parenthesis following the disorder name. b, Quantification of the polygenic 
overlap between PTSD and other psychiatric disorders. Euler diagrams depict Bivariate 
MiXeR analysis of PTSD (blue circles) and bipolar disorder (BIP), major depression 
(MDD), and schizophrenia (SCZ) (red circles). Values in the overlapping part of the Euler 
diagrams denote shared polygenicity (expressed as the number of influential variants, in 
thousands, with SE in parenthesis), and values in the non-overlapping part indicate dataset-
specific variation. rg between dataset pairs are noted in the boxes below the Euler plots. 
Abbreviations: ADHD, attention deficit hyperactive disorder; alc. dep, alcohol dependence; 
BIP, bipolar disorder; MDD, major depression; OCD, obsessive compulsive disorder; SCZ, 
schizophrenia.

Nat Genet. Author manuscript; available in PMC 2024 September 13.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Nievergelt et al.

Page 25

Supplementary Material

Refer to Web version on PubMed Central for supplementary material.

Authors 

Caroline M. Nievergelt1,2,3,*, Adam X. Maihofer1,2,3,*, Elizabeth G. Atkinson4, 
Chia-Yen Chen5, Karmel W. Choi6,7, Jonathan R. I. Coleman8,9, Nikolaos P. 
Daskalakis10,11,12, Laramie E. Duncan13, Renato Polimanti14,15, Cindy Aaronson16, 
Ananda B. Amstadter17, Soren B. Andersen18, Ole A. Andreassen19,20, Paul A. 
Arbisi21,22, Allison E. Ashley-Koch23, S. Bryn Austin24,25,26, Esmina Avdibegoviç27, 
Dragan Babić28, Silviu-Alin Bacanu29, Dewleen G. Baker1,2,30, Anthony Batzler31, 
Jean C. Beckham32,33,34, Sintia Belangero35,36, Corina Benjet37, Carisa Bergner38, 
Linda M. Bierer39, Joanna M. Biernacka31,40, Laura J. Bierut41, Jonathan I. 
Bisson42, Marco P. Boks43, Elizabeth A. Bolger11,44, Amber Brandolino45, Gerome 
Breen9,46, Rodrigo Affonseca Bressan47,48, Richard A. Bryant49, Angela C. 
Bustamante50, Jonas Bybjerg-Grauholm51,52, Marie Bækvad-Hansen51,52, Anders 
D. Børglum52,53,54, Sigrid Børte55,56, Leah Cahn16, Joseph R. Calabrese57,58, 
Jose Miguel Caldas-de-Almeida59, Chris Chatzinakos10,11,60, Sheraz Cheema61, 
Sean A. P. Clouston62,63, Lucía Colodro-Conde64, Brandon J. Coombes31, 
Carlos S. Cruz-Fuentes65, Anders M. Dale66, Shareefa Dalvie67, Lea K. 
Davis68, Jürgen Deckert69, Douglas L. Delahanty70, Michelle F. Dennis32,33,34, 
Frank Desarnaud16, Christopher P. DiPietro10,60, Seth G. Disner71,72, Anna R. 
Docherty73,74, Katharina Domschke75,76, Grete Dyb20,77, Alma Džubur Kulenović78, 
Howard J. Edenberg79,80, Alexandra Evans42, Chiara Fabbri9,81, Negar Fani82, 
Lindsay A. Farrer83,84,85,86,87, Adriana Feder16, Norah C. Feeny88, Janine D. 
Flory16, David Forbes89, Carol E. Franz1, Sandro Galea90, Melanie E. Garrett23, 
Bizu Gelaye6, Joel Gelernter91,92, Elbert Geuze93,94, Charles F. Gillespie82, Slavina 
B. Goleva68,95, Scott D. Gordon64, Aferdita Goçi96, Lana Ruvolo Grasser97, 
Camila Guindalini98, Magali Haas99, Saskia Hagenaars8,9, Michael A. Hauser32, 
Andrew C. Heath100, Sian M. J. Hemmings101,102, Victor Hesselbrock103, Ian B. 
Hickie104, Kelleigh Hogan1,2,3, David Michael Hougaard51,52, Hailiang Huang10,105, 
Laura M. Huckins106, Kristian Hveem55, Miro Jakovljević107, Arash Javanbakht97, 
Gregory D. Jenkins31, Jessica Johnson108, Ian Jones109, Tanja Jovanovic82, Karen-
Inge Karstoft18,110, Milissa L. Kaufman11,44, James L. Kennedy111,112,113,114, 
Ronald C. Kessler115, Alaptagin Khan11,44, Nathan A. Kimbrel32,34,116, Anthony 
P. King117, Nastassja Koen118, Roman Kotov119, Henry R. Kranzler120,121, Kristi 
Krebs122, William S. Kremen1, Pei-Fen Kuan123, Bruce R. Lawford124, Lauren 
A. M. Lebois11,12, Kelli Lehto122, Daniel F. Levey14,15, Catrin Lewis42, Israel 
Liberzon125, Sarah D. Linnstaedt126, Mark W. Logue86,127,128, Adriana Lori82, 
Yi Lu129, Benjamin J. Luft130, Michelle K. Lupton64, Jurjen J. Luykx94,131, Iouri 
Makotkine16, Jessica L. Maples-Keller82, Shelby Marchese132, Charles Marmar133, 
Nicholas G. Martin134, Gabriela A. Martínez-Levy65, Kerrie McAloney64, Alexander 
McFarlane135, Katie A. McLaughlin136, Samuel A. McLean126,137, Sarah E. 
Medland64, Divya Mehta124,138, Jacquelyn Meyers139, Vasiliki Michopoulos82, 
Elizabeth A. Mikita1,2,3, Lili Milani122, William Milberg140, Mark W. Miller127,128, 

Nat Genet. Author manuscript; available in PMC 2024 September 13.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Nievergelt et al.

Page 26

Rajendra A. Morey141, Charles Phillip Morris124, Ole Mors52,142, Preben 
Bo Mortensen52,53,143,144, Mary S. Mufford145, Elliot C. Nelson41, Merete 
Nordentoft52,146, Sonya B. Norman1,2,147, Nicole R. Nugent148,149,150, Meaghan 
O'Donnell151, Holly K. Orcutt152, Pedro M. Pan153, Matthew S. Panizzon1, Gita 
A. Pathak14,15, Edward S. Peters154, Alan L. Peterson155,156, Matthew Peverill157, 
Robert H. Pietrzak15,158, Melissa A. Polusny21,72,159, Bernice Porjesz139, Abigail 
Powers82, Xue-Jun Qin23, Andrew Ratanatharathorn6,160, Victoria B. Risbrough1,2,3, 
Andrea L. Roberts161, Alex O. Rothbaum162,163, Barbara O. Rothbaum82, Peter 
Roy-Byrne164, Kenneth J. Ruggiero165, Ariane Rung166, Heiko Runz167, Bart 
P. F. Rutten168, Stacey Saenz de Viteri169, Giovanni Abrahão Salum170,171, 
Laura Sampson6,87, Sixto E. Sanchez172, Marcos Santoro173, Carina Seah132, 
Soraya Seedat174,175, Julia S. Seng176,177,178,179, Andrey Shabalin74, Christina M. 
Sheerin17, Derrick Silove180, Alicia K. Smith82,181, Jordan W. Smoller7,10,182, Scott 
R. Sponheim21,183, Dan J. Stein118, Synne Stensland56,77, Jennifer S. Stevens82, 
Jennifer A. Sumner184, Martin H. Teicher11,185, Wesley K. Thompson186,187, Arun 
K. Tiwari111,112,113, Edward Trapido166, Monica Uddin188, Robert J. Ursano189, 
Unnur Valdimarsdóttir190,191, Miranda Van Hooff192, Eric Vermetten193,194,195, 
Christiaan H. Vinkers196,197,198, Joanne Voisey124,138, Yunpeng Wang199, Zhewu 
Wang200,201, Monika Waszczuk202, Heike Weber69, Frank R. Wendt203, Thomas 
Werge52,204,205,206, Michelle A. Williams6, Douglas E. Williamson32,33, Bendik S. 
Winsvold55,56,207, Sherry Winternitz11,44, Christiane Wolf69, Erika J. Wolf128,208, 
Yan Xia10,105, Ying Xiong129, Rachel Yehuda16,209, Keith A. Young210,211, 
Ross McD Young212,213, Clement C. Zai10,111,112,113,114,214, Gwyneth C. 
Zai111,112,113,114,215, Mark Zervas99, Hongyu Zhao216, Lori A. Zoellner157, John-
Anker Zwart20,55,56, Terri deRoon-Cassini45, Sanne J. H. van Rooij82, Leigh 
L. van den Heuvel101,102, AURORA Study, Estonian Biobank Research Team, 
FinnGen Investigators, HUNT All-In Psychiatry, Murray B. Stein1,30,217, Kerry J. 
Ressler11,44,82, Karestan C. Koenen6,10,182

Affiliations

1University of California San Diego, Department of Psychiatry, La Jolla, California, 
United States of America.

2Veterans Affairs San Diego Healthcare System, Center of Excellence for Stress 
and Mental Health, San Diego, California, United States of America.

3Veterans Affairs San Diego Healthcare System, Research Service, San Diego, 
California, United States of America.

4Baylor College of Medicine, Department of Molecular and Human Genetics, 
Houston, Texas, United States of America.

5Biogen Inc., Translational Sciences, Cambridge, Massachusetts, United States of 
America.

6Harvard T.H. Chan School of Public Health, Department of Epidemiology, Boston, 
Massachusetts, United States of America.

Nat Genet. Author manuscript; available in PMC 2024 September 13.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Nievergelt et al.

Page 27

7Massachusetts General Hospital, Department of Psychiatry, Boston, 
Massachusetts, United States of America.

8King's College London, National Institute for Health and Care Research Maudsley 
Biomedical Research Centre, South London and Maudsley NHS Foundation Trust, 
London, United Kingdom.

9King's College London, Social, Genetic and Developmental Psychiatry Centre, 
Institute of Psychiatry, Psychology and Neuroscience, London, United Kingdom.

10Broad Institute of MIT and Harvard, Stanley Center for Psychiatric Research, 
Cambridge, Massachusetts, United States of America.

11Harvard Medical School, Department of Psychiatry, Boston, Massachusetts, 
United States of America.

12McLean Hospital, Center of Excellence in Depression and Anxiety Disorders, 
Belmont, Massachusetts, United States of America.

13Stanford University, Department of Psychiatry and Behavioral Sciences, Stanford, 
California, United States of America.

14VA Connecticut Healthcare Center, West Haven, Connecticut, United States of 
America.

15Yale University School of Medicine, Department of Psychiatry, New Haven, 
Connecticut, United States of America.

16Icahn School of Medicine at Mount Sinai, Department of Psychiatry, New York, 
New York, United States of America.

17Virginia Institute for Psychiatric and Behavioral Genetics, Department of 
Psychiatry, Richmond, Virginia, United States of America.

18The Danish Veteran Centre, Research and Knowledge Centre, Ringsted, 
Sjaelland, Denmark.

19Oslo University Hospital, Division of Mental Health and Addiction, Oslo, Norway.

20University of Oslo, Institute of Clinical Medicine, Oslo, Norway.

21Minneapolis VA Health Care System, Mental Health Service Line, Minneapolis, 
Minnesota, United States of America.

22University of Minnesota, Department of Psychiatry, Minneapolis, Minnesota, 
United States of America.

23Duke University, Duke Molecular Physiology Institute, Durham, North Carolina, 
United States of America.

24Boston Children's Hospital, Division of Adolescent and Young Adult Medicine, 
Boston, Massachusetts, United States of America.

25Harvard Medical School, Department of Pediatrics, Boston, Massachusetts, 
United States of America.

Nat Genet. Author manuscript; available in PMC 2024 September 13.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Nievergelt et al.

Page 28

26Harvard T.H. Chan School of Public Health, Department of Social and Behavioral 
Sciences, Boston, Massachusetts, United States of America.

27University Clinical Center of Tuzla, Department of Psychiatry, Tuzla, Bosnia and 
Herzegovina.

28University Clinical Center of Mostar, Department of Psychiatry, Mostar, Bosnia and 
Herzegovina.

29Virginia Commonwealth University, Department of Psychiatry, Richmond, Virginia, 
United States of America.

30Veterans Affairs San Diego Healthcare System, Psychiatry Service, San Diego, 
California, United States of America.

31Mayo Clinic, Department of Quantitative Health Sciences, Rochester, Minnesota, 
United States of America.

32Duke University School of Medicine, Department of Psychiatry and Behavioral 
Sciences, Durham, North Carolina, United States of America.

33Durham VA Health Care System, Research, Durham, North Carolina, United 
States of America.

34VA Mid-Atlantic Mental Illness Research, Education, and Clinical Center 
(MIRECC), Genetics Research Laboratory, Durham, North Carolina, United States 
of America.

35Universidade Federal de São Paulo, Department of Morphology and Genetics, 
São Paulo, São Paulo, Brazil.

36Universidade Federal de São Paulo, Laboratory of Integrative Neuroscience, 
Department of Psychiatry, São Paulo, São Paulo, Brazil.

37Instituto Nacional de Psiquiatraía Ramón de la Fuente Muñiz, Center for Global 
Mental Health, Mexico City, Mexico City, Mexico.

38Medical College of Wisconsin, Comprehensive Injury Center, Milwaukee, 
Wisconsin, United States of America.

39James J. Peters VA Medical Center, Department of Psychiatry, Bronx, New York, 
United States of America.

40Mayo Clinic, Department of Psychiatry and Psychology, Rochester, Minnesota, 
United States of America.

41Washington University in Saint Louis School of Medicine, Department of 
Psychiatry, Saint Louis, Missouri, United States of America.

42Cardiff University, National Centre for Mental Health, MRC Centre for Psychiatric 
Genetics and Genomics, Cardiff, South Glamorgan, United Kingdom.

43Brain Center University Medical Center Utrecht, Department of Psychiatry, 
Utrecht, Utrecht, The Netherlands.

Nat Genet. Author manuscript; available in PMC 2024 September 13.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Nievergelt et al.

Page 29

44McLean Hospital, Belmont, Massachusetts, United States of America.

45Medical College of Wisconsin, Department of Surgery, Division of Trauma & Acute 
Care Surgery, Milwaukee, Wisconsin, United States of America.

46King's College London, NIHR Maudsley BRC, London, United Kingdom.

47Universidade Federal de São Paulo, Department of Psychiatry, São Paulo, São 
Paulo, Brazil.

48Universidade Federal de São Paulo, Laboratory of Integrative Neuroscience, 
Department of Psychiatry, São Paulo, São Paulo, Brazil.

49University of New South Wales, School of Psychology, Sydney, New South Wales, 
Australia.

50University of Michigan Medical School, Division of Pulmonary and Critical Care 
Medicine, Department of Internal Medicine, Ann Arbor, Michigan, United States of 
America.

51Statens Serum Institut, Department for Congenital Disorders, Copenhagen, 
Denmark.

52The Lundbeck Foundation Initiative for Integrative Psychiatric Research, iPSYCH, 
Aarhus, Denmark.

53Aarhus University, Centre for Integrative Sequencing, iSEQ, Aarhus, Denmark.

54Aarhus University, Department of Biomedicine - Human Genetics, Aarhus, 
Denmark.

55Norwegian University of Science and Technology, K. G. Jebsen Center for Genetic 
Epidemiology, Department of Public Health and Nursing, Faculty of Medicine and 
Health Sciences, Trondheim, Norway.

56Oslo University Hospital, Department of Research, Innovation and Education, 
Division of Clinical Neuroscience, Oslo, Norway.

57Case Western Reserve University, School of Medicine, Cleveland, Ohio, United 
States of America.

58University Hospitals, Department of Psychiatry, Cleveland, Ohio, United States of 
America.

59Chronic Diseases Research Centre (CEDOC), Lisbon Institute of Global Mental 
Health, Lisbon, Portugal.

60McLean Hospital, Division of Depression and Anxiety Disorders, Belmont, 
Massachusetts, United States of America.

61University of Toronto, CanPath National Coordinating Center, Toronto, Ontario, 
Canada.

62Stony Brook University, Family, Population, and Preventive Medicine, Stony Brook, 
New York, United States of America.

Nat Genet. Author manuscript; available in PMC 2024 September 13.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Nievergelt et al.

Page 30

63Stony Brook University, Public Health, Stony Brook, New York, United States of 
America.

64QIMR Berghofer Medical Research Institute, Mental Health & Neuroscience 
Program, Brisbane, Queensland, Australia.

65Instituto Nacional de Psiquiatraía Ramón de la Fuente Muñiz, Department of 
Genetics, Mexico City, Mexico City, Mexico.

66University of California San Diego, Department of Radiology, Department of 
Neurosciences, La Jolla, California, United States of America.

67University of Cape Town, Division of Human Genetics, Department of Pathology, 
Cape Town, Western Province, South Africa.

68Vanderbilt University Medical Center, Vanderbilt Genetics Institute, Nashville, 
Tennessee, United States of America.

69University Hospital of Würzburg, Center of Mental Health, Psychiatry, 
Psychosomatics and Psychotherapy, Würzburg, Denmark.

70Kent State University, Department of Psychological Sciences, Kent, Ohio, United 
States of America.

71Minneapolis VA Health Care System, Research Service Line, Minneapolis, 
Minnesota, United States of America.

72University of Minnesota Medical School, Department of Psychiatry & Behavioral 
Sciences, Minneapolis, Minnesota, United States of America.

73Huntsman Mental Health Institute, Salt Lake City, Utah, United States of America.

74University of Utah School of Medicine, Department of Psychiatry, Salt Lake City, 
Utah, United States of America.

75University of Freiburg, Faculty of Medicine, Centre for Basics in Neuromodulation, 
Freiburg, Denmark.

76University of Freiburg, Faculty of Medicine, Department of Psychiatry and 
Psychotherapy, Freiburg, Denmark.

77Norwegian Centre for Violence and Traumatic Stress Studies, Oslo, Norway.

78University Clinical Center of Sarajevo, Department of Psychiatry, Sarajevo, Bosnia 
and Herzegovina.

79Indiana University School of Medicine, Biochemistry and Molecular Biology, 
Indianapolis, Indiana, United States of America.

80Indiana University School of Medicine, Medical and Molecular Genetics, 
Indianapolis, Indiana, United States of America.

81University of Bologna, Department of Biomedical and Neuromotor Sciences, 
Bologna, Italy.

Nat Genet. Author manuscript; available in PMC 2024 September 13.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Nievergelt et al.

Page 31

82Emory University, Department of Psychiatry and Behavioral Sciences, Atlanta, 
Georgia, United States of America.

83Boston University Chobanian & Avedisian School of Medicine, Department of 
Medicine (Biomedical Genetics), Boston, Massachusetts, United States of America.

84Boston University Chobanian & Avedisian School of Medicine, Department of 
Neurology, Boston, Massachusetts, United States of America.

85Boston University Chobanian & Avedisian School of Medicine, Department of 
Ophthalmology, Boston, Massachusetts, United States of America.

86Boston University School of Public Health, Department of Biostatistics, Boston, 
Massachusetts, United States of America.

87Boston University School of Public Health, Department of Epidemiology, Boston, 
Massachusetts, United States of America.

88Case Western Reserve University, Department of Psychological Sciences, 
Cleveland, Ohio, United States of America.

89University of Melbourne, Department of Psychiatry, Melbourne, Victoria, Australia.

90Boston University School of Public Health, Boston, Massachusetts, United States 
of America.

91VA Connecticut Healthcare Center, Psychiatry Service, West Haven, Connecticut, 
United States of America.

92Yale University School of Medicine, Department of Genetics and Neuroscience, 
New Haven, Connecticut, United States of America.

93Netherlands Ministry of Defence, Brain Research and Innovation Centre, Utrecht, 
Utrecht, The Netherlands.

94UMC Utrecht Brain Center Rudolf Magnus, Department of Psychiatry, Utrecht, 
Utrecht, The Netherlands.

95National Institutes of Health, National Human Genome Research Institute, 
Bethesda, Maryland, United States of America.

96University Clinical Centre of Kosovo, Department of Psychiatry, Prishtina, Kosovo.

97Wayne State University School of Medicine, Psychiatry and Behavioral 
Neurosciences, Detroit, Michigan, United States of America.

98Gallipoli Medical Research Foundation, Greenslopes Private Hospital, 
Greenslopes, Queensland, Australia.

99Cohen Veterans Bioscience, New York, New York, United States of America.

100Washington University in Saint Louis School of Medicine, Department of 
Genetics, Saint Louis, Missouri, United States of America.

101Stellenbosch University, Department of Psychiatry, Faculty of Medicine and 
Health Sciences, Cape Town, Western Cape, South Africa.

Nat Genet. Author manuscript; available in PMC 2024 September 13.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Nievergelt et al.

Page 32

102Stellenbosch University, SAMRC Genomics of Brain Disorders Research Unit, 
Cape Town, Western Cape, South Africa.

103University of Connecticut School of Medicine, Psychiatry, Farmington, 
Connecticut, United States of America.

104University of Sydney, Brain and Mind Centre, Sydney, New South Wales, 
Australia.

105Massachusetts General Hospital, Analytic and Translational Genetics Unit, 
Department of Medicine, Boston, Massachusetts, United States of America.

106Yale University, Department of Psychiatry, New Haven, Connecticut, United 
States of America.

107University Hospital Center of Zagreb, Department of Psychiatry, Zagreb, Croatia.

108Icahn School of Medicine at Mount Sinai, Genetics and Genomic Sciences, New 
York, New York, United States of America.

109Cardiff University, National Centre for Mental Health, Cardiff University Centre for 
Psychiatric Genetics and Genomics, Cardiff, South Glamorgan, United Kingdom.

110University of Copenhagen, Department of Psychology, Copenhagen, Denmark.

111Centre for Addiction and Mental Health, Neurogenetics Section, Molecular Brain 
Science Department, Campbell Family Mental Health Research Institute, Toronto, 
Ontario, Canada

112Centre for Addiction and Mental Health, Tanenbaum Centre for 
Pharmacogenetics, Toronto, Ontario, Canada.

113University of Toronto, Department of Psychiatry, Toronto, Ontario, Canada

114University of Toronto, Institute of Medical Sciences, Toronto, Ontario, Canada.

115Harvard Medical School, Department of Health Care Policy, Boston, 
Massachusetts, United States of America.

116Durham VA Health Care System, Mental Health Service Line, Durham, North 
Carolina, United States of America.

117The Ohio State University, College of Medicine, Institute for Behavioral Medicine 
Research, Columbus, Ohio, United States of America.

118University of Cape Town, Department of Psychiatry & Neuroscience Institute, SA 
MRC Unit on Risk & Resilience in Mental Disorders, Cape Town, Western Province, 
South Africa.

119Stony Brook University, Department of Psychiatry, Stony Brook, New York, United 
States of America.

120Mental Illness Research, Education and Clinical Center, Crescenz VAMC, 
Philadelphia, Pennsylvania, United States of America.

Nat Genet. Author manuscript; available in PMC 2024 September 13.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Nievergelt et al.

Page 33

121University of Pennsylvania Perelman School of Medicine, Department of 
Psychiatry, Philadelphia, Pennsylvania, United States of America.

122University of Tartu, Institute of Genomics, Estonian Genome Center, Tartu, 
Estonia.

123Stony Brook University, Department of Applied Mathematics and Statistics, Stony 
Brook, New York, United States of America.

124Queensland University of Technology, School of Biomedical Sciences, Kelvin 
Grove, Queensland, Australia.

125Texas A&M University College of Medicine, Department of Psychiatry and 
Behavioral Sciences, Bryan, Texas, United States of America.

126UNC Institute for Trauma Recovery, Department of Anesthesiology, Chapel Hill, 
North Carolina, United States of America.

127Boston University School of Medicine, Psychiatry, Biomedical Genetics, Boston, 
Massachusetts, United States of America.

128VA Boston Healthcare System, National Center for PTSD, Boston, 
Massachusetts, United States of America.

129Karolinska Institutet, Department of Medical Epidemiology and Biostatistics, 
Stockholm, Sweden.

130Stony Brook University, Department of Medicine, Stony Brook, New York, United 
States of America.

131UMC Utrecht Brain Center Rudolf Magnus, Department of Translational 
Neuroscience, Utrecht, Utrecht, The Netherlands.

132Icahn School of Medicine at Mount Sinai, Department of Genetic and Genomic 
Sciences, New York, New York, United States of America.

133New York University, Grossman School of Medicine, New York, New York, United 
States of America.

134QIMR Berghofer Medical Research Institute, Genetics, Brisbane, Queensland, 
Australia.

135University of Adelaide, Discipline of Psychiatry, Adelaide, South Australia, 
Australia.

136Harvard University, Department of Psychology, Boston, Massachusetts, United 
States of America.

137UNC Institute for Trauma Recovery, Department of Emergency Medicine, Chapel 
Hill, North Carolina, United States of America.

138Queensland University of Technology, Centre for Genomics and Personalised 
Health, Kelvin Grove, Queensland, Australia.

Nat Genet. Author manuscript; available in PMC 2024 September 13.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Nievergelt et al.

Page 34

139SUNY Downstate Health Sciences University, Department of Psychiatry and 
Behavioral Sciences, Brooklyn, New York, United States of America.

140VA Boston Healthcare System, GRECC/TRACTS, Boston, Massachusetts, 
United States of America.

141Duke University School of Medicine, Duke Brain Imaging and Analysis Center, 
Durham, North Carolina, United States of America.

142Aarhus University Hospital - Psychiatry, Psychosis Research Unit, Aarhus, 
Denmark.

143Aarhus University, Centre for Integrated Register-based Research, Aarhus, 
Denmark.

144Aarhus University, National Centre for Register-Based Research, Aarhus, 
Denmark.

145University of Cape Town, Division of Human Genetics, Department of Pathology, 
Cape Town, Western Province, South Africa.

146University of Copenhagen, Mental Health Services in the Capital Region of 
Denmark, Copenhagen, Denmark.

147National Center for Post Traumatic Stress Disorder, Executive Division, White 
River Junction, Vermont, United States of America.

148Alpert Brown Medical School, Department of Emergency Medicine, Providence, 
Rhode Island, United States of America.

149Alpert Brown Medical School, Department of Pediatrics, Providence, Rhode 
Island, United States of America.

150Alpert Brown Medical School, Department of Psychiatry and Human Behavior, 
Providence, Rhode Island, United States of America.

151University of Melbourne, Phoenix Australia, Department of Psychiatry, 
Melbourne, Victoria, Australia.

152Northern Illinois University, Department of Psychology, DeKalb, Illinois, United 
States of America.

153Universidade Federal de São Paulo, Psychiatry, São Paulo, São Paulo, Brazil.

154University of Nebraska Medical Center, College of Public Health, Omaha, 
Nebraska, United States of America.

155South Texas Veterans Health Care System, Research and Development Service, 
San Antonio, Texas, United States of America.

156University of Texas Health Science Center at San Antonio, Department of 
Psychiatry and Behavioral Sciences, San Antonio, Texas, United States of America.

157University of Washington, Department of Psychology, Seattle, Washington, 
United States of America.

Nat Genet. Author manuscript; available in PMC 2024 September 13.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Nievergelt et al.

Page 35

158U.S. Department of Veterans Affairs National Center for Posttraumatic Stress 
Disorder, West Haven, Connecticut, United States of America.

159Center for Care Delivery and Outcomes Research (CCDOR), Minneapolis, 
Minnesota, United States of America.

160Columbia University Mailmain School of Public Health, Department of 
Epidemiology, New York, New York, United States of America.

161Harvard T.H. Chan School of Public Health, Department of Environmental Health, 
Boston, Massachusetts, United States of America.

162Emory University, Department of Psychological Sciences, Atlanta, Georgia, 
United States of America.

163Skyland Trail, Department of Research and Outcomes, Atlanta, Georgia, United 
States of America.

164University of Washington, Department of Psychiatry, Seattle, Washington, United 
States of America.

165Medical University of South Carolina, Department of Nursing and Department of 
Psychiatry, Charleston, South Carolina, United States of America.

166Louisiana State University Health Sciences Center, School of Public Health and 
Department of Epidemiology, New Orleans, Louisiana, United States of America.

167Biogen Inc., Research & Development, Cambridge, Massachusetts, United 
States of America.

168Maastricht Universitair Medisch Centrum, School for Mental Health and 
Neuroscience, Department of Psychiatry and Neuropsychology, Maastricht, 
Limburg, The Netherlands.

169SUNY Downstate Health Sciences University, School of Public Health, Brooklyn, 
New York, United States of America.

170Child Mind Institute, New York, New York, United States of America.

171Instituto Nacional de Psiquiatria de Desenvolvimento, São Paulo, São Paulo, 
Brazil.

172Universidad Peruana de Ciencias Aplicadas, Department of Medicine, Lima, 
Lima, Peru.

173Universidade Federal de São Paulo, Departamento de Bioquímica - Disciplina de 
Biologia Molecular, São Paulo, São Paulo, Brazil.

174Stellenbosch University, Department of Psychiatry, Faculty of Medicine and 
Health Sciences, Stellenbosch University, Cape Town, Western Cape, South Africa.

175Stellenbosch University, SAMRC Extramural Genomics of Brain Disorders 
Research Unit, Cape Town, Western Cape, South Africa.

Nat Genet. Author manuscript; available in PMC 2024 September 13.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Nievergelt et al.

Page 36

176University of Michigan, Department of Obstetrics and Gynecology, Ann Arbor, 
Michigan, United States of America.

177University of Michigan, Department of Women's and Gender Studies, Ann Arbor, 
Michigan, United States of America.

178University of Michigan, Institute for Research on Women and Gender, Ann Arbor, 
Michigan, United States of America.

179University of Michigan, School of Nursing, Ann Arbor, Michigan, United States of 
America.

180University of New South Wales, Department of Psychiatry, Sydney, New South 
Wales, Australia.

181Emory University, Department of Gynecology and Obstetrics; Department of 
Psychiatry and Behavioral Sciences; Department of Human Genetics, Atlanta, 
Georgia, United States of America.

182Massachusetts General Hospital, Psychiatric and Neurodevelopmental Genetics 
Unit (PNGU), Boston, Massachusetts, United States of America.

183University of Minnesota Medical School, Department of Psychiatry and 
Behavioral Sciences, Minneapolis, Minnesota, United States of America.

184University of California, Los Angeles, Department of Psychology, Los Angeles, 
California, United States of America.

185McLean Hospital, Developmental Biopsychiatry Research Program, Belmont, 
Massachusetts, United States of America.

186Mental Health Centre Sct. Hans, Institute of Biological Psychiatry, Roskilde, 
Denmark.

187University of California San Diego, Herbert Wertheim School of Public Health and 
Human Longevity Science, La Jolla, California, United States of America.

188University of South Florida College of Public Health, Genomics Program, Tampa, 
Florida, United States of America.

189Uniformed Services University, Department of Psychiatry, Bethesda, Maryland, 
United States of America.

190Karolinska Institutet, Unit of Integrative Epidemiology, Institute of Environmental 
Medicine, Stockholm, Sweden.

191University of Iceland, Faculty of Medicine, Center of Public Health Sciences, 
School of Health Sciences, Reykjavik, Iceland.

192University of Adelaide, Adelaide Medical School, Adelaide, South Australia, 
Australia.

193ARQ Nationaal Psychotrauma Centrum, Psychotrauma Reseach Expert Group, 
Diemen, North Holland, The Netherlands

Nat Genet. Author manuscript; available in PMC 2024 September 13.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Nievergelt et al.

Page 37

194Leiden University Medical Center, Department of Psychiatry, Leiden, South 
Holland, The Netherlands.

195New York University School of Medicine, Department of Psychiatry, New York, 
New York, United States of America.

196Amsterdam Neuroscience, Mood, Anxiety, Psychosis, Sleep & Stress Program, 
Amsterdam, North Holland, The Netherlands.

197Amsterdam UMC location Vrije Universiteit Amsterdam, Department of Anatomy 
and Neurosciences, Amsterdam, North Holland, The Netherlands.

198Amsterdam UMC location Vrije Universiteit Amsterdam, Department of 
Psychiatry, Amsterdam, North Holland, The Netherlands.

199University of Oslo, Lifespan Changes in Brain and Cognition (LCBC), Department 
of Psychology, Oslo, Norway.

200Medical University of South Carolina, Department of Psychiatry and Behavioral 
Sciences, Charleston, South Carolina, United States of America.

201Ralph H Johnson VA Medical Center, Department of Mental Health, Charleston, 
South Carolina, United States of America.

202Rosalind Franklin University of Medicine and Science, Department of Psychology, 
North Chicago, Illinois, United States of America.

203University of Toronto, Dalla Lana School of Public Health, Department of 
Anthropology, Toronto, Ontario, Canada.

204Copenhagen University Hospital, Institute of Biological Psychiatry, Mental Health 
Services, Copenhagen, Denmark.

205University of Copenhagen, Department of Clinical Medicine, Copenhagen, 
Denmark.

206University of Copenhagen, The Globe Institute, Lundbeck Foundation Center for 
Geogenetics, Copenhagen, Denmark.

207Oslo University Hospital, Department of Neurology, Oslo, Norway.

208Boston University Chobanian & Avedisian School of Medicine, Department of 
Psychiatry, Boston, Massachusetts, United States of America.

209James J. Peters VA Medical Center, Department of Mental Health, Bronx, New 
York, United States of America.

210Central Texas Veterans Health Care System, Research Service, Temple, Texas, 
United States of America.

211Texas A&M University School of Medicine, Department of Psychiatry and 
Behavioral Sciences, Bryan, Texas, United States of America.

212Queensland University of Technology, School of Clinical Sciences, Kelvin Grove, 
Queensland, Australia.

Nat Genet. Author manuscript; available in PMC 2024 September 13.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Nievergelt et al.

Page 38

213University of the Sunshine Coast, The Chancellory, Sippy Downs, Queensland, 
Australia.

214University of Toronto, Department of Laboratory Medicine and Pathology, Toronto, 
Ontario, Canada.

215Centre for Addiction and Mental Health, General Adult Psychiatry and Health 
Systems Division, Toronto, Ontario, Canada.

216Yale University, Department of Biostatistics, New Haven, Connecticut, United 
States of America.

217University of California San Diego, School of Public Health, La Jolla, California, 
United States of America.

Acknowledgements

Major financial support for the PTSD-PGC was provided by the Cohen Veterans Bioscience, Stanley Center for 
Psychiatric Research at the Broad Institute, and the National Institute of Mental Health (NIMH; R01MH106595 to 
K.C.K., C.M.N., K.J.R., and M.B.S.; R01MH124847 to C.M.N.; R01MH124851 to A.D.B., L.K.D., and K.C.K.). 
Statistical analyses were carried out on the NL Genetic Cluster computer (URL) hosted by SURFsara. Genotyping 
of samples was supported in part through the Stanley Center for Psychiatric Genetics at the Broad Institute of 
MIT and Harvard. This research has been conducted using the UK Biobank resource under application number 
41209. This work would not have been possible without the contributions of the investigators who comprise the 
PGC-PTSD working group, and especially the more than 1,307,247 research participants worldwide who shared 
their life experiences and biological samples with PGC-PTSD investigators. We thank Allison E. Aiello, Bekh 
Bradley, Aarti Gautam, Rasha Hammamieh, Marti Jett, Michael J. Lyons, Douglas Maurer, Matig R. Mavissakalian, 
and the late Christopher R. Erbes and Regina E. McGlinchey for their contributions to this study.

Competing Interests

L.J.B. is listed as an inventor on Issued U.S. Patent 8,080,371, “Markers for Addiction” covering the use 
of certain SNPs in determining the diagnosis, prognosis, and treatment of addiction. C.-Y.C. and H.R. are 
employees of Biogen. A.M.D. holds equity in CorTechs Labs, Inc., and serves on the Scientific Advisory Board 
of Human Longevity, Inc., and the Mohn Medical Imaging and Visualization Centre; A.M.D. receives funding 
through research grants with General Electric Healthcare. C.F. was a speaker for Janssen in 2021. I.B.H. is the 
Co-Director, Health and Policy at the Brain and Mind Centre (BMC) University of Sydney; the BMC operates 
an early-intervention youth services at Camperdown under contract to headspace. I.B.H. is the Chief Scientific 
Advisor to, and a 3.2% equity shareholder in, InnoWell Pty Ltd; InnoWell was formed by the University of 
Sydney (45% equity) and PwC (Australia; 45% equity) to deliver the $30 M Australian Government-funded 
Project Synergy. H.H. received consultancy fees from Ono Pharmaceutical and honorarium from Xian Janssen 
Pharmaceutical. In the past 3 years, R.C.K. was a consultant for Cambridge Health Alliance, Canandaigua VA 
Medical Center, Holmusk, Partners Healthcare, Inc., RallyPoint Networks, Inc., and Sage Therapeutics. He has 
stock options in Cerebral Inc., Mirah, PYM, Roga Sciences and Verisense Health. L.A.M.L. reports spousal IP 
payments from Vanderbilt University for technology licensed to Acadia Pharmaceuticals unrelated to the present 
work. C.M. has served on advisory boards of Receptor Life Sciences, Otsuka Pharmaceuticals and Roche Products 
Limited and has received support from National Institute on Alcohol Abuse and Alcoholism, National Institute 
of Mental Health, Department of Defense- CDMRP * US Army Research Office * DARPA, Bank of America 
Foundation, Brockman Foundation, Cohen Veterans Bioscience, Cohen Veterans Network, McCormick Foundation, 
Home Depot Foundation, New York City Council, New York State Health, Mother Cabrini Foundation, Tilray 
Pharmaceuticals, and Ananda Scientific. P.M.P. received payment or honoraria for lectures and presentations in 
educational events for Sandoz, Daiichi Sankyo, Eurofarma, Abbot, Libbs, Instituto Israelita de Pesquisa e Ensino 
Albert Einstein, Instituto D’Or de Pesquisa e Ensino. R.P. has been paid for his editorial work on the journal 
Complex Psychiatry and received a research grant outside the scope of this study from Alkermes. J.W.S. is a 
member of the Scientific Advisory Board of Sensorium Therapeutics (with equity), and has received grant support 
from Biogen, Inc.; J.W.S. is PI of a collaborative study of the genetics of depression and bipolar disorder sponsored 
by 23andMe for which 23andMe provides analysis time as in-kind support but no payments. M.B.S. has in the past 
3 years received consulting income from Acadia Pharmaceuticals, Aptinyx, atai Life Sciences, BigHealth, Biogen, 
Bionomics, BioXcel Therapeutics, Boehringer Ingelheim, Clexio, Eisai, EmpowerPharm, Engrail Therapeutics, 
Janssen, Jazz Pharmaceuticals, NeuroTrauma Sciences, PureTech Health, Sage Therapeutics, Sumitomo Pharma, 
and Roche/Genentech. M.B.S. has stock options in Oxeia Biopharmaceuticals and EpiVario. M.B.S. has been paid 
for his editorial work on Depression and Anxiety (Editor-in-Chief), Biological Psychiatry (Deputy Editor), and 

Nat Genet. Author manuscript; available in PMC 2024 September 13.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Nievergelt et al.

Page 39

UpToDate (Co-Editor-in-Chief for Psychiatry). M.B.S. has also received research support from NIH, Department 
of Veterans Affairs, and the Department of Defense. M.B.S. is on the scientific advisory board for the Brain and 
Behavior Research Foundation and the Anxiety and Depression Association of America. In the past 3 years, D.J.S. 
has received consultancy honoraria from Discovery Vitality, Johnson & Johnson, Kanna, L’Oreal, Lundbeck, Orion, 
Sanofi, Servier, Takeda and Vistagen. M.L.K. reports unpaid membership on the Scientific Committee for the 
ISSTD. All other authors declare no competing interests.

Data availability

Summary statistics for PGC PTSD Freeze 3 will be made available upon publication 
under the accession ID ptsd2024 via the PGC website (https://pgc.unc.edu/for-researchers/
download-results/). Access to study level summary statistics and genotype data can be 
applied for by using the PGC data access portal (https://pgc.unc.edu/for-researchers/data-
access-committee/data-access-portal/).

Consortia member authors

AURORA Study

Sarah D. Linnstaedt126, Karestan C. Koenen6,10,182, Kerry J. Ressler11,44,82, Samuel A. 
McLean126,137, and Ronald C. Kessler115

Estonian Biobank Research Team

Kristi Krebs122, Kelli Lehto122, and Lili Milani122

FinnGen Investigators

Chia-Yen Chen5 and Heiko Runz167

HUNT All-In Psychiatry

Bendik S. Winsvold55,56,207, Synne Stensland56,77, Sigrid Børte55,56, Kristian Hveem55, 
John-Anker Zwart20,55,56, and Grete Dyb20,77

5 Biogen Inc., Translational Sciences, Cambridge, Massachusetts, United States of America.

6 Harvard T.H. Chan School of Public Health, Department of Epidemiology, Boston, 
Massachusetts, United States of America.

10 Broad Institute of MIT and Harvard, Stanley Center for Psychiatric Research, Cambridge, 
Massachusetts, United States of America.

11 Harvard Medical School, Department of Psychiatry, Boston, Massachusetts, United States 
of America.

20 University of Oslo, Institute of Clinical Medicine, Oslo, Norway.

44 McLean Hospital, Belmont, Massachusetts, United States of America.

Nat Genet. Author manuscript; available in PMC 2024 September 13.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Nievergelt et al.

Page 40

55 Norwegian University of Science and Technology, K. G. Jebsen Center for Genetic 
Epidemiology, Department of Public Health and Nursing, Faculty of Medicine and Health 
Sciences, Trondheim, Norway.

56 Oslo University Hospital, Department of Research, Innovation and Education, Division of 
Clinical Neuroscience, Oslo, Norway.

77 Norwegian Centre for Violence and Traumatic Stress Studies, Oslo, Norway.

82 Emory University, Department of Psychiatry and Behavioral Sciences, Atlanta, Georgia, 
United States of America.

115 Harvard Medical School, Department of Health Care Policy, Boston, Massachusetts, 
United States of America.

122 University of Tartu, Institute of Genomics, Estonian Genome Center, Tartu, Estonia.

126 UNC Institute for Trauma Recovery, Department of Anesthesiology, Chapel Hill, North 
Carolina, United States of America.

137 UNC Institute for Trauma Recovery, Department of Emergency Medicine, Chapel Hill, 
North Carolina, United States of America.

167 Biogen Inc., Research & Development, Cambridge, Massachusetts, United States of 
America.

182 Massachusetts General Hospital, Psychiatric and Neurodevelopmental Genetics Unit 
(PNGU), Boston, Massachusetts, United States of America.

207 Oslo University Hospital, Department of Neurology, Oslo, Norway.

A full list of all consortia members is available in the Supplementary Note.

References

1. Koenen KC et al. Posttraumatic stress disorder in the World Mental Health Surveys. Psychol Med 

47, 2260–2274 (2017). [PubMed: 28385165] 

2. Davis LL et al. The Economic Burden of Posttraumatic Stress Disorder in the United States From a 

Societal Perspective. J Clin Psychiatry 83, 21m14116 (2022).

3. Ferland-Beckham C et al. Systematic review and methodological considerations for the use of single 
prolonged stress and fear extinction retention in rodents. Front Behav Neurosci 15, 652636 (2021).
4. Ressler KJ et al. Post-traumatic stress disorder: clinical and translational neuroscience from cells to 

circuits. Nat Rev Neurol 18, 273–288 (2022). [PubMed: 35352034] 

5. McClellan France J & Jovanovic T Human fear neurobiology reimagined: Can brain-derived 

biotypes predict fear-based disorders after trauma? Neurosci Biobehav Rev 144, 104988 (2023).

6. Dunsmoor JE, Cisler JM, Fonzo GA, Creech SK & Nemeroff CB Laboratory models of post-
traumatic stress disorder: The elusive bridge to translation. Neuron 110, 1754–1776 (2022). 
[PubMed: 35325617] 

7. Bassil K et al. In vitro modeling of the neurobiological effects of glucocorticoids: A review. 

Neurobiol Stress 23, 100530 (2023).

Nat Genet. Author manuscript; available in PMC 2024 September 13.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Nievergelt et al.

Page 41

8. Seah C et al. Modeling gene × environment interactions in PTSD using human neurons reveals 

diagnosis-specific glucocorticoid-induced gene expression. Nat Neurosci 25, 1434–1445 (2022). 
[PubMed: 36266471] 

9. Kremen WS, Koenen KC, Afari N & Lyons MJ Twin studies of posttraumatic stress disorder: 
differentiating vulnerability factors from sequelae. Neuropharmacology 62, 647–653 (2012). 
[PubMed: 21443892] 

10. Wolf EJ et al. A classical twin study of PTSD symptoms and resilience: Evidence for a single 
spectrum of vulnerability to traumatic stress. Depress Anxiety 35, 132–139 (2018). [PubMed: 
29283198] 

11. Nievergelt CM et al. International meta-analysis of PTSD genome-wide association studies 

identifies sex- and ancestry-specific genetic risk loci. Nat Commun 10, 4558 (2019). [PubMed: 
31594949] 

12. Maihofer AX et al. Enhancing Discovery of Genetic Variants for Posttraumatic Stress Disorder 
Through Integration of Quantitative Phenotypes and Trauma Exposure Information. Biol 
Psychiatry 91, 626–636 (2022). [PubMed: 34865855] 

13. Stein MB et al. Genome-wide association analyses of post-traumatic stress disorder and its 

symptom subdomains in the Million Veteran Program. Nat Genet 53, 174–184 (2021). [PubMed: 
33510476] 

14. Wendt FR et al. The Relationship of Attention-Deficit/Hyperactivity Disorder With Posttraumatic 

Stress Disorder: A Two-Sample Mendelian Randomization and Population-Based Sibling 
Comparison Study. Biol Psychiatry 93, 362–369 (2023). [PubMed: 36335070] 

15. Polimanti R et al. Understanding the comorbidity between posttraumatic stress severity and 
coronary artery disease using genome-wide information and electronic health records. Mol 
Psychiatry 27, 3961–3969 (2022). [PubMed: 35986173] 

16. Campbell-Sills L et al. Dissecting the heterogeneity of posttraumatic stress disorder: differences in 
polygenic risk, stress exposures, and course of PTSD subtypes. Psychol Med 52, 1–9 (2021).
17. Choi KW et al. Prospective study of polygenic risk, protective factors, and incident depression 

following combat deployment in US Army soldiers. Psychol Med 50, 737–745 (2020). [PubMed: 
30982473] 

18. Lobo JJ et al. Polygenic risk scoring to assess genetic overlap and protective factors influencing 

posttraumatic stress, depression, and chronic pain after motor vehicle collision trauma. Transl 
Psychiatry 11, 359 (2021). [PubMed: 34226500] 

19. Roberts AL, Gilman SE, Breslau J, Breslau N & Koenen KC Race/ethnic differences in exposure 
to traumatic events, development of post-traumatic stress disorder, and treatment-seeking for 
post-traumatic stress disorder in the United States. Psychol Med 41, 71–83 (2011). [PubMed: 
20346193] 

20. Bassett D, Buchwald D & Manson S Posttraumatic stress disorder and symptoms among American 
Indians and Alaska Natives: a review of the literature. Soc Psychiatry Psychiatr Epidemiol 49, 
417–433 (2014). [PubMed: 24022752] 

21. Logue MW et al. The Psychiatric Genomics Consortium Posttraumatic Stress Disorder 

Workgroup: Posttraumatic Stress Disorder Enters the Age of Large-Scale Genomic Collaboration. 
Neuropsychopharmacology 40, 2287–2297 (2015). [PubMed: 25904361] 

22. Holland D et al. Beyond SNP heritability: Polygenicity and discoverability of phenotypes 
estimated with a univariate Gaussian mixture model. PLoS Genet 16, e1008612 (2020).

23. Frei O et al. Bivariate causal mixture model quantifies polygenic overlap between complex traits 

beyond genetic correlation. Nat Commun 10, 2417 (2019). [PubMed: 31160569] 

24. Bulik-Sullivan BK et al. LD Score regression distinguishes confounding from polygenicity in 
genome-wide association studies. Nat Genet 47, 291–295 (2015). [PubMed: 25642630] 

25. Watanabe K, Taskesen E, van Bochoven A & Posthuma D Functional mapping and annotation of 

genetic associations with FUMA. Nat Commun 8, 1826 (2017). [PubMed: 29184056] 
26. Yang J, Lee SH, Goddard ME & Visscher PM GCTA: a tool for genome-wide complex trait 

analysis. Am J Hum Genet 88, 76–82 (2011). [PubMed: 21167468] 

27. de Bakker PI & Raychaudhuri S Interrogating the major histocompatibility complex with high-

throughput genomics. Hum Mol Genet 21, R29–R36 (2012). [PubMed: 22976473] 

Nat Genet. Author manuscript; available in PMC 2024 September 13.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Nievergelt et al.

Page 42

28. Rentzsch P, Witten D, Cooper GM, Shendure J & Kircher M CADD: predicting the deleteriousness 
of variants throughout the human genome. Nucleic Acids Res 47, D886–D894 (2019). [PubMed: 
30371827] 

29. Boyle AP et al. Annotation of functional variation in personal genomes using RegulomeDB. 

Genome Res 22, 1790–1797 (2012). [PubMed: 22955989] 

30. Weissbrod O et al. Functionally informed fine-mapping and polygenic localization of complex trait 

heritability. Nat Genet 52, 1355–1363 (2020). [PubMed: 33199916] 

31. de Leeuw CA, Mooij JM, Heskes T & Posthuma D MAGMA: Generalized Gene-Set Analysis of 

GWAS Data. PLoS Comput Biol 11, e1004219 (2015).

32. Trubetskoy V et al. Mapping genomic loci implicates genes and synaptic biology in schizophrenia. 

Nature 604, 502–508 (2022). [PubMed: 35396580] 

33. Liberzon A et al. The Molecular Signatures Database (MSigDB) hallmark gene set collection. Cell 

Syst 1, 417–425 (2015). [PubMed: 26771021] 

34. La Manno G et al. Molecular Diversity of Midbrain Development in Mouse, Human, and Stem 

Cells. Cell 167, 566–580.e19 (2016). [PubMed: 27716510] 

35. Barbeira AN et al. Exploiting the GTEx resources to decipher the mechanisms at GWAS loci. 

Genome Biol 22, 49 (2021). [PubMed: 33499903] 

36. Zhu Z et al. Integration of summary data from GWAS and eQTL studies predicts complex trait 

gene targets. Nat Genet 48, 481–487 (2016). [PubMed: 27019110] 

37. Pathak GA et al. Genetically regulated multi-omics study for symptom clusters of posttraumatic 

stress disorder highlights pleiotropy with hematologic and cardio-metabolic traits. Mol Psychiatry 
27, 1394–1404 (2022). [PubMed: 35241783] 

38. Waszczuk MA et al. Discovery and replication of blood-based proteomic signature of PTSD in 

9/11 responders. Transl Psychiatry 13, 8 (2023). [PubMed: 36631443] 

39. Wingo TS et al. Integrating human brain proteomes with genome-wide association data implicates 
novel proteins in post-traumatic stress disorder. Mol Psychiatry 27, 3075–3084 (2022). [PubMed: 
35449297] 

40. Sun BB et al. Plasma proteomic associations with genetics and health in the UK Biobank. Nature 

622, 329–338 (2022).

41. Bellenguez C et al. New insights into the genetic etiology of Alzheimer’s disease and related 

dementias. Nat Genet 54, 412–436 (2022). [PubMed: 35379992] 

42. Romero C et al. Exploring the genetic overlap between twelve psychiatric disorders. Nat Genet 54, 

1795–1802 (2022). [PubMed: 36471075] 

43. Demontis D et al. Genome-wide analyses of ADHD identify 27 risk loci, refine the genetic 

architecture and implicate several cognitive domains. Nat Genet 55, 198–208 (2023). [PubMed: 
36702997] 

44. Walters RK et al. Transancestral GWAS of alcohol dependence reveals common genetic 

underpinnings with psychiatric disorders. Nat Neurosci 21, 1656–1669 (2018). [PubMed: 
30482948] 

45. Watson HJ et al. Genome-wide association study identifies eight risk loci and implicates metabo-
psychiatric origins for anorexia nervosa. Nat Genet 51, 1207–1214 (2019). [PubMed: 31308545] 

46. Otowa T et al. Meta-analysis of genome-wide association studies of anxiety disorders. Mol 

Psychiatry 21, 1391–1399 (2016). [PubMed: 26754954] 

47. Grove J et al. Identification of common genetic risk variants for autism spectrum disorder. Nat 

Genet 51, 431–444 (2019). [PubMed: 30804558] 

48. Mullins N et al. Genome-wide association study of more than 40,000 bipolar disorder cases 

provides new insights into the underlying biology. Nat Genet 53, 817–829 (2021). [PubMed: 
34002096] 

49. Pasman JA et al. GWAS of lifetime cannabis use reveals new risk loci, genetic overlap with 
psychiatric traits, and a causal effect of schizophrenia liability. Nat Neurosci 21, 1161–1170 
(2018). [PubMed: 30150663] 

Nat Genet. Author manuscript; available in PMC 2024 September 13.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Nievergelt et al.

Page 43

50. Howard DM et al. Genome-wide meta-analysis of depression identifies 102 independent variants 
and highlights the importance of the prefrontal brain regions. Nat Neurosci 22, 343–352 (2019). 
[PubMed: 30718901] 

51. Revealing the complex genetic architecture of obsessive-compulsive disorder using meta-analysis. 

Mol Psychiatry 23, 1181–1188 (2018). [PubMed: 28761083] 

52. Yu D et al. Interrogating the Genetic Determinants of Tourette’s Syndrome and Other Tic 

Disorders Through Genome-Wide Association Studies. Am J Psychiatry 176, 217–227 (2019). 
[PubMed: 30818990] 

53. Werme J, van der Sluis S, Posthuma D & de Leeuw CA An integrated framework for local genetic 

correlation analysis. Nat Genet 54, 274–282 (2022). [PubMed: 35288712] 

54. Bycroft C et al. The UK Biobank resource with deep phenotyping and genomic data. Nature 562, 

203–209 (2018). [PubMed: 30305743] 

55. Zoellner LA, Roy-Byrne PP, Mavissakalian M & Feeny NC Doubly Randomized Preference Trial 
of Prolonged Exposure Versus Sertraline for Treatment of PTSD. Am J Psychiatry 176, 287–296 
(2019). [PubMed: 30336702] 

56. Bullman TA & Kang HK Posttraumatic stress disorder and the risk of traumatic deaths among 

Vietnam veterans. J Nerv Ment Dis 182, 604–610 (1994). [PubMed: 7964667] 

57. Clover K, Carter GL & Whyte IM Posttraumatic stress disorder among deliberate self-poisoning 

patients. J Trauma Stress 17, 509–517 (2004). [PubMed: 15730070] 

58. Gradus JL et al. Posttraumatic Stress Disorder and Gastrointestinal Disorders in the Danish 

Population. Epidemiology 28, 354–360 (2017). [PubMed: 28099266] 

59. Brady KT, Killeen TK, Brewerton T & Lucerini S Comorbidity of psychiatric disorders and 

posttraumatic stress disorder. J Clin Psychiatry 61 Suppl 7, 22–32 (2000).

60. Kind S & Otis JD The Interaction Between Chronic Pain and PTSD. Curr Pain Headache Rep 23, 

91 (2019). [PubMed: 31781875] 

61. Nishimi K et al. Post-traumatic stress disorder and risk for hospitalization and death following 

COVID-19 infection. Transl Psychiatry 12, 482 (2022). [PubMed: 36411283] 

62. Roberts AL, Kubzansky LD, Chibnik LB, Rimm EB & Koenen KC Association of Posttraumatic 
Stress and Depressive Symptoms With Mortality in Women. JAMA Netw Open 3, e2027935 
(2020).

63. Schlenger WE et al. A Prospective Study of Mortality and Trauma-Related Risk Factors Among 
a Nationally Representative Sample of Vietnam Veterans. Am J Epidemiol 182, 980–990 (2015). 
[PubMed: 26634285] 

64. Martin AR et al. Human Demographic History Impacts Genetic Risk Prediction across Diverse 

Populations. Am J Hum Genet 100, 635–649 (2017). [PubMed: 28366442] 

65. Panagiotou OA, Willer CJ, Hirschhorn JN & Ioannidis JP The power of meta-analysis in 

genome-wide association studies. Annu Rev Genomics Hum Genet 14, 441–65 (2013). [PubMed: 
23724904] 

66. Huggins A et al. Smaller Total and Subregional Cerebellar Volumes in Posttraumatic Stress 

Disorder: A Mega-Analysis by the ENIGMA-PGC PTSD Workgroup. Biol Psychiatry 93, S44 
(2023).

67. Girgenti MJ et al. Transcriptomic organization of the human brain in post-traumatic stress disorder. 

Nat Neurosci 24, 24–33 (2021). [PubMed: 33349712] 

68. Logue MW et al. Gene expression in the dorsolateral and ventromedial prefrontal cortices 
implicates immune-related gene networks in PTSD. Neurobiol Stress 15, 100398 (2021).
69. Jaffe AE et al. Decoding Shared Versus Divergent Transcriptomic Signatures Across Cortico-

Amygdala Circuitry in PTSD and Depressive Disorders. Am J Psychiatry 179, 673–686 (2022). 
[PubMed: 35791611] 

70. Chatzinakos C et al. Single-Nucleus Transcriptome Profiling of Dorsolateral Prefrontal Cortex: 

Mechanistic Roles for Neuronal Gene Expression, Including the 17q21.31 Locus, in PTSD Stress 
Response. Am J Psychiatry 180, 739–754 (2023). [PubMed: 37491937] 

71. Kessler RC, Sonnega A, Bromet E, Hughes M & Nelson CB Posttraumatic Stress Disorder in the 
National Comorbidity Survey. Arch Gen Psychiatry 52, 1048–1060 (1995). [PubMed: 7492257] 

Nat Genet. Author manuscript; available in PMC 2024 September 13.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Nievergelt et al.

Page 44

72. Ravi M, Stevens JS & Michopoulos V Neuroendocrine pathways underlying risk and resilience to 

PTSD in women. Front Neuroendocrinol 55, 100790 (2019).

73. Hodes GE & Epperson CN Sex Differences in Vulnerability and Resilience to Stress Across the 

Life Span. Biol Psychiatry 86, 421–432 (2019). [PubMed: 31221426] 

74. Gelernter J et al. Genome-wide association study of post-traumatic stress disorder reexperiencing 
symptoms in >165,000 US veterans. Nat Neurosci 22, 1394–1401 (2019). [PubMed: 31358989] 

75. Nachtigall EG, de Freitas JDR, de CMJ & Furini CRG Role of Hippocampal Wnt Signaling 
Pathways on Contextual Fear Memory Reconsolidation. Neuroscience 524, 108–119 (2023). 
[PubMed: 37286160] 

76. Lv T et al. Electroacupuncture alleviates PTSD-like behaviors by modulating hippocampal 

synaptic plasticity via Wnt/β-catenin signaling pathway. Brain Res Bull 202, 110734 (2023).
77. Herrero MJ et al. Sex-Specific Social Behavior and Amygdala Proteomic Deficits in Foxp2 (+/−) 

Mutant Mice. Front Behav Neurosci 15, 706079 (2021).

78. Dalvie S et al. Genomic influences on self-reported childhood maltreatment. Transl Psychiatry 10, 

38 (2020). [PubMed: 32066696] 

79. Grotzinger AD et al. Genetic architecture of 11 major psychiatric disorders at biobehavioral, 
functional genomic and molecular genetic levels of analysis. Nat Genet 54, 548–559 (2022). 
[PubMed: 35513722] 

80. Kessler RC, Chiu WT, Demler O & Walters EE Prevalence, Severity, and Comorbidity of 12-

Month DSM-IV Disorders in the National Comorbidity Survey Replication. Arch Gen Psychiatry 
62, 617–627 (2005). [PubMed: 15939839] 

81. Breen G et al. Translating genome-wide association findings into new therapeutics for psychiatry. 

Nat Neurosci 19, 1392–1396 (2016). [PubMed: 27786187] 

82. Stein MB & Rothbaum BO 175 Years of Progress in PTSD Therapeutics: Learning From the Past. 

Am J Psychiatry 175, 508–516 (2018). [PubMed: 29869547] 

83. Mahoney CT, Moshier SJ, Keane TM & Marx BP Heightened healthcare utilization & risk 

of mental disorders among Veterans with comorbid opioid use disorder & posttraumatic stress 
disorder. Addict Behav 112, 106572 (2021).

84. Upadhyay J et al. Neurocircuitry basis of the opioid use disorder-post-traumatic stress disorder 

comorbid state: conceptual analyses using a dimensional framework. Lancet Psychiatry 9, 84–96 
(2022). [PubMed: 34774203] 

Methods-only references

85. Lam M et al. RICOPILI: Rapid Imputation for COnsortias PIpeLIne. Bioinformatics 36, 930–933 

(2020). [PubMed: 31393554] 

86. McCarthy S et al. A reference panel of 64,976 haplotypes for genotype imputation. Nat Genet 48, 

1279–1283 (2016). [PubMed: 27548312] 

87. Chen CY et al. Improved ancestry inference using weights from external reference panels. 

Bioinformatics 29, 1399–1406 (2013). [PubMed: 23539302] 

88. Chang CC et al. Second-generation PLINK: rising to the challenge of larger and richer datasets. 

Gigascience 4, 7 (2015). [PubMed: 25722852] 

89. Zhou X & Stephens M Genome-wide efficient mixed-model analysis for association studies. Nat 

Genet 44, 821–824 (2012). [PubMed: 22706312] 

90. Chen WM, Manichaikul A & Rich SS A generalized family-based association test for dichotomous 

traits. Am J Hum Genet 85, 364–376 (2009). [PubMed: 19732865] 

91. Loh P-R et al. Efficient Bayesian mixed-model analysis increases association power in large 

cohorts. Nat Genet 47, 284–290 (2015). [PubMed: 25642633] 

92. Zhou W et al. Efficiently controlling for case-control imbalance and sample relatedness in large-
scale genetic association studies. Nat Genet 50, 1335–1341 (2018). [PubMed: 30104761] 

93. Mbatchou J et al. Computationally efficient whole-genome regression for quantitative and binary 

traits. Nat Genet 53, 1097–1103 (2021). [PubMed: 34017140] 

Nat Genet. Author manuscript; available in PMC 2024 September 13.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Nievergelt et al.

Page 45

94. Willer CJ, Li Y & Abecasis GR METAL: fast and efficient meta-analysis of genomewide 

association scans. Bioinformatics 26, 2190–2191 (2010). [PubMed: 20616382] 

95. Pruim RJ et al. LocusZoom: regional visualization of genome-wide association scan results. 

Bioinformatics 26, 2336–2337 (2010). [PubMed: 20634204] 

96. Auton A et al. A global reference for human genetic variation. Nature 526, 68–74 (2015). 

[PubMed: 26432245] 

97. Finucane HK et al. Partitioning heritability by functional annotation using genome-wide 
association summary statistics. Nat Genet 47, 1228–1235 (2015). [PubMed: 26414678] 

98. Ge T, Chen C-Y, Ni Y, Feng Y-CA & Smoller JW Polygenic prediction via Bayesian regression 

and continuous shrinkage priors. Nat Commun 10, 1776 (2019). [PubMed: 30992449] 

99. GTEx Consortium. The GTEx Consortium atlas of genetic regulatory effects across human tissues. 

Science 369, 1318–1330 (2020). [PubMed: 32913098] 

100. Ramasamy A et al. Genetic variability in the regulation of gene expression in ten regions of the 

human brain. Nat Neurosci 17, 1418–1428 (2014). [PubMed: 25174004] 

101. Hoffman GE et al. CommonMind Consortium provides transcriptomic and epigenomic data for 
Schizophrenia and Bipolar Disorder. Scientific Data 6, 180 (2019). [PubMed: 31551426] 
102. Wang D et al. Comprehensive functional genomic resource and integrative model for the human 

brain. Science 362, eaat8464 (2018).

103. Paola G-R et al. Using three-dimensional regulatory chromatin interactions from adult and fetal 
cortex to interpret genetic results for psychiatric disorders and cognitive traits. bioRxiv, 406330 
(2019).

104. Schmitt AD et al. A Compendium of Chromatin Contact Maps Reveals Spatially Active Regions 

in the Human Genome. Cell Rep 17, 2042–2059 (2016). [PubMed: 27851967] 

105. Wang K, Li M & Hakonarson H ANNOVAR: functional annotation of genetic variants from high-
throughput sequencing data. Nucleic Acids Res 38, e164–e164 (2010). [PubMed: 20601685] 
106. Zou Y, Carbonetto P, Wang G & Stephens M Fine-mapping from summary data with the “Sum of 

Single Effects” model. PLoS Genet 18, e1010299 (2022).

107. Bryois J et al. Cell-type-specific cis-eQTLs in eight human brain cell types identify novel risk 

genes for psychiatric and neurological disorders. Nat Neurosci 25, 1104–1112 (2022). [PubMed: 
35915177] 

108. Chatzinakos C et al. TWAS pathway method greatly enhances the number of leads for uncovering 
the molecular underpinnings of psychiatric disorders. Am J Med Genet B Neuropsychiatr Genet 
183, 454–463 (2020). [PubMed: 32954640] 

109. Barbeira AN et al. Exploring the phenotypic consequences of tissue specific gene expression 
variation inferred from GWAS summary statistics. Nat Commun 9, 1825 (2018). [PubMed: 
29739930] 

110. Koopmans F et al. SynGO: An Evidence-Based, Expert-Curated Knowledge Base for the 

Synapse. Neuron 103, 217–234.e4 (2019). [PubMed: 31171447] 

111. Gaspar HA & Breen G Drug enrichment and discovery from schizophrenia genome-wide 

association results: an analysis and visualisation approach. Sci Rep 7, 12460 (2017). [PubMed: 
28963561] 

112. Freshour SL et al. Integration of the Drug-Gene Interaction Database (DGIdb 4.0) with open 
crowdsource efforts. Nucleic Acids Res 49, D1144–D1151 (2021). [PubMed: 33237278] 
113. Roth BL, Lopez E, Patel S & Kroeze WK The Multiplicity of Serotonin Receptors: Uselessly 

Diverse Molecules or an Embarrassment of Riches? The Neuroscientist 6, 252–262 (2000).

114. Mendez D et al. ChEMBL: towards direct deposition of bioassay data. Nucleic Acids Res 47, 

D930–D940 (2019). [PubMed: 30398643] 

115. Sheils TK et al. TCRD and Pharos 2021: mining the human proteome for disease biology. Nucleic 

Acids Res 49, D1334–D1346 (2021). [PubMed: 33156327] 

116. Yoo M et al. DSigDB: drug signatures database for gene set analysis. Bioinformatics 31, 3069–

3071 (2015). [PubMed: 25990557] 

117. Bulik-Sullivan B et al. An atlas of genetic correlations across human diseases and traits. Nat 

Genet 47, 1236–1241 (2015). [PubMed: 26414676] 

Nat Genet. Author manuscript; available in PMC 2024 September 13.

 
 
 
 
Nievergelt et al.

Page 46

118. Maihofer AX nievergeltlab/PTSDF3: Release V0.99. Zenodo. 10.5281/zenodo.10182702 (2023).

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Nat Genet. Author manuscript; available in PMC 2024 September 13.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Nievergelt et al.

Page 47

Figure 1 |. Data sources and analyses in PTSD Freeze 3.
a, Data sources of genome-wide association studies (GWAS) included in PGC-PTSD 
Freeze 3. Collections of contributing studies are pictured as bubble plots where each 
circle represents a contributing study. Circle areas are proportional to sample size and 
colors indicate the ancestry classification of participants (blue, EA; red, AA; purple, LAT). 
Arrowed lines indicate data sources being pooled together to perform GWAS meta-analyses 
stratified by ancestry. b, Methods applied for genetic characterization of PTSD, gene 
prioritization analyses, and translational applications. Abbreviations: EA, European ancestry, 
AA, African ancestry, LAT, Native American (Latin American) ancestry; EHR, electronic 
health record

Nat Genet. Author manuscript; available in PMC 2024 September 13.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Nievergelt et al.

Page 48

Figure 2 |. GWAS meta-analyses in European and multi-ancestry individuals identify a total of 
95 PTSD risk loci.
Overlaid Manhattan plots of European ancestry (EA; 137,136 cases and 1,085,746 controls) 
and multi-ancestry meta-analyses (150,760 cases and 1,130,173 controls), showing 81 
genome-wide significant (GWS) loci for the EA (full circles) and 85 GWS loci for the 
multi-ancestry (hollow circles) analyses. Circle colors alternate between chromosomes, with 
even chromosomes colored blue and odd chromosomes colored black. The y-axis refers to 
−log10 P-values from two-sided z-tests for meta-analysis effect estimates. The horizontal red 
bar indicates the threshold for GWS associations (P < 5 × 10−8).

Nat Genet. Author manuscript; available in PMC 2024 September 13.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Nievergelt et al.

Page 49

Figure 3 |. Manhattan plots of PTSD associations in multi-omic analyses.
a,b, Gene expression data from 13 brain tissue types and the pituitary were used to conduct 
transcriptome-wide association study (TWAS) identifying 9 loci with differential expression 
between PTSD cases and controls (a) and expression quantitative trait locus summary 
based Mendelian randomization (eQTL SMR) identifying 4 loci where gene expression has 
putative causal effects on PTSD (b). c, Blood protein quantitative trait locus (pQTL) SMR 
identify 16 blood proteins whose abundance has a putative causal effect on PTSD. The 
y-axis refers to −log10 P-values from two-sided z-tests for TWAS, two-sided Chi-square 
tests for eQTL SMR, and two-sided Chi-square tests for pQTL SMR. The horizontal red 
bars indicate gene-wide significance (P < 0.05/14,935 for TWAS, P < 0.05/9,903 for eQTL 
SMR, and P < 0.05/1,209 for pQTL SMR). Significant findings are labeled.

Nat Genet. Author manuscript; available in PMC 2024 September 13.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Nievergelt et al.

Page 50

Figure 4 |. Gene prioritization in PTSD loci.
Summary of evidence categories of prioritized genes (Tier 1 or 2) for the top 20% of PTSD 
loci (as ranked by leading SNP P-value). Locus number, prioritized genes within locus, gene 
locations (in terms of cytogenic band), and gene tier ranks (Tier 1, orange; Tier 2, blue) 
are indicated on the left. Categories of evidence are grouped and colored according to the 
domain they belong to. CADD scores, pLI scores and fine-mapping PIPs are written within 
their respective squares. The total weighted scores taken across all 9 evidence categories 
are shown on the rightmost squares. Abbreviations: eQTL, expression QTL; CI, chromatin 

Nat Genet. Author manuscript; available in PMC 2024 September 13.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Nievergelt et al.

Page 51

interaction; CADD, combined annotation dependent depletion; RDB, regulome DB; pLI, 
predicted loss of impact; PIP, posterior importance probability; TWAS, transcriptome-wide 
association study; SMR, summary Mendelian randomization; pQTL, protein QTL.

Nat Genet. Author manuscript; available in PMC 2024 September 13.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Nievergelt et al.

Page 52

Figure 5 |. Polygenic risk score analysis for PTSD across different data sets and ancestries.
PGC-PTSD Freeze 2 and Freeze 3 European ancestry (EA) based genetic risk score 
(PRS) predictions into independent samples of different ancestries. The y-axis represents 
PTSD risk relative to the lowest quintile of PRS with 95% confidence intervals. For 
EA, predictions based on Freeze 3 training data (10,334 cases and 55,504 controls; blue 
circles) demonstrate a significant performance increase compared to predictions based on 
the previous Freeze 2 training GWAS11 (yellow circles). Based on Freeze 3 EA training 
data, EA individuals in the highest quintile of PRS have 2.40 (95% CI = [2.26,2.56]) fold 

Nat Genet. Author manuscript; available in PMC 2024 September 13.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Nievergelt et al.

Page 53

the risk of PTSD relative to individuals in the lowest quintile PRS (blue circles). Lower 
prediction accuracies are found for individuals of African (AA; 10,151 cases and 22,420 
controls; red circles) and Native American (LAT; 5,346 cases and 10,821 controls; purple 
circles) ancestries, indicating poor PRS transferability across ancestries.

Nat Genet. Author manuscript; available in PMC 2024 September 13.

 
 
 
 
Nievergelt et al.

Page 54

Genome-wide significant loci associated with PTSD in the multi-ancestry and European 
PGC-PTSD Freeze 3 data

Table 1 |

Locusa

Lead 
SNPb

Chr

Start

Stop

A1 A2

A1 
Freq

Z 
score

P-valued

A1 
Freq

Z 
score

P-valued

A1 
Freq

Z 
score

P-
valued

A1 
Freq

Z 

score

P-

valued

Multi-ancestryc (150,793 
cases, 1,130,197 controls)

EAc (137,136 cases, 
1,085,746 controls)

AAc (11,560 cases, 
39,474 controls)

LATc (2,064 cases, 4,953 

controls)

1

2

2

3

3

4

4

5

6

6

7

7

8

9

9

10

11

12

13

13

14

15

15

16

16

17

17

18

rs78201023

rs617099

rs12026766

rs2186120

rs7519259

rs10789373

rs12128161

rs2207285

rs169235

rs4652676

rs9287117

rs9651063

rs2011374

rs10865093

rs6759922

rs1866560

rs10496632

rs6430728

rs28380327

rs10191758

rs197261

rs6800583

rs748832

rs4373086

rs6800637

rs6801151

rs6802567

rs7431106

1

1

1

1

1

1

1

1

1

1

1

1

1

2

2

2

2

2

2

2

2

3

3

3

3

3

3

3

35,664,657

36,375,226

38,198,744

38,459,210

38,198,744

38,459,210

66,392,405

66,584,457

66,392,405

66,547,212

73,279,823

74,108,971

73,275,828

74,099,273

88,790,511

88,836,922

181,698,693

181,747,349

181,698,693

181,747,349

191,154,894

191,418,368

191,154,894

191,418,368

214,094,735

214,139,159

22,430,795

22,613,427

22,430,795

22,613,427

27,186,507

27,345,484

124,953,763

125,053,393

138,097,204

138,334,702

144,145,478

144,263,280

144,145,478

144,272,229

161,866,881

162,095,003

16,843,737

16,879,208

16,843,737

16,879,208

18,611,283

18,824,298

18,611,283

18,824,298

43,249,957

43,591,405

43,249,957

43,594,564

49,734,229

50,644,134

T

A

A

A

A

C

A

A

A

A

T

T

A

T

A

T

C

A

A

A

A

A

A

A

A

A

T

A

C NA

NA

NA

0.038

6.264

3.76E-10 NA

NA

NA

NA

NA

NA

T

G

G

G

G

C

C

G

G

C

C

T

C

G

G

G

G

T

G

G

G

G

G

T

G

C

G

0.692

−5.525

3.30E-08

0.713

−5.283

1.27E-07

0.386

−1.859

0.277

5.425

5.80E-08

0.272

5.538

3.06E-08

0.329

0.358

0.504

5.592

2.24E-08

0.529

5.388

7.12E-08

0.146

2.314

0.511

5.335

9.58E-08

0.532

5.514

3.50E-08

0.198

0.665

0.606

−8.061

7.59E-16

0.619

−7.828

4.95E-15

0.447

−2.193

0.608

−7.946

1.93E-15

0.615

−7.882

3.23E-15

0.539

−1.640

0.06

0.72

0.02

0.51

0.03

0.10

0.585

0.283

0.399

−0.028

0.422

−1.787

0.426

−1.500

0.320

0.290

0.328

0.587

0.157

−5.557

2.75E-08

0.158

−5.311

1.09E-07

0.157

−1.132

0.26

0.144

−1.610

0.11

0.747

−6.509

7.56E-11

0.751

−6.191

6.00E-10

0.702

−1.560

0.251

6.247

4.19E-10

0.248

6.396

1.60E-10

0.280

−0.284

0.384

6.378

1.80E-10

0.369

6.390

1.66E-10

0.630

0.736

0.378

6.337

2.35E-10

0.367

6.409

1.47E-10

0.553

0.602

0.12

0.78

0.46

0.55

0.651

−1.598

0.349

1.553

0.256

0.439

0.253

0.181

0.510

−6.093

1.11E-09

0.505

−5.738

9.58E-09

0.600

−0.760

0.45

0.389

−3.818

0.00

0.545

−8.585

9.08E-18

0.541

−8.736

2.41E-18

0.645

−0.548

0.460

8.572

1.01E-17

0.457

8.821

1.13E-18

0.461

0.163

0.58

0.87

0.255

−0.279

0.746

0.299

0.78

0.76

0.464

−5.448

5.11E-08

0.449

−5.455

4.89E-08

0.658

−1.253

0.21

0.661

1.210

0.23

0.282

−6.698

2.12E-11

0.286

−6.680

2.40E-11

0.206

−0.839

0.40

0.330

−0.528

0.60

0.526

5.683

1.33E-08

0.511

5.996

2.02E-09

0.747

−0.008

0.99

0.548

−0.959

0.34

0.657

5.593

2.23E-08

0.642

5.355

8.58E-08

0.864

1.711

0.637

5.517

3.44E-08

0.626

5.487

4.08E-08

0.789

0.860

0.727

5.373

7.74E-08

0.718

5.994

2.05E-09

0.837

−1.129

0.383

6.117

9.54E-10

0.373

6.366

1.95E-10

0.541

0.157

0.618

−5.785

7.25E-09

0.627

−6.377

1.80E-10

0.483

1.219

0.732

5.648

1.62E-08

0.722

5.765

8.19E-09

0.878

1.166

0.722

5.599

2.15E-08

0.711

5.923

3.16E-09

0.853

0.520

0.163

6.326

2.52E-10

0.156

6.098

1.08E-09

0.275

1.860

0.876

−6.263

3.79E-10

0.883

−6.726

1.74E-11

0.763

0.959

0.09

0.39

0.26

0.88

0.22

0.24

0.60

0.06

0.34

0.811

0.085

0.803

0.150

0.851

−1.276

0.360

−0.578

0.641

0.583

0.758

−2.025

0.747

−2.028

0.065

−0.132

0.953

−0.155

0.78

0.98

0.07

0.13

0.77

0.56

0.11

0.12

0.66

0.86

0.93

0.88

0.20

0.56

0.56

0.04

0.04

0.90

0.88

0.491

7.941

2.00E-15

0.488

7.304

2.78E-13

0.582

3.090

0.00

0.247

1.372

0.17

Nat Genet. Author manuscript; available in PMC 2024 September 13.

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

 
 
 
 
Nievergelt et al.

Page 55

Locusa

Lead 
SNPb

Chr

Start

Stop

A1 A2

A1 
Freq

Z 
score

P-valued

A1 
Freq

Z 
score

P-valued

A1 
Freq

Z 
score

P-
valued

A1 
Freq

Z 

score

P-

valued

Multi-ancestryc (150,793 
cases, 1,130,197 controls)

EAc (137,136 cases, 
1,085,746 controls)

AAc (11,560 cases, 
39,474 controls)

LATc (2,064 cases, 4,953 

controls)

18

19

20

21

22

23

23

24

25

25

26

27

27

28

29

30

31

32

32

33

34

35

35

36

37

38

38

39

39

rs11130221

rs1541903

rs28758576

rs34811474

rs12509393

rs10939933

rs12521971

rs4489042

rs6867409

rs33817

rs295017

rs13161115

rs13161130

rs34425

rs175086

rs251352

rs4257818

rs11167640

rs13168358

rs2135029

rs11957630

rs28986300

rs29242

rs180963

rs9479138

rs868754

rs34809719

rs10264275

rs35791987

3

3

3

4

4

5

5

5

5

5

5

5

5

5

5

5

5

5

5

5

5

6

6

6

6

7

7

7

7

49,734,229

50,644,134

71,303,875

71,344,078

135,476,532

135,602,459

25,342,606

25,408,838

28,273,059

28,347,050

61,398,053

61,683,591

61,398,053

61,683,591

92,362,700

92,538,853

103,684,787

104,055,261

103,791,044

104,055,261

106,118,410

106,215,439

106,918,329

107,084,359

106,918,329

107,060,400

107,349,092

107,769,562

139,517,197

139,700,608

140,225,137

140,331,337

152,505,453

152,610,561

153,085,668

153,241,171

153,085,668

153,255,743

155,730,681

155,912,474

164,467,717

164,678,946

26,748,873

29,607,101

25,846,381

29,607,101

100,914,602

101,339,400

152,201,201

152,264,529

1,833,097

2,110,850

1,809,618

2,110,850

3,521,658

3,715,667

3,521,658

3,715,667

C

T

T

A

T

C

A

C

T

A

A

C

C

A

A

A

A

T

T

A

A

A

T

T

T

C

T

A

C

G

C

C

G

C

G

C

G

C

G

G

G

G

T

G

G

C

C

C

G

G

G

C

C

G

G

G

G

G

0.511

−7.792

6.60E-15

0.513

−7.328

2.33E-13

0.450

−2.410

0.02

0.752

−1.337

0.18

0.129

6.111

9.91E-10

0.134

5.939

2.87E-09

0.077

1.228

0.22

0.044

0.842

0.40

0.898

−5.606

2.08E-08

0.892

−5.305

1.13E-07

0.978

−1.675

0.09

0.968

−0.806

0.42

0.211

−5.395

6.85E-08

0.223

−5.544

2.95E-08

0.054

−0.454

0.65

0.076

0.640

0.52

0.255

−5.456

4.87E-08

0.262

−5.188

2.13E-07

0.136

−1.597

0.11

0.325

−0.619

0.54

0.509

5.956

2.58E-09

0.517

6.161

7.23E-10

0.387

0.358

0.509

5.942

2.81E-09

0.517

6.178

6.49E-10

0.386

0.237

0.72

0.81

0.472

−0.736

0.472

−0.739

0.46

0.46

0.401

−5.983

2.19E-09

0.382

−5.610

2.02E-08

0.662

−2.045

0.04

0.554

−0.700

0.48

0.465

5.961

2.50E-09

0.461

5.973

2.33E-09

0.506

0.688

0.421

5.942

2.82E-09

0.428

6.358

2.04E-10

0.314

−0.440

0.49

0.66

0.597

0.352

0.451

−0.814

0.73

0.42

0.298

5.808

6.31E-09

0.309

5.597

2.19E-08

0.159

1.265

0.21

0.139

1.034

0.30

0.226

−6.961

3.38E-12

0.238

−6.615

3.72E-11

0.060

−2.664

0.232

−6.955

3.54E-12

0.242

−6.650

2.94E-11

0.094

−2.508

0.01

0.01

0.078

0.767

0.081

0.785

0.44

0.43

0.317

6.172

6.75E-10

0.307

6.130

8.80E-10

0.475

1.230

0.22

0.170

−0.313

0.75

0.514

6.182

6.35E-10

0.517

5.817

5.99E-09

0.498

1.860

0.06

0.356

1.142

0.25

0.540

−5.581

2.39E-08

0.555

−5.203

1.96E-07

0.354

−1.476

0.14

0.478

−1.961

0.05

0.388

5.944

2.79E-09

0.405

5.942

2.82E-09

0.198

0.628

0.53

0.164

0.741

0.46

0.776

5.824

5.75E-09

0.778

5.869

4.39E-09

0.791

0.617

0.224

−5.823

5.79E-09

0.222

−5.873

4.27E-09

0.209

−0.594

0.54

0.55

0.463

0.138

0.537

−0.138

0.89

0.89

0.615

−6.812

9.64E-12

0.606

−7.251

4.14E-13

0.774

0.790

0.43

0.435

−0.226

0.82

0.446

−6.249

4.14E-10

0.458

−5.847

5.01E-09

0.298

−2.021

0.04

0.250

−1.147

0.25

0.936

−7.468

8.14E-14

0.939

−7.343

2.09E-13

0.908

−1.056

0.947

−7.368

1.73E-13

0.946

−7.464

8.39E-14

0.979

−0.272

0.29

0.79

0.851

−1.334

0.865

−1.117

0.18

0.26

0.493

5.848

4.99E-09

0.496

5.468

4.56E-08

0.465

2.030

0.04

0.383

0.756

0.45

0.372

7.282

3.30E-13

0.349

7.272

3.54E-13

0.670

1.377

0.17

0.599

−0.668

0.50

0.195

−6.954

3.54E-12

0.205

−6.791

1.11E-11

0.067

−1.506

0.199

−6.943

3.85E-12

0.209

−6.849

7.43E-12

0.067

−1.228

0.763

5.663

1.49E-08

0.762

5.287

1.24E-07

0.796

2.115

0.239

−5.653

1.58E-08

0.237

−5.519

3.40E-08

0.247

−1.078

0.13

0.22

0.03

0.28

0.098

−0.363

0.098

−0.372

0.618

0.446

0.380

−0.670

0.72

0.71

0.66

0.50

Nat Genet. Author manuscript; available in PMC 2024 September 13.

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

 
 
 
 
Nievergelt et al.

Page 56

Locusa

Lead 
SNPb

Chr

Start

Stop

A1 A2

A1 
Freq

Z 
score

P-valued

A1 
Freq

Z 
score

P-valued

A1 
Freq

Z 
score

P-
valued

A1 
Freq

Z 

score

P-

valued

Multi-ancestryc (150,793 
cases, 1,130,197 controls)

EAc (137,136 cases, 
1,085,746 controls)

AAc (11,560 cases, 
39,474 controls)

LATc (2,064 cases, 4,953 

controls)

40

41

41

42

43

44

45

46

47

47

48

49

50

51

52

53

54

54

55

56

56

57

58

59

60

60

61

61

62

rs13237518

rs4722031

rs2107448

rs4732514

rs58043442

rs2470937

rs1476535

rs7806900

rs2214230

rs61702433

rs34618371

rs10487459

rs10104247

rs4129585

rs13284172

rs13290462

rs10821163

rs10992779

rs4838242

rs515805

rs67736073

rs1114148

rs1124372

rs2771265

rs11192260

rs7914674

rs4350350

rs11529859

rs488769

7

7

7

7

7

7

7

7

7

7

7

7

8

8

9

9

9

9

9

9

9

10

10

10

10

10

11

11

11

12,233,848

12,285,140

21,468,640

21,555,536

21,468,640

21,555,536

75,607,155

75,852,480

82,386,297

82,641,937

104,558,434

105,063,372

113,858,363

114,290,415

114,940,147

115,113,279

117,502,574

117,668,235

117,502,574

117,636,111

124,392,512

124,710,858

126,371,011

126,507,903

9,397,184

9,641,034

143,297,329

143,479,815

14,578,127

14,688,114

31,124,452

31,251,063

96,106,521

96,386,972

96,181,075

96,381,916

127,765,978

127,898,032

137,932,302

137,965,157

137,932,302

137,965,157

67,521,802

67,548,278

77,537,562

77,660,164

93,791,409

94,134,467

106,563,924

106,830,537

106,563,924

106,830,537

28,591,168

28,709,434

28,642,381

28,695,181

A

A

T

T

T

A

T

A

T

T

T

A

A

A

T

T

C

A

A

A

T

T

A

T

T

T

T

T

57,369,008

57,681,828

A

C

T

C

C

C

T

C

G

G

G

C

G

C

C

C

G

G

G

G

G

C

C

G

C

C

C

C

C

C

0.435

5.006

5.55E-07

0.414

5.457

4.83E-08

0.689

−0.699

0.48

0.627

−0.445

0.66

0.294

6.325

2.53E-10

0.308

5.798

6.69E-09

0.117

3.441

0.572

−5.958

2.55E-09

0.581

−5.823

5.78E-09

0.492

−1.725

0.00

0.08

0.228

−1.389

0.273

0.846

0.16

0.40

0.645

5.470

4.51E-08

0.660

5.026

5.00E-07

0.425

2.385

0.02

0.575

0.347

0.73

0.474

−5.753

8.79E-09

0.488

−6.179

6.44E-10

0.278

0.370

0.71

0.394

1.116

0.26

0.454

6.391

1.65E-10

0.450

6.246

4.22E-10

0.506

1.469

0.14

0.493

0.069

0.95

0.576

9.183

4.18E-20

0.561

8.807

1.28E-18

0.765

2.564

0.01

0.605

0.589

0.56

0.535

6.258

3.91E-10

0.519

5.981

2.22E-09

0.769

1.786

0.07

0.531

0.507

0.61

0.374

6.166

7.01E-10

0.385

5.885

3.98E-09

0.229

1.257

0.348

6.053

1.42E-09

0.362

5.994

2.05E-09

0.160

0.507

0.21

0.61

0.228

1.859

0.220

1.568

0.06

0.12

0.607

−5.500

3.80E-08

0.602

−5.054

4.33E-07

0.703

−1.551

0.12

0.553

−2.510

0.01

0.622

−6.441

1.19E-10

0.612

−6.460

1.05E-10

0.768

−1.219

0.22

0.638

0.912

0.36

0.814

−5.501

3.77E-08

0.816

−5.462

4.71E-08

0.762

−1.258

0.21

0.906

0.681

0.50

0.412

6.705

2.02E-11

0.432

6.654

2.85E-11

0.135

1.346

0.18

0.209

−0.438

0.66

0.628

−5.478

4.29E-08

0.626

−5.211

1.88E-07

0.642

−1.846

0.06

0.783

0.023

0.98

0.679

5.610

2.02E-08

0.669

6.037

1.58E-09

0.791

−1.095

0.27

0.709

1.196

0.23

0.339

−7.255

4.00E-13

0.355

−6.915

4.67E-12

0.102

−1.956

0.661

7.131

9.96E-13

0.647

6.982

2.92E-12

0.874

1.239

0.05

0.22

0.313

−1.067

0.688

0.970

0.29

0.33

0.470

5.546

2.92E-08

0.472

5.080

3.78E-07

0.410

1.805

0.07

0.674

2.080

0.04

0.185

5.824

5.75E-09

0.187

5.567

2.60E-08

0.144

1.734

0.150

5.682

1.33E-08

0.147

5.795

6.84E-09

0.194

0.352

0.08

0.72

0.238

0.294

0.102

0.226

0.77

0.82

0.189

5.674

1.40E-08

0.156

5.384

7.30E-08

0.666

1.469

0.14

0.245

1.222

0.22

0.730

5.015

5.30E-07

0.720

5.539

3.05E-08

0.858

−1.715

0.09

0.854

1.078

0.28

0.327

−5.503

3.74E-08

0.343

−5.393

6.94E-08

0.095

−1.023

0.31

0.255

−0.522

0.60

0.221

−5.571

2.54E-08

0.212

−5.444

5.20E-08

0.370

−1.213

0.755

5.515

3.49E-08

0.789

5.694

1.24E-08

0.252

0.127

0.391

−5.813

6.13E-09

0.380

−5.444

5.20E-08

0.564

−2.454

0.724

5.278

1.31E-07

0.713

5.579

2.41E-08

0.862

−0.218

0.23

0.90

0.01

0.83

0.092

−0.239

0.894

0.045

0.325

0.435

0.877

−0.484

0.81

0.96

0.66

0.63

0.658

−8.577

9.72E-18

0.656

−8.815

1.20E-18

0.653

−0.842

0.40

0.865

1.328

0.18

Nat Genet. Author manuscript; available in PMC 2024 September 13.

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

 
 
 
 
Nievergelt et al.

Page 57

Locusa

Lead 
SNPb

Chr

Start

Stop

A1 A2

A1 
Freq

Z 
score

P-valued

A1 
Freq

Z 
score

P-valued

A1 
Freq

Z 
score

P-
valued

A1 
Freq

Z 

score

P-

valued

Multi-ancestryc (150,793 
cases, 1,130,197 controls)

EAc (137,136 cases, 
1,085,746 controls)

AAc (11,560 cases, 
39,474 controls)

LATc (2,064 cases, 4,953 

controls)

63

64

64

65

66

67

68

68

69

70

70

71

72

72

73

74

74

75

76

77

78

79

80

81

81

82

82

83

rs559566

rs7106434

rs2186710

11

11

11

64,567,047

64,584,231

112,826,867

113,034,787

112,826,867

113,034,787

rs10842260

12

24,166,426

24,225,819

rs2292996

12

103,447,647

103,556,972

rs816363

12

117,649,880

117,700,047

rs16948230

rs61946067

rs1373273

rs17084460

rs7333625

12

12

13

13

13

118,585,698

118,888,131

118,585,698

118,888,131

53,865,141

54,039,629

69,561,090

69,687,825

69,561,090

69,687,825

rs11628299

14

42,036,322

42,697,579

rs57167554

rs2899991

rs7141058

rs11552464

rs10132977

rs7170398

14

14

14

14

14

15

47,238,606

47,448,072

47,238,606

47,448,072

69,429,386

69,765,644

103,229,696

103,387,971

103,230,005

103,387,971

77,995,949

78,146,382

rs17514846

15

91,412,850

91,429,042

rs1861188

rs6416794

16

16

6,310,645

6,345,984

51,172,677

51,202,778

rs12930480

16

52,232,367

52,327,267

rs7200432

rs7224932

rs14313371
7

rs199526

rs2684641

16

17

17

17

17

60,644,510

60,745,208

30,173,581

30,571,416

30,173,581

30,571,416

43,460,181

44,865,603

43,460,181

44,865,603

rs73338706

17

65,822,573

66,098,979

A

T

C

A

T

C

A

T

A

A

A

A

A

T

A

T

T

T

A

A

T

A

A

C

T

C

A

T

G

C

G

G

C

G

G

C

C

T

T

G

G

C

G

G

C

C

C

G

C

C

G

G

C

G

G

C

0.548

−5.665

1.47E-08

0.560

−5.786

7.19E-09

0.364

0.070

0.94

0.612

−1.148

0.25

0.462

8.125

4.46E-16

0.450

8.410

4.09E-17

0.593

−0.745

0.540

−7.992

1.33E-15

0.545

−8.456

2.78E-17

0.501

1.327

0.46

0.18

0.674

2.180

0.328

−1.896

0.03

0.06

0.467

−5.210

1.89E-07

0.466

−5.577

2.44E-08

0.510

0.554

0.58

0.284

0.305

0.76

0.476

−5.732

9.92E-09

0.496

−5.817

6.00E-09

0.191

−0.355

0.72

0.382

−0.397

0.69

0.580

−5.870

4.37E-09

0.597

−5.762

8.29E-09

0.397

−1.415

0.16

0.334

0.270

0.79

0.858

−6.637

3.20E-11

0.855

−6.522

6.95E-11

0.894

−1.338

0.862

−6.119

9.44E-10

0.855

−6.536

6.34E-11

0.960

0.960

0.18

0.34

0.943

−0.197

0.943

−0.562

0.84

0.57

0.565

−5.838

5.28E-09

0.566

−5.583

2.37E-08

0.544

−0.938

0.35

0.616

−2.310

0.02

0.916

5.714

1.11E-08

0.919

5.623

1.88E-08

0.858

0.985

0.900

5.203

1.96E-07

0.910

5.680

1.34E-08

0.740

−1.397

0.32

0.16

0.963

0.510

0.956

0.790

0.61

0.43

0.448

5.679

1.36E-08

0.427

5.708

1.14E-08

0.760

0.608

0.54

NA

NA

NA

0.529

−6.654

2.86E-11

0.524

−6.823

8.91E-12

0.659

−0.022

0.470

6.617

3.67E-11

0.475

6.840

7.91E-12

0.339

−0.153

0.98

0.88

0.207

−0.753

0.793

0.655

0.45

0.51

0.472

−5.845

5.08E-09

0.480

−5.987

2.14E-09

0.398

−0.154

0.88

0.205

−0.380

0.70

0.838

−6.427

1.31E-10

0.834

−6.009

1.86E-09

0.937

−2.548

0.203

6.074

1.25E-09

0.174

6.227

4.74E-10

0.605

0.328

0.01

0.74

0.578

−0.004

0.442

−0.092

1.00

0.93

0.400

6.583

4.62E-11

0.412

6.459

1.05E-10

0.254

1.242

0.21

0.307

0.528

0.60

0.484

−6.925

4.36E-12

0.459

−6.870

6.44E-12

0.801

−1.151

0.25

NA

NA

NA

0.654

−5.733

9.88E-09

0.683

−5.514

3.50E-08

0.228

−1.544

0.12

0.680

−0.358

0.72

0.234

5.572

2.52E-08

0.214

5.252

1.51E-07

0.501

1.474

0.14

0.140

1.405

0.16

0.821

5.837

5.30E-09

0.811

5.216

1.82E-07

0.950

2.459

0.01

0.923

1.841

0.07

0.306

5.121

3.03E-07

0.306

5.456

4.87E-08

0.311

−0.573

0.57

0.249

0.049

0.96

0.165

−5.778

7.55E-09

0.165

−5.599

2.16E-08

0.175

−1.418

0.16

0.148

−0.312

0.75

0.130

−5.511

3.56E-08

0.139

−5.771

7.87E-09

0.033

0.115

0.91

0.049

−0.154

0.88

0.212

6.820

9.11E-12

0.211

6.414

1.42E-10

0.193

1.927

0.824

−6.253

4.03E-10

0.820

−6.646

3.01E-11

0.850

0.450

0.05

0.65

0.482

1.609

0.910

−0.141

0.11

0.89

0.761

−6.435

1.24E-10

0.789

−6.524

6.84E-11

0.406

−0.512

0.61

0.671

−0.399

0.69

Nat Genet. Author manuscript; available in PMC 2024 September 13.

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

 
 
 
 
Nievergelt et al.

Page 58

Locusa

Lead 
SNPb

Chr

Start

Stop

A1 A2

A1 
Freq

Z 
score

P-valued

A1 
Freq

Z 
score

P-valued

A1 
Freq

Z 
score

P-
valued

A1 
Freq

Z 

score

P-

valued

Multi-ancestryc (150,793 
cases, 1,130,197 controls)

EAc (137,136 cases, 
1,085,746 controls)

AAc (11,560 cases, 
39,474 controls)

LATc (2,064 cases, 4,953 

controls)

rs74515851

17

73,431,367

73,497,272

84

85

86

87

88

89

90

90

91

92

93

94

95

rs7243332

rs9954874

rs4632195

rs896686

rs7408312

rs13037326

rs6032660

rs1378559

rs10284205

rs1320317

rs11205253
4

rs2266850

18

18

18

18

19

20

20

X

X

X

X

X

26,570,584

26,611,564

42,843,373

42,843,373

50,555,225

51,055,069

53,072,319

53,464,917

18,412,122

18,444,809

44,680,412

44,747,947

44,680,853

44,749,251

21,074,049

21,696,222

24,914,760

24,927,706

117,333,327

117,339,104

A

A

T

T

T

T

T

A

T

T

T

G

G

C

C

G

G

C

G

C

C

C

0.044

−5.730

1.00E-08

0.046

−5.323

1.02E-07

0.017

−2.326

0.02

NA

NA

NA

0.666

−5.602

2.12E-08

0.680

−5.591

2.26E-08

0.455

−1.043

0.30

0.723

0.431

0.67

0.594

5.715

1.10E-08

0.573

5.369

7.91E-08

0.898

1.723

0.08

0.641

1.152

0.25

0.524

7.064

1.62E-12

0.522

7.029

2.07E-12

0.561

1.030

0.30

NA

NA

NA

0.839

6.232

4.60E-10

0.830

6.195

5.82E-10

0.958

0.820

0.41

0.948

0.584

0.56

0.440

−5.885

3.98E-09

0.454

−6.072

1.26E-09

0.198

−0.133

0.89

0.690

−0.102

0.92

0.250

5.724

1.04E-08

0.262

5.590

2.27E-08

0.074

0.642

0.763

−5.704

1.17E-08

0.751

−5.744

9.23E-09

0.931

0.059

0.52

0.95

0.152

1.806

0.850

−1.888

0.07

0.06

0.852

5.539

3.05E-08

0.846

5.609

2.04E-08

0.978

−0.484

0.63

0.957

1.178

0.24

0.591

−5.567

2.59E-08

0.587

−5.513

3.53E-08

0.705

−0.999

0.32

0.605

−0.048

0.96

0.050

−5.328

9.94E-08

0.031

−5.519

3.40E-08

0.642

0.408

0.68

0.028

−0.108

0.91

130,428,965

130,432,493

A

G

0.696

−5.929

3.05E-09

0.685

−5.971

2.35E-09

0.930

−0.881

0.38

0.861

0.454

0.65

149,791,188

149,798,641

C

G

0.478

5.385

7.26E-08

0.492

5.678

1.36E-08

0.089

−0.219

0.83

0.352

−0.958

0.34

Abbreviations: EA, European ancestry; AA, African ancestry; LAT, Native American (Latin American) ancestry; Chr, chromosome; Start, locus 
start position in base pairs (GR37 Human Genome Build/h19 coordinates; Stop, locus stop position; A1, coded allele (effects and allele frequencies 
are coded in terms of copies of this allele); A2, non-coded allele; A, adenosine; C, cytosine; G, guanidine; T, thymidine; A1 Freq, frequency of 
allele 1.

a

Loci number designations used in manuscript and gene-mapping tables.

b

Where leading marker varied between EA and multi-ancestry, results for both leading markers shown.

c

Results highlighted in color indicate leading SNPs for a specific locus and ancestry.

d
Meta-analysis effect estimates were tested for significance using two-sided z-tests. Results bolded where genome-wide significant (P < 5 × 10−8).

Nat Genet. Author manuscript; available in PMC 2024 September 13.

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t
