# Ou_2024_Lithium response in bipolar disorder is associated with focal adhesion and PI3K-Akt networks a multi-omics replication study.

:
,
;
)
(
0
9
8
7
6
5
4
3
2
1

Translational Psychiatry

www.nature.com/tp

OPEN

ARTICLE
Lithium response in bipolar disorder is associated with focal
adhesion and PI3K-Akt networks: a multi-omics
replication study

9,

9, Francesc Colom18,
28, Clarissa R. Dantas29, Alexandre Dayer13, Maria Del Zompo 10,30,

7,8, Azmeraw T. Amare
9,16,

20,21,

42, Joanna Hauser

27, Piotr M. Czerski

15, Bernhard T. Baune

11,12, Jean-Michel Aubry13, Lena Backlund14, Michael Bauer

31, Bruno Étain 17, Peter Falkai32, Frederike Tabea Fellendorf19,

1, Sara B. Rosenthal2, Mazda Adli3,4, Kazufumi Akiyama5, Nirmala Akula6, Martin Alda

6, Stephane Jamain 43, Esther Jiménez18, Jean-Pierre Kahn44, Layla Kassem6, Tadafumi Kato

Anna H. Ou
Raffaella Ardau10, Bárbara Arias
Frank Bellivier17, Antonio Benabarre18, Susanne Bengesser19, Abesh Kumar Bhattacharjee1, Joanna M. Biernacka
Pablo Cervantes22, Guo-Bo Chen22, Hsi-Chung Chen 23, Caterina Chillotti10, Sven Cichon24,25, Scott R. Clark
David A. Cousins26, Cristiana Cruceanu
Franziska Degenhardt24, J. Raymond DePaulo
Ewa Ferensztajn-Rochowiak33, Andreas J. Forstner
24, Louise Frisén14,34,35, Mark A. Frye
Sébastien Gard38, Julie S. Garnham 7, Fernando S. Goes31, Maria Grigoroiu-Serbanescu
Ryota Hashimoto
Liping Hou
Sarah Kittel-Schneider46, Barbara König47, Po-Hsiu Kuo
Catharina Lavebratt
Mirko Manchia
Francis J. McMahon 6, Philip B. Mitchell
8, Urban Ösby66, Norio Ozaki
Caroline M. Nievergelt
Claudia Pisanu30, James B. Potash70, Andrea Pfennig15, Daniela Reich-Erkelenz32, Andreas Reif
71, Guy A. Rouleau
Marcella Rietschel
9, Thomas G. Schulze
K. Oliver Schubert
Tatyana Shekhtman1,73, Paul D. Shilling 1, Kazutaka Shimoda
Thomas Stamm3, Pavla Stopkova8, Sarah K. Tighe70,76, Alfonso Tortorella56, Gustavo Turecki
Stephanie Witt

67, Sergi Papiol
46, Eva Z. Reininghaus
33, Martin Schalling 14, Peter R. Schoﬁeld 36,37,
6,31,32,41,71, Barbara W. Schweizer31, Florian Seemüller32, Giovanni Severino

77, Adam Wright63, L. Trevor Young78, Peter P. Zandi79 and John R. Kelsoe1,73 ✉

21, Janice M. Fullerton 36,37,
39, Paul Grof40, Oliver Gruber41,
32, Stefan Herms24,25, Per Hoffmann24,25, Andrea Hofmann24,

64, Francis M. Mondimore31, Palmiero Monteleone65,

1, Markus M. Nöthen 24, Tomas Novák

57,58, Cynthia Marie-Claire

72, Janusz K. Rybakowski

28, Urs Heilbronner

14, Marion Leboyer

63, Marina Mitjans

71, Naomi R. Wray

48, Ichiro Kusumi

27, Eduard Vieta

74, Christian Simhandl75, Claire M. Slaney7, Alessio Squassina

52, Susan G. Leckband53, Carlos A. López Jaramillo54, Glenda MacQueen55, Mario Maj56,
17, Lina Martinsson59, Manuel Mattheisen60, Michael J. McCarthy61, Susan L. McElroy62,

49, Nina Lackner19, Gonzalo Laje6, Mikael Landén 50,51,

30,
18, Julia Volkert46,

68, Roy H. Perlis69,

19,

30,

45,

© The Author(s) 2024

Lithium is the gold standard treatment for bipolar disorder (BD). However, its mechanism of action is incompletely understood, and
prediction of treatment outcomes is limited. In our previous multi-omics study of the Pharmacogenomics of Bipolar Disorder
(PGBD) sample combining transcriptomic and genomic data, we found that focal adhesion, the extracellular matrix (ECM), and PI3K-
Akt signaling networks were associated with response to lithium. In this study, we replicated the results of our previous study using
network propagation methods in a genome-wide association study of an independent sample of 2039 patients from the
International Consortium on Lithium Genetics (ConLiGen) study. We identiﬁed functional enrichment in focal adhesion and PI3K-Akt
pathways, but we did not ﬁnd an association with the ECM pathway. Our results suggest that deﬁcits in the neuronal growth cone
and PI3K-Akt signaling, but not in ECM proteins, may inﬂuence response to lithium in BD.

Translational Psychiatry  

 (2024) 14:109  ; https://doi.org/10.1038/s41398-024-02811-4

INTRODUCTION
Bipolar disorder (BD) is a chronic psychiatric illness that presents
with episodes of mania, depression, and sometimes psychosis.
Globally, it is the sixth leading cause of medical disability among
people from 15 to 44 years old. Patients with BD are at a higher
risk of suicide than those with any other psychiatric or medical
illness. Some studies report that roughly 50% of patients will
attempt suicide, and up to 20% of untreated patients will

complete suicide [1], while treatment by lithium reduces that risk
signiﬁcantly [2, 3]. Unfortunately, misdiagnosis is common and
often delays an accurate treatment. Up to 70% of patients are
initially misdiagnosed, usually with major depressive disorder. On
average, there is a delay of 8 years before the correct diagnosis of
BD is made [4]. During this time, patients continue to suffer, may
be treated with medications that make their illness course worse,
and are at risk of suicide.

A full list of author afﬁliations appears at the end of the paper.

Received: 12 August 2023 Revised: 6 December 2023 Accepted: 29 January 2024

 
 
 
 
 
 
 
2

A.H. Ou et al.

for BD [5].

In the PI cycle,

Lithium is the gold standard treatment

Its
mechanism of action is still not completely understood [6].
Many studies have investigated the neurotrophic effect of
lithium. One theory posits that chronic administration of lithium
inhibits glycogen synthase kinase 3 (GSK3β), a serine/threonine
kinase. This leads to anti-apoptotic effects and improved cell
structural stability [7–10]. GSK3β has also been shown to exhibit
interactions with many pathways, including phosphorylation of
several components of the PI3K/AKT/mTOR signaling network,
as well as regulation of transcription for proteins bound to
microtubules [11]. Another theory involves the phosphoinositol
(PI) cycle.
lithium inhibits inositol monopho-
sphatase, which ultimately downregulates protein kinase C
isozymes such as myristoylated alanine-rich C-kinase substrate
(MARCKS). MARCKS is an actin-binding protein found in
neuronal processes that is implicated in cytoskeletal restructur-
ing. Its downregulation stabilizes the neuronal membrane and
results in neurotrophic effects [7, 12]. A more recent theory
proposes that
lithium alters the phosphorylation state of
collapsin response mediator protein-2 (CRMP2). CRMP2 reg-
ulates cytoskeletal organization, particularly in dendritic spines
[13, 14]. Finally, a study using polygenic score modeling has
indicated that the cholinergic and glutamatergic pathways may
potentially serve as targets for lithium [15]. It is possible that
lithium exerts its effects through multiple or all of
these
pathways. A single deﬁnitive model
remains elusive, but
interactions with neuronal cytoskeleton are possibly involved.
Interestingly, there is a range of responses to treatment with
studies have reported that 20–30% of
lithium. Previous
patients with BD are excellent responders, whereas over 40%
fail to demonstrate any signiﬁcant clinical improvement. These
patient populations have been shown to differ from each other
both phenotypically and genetically [16]. A differential
response to lithium has been previously demonstrated
between induced pluripotent stem cell (iPSC) neurons derived
from lithium responders and non-responders. The hyperexcit-
ability of
in vitro neurons derived from BD patients was
reversed by lithium treatment, but only in those from patients
[17]. This ﬁnding is also
who were lithium responders
supported by family studies, which found that the relatives of
lithium responders were signiﬁcantly more likely to be lithium
responders as well [18, 19]. These studies imply that patients
with BD could be subcategorized based on biological
differences which induce a divergent lithium response. There
is a great need to better understand these differences in order
to identify possible predictors of treatment response. However,
candidate-gene association studies,
dozens of previous
(GWAS), and polygenic
genome-wide association studies
risk score analyses of lithium response in BD have failed to
identify genetic variants with major effects. Given this pressing
need to ﬁnd pharmacogenetic predictors of response, more
advanced methods
in integrative genomic analysis are
necessary [16].

GWAS inherently face several

including the challenge of genetic heterogeneity.

limitations when used in
isolation,
In
many disease processes with genetic associations, patients may
carry diverse combinations of causal variants that impact multiple
genes, creating a net effect across a particular pathway. GWAS of
BD primarily detect variants of very small effect size consistent
with a polygenic mode of
transmission. Since each single
nucleotide polymorphism (SNP) contributes only a tiny amount
to the overall predisposition to BD, enormous sample sizes are
required, and it can be difﬁcult to surmise mechanisms of disease.
Network approaches seek to address this biological reality by
integrating GWAS results with known protein-protein interactions
and other molecular networks. New causal genes may be
identiﬁed by boosting their interactions with products of known
causal genes [20, 21].

We have recently reported a combined analysis of transcrip-
tomic and GWAS data from the Pharmacogenomics of Bipolar
Disorder (PGBD) study [22] of treatment response to lithium. After
using network propagation to reprioritize candidate genes from
GWAS data, we found signiﬁcant overlap between both tran-
scriptomic and GWAS results. The joint analysis yielded a 500 gene
network signiﬁcantly enriched in the following Kyoto Encyclope-
dia of Genes and Genomes (KEGG) pathways: focal adhesion, ECM-
receptor interaction, and PI3K-Akt signaling [23]. All three path-
ways play a role in axon growth and neuronal development [24].
Consistent with these results, post-mortem studies have found
that in BD, neuronal populations may exhibit a decrease in
number, size, and/or amount of dendritic spines [13, 25]. Given
that lithium may have downstream effects on these pathways, it is
possible that genetic defects in focal adhesion pathways may
provide both a mechanism for susceptibility to BD as well as a
target for lithium treatment.

In this study, we aimed to replicate the results of our previous
multi-omics study on a larger dataset of over 2000 patients from
the International Consortium on Lithium Genetics (ConLiGen) [26].
We reprioritized GWAS results using network methods
to
determine overlap with focal adhesion, ECM-receptor interaction,
and PI3K-Akt signaling pathways.

METHODS
Summary statistics were downloaded from the NHGRI-EBI GWAS Catalog
[27] on 12/12/2022 for study GCST012487 [26]. The data resulted from a
GWAS of lithium response in 2563 patients at 22 sites participating in the
International Consortium on Lithium Genetics (ConLiGen). We utilized the
summary statistics from a combined sample of 2039 European ancestry
individuals.
In the ConLiGen study, data from over 6 million single
nucleotide polymorphisms (SNPs) were tested for association with
categorical and continuous retrospective ratings of lithium response using
the Alda scale [28, 29]. The Alda scale includes two scores: score A is a 0–10
retrospective rating of lifetime response, while score B captures factors
reducing the conﬁdence in score A such as lack of a documented lithium
level, etc.
In the ConLiGen study, under the continuous phenotype,
participants were rated with the Alda A score, and individuals with a B
score greater than 4 were excluded. We used the continuous rather than
the dichotomous phenotype as a measure of treatment response because
genome-wide signiﬁcant association was detected with the continuous
phenotype in the original GWAS. Quality control and statistical analysis
methods are described in the original paper.

SNP, gene, and gene-set analysis
We imported the ConLiGen summary statistics into FUMA (Functional
Mapping and Annotation of Genome-Wide Association Studies—https://
fuma.ctglab.nl) [30], a web-based platform for annotating, prioritizing,
visualizing and interpreting GWAS results. We utilized the SNP2GENE
function to map SNPs to genes and conduct SNP, gene-based, and gene-
set analysis. We used all default settings, except for setting the maximum
lead SNP p value to 1 × 10e−5.

Network analysis
We input the ConLiGen summary statistics into NAGA (Network Assisted
Genomic Analysis), an online network propagation tool
for pathway
boosting and interpretation of genome-wide association studies [21].
NAGA provided a reprioritized ranked list of 19,781 genes as output. We
then entered the top 500 genes with the highest ﬁnal heat scores into
STRING, an online database that generates mapped networks based on
protein-protein interactions [31]. STRING additionally analyzes for over-
representation of user-inputted gene lists in established pathways, using
the hypergeometric test [32]. Using this function, we tested our a priori
hypotheses to identify functional enrichment of the NAGA-generated top
500 gene list in the KEGG hsa04510 focal adhesion pathway, KEGG
hsa04512 ECM-receptor interaction, and KEGG hsa04151 PI3K-Akt signaling
pathway [33]. p values were corrected for multiple testing by STRING using
the Benjamini–Hochberg procedure [34].

Overlap between the NAGA-generated top 500 gene list and the KEGG
pathways was visualized using Cytoscape [35]. A hypergeometric test was

Translational Psychiatry  

 (2024) 14:109 

 
 
 
 
 
 
 
A.H. Ou et al.

3

conducted to test for overrepresentation of the NAGA-generated 500 gene
network in the 500 gene network generated in our previous study [23].

RESULTS
Demographics
The demographics of the sample can be found in the original
ConLiGen study [26]. The study was conducted in two phases:
GWAS 1 (n = 1065) and GWAS 2 (n = 1168). Sex and age were
similar across both cohorts. Mean Alda scale A scores were 6.13
(SD = 3.13) and 6.52 (SD = 2.87), respectively. Mean Alda scale B
scores were 1.78 (SD = 1.26) and 2.35 (SD = 1.16), respectively.

SNP, gene, and gene-set analysis
As reported in the original ConLiGen study, the only SNPs that
were signiﬁcant at a genome-wide signiﬁcance level of 5e-08 were
in linkage disequilibrium with the SNP rs74795342 on chromo-
some 21 (Supplementary Fig. 1). Using FUMA in our gene-wise
analysis, no signiﬁcant genes were found at a signiﬁcance level of
p < 0.05/18314 = 2.730e−6 (Supplementary Fig. 2). No gene-sets
were found to be signiﬁcant either, using p < 0.05 after Bonferroni
correction. The most highly associated genes and gene-sets are
listed in Supplementary Tables 1 and 2.

Table 1.
adhesion, ECM, and PI3K-Akt pathways.

Functional enrichment of NAGA top 500 gene list in focal

Pathway

p value

Number of genes
overlapped

KEGG focal adhesion

KEGG ECM-receptor
interaction

1.74e−06*
0.1494

21 of 198

5 of 88

1.90e−07*
KEGG PI3k-Akt
All p values corrected for multiple testing using the Benjamini–Hochberg
procedure.
*Signiﬁcant at p < 0.05.

31 of 350

Network analysis
We ﬁrst tested the three a priori pathways that were signiﬁcant in
our previous study, which had examined an independent sample
[23]. Using the STRING analysis function, the top 500 reprioritized
gene list generated by NAGA was found to be signiﬁcantly
enriched in both the KEGG hsa04510 focal adhesion pathway
(p = 1.74e−06) and KEGG hsa04151 PI3K-Akt signaling pathway
(p = 1.90e−07) (Table 1). Given the goal of replication and the
small number of statistical
this was considered as a
tests,
signiﬁcant replication of our previous results in an independent
sample for the focal adhesion and PI3K-Akt pathways. However,
the KEGG hsa04512 ECM-receptor interaction pathway was not
found to be signiﬁcantly enriched (Table 1). The overlapping
genes in all three networks can be seen in Figs. 1–3.

A hypergeometric test found signiﬁcant overlap (p = 5.699e−07)
between the 500 gene network generated by NAGA and the 500
gene network generated by network propagation analysis in our
previous study [23]. There were 33 genes that were common to
both networks. The top 25 reprioritized genes produced by NAGA
are listed in Table 2. All top 500 reprioritized NAGA genes are listed
in Supplementary Table 4.

After testing the three a priori hypotheses based on previous
results, we tested the top 500 NAGA gene list for enrichment in all
pathways in STRING. The top 10 KEGG pathways found to be most
strongly enriched are found in Supplementary Table 3. These
include cancer and growth pathways (such as Pathways in Cancer,
Estrogen Signaling Pathway, Ras Signaling Pathway) as well as the
dopaminergic synapse pathway.

We additionally used the STRING analysis function to test for
functional enrichment of
the top 100, 200, 300, and 400
reprioritized gene lists generated by NAGA in all three a priori
KEGG pathways. The results agreed with the primary analysis,
since all gene lists were signiﬁcantly enriched in the KEGG
hsa04510 focal adhesion pathway and KEGG hsa04151 PI3K-Akt
signaling pathways at a level of p < 0.05. Only the top 100
reprioritized gene list was found to be signiﬁcantly enriched in the
KEGG hsa04512 ECM-receptor interaction pathway (p = 0.0050)
inconsistent with a robust result. (Supplementary Table 5).

Fig. 1 Overlap between KEGG focal adhesion and top 500 genes. KEGG hsa04510 pathway for focal adhesion adapted to illustrate gene
overlap. Genes in yellow overlap with the 500 gene NAGA network.

Translational Psychiatry  

 (2024) 14:109 

 
 
 
 
 
 
 
4

A.H. Ou et al.

Fig. 2 Overlap between KEGG ECM-receptor interaction and top 500 genes. KEGG hsa04512 pathway for ECM-receptor interaction adapted
to illustrate gene overlap. Genes in yellow overlap with the 500 gene NAGA network.

DISCUSSION
In this study, we attempted to replicate our previous results which
were from an independent sample [23]. We used network
methods via NAGA to reprioritize GWAS results from the ConLiGen
study on lithium response and used STRING to test three a priori
network hypotheses: KEGG focal adhesion, ECM-receptor interac-
tion and PI3K-Akt signaling. Two of these three networks, KEGG
focal adhesion and PI3K-Akt signaling, were enriched in our top
500 reprioritized genes. However, we did not ﬁnd signiﬁcant
enrichment for the ECM-receptor interaction pathway in the 500
gene network. Besides this pathway, we were otherwise able to
replicate the results of our previous paper in a larger, independent
sample of patients with BD. We found highly signiﬁcant overlap
between the top 500 gene network generated by NAGA in this
study and the 500 gene network generated in the previous study,
providing further evidence for replication.

Focal adhesions are points of contact between cells and
proteins in the ECM. The formation of cell-ECM adhesion
structures is initiated by cell surface integrins and driven by local

actin polymerization. These structures function to not only
mediate cell attachment to ECM, but also mediate transmembrane
signaling. Integrin-ECM ligand binding can induce a number of
downstream changes affecting cell shape, growth, and prolifera-
tion [36]. In neurons, speciﬁcally, the actin cytoskeleton of growth
cones interacts with the ECM to guide axon development and
extension [24, 37].

We had originally hypothesized that genetic deﬁcits in focal
adhesion, ECM, and PI3K-Akt pathways may impair axonal growth
in neurons and determine response to lithium. Though one
integrin protein was included in our top 500 genes, in general
ECM proteins did not overlap with the top 500 gene list
(Supplementary Table 4)
(Fig. 2), and the pathway was not
signiﬁcant. This result is inconsistent with our previous study.
the deﬁcits
However,
inﬂuencing lithium response may be inherent to the growth cone
rather than components of the ECM. This is supported by a
number of studies, which have shown that lithium prevents
collapse and induces growth of growth cones [38–40].

the possibility that

it may suggest

Translational Psychiatry  

 (2024) 14:109 

 
 
 
 
 
 
 
A.H. Ou et al.

5

Fig. 3 Overlap between KEGG PI3k-Akt and top 500 genes. KEGG hsa04151 pathway for PI3k-Akt signaling adapted to illustrate gene
overlap. Genes in yellow overlap with the 500 gene NAGA network.

Previously, neurons derived from induced pluripotent stem cells
of patients with BD have been shown to exhibit hyperexcitability
in vitro. This hyperexcitable phenotype was rescued by lithium
only in neurons derived from lithium good responders [17].
Elevated neuroactivity in BD may induce vulnerability in neurons
through impairment of focal adhesion pathways. Chronic eleva-
tion of neuroactivity has been shown to dramatically reduce
surface expression of integrin β1 in animal models,
leading to
axonal and dendritic degeneration and eventually cell death [41].
Unsurprisingly, neurons in patients with BD have been shown to
be present with smaller size, fewer numbers, and more limited
branching. We had previously proposed that in lithium respon-
ders, this deﬁcit is caused by deﬁcits in focal adhesion and is
rescued by lithium treatment. Furthermore, we proposed that in
patients who are not lithium responders, focal adhesion is not
dysregulated, and lithium is unable to address the relevant
impairments [42–44]. Our results in this study are consistent with
this hypothesis.

After testing our three a priori hypotheses, we conducted
exploratory analyses using network methods. We listed the top 10
most signiﬁcant KEGG pathways that were associated by STRING
with the NAGA generated gene list in Supplementary Table 3.
These pathways are mostly cancer pathways associated with cell
growth and proliferation or pathways of addiction and other
dopamine-related processes. Dopamine neurotransmission has
previously been associated with response to lithium treatment in
BD [45]. Genes in associated cancer pathways show some overlap

with focal adhesion as well, which suggests the possibility of
shared mechanisms (Fig. 1).

Limitations of our study include the relatively small sample size
(N = 2039) and the generalizability of the dataset, given that all
participants were of European descent. Additionally, data was
collected retrospectively. As a result, outcomes may be less
accurate in determining response phenotypes [46] which can blur
our ﬁndings due to false negatives.

This study also demonstrates the utility of network propagation
methods, which can add power to GWAS with limited sample
sizes. These methods are beneﬁcial in identifying which genes and
gene-sets are of interest to a disease process, but future research
is still indicated for conﬁrmation [20, 21].

In summary, we replicated our previous results reinforcing that
genetic deﬁcits in focal adhesion and PI3K-Akt signaling are
associated with lithium response in BD patients. We hypothesize,
as before, that malformed axonal growth cones result in shorter
and less branched axons and susceptibility to BD in a subpopula-
tion of patients who are lithium responders. This is also consistent
with the idea that response to lithium results from a disease
mechanism distinct from that of lithium non-responders. Further-
more, we propose that lithium rescues disrupted neuronal growth
and axon extension processes by addressing deﬁcits in focal
adhesion. A better understanding of the pathophysiology of BD
and lithium treatment may lead to the future development of
drugs similar to lithium, as well as possible clinical predictors for
treatment response.

Translational Psychiatry  

 (2024) 14:109 

 
 
 
 
 
 
 
6

Table 2. NAGA top 25 gene list.

NAGA

Gene

UBC

GNB1

PRKACB

GNAL

GNGT1

REEP1

ARRB1

RTP2

RTP1

PRKACA

ARRB2

PRKACG

GRK2

GNG13

GNG7

GRK3

TAF1

APP

JUN

HNF4A

ELAVL1

C1orf94

CSMD2

KCNJ5

Input heat

0

2.624306658

6.770605441

0

0

0

7.336233161

0

0

0

0

0

0

0

0

0

0

0

5.596453897

0

0

14.45796462

14.45796462

11.85176049

INS
aData does not exist as gene was not evaluated in FUMA.

3.272540349

A.H. Ou et al.

Final heat

37.88310418

21.36254772

19.12717575

18.71021909

18.61047924

17.83396241

17.55148794

17.50819164

17.50727661

15.7274719

15.43877781

15.38554785

15.30495263

14.28098978

14.26968321

13.36097413

10.51679188

9.571290241

9.123060759

7.372223527

7.080313649

7.001838181

6.383962772

6.256678752

5.764244643

FUMA gene-wise analysis

Rank

1

2

3

4

5

6

7

8

9

10

11

12

13

14

15

16

17

18

19

20

21

22

23

24

25

Rank

4335

12993

5778

16294

12956

13325

12430

11202

13284

5506

14216

10721
a

9444

2636
a

a

11903

6091

13583

2194

9155

36

13972

10662

p value
0.22849

0.69873

0.30963

0.88169

0.69712

0.71728

0.66845

0.60084

0.71484

0.29380

0.76615

0.57518
a

0.50737

0.13696
a

a

0.63889

0.32547

0.73157

0.1154

0.49203

0.0016402

0.75419

0.57147

DATA AVAILABILITY
Summary statistics used in this study are available through the NHGRI-EBI GWAS
Catalog as study number GCST012487.

REFERENCES
1. Dome P, Rihmer Z, Gonda X. Suicide risk in bipolar disorder: a brief review.

Medicina. 2019;55. https://doi.org/10.3390/medicina55080403.

2. Cipriani A, Hawton K, Stockton S, Geddes JR. Lithium in the prevention of suicide
in mood disorders: updated systematic review and meta-analysis. BMJ.
2013;346:f3646.

3. Plans L, Barrot C, Nieto E, Rios J, Schulze TG, Papiol S, et al. Association between
completed suicide and bipolar disorder: a systematic review of the literature. J
Affect Disord. 2019;242:111–22.

4. Sajatovic M. Bipolar disorder: disease burden. Am J Manag Care. 2005;11:S80–4.
5. Rybakowski JK. Lithium. Eur Neuropsychopharmacol. 2022;57:86–7.
6. Kato T. Mechanisms of action of anti-bipolar drugs. Eur Neuropsychopharmacol.

2022;59:23–5.

7. Machado-Vieira R, Manji HK, Zarate CA Jr. The role of lithium in the treatment of
bipolar disorder: convergent evidence for neurotrophic effects as a unifying
hypothesis. Bipolar Disord. 2009;11(Suppl 2):92–109.

8. Freland L, Beaulieu J-M. Inhibition of GSK3 by lithium, from single molecules to

signaling networks. Front Mol Neurosci. 2012;5:14.

9. Jope RS. Lithium and GSK-3: one inhibitor, two inhibitory actions, multiple out-

comes. Trends Pharm Sci. 2003;24:441–3.

10. Mishra HK, Wei H, Rohr KE, Ko I, Nievergelt CM, Maihofer AX, et al. Contributions
of circadian clock genes to cell survival in ﬁbroblast models of lithium-responsive
bipolar disorder. Eur Neuropsychopharmacol. 2023;74:1–14.

11. Hermida MA, Dinesh Kumar J, Leslie NR. GSK3 and its interactions with the PI3K/

AKT/mTOR signalling network. Adv Biol Regul. 2017;65:5–15.

12. Watson DG, Lenox RH. Chronic lithium-induced down-regulation of MARCKS in
immortalized hippocampal cells: potentiation by muscarinic receptor activation. J
Neurochem. 1996;67:767–77.

13. Tobe BTD, Crain AM, Winquist AM, Calabrese B, Makihara H, Zhao W-N, et al.
Probing the lithium-response pathway in hiPSCs implicates the phosphor-
egulatory set-point for a cytoskeletal modulator in bipolar pathogenesis. Proc
Natl Acad Sci USA. 2017;114:E4462–71.

14. Zhao W-N, Tobe BTD, Udeshi ND, Xuan LL, Pernia CD, Zolg DP, et al. Discovery of
suppressors of CRMP2 phosphorylation reveals compounds that mimic the
behavioral effects of lithium on amphetamine-induced hyperlocomotion. Transl
Psychiatry. 2020;10:76.

15. Amare AT, Thalamuthu A, Schubert KO, Fullerton JM, Ahmed M, Hartmann S, et al.
Association of polygenic score and the involvement of cholinergic and gluta-
matergic pathways with lithium treatment response in patients with bipolar
disorder. Mol Psychiatry. 2023. https://doi.org/10.1038/s41380-023-02149-1.
16. Papiol S, Schulze TG, Heilbronner U. Lithium response in bipolar disorder:

genetics, genomics, and beyond. Neurosci Lett. 2022;785:136786.

17. Mertens J, Wang Q-W, Kim Y, Yu DX, Pham S, Yang B, et al. Differential responses
to lithium in hyperexcitable neurons from patients with bipolar disorder. Nature.
2015;527:95–9.

18. Grof P, Duffy A, Cavazzoni P, Grof E, Garnham J, MacDougall M, et al. Is response

to prophylactic lithium a familial trait? J Clin Psychiatry. 2002;63:942–7.

19. Cruceanu C, Alda M, Turecki G. Lithium: a key to the genetics of bipolar disorder.

Genome Med. 2009;1:79.

20. Leiserson MDM, Eldridge JV, Ramachandran S, Raphael BJ. Network analysis of

GWAS data. Curr Opin Genet Dev. 2013;23:602–10.

21. Carlin DE, Fong SH, Qin Y, Jia T, Huang JK, Bao B, et al. A fast and ﬂexible
framework for network-assisted genomic association. iScience. 2019;16:155–61.
22. Oedegaard KJ, Alda M, Anand A, Andreassen OA, Balaraman Y, Berrettini WH,
et al. The Pharmacogenomics of Bipolar Disorder study (PGBD): identiﬁcation of
genes for lithium response in a prospective sample. BMC Psychiatry. 2016;16:129.

Translational Psychiatry  

 (2024) 14:109 

 
 
 
 
 
 
 
23. Niemsiri V, Rosenthal SB, Nievergelt CM, Maihofer AX, Marchetto MC, Santos R,
et al. Focal adhesion is associated with lithium response in bipolar disorder:
evidence from a network-based multi-omics analysis. Mol Psychiatry. 2023.
https://doi.org/10.1038/s41380-022-01909-9.

24. Short CA, Suarez-Zayas EA, Gomez TM. Cell adhesion and invasion mechanisms

that guide developing axons. Curr Opin Neurobiol. 2016;39:77–85.

25. Maletic V, Raison C. Integrated neurobiology of bipolar disorder. Front Psychiatry.

2014;5:98.

26. Hou L, Heilbronner U, Degenhardt F, Adli M, Akiyama K, Akula N, et al. Genetic
variants associated with response to lithium treatment in bipolar disorder: a
genome-wide association study. Lancet. 2016;387:1085–93.

27. Sollis E, Mosaku A, Abid A, Buniello A, Cerezo M, Gil L, et al. The NHGRI-EBI GWAS
and deposition resource. Nucleic Acids Res.

knowledgebase

Catalog:
2023;51:D977–85.

28. Manchia M, Adli M, Akula N, Ardau R, Aubry J-M, Backlund L, et al. Assessment of
response to lithium maintenance treatment in bipolar disorder: a Consortium on
Lithium Genetics (ConLiGen) Report. PLoS ONE. 2013;8:e65636.

29. Marie-Claire C, Courtin C, Bellivier F, Scott J, Etain B. Methylomic biomarkers of
lithium response in bipolar disorder: a proof of transferability study. Pharma-
ceuticals. 2022;15. https://doi.org/10.3390/ph15020133.

30. Watanabe K, Taskesen E, van Bochoven A, Posthuma D. Functional mapping and
annotation of genetic associations with FUMA. Nat Commun. 2017;8:1826.
31. Szklarczyk D, Kirsch R, Koutrouli M, Nastou K, Mehryary F, Hachilif R, et al. The
STRING database in 2023: protein-protein association networks and functional
enrichment analyses for any sequenced genome of interest. Nucleic Acids Res.
2023;51:D638–46.

32. Szklarczyk D, Gable AL, Nastou KC, Lyon D, Kirsch R, Pyysalo S, et al. The STRING
database in 2021: customizable protein-protein networks, and functional char-
acterization of user-uploaded gene/measurement sets. Nucleic Acids Res.
2021;49:D605–12.

33. Kanehisa M, Goto S. KEGG: kyoto encyclopedia of genes and genomes. Nucleic

Acids Res. 2000;28:27–30.

34. Benjamini Y, Hochberg Y. Controlling the false discovery rate: a practical and

powerful approach to multiple testing. J R Stat Soc. 1995;57:289–300.

35. Shannon P, Markiel A, Ozier O, Baliga NS, Wang JT, Ramage D, et al. Cytoscape: a
software environment for integrated models of biomolecular interaction net-
works. Genome Res. 2003;13:2498–504.

36. Wu C. Focal adhesion: a focal point in current cell biology and molecular med-

icine. Cell Adh Migr. 2007;1:13–8.

37. Omotade OF, Pollitt SL, Zheng JQ. Actin-based growth cone motility and gui-

dance. Mol Cell Neurosci. 2017;84:4–10.

38. Williams RSB, Cheng L, Mudge AW, Harwood AJ. A common mechanism of action

for three mood-stabilizing drugs. Nature. 2002;417:292–5.

39. Shah SM, Patel CH, Feng AS, Kollmar R. Lithium alters the morphology of neurites
spiral ganglion neurons. Hear Res.

regenerating from cultured adult
2013;304:137–44.

40. Owen R, Gordon-Weeks PR. Inhibition of glycogen synthase kinase 3β in sensory
neurons in culture alters ﬁlopodia dynamics and microtubule distribution in
growth cones. Mol Cell Neurosci. 2003;23:626–37.

41. Murase S.

Impaired focal adhesion kinase-Grb2 interaction during elevated

activity in hippocampal neurons. Int J Mol Sci. 2015;16:15659–69.

42. Gigante AD, Young LT, Yatham LN, Andreazza AC, Nery FG, Grinberg LT, et al.
Morphometric post-mortem studies in bipolar disorder: possible association with
oxidative stress and apoptosis. Int J Neuropsychopharmacol. 2011;14:1075–89.
43. Konradi C, Zimmerman EI, Yang CK, Lohmann KM, Gresch P, Pantazopoulos H,
interneurons in bipolar disorder. Arch Gen Psychiatry.

et al. Hippocampal
2011;68:340–50.

44. Rajkowska G. Cell pathology in bipolar disorder. Bipolar Disord. 2002;4:105–16.
45. Mohamadian M, Fallah H, Ghofrani-Jahromi Z, Rahimi-Danesh M, Shokouhi Qare
Saadlou M-S, Vaseghi S. Mood and behavior regulation: interaction of lithium and
dopaminergic system. Naunyn Schmiedebergs Arch Pharmacol. 2023. https://
doi.org/10.1007/s00210-023-02437-1.

46. Talari K, Goyal M. Retrospective studies—utility and caveats. J R Coll Physicians

Edinb. 2020;50:398–402.

A.H. Ou et al.

7

Indianapolis,

Initiative. From 2003–2007,
Indiana University,

Forschungsgemeinschaft (DFG; grant no RI 908/7-1; grant FOR2107, RI 908/11-1 to
MR, MB, and TGS, NO 246/10-1 to MMN) and the Intramural Research Program of the
National Institute of Mental Health (ZIA-MH00284311; NCT00001174). The genotyp-
ing was in part funded by the German Federal Ministry of Education and Research
(BMBF) through the Integrated Network IntegraMent (Integrated Understanding of
Causes and Mechanisms in Mental Disorders), under the auspices of the e:Med
Programme (grants awarded to TGS, MR, and MMN). OG, AP, TSt, MB, AR, and TGS
received support from the German Federal Ministry of Education and Research
(BMBF) within the framework of the BipolLife network. MMN received support from
the Alfried Krupp von Bohlen und Halbach-Stiftung. FD received support from the
BONFOR Programme of the University of Bonn, Germany. EZR received funding from
the Land Steiermark as principal investigator. MS received funds from the Swedish
Research Council, Swedish Brain Foundation and funds from Karolinska Institutet and
Karolinska University Hospital. Some data and biomaterials were collected as part of
eleven projects (Study 40) that participated in the National Institute of Mental Health
the principal
(NIMH) Bipolar Disorder Genetics
investigators and co-investigators were:
IN, R01
MH59545 (John Nurnberger, Marvin J Miller, Elizabeth S Bowman, N Leela Rau, P Ryan
Moe, Nalini Samavedy, Rif El-Mallakh [University of Louisville], Husseini Manji
[Johnson and Johnson], Debra A Glitz [Wayne State University], Eric T Meyer [Oxford
University, UK], Carrie Smiley, Tatiana Foroud, Leah Flury, Danielle M Dick [Virginia
Commonwealth University], Howard Edenberg); Washington University, St Louis, MO,
R01 MH059534 (John Rice, Theodore Reich, Allison Goate, Laura Bierut
[K02
DA21237]); Johns Hopkins University, Baltimore, R01 MH59533 (Melvin McInnis, J
Raymond DePaulo Jr, Dean F MacKinnon, Francis M Mondimore, James B Potash,
Peter P Zandi, Dimitrios Avramopoulos, Jennifer Payne); University of Pennsylvania,
PA, R01 MH59553 (Wade Berrettini); University of California at San Francisco, CA, R01
MH60068 (William Byerley, Sophia Vinogradov); University of Iowa, IA, R01 MH059548
(William Coryell, Raymond Crowe); University of Chicago,
IL, R01 MH59535 (Elliot
Gershon, Judith Badner, Francis McMahon, Chunyu Liu, Alan Sanders, Maria Caserta,
Steven Dinwiddie, Tu Nguyen, Donna Harakal); University of California at San Diego,
CA, R01 MH59567 (John Kelsoe, Rebecca McKinney); Rush University,
IL, R01
MH059556 (William Scheftner, Howard M Kravitz, Diana Marta, Annette Vaughn-
Brown, Laurie Bederow); and NIMH Intramural Research Program, Bethesda,
1Z01MH002810-01 (Francis J McMahon, Layla Kassem, PhD, Sevilla Detera-Wadleigh,
Lisa Austin, Dennis L Murphy [Howard University], William B Lawson, Evarista Nwulia,
Maria Hipolito). This work was supported by the NIH grants P50CA89392 from the
National Cancer Institute and 5K02DA021237 from the National
Institute of Drug
Abuse. The Canadian part of the study was supported by a grant #166098 from the
Canadian Institutes of Health Research, by CIHR under the frame of ERA PerMed
(grant PLOT-BD), and Genome Canada grant RP3 to MAl. We wish to thank Joanne
Petite and Giselle Kraus for assistance with data collection. Collection and
phenotyping of the Australian UNSW sample, by PBM, PRS, JMF, and AW, was
funded by an Australian NHMRC Program Grant (No. 1037196). The collection of the
Barcelona sample was supported by the Centro de Investigación en Red de Salud
Mental (CIBERSAM) IDIBAPS (grant numbers PI080247, PI1200906, PI12/00018), and
Secretaria d’Universitats i Recerca del Departament d’Economia i Coneixement
(2021SGR1358 and 2014SGR398). J-MA and AD were supported by the Swiss National
Science Foundation (grant number 32003B_125469 and NCCR Synapsy). DAC was
supported by a Medical Research Council Clinician Scientist Fellowship Award (MR/
L006642/1). LF was supported by the Swedish Research Council (grant no 523-2011-
3807). MG-S was supported by UEFISCDI, Romania, grant no. 89/2012 and grant no.
203/2021. P-HK was funded by the Taiwan Ministry of Science and Technology (grant
no. MST 99-2314-B-002-140-MY3 and 102-2314-B-002-117-MY3). CALJ was funded by
the “Estrategia de Sostenibilidad 2014-2015” program of the University of Antioquia.
TN was supported by the Ministry of Health of the Czech Republic (grant no. IGA
NT13891). JBP was supported by the Reuben Stoltzfus Bipolar Research Fund and
with SKT received funding from the James Wah Fund and Project MATCH. TGS and
UH received support from the Dr-Lisa-Oehler-Foundation (Kassel, Germany). This
study used the high-performance computational capabilities of the Biowulf Linux
cluster at the National Institutes of Health, Bethesda, MD. Genotyping for part of the
Swedish sample was funded by the Stanley Center for Psychiatric Research at the
Broad Institute.

ACKNOWLEDGEMENTS
We thank the subjects who participated in the original study, without whom this
work would not be possible. Additionally, JRK was supported by the NIMH (U01
MH92758), and AHO was supported by the UCSD SOM Summer Research Training
Program. ATA received the 2019–2021 National Alliance for Research on
Schizophrenia and Depression (NARSAD) Young Investigator Grant from the Brain
& Behaviour Research Foundation (BBRF) and is currently supported by the National
Health and Medical Research Council
(NHMRC) Emerging Leadership (EL1)
Investigator Grant (APP2008000). This work was in part funded by the Deutsche

AUTHOR CONTRIBUTIONS
AHO and JRK designed the study, analyzed data, and wrote the manuscript. SBR
analyzed data and reviewed the manuscript. MAd, KA, NA, MAl, ATA, RA, BA, JMA, LB,
MB, BTB, FB, AB, SB, AKB, JMB, PC, GBC, HCC, CCh, SC, SRC, FC, DAC, CCr, PMC, CRD,
AD, MDZ, FD, JRD, BE, PF, FTF, EFR, AJF, LF, MAF, JMF, SG, JSG, FSG, MGS, PG, OG, RH,
JH, UH, SH, PH, AH, LH, SJ, EJ, JPK, LK, TK, SKS, BK, PHK, IK, NL, GL, ML, CL, ML, SGL,
CALJ, GM, MMaj, MMan, CMC, LM, MMat, MJM, SLM, FJM, PBM, MMi, FMM, PM, CMN,
MMN, TN, UO, NO, SP, RHP, CP, JBP, AP, DRE, AR, EZR, MR, GAR, JKR, MS, PRS, KOS,
TGS, BWS, FS, GS, TSh, PDS, KS, CS, CMS, AS, TSt, PS, SKT, AT, GT, EV, JV, SW, NRW, AW,
LTY, and PPZ collected samples and data and reviewed the manuscript.

Translational Psychiatry  

 (2024) 14:109 

 
 
 
 
 
 
 
8

A.H. Ou et al.

COMPETING INTERESTS
MAd has received a grant from Servier, speaker’s fees from Servier, Lundbeck, Aristo,
Parexel, Gilead, ViiV, Deutsche Bank, MSD, and MyTomorrows, plus a non-ﬁnancial
from Lundbeck. KA has received speaker’s fees from Taisho Toyama
support
Pharmaceutical. MAl
is funded by a grant of the Canadian Institutes of Health
Research. MB has received speaker’s fees from AstraZeneca, Pﬁzer, Lilly, Lundbeck,
GlaxoSmithKline, Servier, and Ferrer Internacional. BÉ received non-ﬁnancial support
from Labex Biopsy and Fondation Fondamental. RH received grants and speaker
honoraria from Dainippon Sumitomo Pharma and Novartis plus speaker honoraria
from Eli Lilly Japan, GlaxoSmithKline, Hisamitsu Pharmaceutical, Janssen Pharmaceu-
tical, Nippon Zoki Pharmaceutical, Otsuka Pharmaceutical, Astellas Pharma, Pﬁzer,
and the Yoshitomiyakuhin Corporation. TK received a grant
from Takeda
Pharmaceutical and fees
from Kyowa Hakko Kirin, Eli Lilly Japan, Otsuka
Pharmaceutical, GlaxoSmithKline, Taisho Toyama Pharmaceutical, Dainippon Sumi-
tomo Pharma, Meiji Seika Pharma, Pﬁzer Japan, Mochida Pharmaceutical, Shionogi &
Co, Janssen Pharmaceutical, Yoshitomiyakuhin Corporation, Agilent Technologies,
Astellas Pharma, and Wako Pure Chemical Industries. IK received grants and fees from
Dainippon Sumitomo Pharma, Eisai, Eli Lilly, GlaxoSmithKline, Kyowa Hakko Kirin,
Meiji Seika Pharma, MSD, Novartis, Otsuka, Ono Pharmaceutical, Pﬁzer, Tanabe
Mitsubishi Pharma, Takeda Pharmaceutical, Shionogi, and Yoshitomi Pharmaceutical;
he received grants from AbbVie GK, Asahi Kasei Pharma, Boehringer Ingelheim,
Chugai Pharmaceutical, and Daiichi Sankyo and fees from Astellas Pharma and
Janssen Pharmaceutical. MJM served as unpaid consultant for Pathway Genomic (San
Diego, USA). SLM received a grant and fees from Naurex and Shire, further grants
from Alkermes, Cephalon, Forest, Marriott Foundation, Orexigen Therapeutics, and
Takeda Pharmaceutical, he further has served on the advisory boards for Bracket,
Hoffmann-La Roche, MedAvante, Sunovion and received fees from Novo Nordisk.
RHP received personal fees from RID Ventures, Genomind LLC, Healthrageous, Pﬁzer,
Perfect Health, Proteus, and Psybrain. PRS received a grant from NHMRC. TGS
received a grant and fees from Roche Pharmaceuticals. TSt received personal fees
from Servier, Lundbeck, and Bristol-Myers Squibb. MMan has received grants from
Lundbeck and Angelini. EV has received grants and served as consultant, advisor or
CME speaker for the following entities: AB-Biotics, AbbVie, Adamed, Angelini, Biogen,
Biohaven, Boehringer-Ingelheim, Celon Pharma, Compass, Dainippon Sumitomo
Pharma, Ethypharm, Ferrer, Gedeon Richter, GH Research, Glaxo-Smith Kline, HMNC,
Idorsia, Janssen, Lundbeck, Medincell, Merck, Novartis, Orion Corporation, Organon,
Otsuka, Roche, Rovi, Sage, Sanoﬁ-Aventis, Sunovion, Takeda, and Viatris, outside the
submitted work. SRC has participated in advisory and educational boards and

received speaker’s fees from Janssen-Cilag, Lundbeck, Otsuka, and Servier; research
funding from Janssen-Cilag, Lundbeck, Otsuka, and Gilead; and data sharing from
Viatris Australia. ML has received lecture honoraria from Lundbeck pharmaceuticals.
All above listed interests are outside of the submitted work. All other authors declare
no competing interests.

ADDITIONAL INFORMATION
Supplementary information The online version contains supplementary material
available at https://doi.org/10.1038/s41398-024-02811-4.

Correspondence and requests for materials should be addressed to John R. Kelsoe.

Reprints and permission information is available at http://www.nature.com/
reprints

Publisher’s note Springer Nature remains neutral with regard to jurisdictional claims
in published maps and institutional afﬁliations.

Open Access This article is licensed under a Creative Commons
Attribution 4.0 International License, which permits use, sharing,
adaptation, distribution and reproduction in any medium or format, as long as you give
appropriate credit to the original author(s) and the source, provide a link to the Creative
Commons licence, and indicate if changes were made. The images or other third party
material in this article are included in the article’s Creative Commons licence, unless
indicated otherwise in a credit line to the material. If material is not included in the
article’s Creative Commons licence and your intended use is not permitted by statutory
regulation or exceeds the permitted use, you will need to obtain permission directly
from the copyright holder. To view a copy of
licence, visit http://
creativecommons.org/licenses/by/4.0/.

this

© The Author(s) 2024

1Department of Psychiatry, University of California San Diego, La Jolla, CA, USA. 2Center for Computational Biology and Bioinformatics, Department of Medicine, University
of California San Diego, La Jolla, CA, USA. 3Department of Psychiatry and Psychotherapy, Charité— Universitätsmedizin Berlin, Campus Charité Mitte, Berlin, Germany.
4Fliedner Klinik Berlin, Center for Psychiatry, Psychotherapy and Psychosomatic Medicine, Berlin, Germany. 5Department of Biological Psychiatry and Neuroscience, Dokkyo
Medical University School of Medicine, Mibu, Japan. 6Intramural Research Program, National Institute of Mental Health, National Institutes of Health, US Department of
Health & Human Services, Bethesda, MD, USA. 7Department of Psychiatry, Dalhousie University, Halifax, NS, Canada. 8National Institute of Mental Health, Klecany, Czech
Republic. 9Discipline of Psychiatry, University of Adelaide, Adelaide, SA, Australia. 10Unit of Clinical Pharmacology, Hospital University Agency of Cagliari, Cagliari, Italy.
11Department of Evolutive Biology, Ecology and Environmental Sciences, Facultat de Biologia and Institut de Biomedicina (IBUB), Universitat de Barcelona, Barcelona, Spain.
12CIBER de Salud Mental, ISCIII, Madrid, Barcelona, Catalonia, Spain. 13Department of Mental Health and Psychiatry, Mood Disorders Unit—Geneva University Hospitals,
Geneva, Switzerland. 14Department of Molecular Medicine and Surgery, Karolinska Institutet and Center for Molecular Medicine, Karolinska University Hospital, Stockholm,
Sweden. 15Department of Psychiatry and Psychotherapy, University Hospital Carl Gustav Carus, Medical Faculty, Technische Universität Dresden, Dresden, Germany.
16Department of Psychiatry, University of Münster, Münster, Germany. 17INSERM UMR-S 1144—Université Paris Cité Département de Psychiatrie et de Médecine
Addictologique, AP-HP, Groupe Hospitalier Lariboisière-F Widal, Paris, France. 18Bipolar and Depressive Disorders Unit, Institute of Neuroscience, Hospital Clinic, University of
Barcelona, IDIBAPS, CIBERSAM, ISCIII, Barcelona, Catalonia, Spain. 19Neurobiological Background and Anthropometrics in Bipolar Affective Disorder, Division of Psychiatry
and Psychotherapeutic Medicine, Medical University of Graz, Graz, Austria. 20Department of Health Sciences Research, Mayo Clinic, Rochester, MN, USA. 21Department of
Psychiatry and Psychology, Mayo Clinic, Rochester, MN, USA. 22The Neuromodulation Unit, McGill University Health Centre, Montreal, QC, Canada. 23Department of
Psychiatry & Center of Sleep Disorders, National Taiwan University Hospital, Taipei, Taiwan. 24Institute of Human Genetics, University of Bonn and Department of Genomics,
Life & Brain Center, Bonn, Germany. 25Human Genomics Research Group, Department of Biomedicine, University Hospital Basel, Basel, Switzerland. 26Campus for Ageing and
Vitality, Newcastle University, Newcastle upon Tyne, UK. 27Douglas Mental Health University Institute, McGill University, Montreal, QC, Canada. 28Psychiatric Genetic Unit,
Poznan University of Medical Sciences, Poznan, Poland. 29Department of Psychiatry, University of Campinas (Unicamp), Campinas, Brazil. 30Department of Biomedical
Sciences, University of Cagliari, Cagliari, Italy. 31Department of Psychiatry and Behavioral Sciences, Johns Hopkins University, Baltimore, MD, USA. 32Institute of Psychiatric
Phenomics and Genomics (IPPG) and Department of Psychiatry and Psychotherapy, Ludwig-Maximilians-University of Munich, Munich, Germany. 33Department of Adult
Psychiatry, Poznan University of Medical Sciences, Poznan, Poland. 34Department of Clinical Neuroscience, Karolinska Institutet and Center for Molecular Medicine,
Karolinska University Hospital, Stockholm, Sweden. 35Child and Adolescent Psychiatry Research Center, Stockholm, Sweden. 36Mental Illness Research Theme, Neuroscience
Research Australia, Sydney, NSW, Australia. 37School of Medical Sciences, University of New South Wales, Sydney, NSW, Australia. 38Pôle de Psychiatrie Générale Universitaire,
Centre Hospitalier Charles Perrens, Bordeaux, France. 39Biometric Psychiatric Genetics Research Unit, Alexandru Obregia Psychiatric Hospital, Bucharest, Romania. 40Mood
Disorders Center of Ottawa, Ottawa, ON, Canada. 41Department of Psychiatry and Psychotherapy, University Medical Center (UMG), Georg-August University Göttingen,
Göttingen, Germany. 42Department of Pathology of Mental Diseases, National Institute of Mental Health, National Center of Neurology and Psychiatry, Tokyo, Japan.
43Inserm U955, Psychiatrie Translationnelle, Créteil, France. 44Service de Psychiatrie et Psychologie Clinique, Centre Psychothérapique de Nancy-Laxou—Université de
Lorraine, Nancy, France. 45Laboratory for Molecular Dynamics of Mental Disorders, RIKEN Brain Science Institute, Saitama, Japan. 46Department of Psychiatry, Psychosomatic
Medicine and Psychotherapy, University Hospital Frankfurt—Goethe University, Frankfurt am Main, Germany. 47Department of Psychiatry and Psychotherapeutic Medicine,
Landesklinikum Neunkirchen, Neunkirchen, Austria. 48Institute of Epidemiology and Preventive Medicine, National Taiwan University, Taipei, Taiwan. 49Department of
Psychiatry, Hokkaido University Graduate School of Medicine, Sapporo, Japan. 50Institute of Neuroscience and Physiology, The Sahlgrenska Academy at the Gothenburg
University, Gothenburg, Sweden. 51Department of Medical Epidemiology and Biostatistics, Karolinska Institutet, Stockholm, Sweden. 52Assistance Publique-Hôpitaux de
Paris, Hôpital Albert Chenevier—Henri Mondor, Pôle de Psychiatrie, Créteil, France. 53Department of Pharmacy, VA San Diego Healthcare System, La Jolla, CA, USA.
54Department of Psychiatry, University of Antioquia, Medellín, Medellín, Colombia. 55Department of Psychiatry, University of Calgary, Calgary, AB, Canada. 56Department of

Translational Psychiatry  

 (2024) 14:109 

 
 
 
 
 
 
 
A.H. Ou et al.

9

Psychiatry, University of Naples SUN, Naples, Italy. 57Department of Medical Sciences and Public Health, University of Cagliari, Cagliari, Italy. 58Department of Pharmacology,
Dalhousie University, Halifax, NS, Canada. 59Department of Clinical Neurosciences, Karolinska Institutet, Stockholm, Sweden. 60Department of Biomedicine, Aarhus
University, Aarhus, Denmark. 61Department of Psychiatry, VA San Diego Healthcare System, La Jolla, CA, USA. 62Department of Psychiatry, Lindner Center of Hope, University
of Cincinnati, Mason, OH, USA. 63School of Psychiatry, University of New South Wales, and Black Dog Institute, Sydney, NSW, Australia. 64Department of Genetics,
Microbiology, and Statistics, Faculty of Biology and Institut de Biomedicina (IBUB), Universitat de Barcelona, Barcelona, CIBER de Salud Mental, ISCIII, Madrid, Spain.
65Neurosciences Section, Department of Medicine and Surgery, University of Salerno, Salerno, Italy. 66Department of Neurobiology, Care Sciences, and Society, Karolinska
Institutet and Center for Molecular Medicine, Karolinska University Hospital, Stockholm, Sweden. 67Department of Psychiatry, Nagoya University Graduate School of
Medicine, Nagoya, Japan. 68Ludwig-Maximilians-University of Munich, Munich, Germany. 69Department of Psychiatry, Massachusetts General Hospital and Harvard Medical
School, Boston, MA, USA. 70Department of Psychiatry, University of Iowa, Iowa City, IA, USA. 71Department of Genetic Epidemiology in Psychiatry, Central Institute of Mental
Health, Medical Faculty Mannheim, University of Heidelberg, Mannheim, Germany. 72Montreal Neurological Institute and Hospital, McGill University, Montreal, QC, Canada.
73Veterans Administration, San Diego Healthcare System, San Diego, CA, USA. 74Department of Psychiatry, Dokkyo Medical University School of Medicine, Mibu, Japan.
75Bipolar Center, Wiener Neustadt, Austria. 76University of Iowa Carver College of Medicine and University of Iowa College of Public Health, VA Quality Scholars Program,
Iowa City VA Hospital, Iowa City, IA, USA. 77The University of Queensland, Queensland Brain Institute, Brisbane, QLD, Australia. 78Department of Psychiatry, University of
Toronto, Toronto, ON, Canada. 79Department of Mental Health, Johns Hopkins Bloomberg School of Public Health, Baltimore, MD, USA.
email: jkelsoe@health.ucsd.edu

✉

Translational Psychiatry  

 (2024) 14:109
