# Soto_2024_Astrocyte Gi-GPCR signaling corrects compulsive-like grooming and anxiety-related behaviors in Sapap3 knockout mice.

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

HHS Public Access
Author manuscript
Neuron. Author manuscript; available in PMC 2025 October 23.

Published in final edited form as:

Neuron. 2024 October 23; 112(20): 3412–3423.e6. doi:10.1016/j.neuron.2024.07.019.

Astrocyte Gi-GPCR signaling corrects compulsive-like grooming 
and anxiety-related behaviors in Sapap3 knockout mice

Joselyn S. Soto1,#, Chiranjivi Neupane1, Muskan Kaur1, Vijaya Pandey3, James A. 
Wohlschlegel3, Baljit S. Khakh1,2,*
1Department of Physiology, David Geffen School of Medicine, University of California Los 
Angeles, Los Angeles USA CA 90095-1751

2Department of Neurobiology, David Geffen School of Medicine, University of California Los 
Angeles, Los Angeles USA CA 90095-1751

3Department of Biological Chemistry, David Geffen School of Medicine, University of California 
Los Angeles, Los Angeles USA CA 90095-1751

Summary

Astrocytes are morphologically complex cells that serve essential roles. They are widely 
implicated in CNS disorders, with changes in astrocyte morphology and gene expression 
accompanying disease. In the Sapap3 knockout mouse model of compulsive and anxiety-
related behaviors related to obsessive-compulsive disorder (OCD), striatal astrocytes display 
reduced morphology and altered actin cytoskeleton and Gi-G protein-coupled receptor (Gi-
GPCR) signaling proteins. Here, we show that normalizing striatal astrocyte morphology, actin 
cytoskeleton, and essential homeostatic support functions by targeting the astrocyte Gi-GPCR 
pathway using chemogenetics corrected phenotypes in Sapap3 knockout mice, including anxiety-
related and compulsive behaviors. Our data portend an astrocytic pharmacological strategy for 
rescuing phenotypes in brain disorders that include compromised astrocyte morphology and tissue 
support.

Graphical Abstract

Correspondence: bkhakh@mednet.ucla.edu and joselynsoto@fullerton.edu.
#Current address for JSS: Department of Biological Science, California State University Fullerton, Fullerton, CA 92831
*Lead contact (BSK)
Author contributions
JSS performed most experiments and analyses. CN performed electrophysiology and analyses. MK helped with RNA-scope and 
analyses. VP performed mass spectrometry and JAW helped with analysis. BSK conceived the study and supervised JSS and CN. BSK 
and JSS wrote the manuscript. All authors commented on it.

Publisher's Disclaimer: This is a PDF file of an unedited manuscript that has been accepted for publication. As a service to our 
customers we are providing this early version of the manuscript. The manuscript will undergo copyediting, typesetting, and review 
of the resulting proof before it is published in its final form. Please note that during the production process errors may be discovered 
which could affect the content, and all legal disclaimers that apply to the journal pertain.

Declaration of interests
UCLA filed a US Provisional Patent (No. 63/658,760) based on this work. BSK is on the editorial advisory board of Neuron.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Soto et al.

Page 2

eTOC Blurb

Soto et al., used multiple approaches to assess if and how function is affected following astrocyte 
Gi-DREADD activation in a mouse model of compulsive and anxiety-related behaviors that 
is accompanied with astrocyte dysfunction. They found that Gi-DREADD activation restored 
decreased astrocyte morphology and homeostatic functions, and lead to correction of compulsive 
and anxiety behaviors in obsessive-compulsive disorder (OCD) related Sapap3 knockout mice. 
Soto et al., provide an astrocyte-specific pharmacological strategy for normalizing detrimental 
compulsive phenotypes related to OCD and potentially other disorders in which astrocyte 
morphology and homeostatic support are compromised.

Keywords

astrocyte; striatum; proteomics; RNA-sequencing; OCD; GPCR; behavior

Introduction

Astrocytes are an abundant type of glia1,2. Their complex morphology allows astrocytes 
to contact diverse cells3,4, thus participating in tissue homeostasis, signaling, and disease 
pathogenesis4–6. Astrocytes are widely implicated in CNS disorders including psychiatric 
diseases5,7–9 and their alterations are associated with repetitive behaviors and obsessive-
compulsive disorder (OCD)10–13.

Neuron. Author manuscript; available in PMC 2025 October 23.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Soto et al.

Page 3

OCD is a severe psychiatric disorder that affects about 1 in 50 people14. OCD is 
characterized by obsessive thoughts that give rise to severe anxiety and behavioral 
compulsions, which manifest as repetitive actions14,15. Although the molecular and cellular 
mechanisms that give rise to OCD remain to be established, basal ganglia circuits and the 
striatum are a main locus for OCD phenotypes16–19. Our understanding of how astrocytes 
contribute to neural circuits including those involved in compulsivity is developing10–12,20–
25, but whether astrocytes can be exploited for therapeutic strategies for psychiatric diseases 
remains incompletely explored.

Astrocyte morphology-related genes are implicated in CNS disorders9,13 that may 
encompass astrocytes with reduced morphology. We found that correcting reduced 
astrocyte morphology and homeostasis in Sapap3 knockout (KO) mice10 that model 
compulsive behaviors26,27, by targeting the striatal astrocyte Gi-GPCR pathway, improved 
several disease-related phenotypes. We thus report an astrocytic strategy for exploring 
pathophysiological mechanisms in OCD10, and other brain disorders9,13 in which astrocyte 
functions are compromised.

Results

Experimental rationale

Astrocyte morphology-related genes such as those correlated with territory size are 
implicated in CNS disorders9,13 implying altered morphology may be a feature of diseases 
such as OCD (Fig. 1Ai). Furthermore, some genes causatively or correlatively associated 
with OCD and Huntington’s disease (HD) map to astrocytes28 (Fig. 1Aii), and astrocytes 
display reduced morphology in HD mouse models28,29 and in the Sapap3 KO mouse model 
of compulsive-like behaviors26,27. In Sapap3 KO mice10, such changes involve the actin 
cytoskeleton and G-proteins including the Gαi protein Gnai2 that is part of Gi-GPCR 
signaling (Fig. 1B). We hypothesized that disrupted astrocyte morphology may contribute 
to pathophysiology and correcting it may be beneficial (Fig. 1). To test this, we targeted 
the Gαi-GPCR and Gαq-GPCR pathways within striatal astrocytes28,30–32, first assessing 
behavioral outcomes followed by molecular and cellular assessments, including in relation 
to available human OCD data (Fig. 1C).

Behavioral improvements in Sapap3 KO mice

Using astrocyte selective adeno-associated viruses (AAV)10,11,28,30,32–34, we expressed 
the Gαi-GPCR hM4Di or Gαq-GPCR hM3Dq Designer Receptor Exclusively Activated 
by Designer Drugs (DREADD)35 in astrocytes of the striatum in wild-type (WT) and 
Sapap3 KO mice at P28. Three weeks later, we administered the DREADD agonist 
deschloroclozapine (DCZ; 10 μg/kg intraperitoneal [i.p.]) every three days until P180 when 
downstream assessments were performed (Fig. 1C) and when repetitive and anxiety-related 
phenotypes are observed27,36.

We performed experiments for four groups of mice. The first group comprised WT mice 
that received astrocyte-targeted tdTomato (tdT) AAVs (WT + tdT), whereas the second 
group comprised WT mice that received astrocyte-targeted hM4Di or hM3Dq AAVs (WT 

Neuron. Author manuscript; available in PMC 2025 October 23.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Soto et al.

Page 4

+ hM4Di or hM3Dq). The third group comprised Sapap3 KO mice that received astrocyte-
targeted tdTomato AAVs (KO + tdT) and the fourth group comprised the test group where 
Sapap3 KO mice received astrocyte-targeted hM4Di or hM3Dq AAVs (KO + hM4Di or 
hM3Dq). All mice were treated with DCZ, permitting assessments due to hM4Di or hM3Dq 
GPCR signaling activation. As expected28,30,32, AAV-delivered DREADDs were astrocyte 
selective (Fig. S1A). We initially evaluated WT + tdT and KO + tdT groups, and then 
compared KO + hM4Di or hM3Dq with KO + tdT for repetitive and anxiety-related 
behaviors.

Sapap3 KO mice (P180) display repetitive and compulsive-like phenotypes including 
increased anxiety and excessive self-grooming, which precede facial lesions27. To assess 
compulsion-like behaviors, we measured the area of facial lesions, the total number of 
lesions, the number of grooming bouts, and the time spent grooming in all experimental 
groups. We found KO + hM4Di mice displayed decreased lesion area, decreased number of 
lesions, and spent less time grooming than KO + tdT mice (Figs. 1D and 1F(i); P < 0.05).

Anxiety-related measurements in the elevated plus maze (EPM) and in the open field 
revealed that KO + hM4Di mice spent more time exploring the open arms of the EPM, more 
time in the center of the open field and displayed decreased speed in the center of the open 
field compared to KO + tdT mice (Figs. 1E, 1F(ii), and S1B; P < 0.05). KO + hM4Di mice 
displayed significantly increased total speed and total distance traveled relative to the KO + 
tdT group (Fig. S1B; P < 0.05).

In contrast to the results with Gαi-GPCR pathway activation, hM3Dq-DREADDs did not 
rescue these behaviors in Sapap3 KO mice (Figs. 1G–I and S1C); this does not negate 
other Gq-GPCR functions37. By comparing KO + hM4Di behavioral data to our previously 
reported astrocyte-selective Sapap3 genetic rescue10, we found that hM4Di activation was 
equally effective at correcting repetitive behaviors and better at correcting anxiety-related 
behaviors (Figs. 1J and 1K). hM4Di activation was comparable to fluoxetine10, the first-line 
therapeutic effective in some OCD patients14 (Fig. 1K). Since the Gαi-GPCR pathway was 
effective in Sapap3 KO mice (Fig. 1), we performed molecular and cellular assessments.

Improvement of astrocyte morphology in Sapap3 KO mice

Since astrocytic hM4Di activation caused improvements in behavior, and since astrocyte 
morphology is reduced in Sapap3 KO mice, we assessed astrocyte morphology using 
LifeAct-GFP to visualize the actin cytoskeleton10 (Fig. 2A). We observed significant 
reduction in the intensity of actin cytoskeleton labeling in KO + tdT astrocytes when 
compared to WT + tdT, especially towards the periphery. The reduced actin intensity was 
improved in KO + hM4Di astrocytes (Fig. 2B; P < 0.05). Assessments revealed significant 
25% reduction in astrocyte territory and a decrease of cell roundness in KO + tdT astrocytes, 
which were improved in KO + hM4Di astrocytes (Fig. 2B; P < 0.05). Other morphological 
parameters13,38 are shown in Fig. 2C, 2D and Fig. S1D. There was no change in the 
number of major branches emanating from somata between experimental groups (Fig. 2E), 
indicating that reduced morphology is associated with astrocytic branchlets and leaflets4 
observed with LifeAct-GFP. Additionally, activation of hM4Di in the WT experimental 

Neuron. Author manuscript; available in PMC 2025 October 23.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Soto et al.

Page 5

group did not alter astrocyte morphology, indicating Gαi-GPCR mediated improvement in 
Sapap3 KO mice comparable to genetic rescue10 of Sapap3 (Fig. 2C).

Improvements in neuronal cell excitability

Since astrocytes regulate neurons23,39, we explored if astrocyte Gαi-GPCR pathway 
activation affected neurons in Sapap3 KO mice. We assessed ΔFosB, a marker of sustained 
neuronal activity40, across the striatum and select cortical areas (Fig. 3A). KO + tdT mice 
displayed increased density of ΔFosB neurons in the striatum and cortex, including the 
motor cortices and the orbitofrontal cortex (Fig. 3A). Such increased density of ΔFosB 
neurons was decreased in striata of KO + hM4Di mice (Fig. 3A), corresponding to striatal 
hM4Di AAV expression (Fig. 1C). We next analyzed ΔFosB+ neurons in the dorsal and 
ventral striatum and found that ΔFosB was significantly decreased in KO + hM4Di mice 
compared to KO + tdT mice (Fig. 3B; P < 0.05). Medium spiny neurons (MSNs) constitute 
95% of striatal neurons and comprise D1 MSNs of the direct pathway and D2 MSNs of 
the indirect pathway41–44. Both populations may contribute to OCD-related phenotypes44,45. 
Using RNAscope for D1 and D2 mRNA in conjunction with ΔFosB immunostaining, we 
found that astrocytic hM4Di activation significantly improved the elevated ΔFosB levels in 
both D1 and D2 MSNs of Sapap3 KO mice (Fig. 3C, D; Fig. S2A). Furthermore, there 
were no differences in the proportions of D1 and D2 MSNs expressing ΔFosB (Fig. 3E), in 
accord with astrocytes contacting both MSN types29. Since ΔFosB likely reflects elevated 
neuronal activity in Sapap3 KO mice46, we assessed excitability of MSNs across all four 
experimental groups (Fig. 3F–J; Table S1). MSNs from KO + tdT mice displayed increased 
excitability when compared to WT + tdT mice (Fig. 3F–J). Consistent with ΔFosB findings, 
astrocyte activation of hM4Di significantly decreased the elevated excitability of MSNs 
from Sapap3 KO mice, which could be seen by measuring the Rheobase and the number of 
action potentials evoked by the depolarizing current steps (Fig. 3F–J; Table S1). In contrast, 
we found no changes in MSN spontaneous excitatory postsynaptic currents (EPSCs) across 
experimental groups due to astrocytic hM4Di activation (Fig. S2). However, in accord with 
effects on MSN excitability, increased spontaneous inhibitory postsynaptic currents that 
likely arrive from action potential firing in surrounding MSNs were reversed by astrocytic 
activation of hM4Di in Sapap3 KO mice (Fig. S2). Thus, behavioral improvements caused 
by Gαi-GPCR signaling (Fig. 1) are associated with improved astrocyte morphology (Fig. 2) 
and neuronal functions (Fig. 3) in Sapap3 KO mice.

Past studies27,47–49 assessed EPSCs in mice at ages when overt behavioral symptoms do 
not occur. From such studies, we could not surmise what happens to EPSCs in the central 
striatum of Sapap3 KO mice at 6 months, but our findings of increased MSN firing are 
consistent with past work46,50 51 at similar ages.

Molecular changes in astrocyte-related tissue homeostasis gene products

To evaluate mechanisms underlying improved behavior and neuronal activity in KO + 
hM4Di mice, we assessed striatal tissue at the proteomic and transcriptomic level (Fig. 
4A). We compared changes that occur in Sapap3 KO versus WT. At the protein level, 
209 proteins were differentially expressed (FDR < 0.05) while at the RNA level, 78 genes 
were differentially expressed (FDR < 0.05) (Fig. 4B). When we compared differentially 

Neuron. Author manuscript; available in PMC 2025 October 23.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Soto et al.

Page 6

expressed proteins (DEPs) to genes (DEGs), 31 were common, therefore, we focused our 
analyses on the DEPs. Enrichr gene ontology (GO) analysis revealed that the dysregulated 
DEPs were involved in processes related to guanyl-ribonucleotide activity, indicative of G-
protein signaling, to actin filament regulation, relating to astrocyte morphology10, glutamate 
transporter activity and ion homeostasis (Fig. 4C). Assignment of the DEPs to protein 
classes revealed that the most represented groups of proteins were transporters and channels 
including Slc1a2, Atp1a2, and Atp1b1 which contribute to tissue glutamate and potassium 
homeostasis (Fig. 4C and Table S2).

To explore how KO + hM4Di mice show behavioral and cellular improvements, we 
compared changes that occur in KO + hM4Di to KO + tdT striata and found 271 DEPs (FDR 
< 0.05). We compared these 271 DEPs to the DEPs observed in Sapap3 KO and found 43 
common DEPs which were improved by hM4Di activation (Fig. 4D): ~10% of these were 
astrocytic13 transporters and channels, including Atp1a2 and Slc1a2, a target in OCD20. 228 
Proteins were unique to KO + hM4Di indicating orthogonal beneficial hM4Di mechanisms. 
When we plotted the top 20 and bottom 20 proteins by enrichment of these unique proteins, 
several were cytoskeletal proteins (e.g., Gfap, Vim, Ankrd63). Membrane receptors were 
also changed including an NMDA receptor subunit (Grin2a) and the G-protein coupled 
GABAB receptor (Gabbr2) that contributes to astrocyte morphology52.

Using GO analyses, we found that the 43 hM4Di-recovered proteins were involved in 
tissue homeostasis including K+ regulation, whereas the 228 hM4Di-unique proteins were 
involved in signaling, including synaptic transmission (Fig. 4E). Thus, the main pathways 
that accompany improvement of MSN and behavioral dysfunction are homeostatic, driven 
by morphologically complex astrocytes4. Our results indicate loss of tissue homeostasis 
due to dysregulation of astrocytic transporters, channels, and receptors may drive altered 
neuronal signaling and thus Sapap3 KO phenotypes. Parsimoniously, recovery of astrocyte 
morphology with hM4Di drives improvement of astrocyte homeostatic mechanisms and 
ultimately behavior (Figures 1–4).

To study whether the aforementioned outcomes were relevant to human OCD, we compared 
the transcriptomic changes in human OCD caudate RNA-seq data that represent postmortem 
tissue from eight OCD patients and eight control patients (P-value < 0.05)12 to Sapap3 KO 
striatum data (Figure S3 and Table S2) and found overlap between mechanisms for mouse 
and human, notwithstanding species differences28. In the case of genes shared between 
human and mouse, the major pathways were extracellular matrix, actin cytoskeleton, 
and actin-based cell projection (Figure S3). In the case of non-shared genes, we found 
shared pathways related to potassium channel activity, cell-substrate junctions, and neuron 
projections (Figure S3).

We mapped the 43 hM4Di recovered proteins to significant DEGs from the human data to 
assess convergence, or signatures that change in the same direction in Sapap3 KO mice and 
human OCD data. Of the 43 recovered proteins, 15 also exist as DEGs in the human. Of 
these, KO + hM4Di showed improvement in 10 and most were G-protein associated (Fig. 
4F). We assessed the 228 proteins that were unique to KO + hM4Di and found 51 that 
exist within the human DEGs with 24 showing improved changes (Fig. 4G). Many DEGs 

Neuron. Author manuscript; available in PMC 2025 October 23.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Soto et al.

Page 7

were also cytoskeletal proteins or associated with GPCR signaling, emphasizing pathways in 
mouse and human (Fig. 4B–G) that are downstream of Gαi-GPCR activation and could be 
exploited.

Mechanistic insights in Sapap3 KO mice and in relation to human data

Given that astrocyte specific functions were improved by hM4Di, we explored mechanisms 
by which astrocyte actin regulation could be regulated by hM4Di activation. Protein kinase 
A (PKA) negatively regulates actin polymerization and activation of the Gi-GPCR pathway 
promotes actin polymerization53. Furthermore, Gi-GPCR signaling increases the activity 
of Rac/Rho GTPases and SCAR/WAVE complex proteins, which promote actin growth54 
(Figure 4F). Thus, we assessed the expression of these actin polymerization effectors 
in the context of human OCD, Sapap3 KO, and Sapap3 KO + hM4Di. Of extant actin 
polymerization effectors within mouse (~300 genes from Mouse Genome Informatics), 38 
were highly expressed within astrocytes30 (FPKM > 10). Of these, 40% were changed in 
human OCD and Sapap3 KO and were improved by hM4Di activation revealing how actin 
polymerization dynamics drive astrocyte morphology changes (Figure 4G).

Since our data demonstrate that astrocytes may be exploited in the context of OCD 
via targeting of the Gαi-GPCR pathway (Figures 1–4), we mined single nucleus-RNA 
sequencing of human striatum to identify Gαi-GPCRs55. These data represented ~155k 
astrocytes, of which 2.3k were striatal55. We identified Gαi-GPCRs enriched in striatal 
astrocytes from mouse RNAseq data28 and mapped them onto human striatal data. We 
identified 15 GPCRs expressed by striatal astrocytes in human including a metabotropic 
glutamate receptor (Grm3), S1PR1 which has been shown to directly regulate actin 
within astrocytes56, and GPR37L1, an astrocyte enriched GPCR. Together, these GPCRs 
may represent drug targets for endogenous Gαi pathway activation to improve astrocyte-
dependent dysfunctions for OCD (Fig. 4J).

A defining feature of astrocytes is their complex morphology1,57, which is compromised 
in disease settings4. Our data provide evidence that activating the Gαi-GPCR pathway 
in striatal astrocytes improved astrocyte morphology and homeostasis and was beneficial 
for behavioral outcomes in Sapap3 KO mice that display anxiety and repetitive behaviors 
also seen in OCD27. This may be pertinent to understanding the human disorder58,59. 
Phenotypic improvements are also measurable at the cellular and molecular level and share 
overlap with available human data55. We thus identified an astrocyte specific approach by 
which compulsive-like and anxiety-related behaviors can be mechanistically explored and 
improved. Consistent with past work31, the Gi-GPCR pathway in control mice did not affect 
such behaviors.

Our molecular data suggest that while neuronal dysfunction ultimately drives altered 
behaviors, ~30% of the molecular changes that occur in the striatum of Sapap3 KO mice 
relate to astrocyte mediated functions such as ion and neurotransmitter homeostasis and 
structural support. Such functions are served by the complex morphology1,57 of astrocytes 
and their interactions with other cells to maintain essential tissue support4,7. GPCR signaling 

Neuron. Author manuscript; available in PMC 2025 October 23.

Discussion

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Soto et al.

Page 8

contributes to multiple aspects of astrocyte biology, including their morphology4,28,52,60. 
We note that while our study shows that Gαi-GPCR pathway activation drives actin-based 
morphological improvements in astrocytes53, additional mechanisms that are engaged after 
morphology is restored also exist (e.g. homeostasis broadly)28,31. We propose astrocyte 
morphology and homeostatic mechanisms are likely to be interrelated and altered in multiple 
diseases including psychiatric disorders and aging5,9,13.

Limitations of the study

While all aspects of OCD cannot be modeled in mice and although further human genetic 
studies for OCD are necessary, Sapap3 is a promising functional candidate for further 
study. Relatedly, DLGAP3 mRNA is decreased in OCD human caudate12, and OCD gene 
expression analyses identify astrocytic alterations12. Earlier genetic studies have implicated 
DLGAP3 in OCD, for example rare DLGAP3 missense variants have been observed in OCD 
and trichotillomania61, and a family-based association study found a link between DLGAP3 
and grooming disorders62. Sapap3 KO mice also display behaviors related to those observed 
in Tourette and trichotillomania26, implying that the mice model phenotypes broader than 
previously thought. While genomic and transcriptomic studies of human OCD have been 
helpful12,25, further work is needed to understand cellular changes, including for astrocytes.

We assessed behavioral changes in Sapap3 KO mice at 6 months of age after 4 months of 
hM4Di activation, but we did not assess shorter stimulations, nor did we assess if hM3Dq 
activation affected astrocyte morphology. Another limitation is that our electrophysiological 
recordings cannot differentiate between D1 and D2 MSNs, although we found no differences 
between D1 and D2 MSN ΔFosB following astrocytic hM4Di activation. Our findings 
should not be overinterpreted as additional studies to explore astrocytes are needed. Our 
studies on astrocytes in compulsive-like behaviors10 recall studies with microglia63,64.

Overall, psychiatric and neurological disorders have been targeted therapeutically with 
strategies that focus on restoring disease-associated neuronal or circuit dysfunctions. 
Such studies are important for OCD51,65. Our findings show that additional restoration 
of astrocyte dysfunctions such as their morphology and tissue homeostasis represent a 
potential therapeutic strategy and that astrocytes could be exploited in these regards for brain 
disorders9,13.

STAR Methods

RESOURCE AVAILABILITY

Lead contact—Further information and requests for resources and reagents should 
be directed to and will be fulfilled by the Lead Contact, Baljit S. Khakh 
(bkhakh@mednet.ucla.edu).

Materials availability—All unique/stable reagents generated in this study are available 
upon request from the Lead Contact.

Data and code availability—All data and codes are available upon request from the 
Lead Contact. Raw proteomics data from all experimental groups have been deposited in 

Neuron. Author manuscript; available in PMC 2025 October 23.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Soto et al.

Page 9

the Mass Spectrometry Interactive Virtual Environment (MassIVE) with accession number 
MSV000093698. Raw and normalized RNA-seq data from all experimental groups have 
been deposited in the Gene Expression Omnibus with accession number GSE247241. RNA-
seq FPKM values are provided in Table S3. The results of statistical comparisons, n numbers 
and P values are shown in the figure panels or figure legends. All statistical tests and 
replicate values are reported in Tables S4–5.

EXPERIMENTAL MODEL AND SUBJECT DETAILS

Mouse models—All experiments described were conducted in accordance with the 
National Institutes of Health’s Guide for the Care and Use of Laboratory Animals and were 
approved by the Chancellor’s Animal Research Committee at the University of California, 
Los Angeles (UCLA). Male and female mice aged between P21-P28 were used for AAV 
microinjections. Mice were housed in the vivarium managed by the UCLA Division of 
Laboratory Animal Medicine with a 12 h light–dark cycle and ad libitum access to food 
and water. Data for experiments were collected from adult mice at P180. Targeted KO 
mice for Dlgap3 (B6.129-Dlgap3tm1Gfng/J) were obtained from the Jackson Laboratory 
and maintained as a heterozygous line.

Bioinformatic analyses from previous studies—Figure 1A(i) was made from 
databases reported by us in a past study on gene modules related to astrocyte morphology13. 
Briefly, gene modules related to astrocyte territory size and genes associated with central 
nervous system diseases were correlated with one another using hypergeometric analysis. 
The significance of over-representation was calculated using Fisher’s exact test. Figure 
1A(ii) was made from our past astrocyte RNAseq data28. In summary, the percentage of 
genes related to human genes associated with HD and OCD were mapped onto mouse 
striatum single cell RNA-sequencing data based on the top 1000 cell type marker genes 
for 6 major brain cell types. Figure 1B was made using proteomic data published by 
us10. Astrocyte Sapap3 interactors were obtained with astrocyte-specific Sapap3-BioID210. 
Proteins related to the actin cytoskeleton and G-protein signaling are shown. To study 
how our mouse data could potentially be relevant to human OCD, we mapped the hM4Di 
recovered proteins to significant DEGs (FDR < 0.05) from human OCD caudate RNA-seq 
data to assess convergence12. We define convergence as gene identities and signatures that 
change in the same direction (i.e., up or down) in both Sapap3 KO mice and human OCD 
data. These human data represent bulk postmortem tissue from eight OCD patients and 
eight control patients with an average age of 47 years and with a 15–16-hour post-mortem 
interval12. When a past multiomic dataset is compared in relation to our data, it is mentioned 
as such in the figure legends.

Stereotaxic injections of adeno-associated viruses—All surgical procedures were 
conducted as previously described 10. Surgery was conducted under general anesthesia 
using continuous isoflurane (induction at 5%, maintenance at 2% v/v) in 4 week old 
Sapap3 KO mice unless otherwise stated. Anesthetic depth was continuously monitored and 
adjusted when necessary. After anesthetic induction, mice were fitted into the stereotaxic 
frame (David Kopf Instruments) with their noses placed into a veterinary-grade anesthesia 
ventilation system (VetEquip) and their heads were secured using blunt ear bars. Mice were 

Neuron. Author manuscript; available in PMC 2025 October 23.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Soto et al.

Page 10

subcutaneously administered with 0.1 mg/kg of buprenorphine (Bupranex) before surgery. 
The incision site was cleaned 3 times with 10% povidone iodine and 70% ethanol (v/v). 
A skin incision was made followed by craniotomies (1–2 mm in diameter) above the left 
and right parietal cortices using a small steel burr (NeoBurr) powered by a high-speed drill 
(Midwest Tradition). Sterile saline (0.9%) was applied onto the skull to reduce heating 
caused by drilling. The injections were carried out using the stereotaxic apparatus to guide 
the placement of bevelled glass pipettes (1B100–4, World Precision Instruments). AAVs 
were bilaterally injected into the central striatum using 3 injection sites per hemisphere 
to achieve maximal striatal tissue coverage. Since the AAVs targeted the central striatum, 
we also performed all cellular assessments within the central striatum. At each of the 
injection sites, the microinjection needle was advanced to the deepest (ventral) position 
for the first injection, whereas the additional injections were made every 0.3 mm while 
withdrawing the injection needle. The coordinates from bregma were as follows: injection 
site 1: anterior, 0.5 mm; medial–lateral, 1.5 mm, dorsal–ventral, 2.9 mm, 2.6 mm and 2.3 
mm from the pial surface; injection site 2: anterior, 0.5 mm, medial–lateral, −1.5 mm; 
dorsal–ventral, 2.9 mm, 2.6 mm and 2.3 mm from the pial surface. For each injection 
location, 100 nl of virus was injected and the needle was left in place for 5 min after each 
injection. These injection procedures were chosen to cover most of the striatum (dorsal and 
ventral). All AAV titers were adjusted to 1.0 × 10E13 genome copies per ml with sterile 0.1 
M PBS. The following viruses were used: AAV 2/5 GfaABC1D-LifeAct-eGFP (Addgene, 
190199); AAV2/5 GfaABC1D-hM4Di-mCherry (Addgene, 92286); AAV2/5 GfaABC1D-
hM3Dq-mCherry (Addgene, 92284); AAV2/5 GfaABC1D-tdTomato (Addgene, 44332)66. 
Litters with multiple wild-type and Sapap3 KO mice were split between experimental 
groups.

In vivo activation of hM4Di in striatal astrocytes—Three weeks after AAV 
microinjection of tdTomato or DREADDs, wild-type and Sapap3 KO mice were 
intraperitoneally injected with deschloroclozapine (DCZ) at a dose of 10 μg/kg 3 times 
a week until 6 months of age. DREADD activation with DCZ began at 7 weeks of 
age prior to the onset of symptoms to benchmark our findings with previous genetic 
rescue experiments10,27. To minimize receptor desensitization with chronic treatment, we 
conducted DCZ injection procedure every three days35,67,68.

Behavioral evaluations—All behavioral tests were performed during the light cycle 
between the hours of 10:00 and 14:00. Mice were assessed at 6 months of age or 5 months 
after AAV microinjection. All experimental mice were extensively handled up to 4 weeks 
prior to the behavioral tests. Mice were transferred to the behavioral testing room at least 30 
min prior to the start of the testing to acclimatize to the environment and reduce stress. The 
temperature and humidity of the experimental rooms were kept at 23 ± 2 °C and 55 ± 5%, 
respectively. The experimental room was kept dimly lit unless otherwise stated. Background 
noise (60–65 dB) was generated using a white noise box (San Diego Instruments). Litters 
with multiple wild-type and Sapap3 KO mice were split between experimental groups. The 
mice were randomly allocated to a group as they became available and of age from the 
breeding colony in alternation. Experimenters were blinded to group allocation during data 
collection and analyses.

Neuron. Author manuscript; available in PMC 2025 October 23.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Soto et al.

Page 11

Self-grooming behavior—The self-grooming behavior test was adapted from previously 
published work 44. The recording was conducted at 35 lux illumination. Mice were placed 
individually into plastic cylinders (15 cm in diameter and 35 cm tall) and allowed to 
habituate for 20 min. Self-grooming behavior was recorded for 10 min. A timer was used 
to assess the cumulative time spent in self-grooming behaviors including paw licking, 
unilateral and bilateral strokes around the nose, mouth and face, paw movement over the 
head and behind the ears, body-fur licking, body scratching with hind paws, tail licking and 
genital cleaning. The number of self-grooming bouts and rearing bouts was also counted. 
Separate grooming bouts were considered when the pause was more than 5 s or if behaviors 
other than self-grooming occurred.

Skin lesion assessment—Mice were anaesthetized with 5% isoflurane and 1% O2 
through a veterinary-grade anesthesia ventilation system (VetEquip). Mice were placed on 
an opaque Plexiglass board, and photos of their head and torso were taken bilaterally. 
Images were scaled with a ruler (Fine Science Tools) and the images were analysed using 
ImageJ software. Measurements were all scaled to the ruler on ImageJ.

Open-field test—The open-field chamber was illuminated at 35 lux. The open-field 
chamber consisted of a square arena (28 × 28 cm) enclosed by walls made of opaque 
Plexiglass (19 cm tall). The periphery of the arena was defined as the area within 2.5 cm 
adjacent to the walls of the chamber and the center of the arena was defined as the area 2.5 
cm away from the chamber walls. Activity was recorded for 20 min using a video camera 
(Logitech) located above the open-field chamber. Anymaze video analysis software was 
used to quantify the time spent in the center, the total distance travelled and speed.

Elevated plus maze—All four arms of the elevated plus maze were illuminated at 25 lux. 
The elevated plus maze consisted of arms that were 30 × 7 cm with 20 cm high closed arm 
walls. The maze was elevated 65 cm above floor level and was placed in the center of the 
room away from other stimuli. Mice were placed in the center of the maze facing an open 
arm. Mice were recorded for 10 min using a video camera (Logitech) located above the 
maze. Anymaze video analysis software was used to quantify time spent in open arms and 
the percent time spent in open arms.

Immunohistochemistry (IHC)—Mice were transcardially perfused with chilled 0.1 M 
PBS followed by 10% formalin. The brains were post-fixed in 10% formalin for 6 h 
after removal of the skull. The brains were then cryoprotected in 30% sucrose with 
0.1 M PBS solution for at least 48 h at 4 °C. Coronal sections (40 μm) containing 
striatum were prepared using a cryostat microtome (Leica) at −20 °C and processed 
for IHC. Sections were washed 3 times in 0.1 M PBS for 10 min each and incubated 
in a blocking solution consisting of 5% normal goat serum (NGS, Vector Laboratories 
S-1000) in 0.1 M PBS with 0.2% Triton-X for 1 h at room temperature with agitation. 
Sections were then incubated in primary antibodies diluted in 5% NGS in 0.1 M PBS 
solution overnight at 4 °C. The following primary antibodies were used: rabbit anti-S100β 
(1:1,000; Abcam, ab13970); rabbit anti-NeuN (1:1,000; Cell Signaling, 12943S); guinea 
pig anti-NeuN (1:1,000; Synaptic Systems, 266004); chicken anti-GFP (1:1,000; Abcam, 

Neuron. Author manuscript; available in PMC 2025 October 23.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Soto et al.

Page 12

ab13970);); mouse anti-RFP (1:500; Rockland, 600906379); and rabbit anti-ΔFosB (1:500; 
Cell Signaling Technology, 14695S). Sections were then incubated with the following 
secondary antibodies for 2 h at room temperature (1:1,000; Molecular Probes): Alexa Fluor 
488 goat anti-chicken (A11039); Alexa Fluor 546 goat anti-mouse (A11003); Alexa Fluor 
647 goat anti-rabbit (A21244); Alexa Fluor 647 goat anti-mouse (A21235); Alexa Fluor 
488 goat anti-rabbit (A11008). The free-floating sections were mounted on microscope 
slides in fluoromount-G. Fluorescence images were taken using a UplanFL ×40 1.30 NA 
oil-immersion or a PlanApo N ×60 1.45 NA oil-immersion objective lens on a confocal 
laser-scanning microscope (FV3000, Olympus) using Fluoview (Olympus) software. Laser 
settings were kept the same within each experiment. Images represent maximum intensity 
projections of optical sections with a step-size of 1.0 μm. Images were processed using 
ImageJ. Cell counting was done on maximum intensity projections using the Cell Counter 
plugin; only cells with somata completely in the region of interest were counted. Analysis 
of astrocyte morphology parameters including astrocyte territory measurements and fractal 
dimension analysis was conducted on maximum intensity projections as previously 
described13,38.

Dual in situ hybridization with immunohistochemistry—Fixed-frozen tissue was 
processed as described above. Serial coronal sections (20 μm) containing striatum were 
prepared using a cryostat microtome (Leica) at −20 °C and mounted immediately onto glass 
slides. Dual ISH-IHC was performed using a Multiplex RNAscope (v.2) with integrated 
co-detection work flow (ACDBio 323180 and 323110). Sections were baked for 45 min at 
60 °C. Sections were then washed for at least 20 min in 0.1 M PBS and then incubated 
in 1× Target Retrieval reagents for 5 min at 95 °C. After washing with ddH2O twice, the 
sections were dehydrated with 100% ethanol and dried at room temperature. Sections were 
then incubated with primary antibody rabbit anti-ΔFosB (1:500; Cell Signaling Technology, 
14695S) overnight at 4 °C. Sections were then incubated with Protease Pretreat-4 solution 
(ACDBio, 322340) for 30 min at 40 °C. The sections were washed with ddH2O twice for 1 
min each and then incubated with probe for 2 h at 40 °C: Mm-Drd1-C3 (ACDBio, 461901-
C3) or Mm-Drd2-C1 (ACDBio, 406501-C3). The sections were incubated in Amp 1-FL for 
30 min, AMP 2-FL for 15 min, AMP 3-FL for 30 min and AMP 4-FL for 15 min at 40 
°C while washing in 1× wash buffer (ACDBio, 310091) between incubations. The HRP-C3 
signal was developed with Opal 690 fluorophore (Akoya Biosciences, FP1497001KT). All 
incubations at 40 °C or 60 °C were performed in a HybEZ hybridization system (ACDBio). 
Last, sections were incubated with Alexa Fluor goat secondary antibodies described in the 
IHC section for 45 min at room temperature. Images were obtained in the same way as for 
IHC (described above) with a step size of 0.5 μm. Images were processed using ImageJ. Cell 
counting was done on maximum intensity projections using the Cell Counter plugin.

Whole striatum ΔFosB mapping— Mice were transcardially perfused with ice-cold 
0.1M PBS with 10 U/mL heparin followed by ice-cold 4% paraformaldehyde (PFA). Brains 
were carefully extracted from the skull and incubated with 4% PFA at 4 °C for 24 hours. The 
brains were then washed in 0.1M PBS and then stored in 0.02% sodium azide in PBS prior 
to shipping to LifeCanvas Technologies. The brains were SHIELD post-fixed by LCT and 
cleared for 5 days with Clear+ delipidation buffer 69. The brains were labeled with 3.5 ug 

Neuron. Author manuscript; available in PMC 2025 October 23.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Soto et al.

Page 13

rabbit anti-ΔFosB and imaged with SmartSPIM at 4 μm z-steps with 1.8 μm x-y pixel size. 
Brains regions were mapped using the Allen Brain Atlas and the number of ΔFosB cells 
were counted to obtain the density for the striatum and select cortical areas.

Brain slice preparation and electrophysiological recordings—300 μm coronal 
slices containing striatum were prepared from 6 month old wild-type and Sapap3 KO mice 
injected with either tdTomato or hM4Di and treated with DCZ. Briefly, the mice were 
anaesthetized with isoflurane and perfused trascardially using ice-cold modified artificial 
cerebral spinal fluid (aCSF) containing (in mM): 194 sucrose, 30 NaCl, 4.5 KCl, 1 
MgCl2, 26 NaHCO3, 1.2 NaH2PO4, and 10 D-glucose, bubbled with 95% O2 and 5% 
CO2. The head was decapitated, and the brain was rapidly extracted. The brain was 
then sliced in same modified aCSF solution using a Vibratome (7000smz-2, Campden 
Instruments). After sectioning, slices were incubated for 20 minutes at 32 °C in normal 
aCSF containing (in mM): 124 NaCl, 4.5 KCl, 2 CaCl2, 1 MgCl2, 26 NaHCO3, 1.2 
NaH2PO4, and 10 D-glucose bubbled with 95% O2 and 5% CO2. The slices were then 
allowed to equilibrate at room temperature for 30 minutes before recording. Slices were 
transferred in recording chamber with continuous flow of oxygenated aCSF. Whole-cell 
patch-clamp recordings were obtained from medium spiny neurons (MSNs) near astrocytes 
containing red fluorescence in the central striatum. All recordings were performed at room 
temperature using pClamp 10.2 (Molecular Devices) and a Multiclamp 700B (Molecular 
Devices). Currents were amplified, filtered at 2 kHz and digitized at 10 kHz using Digidata 
1550 (Molecular Devices). Analyses were performed using ClampFit (v 10.7) software.

Whole-tissue protein extraction from striatum—To assess broad tissue-level changes 
that occur in the striatum, striata from 4 mice per experimental group (Sapap3 WT or KO 
mice microinjected with tdTomato or hM4Di and DCZ injection) were lysed in 200 μl lysis 
buffer (8 M urea, 50 mM Tris-HCl pH 8.2, 75 mM NaCl, 5 mM EDTA, 5 mM EGTA, 10 
mM sodium pyrophosphate and Halt protease inhibitor cocktail (Thermo Scientific, 78429)). 
Tissue was dounce homogenized and extracts were sonicated for 10 min at 80% power in 
a bath sonicator. Samples were then centrifuged at 15,000g for 20 min at 4 °C to remove 
debris. The supernatant was collected, measured with BCA (Pierce 23225), and then further 
processed.

High-performance liquid chromatography and mass spectrometry analysis—
Protein samples were subjected to reduction using 5 mM Tris (2-carboxyethyl) phosphine 
for 30 min, alkylated by 10 mM iodoacetamide for another 30 min and then digested 
with Lys-C and trypsin at a 1:100 protease-to-protein ratio for 4 and 12 h, respectively. 
The digestion reaction was terminated by the addition of formic acid to 5% (v/v) with 
centrifugation. Each sample was then desalted with C18 tips (Thermo Scientific, 87784) 
and dried in a SpeedVac vacuum concentrator. The peptide pellet was reconstituted in 5% 
formic acid before analysis by LC–MS/MS. Tryptic peptide mixtures were loaded onto 
a 25-cm long, 75 μm inner diameter fused-silica capillary, packed in-house with bulk 
1.9 μM ReproSil-Pur beads with 120 Å pores. Peptides were analysed using a 140 min 
water–acetonitrile gradient delivered by a Dionex Ultimate 3000 UHPLC (Thermo Fisher 
Scientific) operated initially at a 400 nl min–1 flow rate with 1% buffer B (acetonitrile 

Neuron. Author manuscript; available in PMC 2025 October 23.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Soto et al.

Page 14

solution with 3% DMSO and 0.1% formic acid) and 99% buffer A (water solution with 
3% DMSO and 0.1% formic acid). Buffer B was increased to 6% over 5 min, at which 
time the flow rate was reduced to 200 nl min–1. A linear gradient from 6 to 28% of buffer 
B was applied to the column over the course of 123 min. The linear gradient of buffer 
B was then further increased to 28–35% for 8 min followed by a rapid ramp-up to 85% 
for column washing. Eluted peptides were ionized using a Nimbus electrospray ionization 
source (Phoenix S&T) by application of a distal voltage of 2.2 kV. Spectra were collected 
using data-dependent acquisition on an Orbitrap Fusion Lumos Tribrid mass spectrometer 
(Thermo Fisher Scientific) with a MS1 resolution of 120,000 followed by sequential MS2 
scans at a resolution of 15,000. Data generated by LC–MS/MS were searched using the 
Andromeda search engine integrated into the MaxQuant52 bioinformatics pipelines against 
the UniProt Mus musculus reference proteome (UP000000589) and then filtered using 
a ‘decoy’ database-estimated FDR < 1%. LFQ was carried out by integrating the total 
extracted ion chromatogram of peptide precursor ions from the MS1 scan. These LFQ 
intensity values were used for protein quantification across samples. Statistical analysis of 
differentially expressed proteins (DEPs) was done using the Bioconductor package ArtMS 
(v.3.18; https://bioconductor.org/packages/release/bioc/html/artMS.html). Protein data has 
been deposited at MassIVE with accession number MSV000093698.

The GO enrichment analysis for protein type was performed using PANTHER 
overrepresentation test (GO database released 30 May 2023) with FDR < 0.05 and with 
all M. musculus genes used as reference. GO pathway analysis for molecular function and 
biological function was performed using Enrichr (https://maayanlab.cloud/Enrichr/).

Whole-tissue RNA extraction from striatum—RNA extraction from striatum was 
performed using standard methods. Briefly, striata from 4 mice per experimental group 
(Sapap3 WT or KO mice microinjected with tdTomato or hM4Di and DCZ injection) 
were dissected and lysed in 350 mL lysis buffer. RNA was extracted using RNeasy Plus 
mini kit (Qiagen, 74134) following the manufacturer’s protocol. RNA concentration and 
quality were assessed with Agilent 2100 Bioanalyzer. RNA samples with RNA integrity 
number (RIN) greater than seven were processed with Ribo-Zero Gold kit (Epicentre, WI) to 
remove ribosomal RNA. Sequencing libraries were prepared using Illumina TruSeq Stranded 
RNA sample prep kit following manufacturer’s protocol. Samples in each experiment were 
multiplexed into a single pool to avoid batch effects and sequencing was performed using 
an Illumina NovaSeq 6000 sequencer (Illumina, San Diego, CA). A yield between 70 and 
142 million reads was obtained per sample. Approximately, 87% of the reads mapped 
uniquely to the mouse genome. Total counts of read fragments aligned to candidate gene 
regions were derived using HTSeq program (www.huber.embl.de/users/anders/HTSeq/doc/
overview.html) with mouse mm10 refSeq (refFlat table) as a reference and used as a 
basis for the quantification of gene expression. Only uniquely mapped reads were used 
for subsequent analyses. Differential expression analysis was conducted with R-project 
and the Bioconductor package limmaVoom using an FDR cutoff of < 0.05. RNA-seq 
data has been deposited to the Gene Expression Omnibus (GEO) repository (https://
www.ncbi.nlm.nih.gov/geo) with accession number GSE247241.

Neuron. Author manuscript; available in PMC 2025 October 23.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Soto et al.

Page 15

Software used—No custom software was used. For proteomics, LFQ was carried out 
using MaxQuant with an integrated search engine, Andromeda (https://www.maxquant.org/). 
Differential protein expression and enrichment analysis was conducted using the 
Bioconductor R package ArtMS (v.3.18; https://bioconductor.org/packages/release/bioc/
html/artMS.html). For RNA-seq, differential gene expression and enrichment analysis used 
the R package limmaVoom (v.3.36) to process RNA counts (https://rdrr.io/bioc/limma/man/
voom.html). All imaging analysis was performed with ImageJ (https://imagej.nih.gov/ij/). 
Electrophysiological analysis was conducted with ClampFit (v 10.7) software.

STATISTICAL ANALYSES

Data from every experiment represent at least three replicates. All statistical tests, unless 
otherwise stated, were run in OriginPro 2018 and GraphPad InStat3. Data are presented 
as the mean ± S.E.M.. along with the individual data points. The results of statistical 
comparisons, n numbers and significance levels are shown in the figure panels or in the 
figure legends along with the average data. N is defined as the number of cells or mice on a 
case-by-case basis throughout the manuscript. We determined whether each set of data was 
normally distributed using GraphPad Instat3 and OriginPro 2018. If the data were normally 
distributed, we used parametric tests, but if they were not normally distributed, we used 
nonparametric tests. Paired and unpaired Student’s two-tailed t-tests (as appropriate), two 
tailed Mann–Whitney tests and one-way and two-way analysis of variance tests were used 
for most statistical analyses with significance declared at P < 0.05. When P values were 
greater than 0.05, they are stated as not significant. When the P value was less than 0.01, 
it is stated as <0.01. All proteomics and transcriptomics analyses used a statistical FDR < 
0.05 unless otherwise stated. All mice were assigned to particular experimental groups at 
random. No data points were excluded from any experiment. Replicate values and the results 
of statistical tests are provided in Tables S4 and S5.

Supplementary Material

Refer to Web version on PubMed Central for supplementary material.

Acknowledgments

Thanks to Fuying Gao and the UCLA Neuroscience Genomics Core for assistance with RNA sequencing. Thanks 
to Leah Chestnut and Zachary Woods (LifeCanvas Technologies) for processing ΔFosB brains. BSK, JSS, and 
CN were supported by the National Institutes of Health (R35NS111583), the Allen Distinguished Investigator 
Award, a Paul G. Allen Frontiers Group advised grant of the Paul G. Allen Family Foundation, and the Ressler 
Family Foundation. JSS was partly supported by a National Science Foundation Graduate Research Fellowship 
(DGE-2034835).

References

1. Salmon CK, Syed TA, Kacerovsky JB, Alivodej N, Schober AL, Sloan TFW, Pratte MT, Rosen MP, 
Green M, Chirgwin-Dasgupta A, et al. (2023). Organizing principles of astrocytic nanoarchitecture 
in the mouse cerebral cortex. Current biology : CB 33, 957–972.e955. 10.1016/j.cub.2023.01.043. 
[PubMed: 36805126] 

2. Allen NJ, and Lyons DA (2018). Glia as architects of central nervous system formation and 

function. Science (New York, N.Y.) 362, 181–185. [PubMed: 30309945] 

Neuron. Author manuscript; available in PMC 2025 October 23.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Soto et al.

Page 16

3. Stein JL, de la Torre-Ubieta L, Tian Y, Parikshak NN, Hernández IA, Marchetto MC, Baker DK, Lu 
D, Hinman CR, Lowe JK, et al. (2014). A quantitative framework to evaluate modeling of cortical 
development by neural stem cells. Neuron 83, 69–86. [PubMed: 24991955] 

4. Baldwin KT, Murai KK, and Khakh BS (2023). Astrocyte morphology. Trends Cell Biol. 10.1016/

j.tcb.2023.09.006.

5. Verkhratsky A, Butt A, Li B, Illes P, Zorec R, Semyanov A, Tang Y, and Sofroniew MV 

(2023). Astrocytes in human central nervous system diseases: a frontier for new therapies. Signal 
transduction and targeted therapy 8, 396. 10.1038/s41392-023-01628-9. [PubMed: 37828019] 
6. Popov A, Brazhe N, Morozova K, Yashin K, Bychkov M, Nosova O, Sutyagina O, Brazhe A, 

Parshina E, Li L, et al. (2023). Mitochondrial malfunction and atrophy of astrocytes in the aged 
human cerebral cortex. Nature communications 14, 8380. 10.1038/s41467-023-44192-0.

7. Escartin C, Galea E, Lakatos A, O’Callaghan JP, Petzold GC, Serrano-Pozo A, Steinhäuser C, 

Volterra A, Carmignoto G, Agarwal A, et al. (2021). Reactive astrocyte nomenclature, definitions, 
and future directions. Nature neuroscience 24, 312–325. 10.1038/s41593-020-00783-4. [PubMed: 
33589835] 

8. Lee HG, Wheeler MA, and Quintana FJ (2022). Function and therapeutic value of astrocytes in 

neurological diseases. Nature reviews. Drug discovery. 10.1038/s41573-022-00390-x.

9. Ling E, Nemesh J, Goldman M, Kamitaki N, Reed N, Handsaker RE, Genovese G, Vogelgsang JS, 

Gerges S, Kashin S, et al. (2024). Concerted neuron-astrocyte gene expression declines in aging and 
schizophrenia. bioRxiv : the preprint server for biology. 10.1101/2024.01.07.574148.

10. Soto JS, Jami-Alahmadi Y, Chacon J, Moye SL, Diaz-Castro B, J.A. W, and Khakh BS (2023). 

Astrocyte and neuron subproteomes and obsessive-compulsive disorder mechanisms. Nature 616, 
764–773. [PubMed: 37046092] 

11. Yu X, Taylor AMW, Nagai J, Golshani P, Evans CJ, Coppola G, and Khakh BS (2018). Reducing 
astrocyte calcium signaling in vivo alters striatal microcircuits and causes repetitive behavior. 
Neuron 99, 1170–1187. [PubMed: 30174118] 

12. Piantadosi SC, McClain LL, Klei L, Wang J, Chamberlain BL, Springer SA, Lewis DA, Devlin 
B, and Ahmari SE (2021). Transcriptome alterations are enriched for synapse-associated genes 
in the striatum of subjects with obsessive-compulsive disorder. Translational psychiatry 11, 171. 
10.1038/s41398-021-01290-1. [PubMed: 33723209] 

13. Endo F, Kasai A, Soto JS, Yu X, Qu Z, Hashimoto H, Gradinaru V, Kawaguchi R, and Khakh 

BS (2022). Molecular basis of astrocyte diversity and morphology across the CNS in health and 
disease. Science (New York, N.Y.) 378, eadc9020. 10.1126/science.adc9020. [PubMed: 36378959] 

14. Stein DJ, Costa DLC, Lochner C, Miguel EC, Reddy YCJ, Shavitt RG, van den Heuvel OA, 

and Simpson HB (2019). Obsessive-compulsive disorder. Nature reviews. Disease primers 5, 52. 
10.1038/s41572-019-0102-3. [PubMed: 31371720] 

15. Goodman WK, Grice DE, Lapidus KA, and Coffey BJ (2014). Obsessive-compulsive disorder. 
The Psychiatric clinics of North America 37, 257–267. 10.1016/j.psc.2014.06.004. [PubMed: 
25150561] 

16. Pauls DL, Abramovitch A, Rauch SL, and Geller DA (2014). Obsessive-compulsive disorder: an 
integrative genetic and neurobiological perspective. Nature reviews. Neuroscience 15, 410–424. 
10.1038/nrn3746. [PubMed: 24840803] 

17. Saxena S, and Rauch SL (2000). Functional neuroimaging and the neuroanatomy of 

obsessive-compulsive disorder. The Psychiatric clinics of North America 23, 563–586. 10.1016/
s0193-953x(05)70181-7. [PubMed: 10986728] 

18. Rotge JY, Guehl D, Dilharreguy B, Cuny E, Tignol J, Bioulac B, Allard M, Burbaud P, and 

Aouizerate B (2008). Provocation of obsessive-compulsive symptoms: a quantitative voxel-based 
meta-analysis of functional neuroimaging studies. Journal of psychiatry & neuroscience : JPN 33, 
405–412. [PubMed: 18787662] 

19. Kwon JS, Jang JH, Choi JS, and Kang DH (2009). Neuroimaging in obsessive-compulsive 

disorder. Expert review of neurotherapeutics 9, 255–269. 10.1586/14737175.9.2.255. [PubMed: 
19210199] 

20. Aida T, Yoshida J, Nomura M, Tanimura A, Iino Y, Soma M, Bai N, Ito Y, Cui W, Aizawa H, et 

al. (2015). Astroglial glutamate transporter deficiency increases synaptic excitability and leads to 

Neuron. Author manuscript; available in PMC 2025 October 23.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Soto et al.

Page 17

pathological repetitive behaviors in mice. Neuropsychopharmacology : official publication of the 
American College of Neuropsychopharmacology 40, 1569–1579. 10.1038/npp.2015.26. [PubMed: 
25662838] 

21. Tanaka K (2021). Astroglia and Obsessive Compulsive Disorder. Advances in neurobiology 26, 

139–149. 10.1007/978-3-030-77375-5_7. [PubMed: 34888834] 

22. Katz M, Corson F, Keil W, Singhal A, Bae A, Lu Y, Liang Y, and Shaham S (2019). Glutamate 
spillover in C. elegans triggers repetitive behavior through presynaptic activation of MGL-2/
mGluR5. Nature communications 10, 1882. 10.1038/s41467-019-09581-4.

23. Nagai J, Yu X, Papouin T, Cheong E, Freeman MR, Monk KR, Hastings MH, Haydon PG, 

Rowitch D, Shaham S, and Khakh BS (2021). Behaviorally consequential astrocytic regulation of 
neural circuits. Neuron 109, 576–596. 10.1016/j.neuron.2020.12.008. [PubMed: 33385325] 
24. Stogsdill JA, Harwell CC, and Goldman SA (2023). Astrocytes as master modulators of neural 

networks: Synaptic functions and disease-associated dysfunction of astrocytes. Annals of the New 
York Academy of Sciences 1525, 41–60. 10.1111/nyas.15004. [PubMed: 37219367] 
25. Lisboa BCG, Oliveira KC, Tahira AC, Barbosa AR, Feltrin AS, Gouveia G, Lima L, Feio 

Dos Santos AC, Martins DC Jr., Puga RD, et al. (2019). Initial findings of striatum tripartite 
model in OCD brain samples based on transcriptome analysis. Scientific reports 9, 3086. 10.1038/
s41598-019-38965-1. [PubMed: 30816141] 

26. Lamothe H, Schreiweis C, Mondragón-González LS, Rebbah S, Lavielle O, Mallet L, and 

Burguière E (2023). The Sapap3(−/−) mouse reconsidered as a comorbid model expressing 
a spectrum of pathological repetitive behaviours. Translational psychiatry 13, 26. 10.1038/
s41398-023-02323-7. [PubMed: 36717540] 

27. Welch JM, Lu J, Rodriguiz RM, Trotta NC, Peca J, Ding JD, Feliciano C, Chen M, Adams JP, Luo 
J, et al. (2007). Cortico-striatal synaptic defects and OCD-like behaviours in Sapap3-mutant mice. 
Nature 448, 894–900. 10.1038/nature06104. [PubMed: 17713528] 

28. Yu X, Nagai J, Marti-Solano M, Soto JS, Coppola G, Babu MM, and Khakh BS (2020). Context-
Specific Striatal Astrocyte Molecular Responses Are Phenotypically Exploitable. Neuron 108, 
1146–1162. 10.1016/j.neuron.2020.09.021. [PubMed: 33086039] 

29. Octeau JC, Chai H, Jiang R, Bonanno SL, Martin KC, and Khakh BS (2018). An Optical Neuron-
Astrocyte Proximity Assay at Synaptic Distance Scales. Neuron 98, 49–66. [PubMed: 29621490] 

30. Chai H, Diaz-Castro B, Shigetomi E, Monte E, Octeau JC, Yu X, Cohn W, Rajendran PS, 

Vondriska TM, Whittelegge JP, et al. (2017). Neural circuit-specialized astrocytes: transcriptomic, 
proteomic, morphological and functional evidence. Neuron 95, 531–549. [PubMed: 28712653] 
31. Nagai J, Rajbhandari AK, Gangwani MR, Hachisuka A, Coppola G, Masmanidis SC, Fanselow 

MS, and Khakh BS (2019). Hyperactivity with Disrupted Attention by Activation of an Astrocyte 
Synaptogenic Cue. Cell 177, 1280–1292 e1220. 10.1016/j.cell.2019.03.019. [PubMed: 31031006] 

32. Yu X, Nagai J, and Khakh BS (2020). Improved tools to study astrocytes. Nature reviews. 

Neuroscience 21, 121–138. [PubMed: 32042146] 

33. Soto JS, Jami-Alahmadi Y, Wohlschlegel JA, and Khakh BS (2023). In vivo identification of 

astrocyte and neuron subproteomes by proximity-dependent biotinylation. Nature protocols in 
press.

34. Srinivasan R, Lu T-Y, Chai H, Xu J, Huang BS, Golshani P, Coppola G, and Khakh BS (2016). 

New Transgenic Mouse Lines for Selectively Targeting Astrocytes and Studying Calcium Signals 
in Astrocyte Processes In Situ and In Vivo. Neuron 92, 1181–1195. [PubMed: 27939582] 
35. Roth BL (2016). DREADDs for Neuroscientists. Neuron 89, 683–694. [PubMed: 26889809] 
36. Hadjas LC, Schartner MM, Cand J, Creed MC, Pascoli V, Lüscher C, and Simmler 

LD (2020). Projection-specific deficits in synaptic transmission in adult Sapap3-knockout 
mice. Neuropsychopharmacology : official publication of the American College of 
Neuropsychopharmacology 45, 2020–2029. 10.1038/s41386-020-0747-3. [PubMed: 32585679] 

37. Nagai J, Bellafard A, Qu Z, Yu X, Ollivier M, Gangwani MR, Diaz-Castro B, Coppola G, 

Schumacher SM, Golshani P, et al. (2021). Specific and behaviorally consequential astrocyte 
G(q) GPCR signaling attenuation in vivo with iβARK. Neuron 109, 2256–2274.e2259. 10.1016/
j.neuron.2021.05.023. [PubMed: 34139149] 

Neuron. Author manuscript; available in PMC 2025 October 23.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Soto et al.

Page 18

38. Shigetomi E, Bushong EA, Haustein MD, Tong X, Jackson-Weaver O, Kracun S, Xu J, Sofroniew 
MV, Ellisman MH, and Khakh BS (2013). Imaging calcium microdomains within entire astrocyte 
territories and endfeet with GCaMPs expressed using adeno-associated viruses. J Gen Physiol 141, 
633–647. 10.1085/jgp.201210949. [PubMed: 23589582] 

39. Oliveira JF, Sardinha VM, Guerra-Gomes S, Araque A, and Sousa N (2015). Do stars govern 
our actions? Astrocyte involvement in rodent behavior. Trends in neurosciences 38, 535–549. 
[PubMed: 26316036] 

40. Nestler EJ, Barrot M, and Self DW (2001). DeltaFosB: a sustained molecular switch for addiction. 
Proceedings of the National Academy of Sciences of the United States of America 98, 11042–
11046. 10.1073/pnas.191352698. [PubMed: 11572966] 

41. Kreitzer AC (2009). Physiology and pharmacology of striatal neurons. Annual review of 

neuroscience 32, 127–147. 10.1146/annurev.neuro.051508.135422.

42. Burguière E, Monteiro P, Mallet L, Feng G, and Graybiel AM (2015). Striatal circuits, habits, 

and implications for obsessive-compulsive disorder. Current opinion in neurobiology 30, 59–65. 
[PubMed: 25241072] 

43. Graybiel AM, and Grafton ST (2015). The striatum: where skills and habits meet. Cold Spring 

Harb Perspect Biol Aug 3;7(8):a021691. doi: 10.1101/cshperspect.a021691. [PubMed: 26238359] 

44. Kalueff AV, Stewart AM, Song C, Berridge KC, Graybiel AM, and Fentress JC (2016). 

Neurobiology of rodent self-grooming and its value for translational neuroscience. Nature reviews. 
Neuroscience 17, 45–59.

45. Xue J, Qian D, Zhang B, Yang J, Li W, Bao Y, Qiu S, Fu Y, Wang S, Yuan TF, and Lu W (2022). 
Midbrain dopamine neurons arbiter OCD-like behavior. Proceedings of the National Academy 
of Sciences of the United States of America 119, e2207545119. 10.1073/pnas.2207545119. 
[PubMed: 36343236] 

46. Burguière E, Monteiro P, Feng G, and Graybiel AM (2013). Optogenetic stimulation of lateral 
orbitofronto-striatal pathway suppresses compulsive behaviors. Science (New York, N.Y.) 340, 
1243–1246. 10.1126/science.1232380. [PubMed: 23744950] 

47. Wan Y, Feng G, and Calakos N (2011). Sapap3 deletion causes mGluR5-dependent silencing 
of AMPAR synapses. The Journal of neuroscience : the official journal of the Society for 
Neuroscience 31, 16685–16691. 10.1523/jneurosci.2533-11.2011. [PubMed: 22090495] 

48. Wan Y, Ade KK, Caffall Z, Ilcim Ozlu M, Eroglu C, Feng G, and Calakos N (2014). 

Circuit-selective striatal synaptic dysfunction in the Sapap3 knockout mouse model of obsessive-
compulsive disorder. Biological psychiatry 75, 623–630. 10.1016/j.biopsych.2013.01.008. 
[PubMed: 23414593] 

49. Corbit VL, Manning EE, Gittis AH, and Ahmari SE (2019). Strengthened Inputs from 

Secondary Motor Cortex to Striatum in a Mouse Model of Compulsive Behavior. The Journal 
of neuroscience : the official journal of the Society for Neuroscience 39, 2965–2975. 10.1523/
jneurosci.1728-18.2018. [PubMed: 30737313] 

50. Malgady JM, Baez A, Hobel ZB, Jimenez K, Goldfried J, Prager EM, Wilking JA, Zhang Q, 

Feng G, and Plotkin JL (2023). Pathway-specific alterations in striatal excitability and cholinergic 
modulation in a SAPAP3 mouse model of compulsive motor behavior. Cell reports 42, 113384. 
10.1016/j.celrep.2023.113384. [PubMed: 37934666] 

51. Piantadosi SC, Manning EE, Chamberlain BL, Hyde J, LaPalombara Z, Bannon NM, Pierson 
JL, VM KN, and Ahmari SE (2024). Hyperactivity of indirect pathway-projecting spiny 
projection neurons promotes compulsive behavior. Nature communications 15, 4434. 10.1038/
s41467-024-48331-z.

52. Cheng YT, Luna-Figueroa E, Woo J, Chen HC, Lee ZF, Harmanci AS, and Deneen B (2023). 

Inhibitory input directs astrocyte morphogenesis through glial GABA(B)R. Nature 617, 369–376. 
10.1038/s41586-023-06010-x. [PubMed: 37100909] 

53. Vázquez-Victorio G, González-Espinosa C, Espinosa-Riquer ZP, and Macías-Silva M (2016). 
GPCRs and actin-cytoskeleton dynamics. Methods in cell biology 132, 165–188. 10.1016/
bs.mcb.2015.10.003. [PubMed: 26928544] 

Neuron. Author manuscript; available in PMC 2025 October 23.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Soto et al.

Page 19

54. Blanchoin L, Boujemaa-Paterski R, Sykes C, and Plastino J (2014). Actin dynamics, architecture, 
and mechanics in cell motility. Physiological reviews 94, 235–263. 10.1152/physrev.00018.2013. 
[PubMed: 24382887] 

55. Siletti K, Hodge R, Mossi Albiach A, Lee KW, Ding SL, Hu L, Lönnerberg P, Bakken T, Casper T, 
Clark M, et al. (2023). Transcriptomic diversity of cell types across the adult human brain. Science 
(New York, N.Y.) 382, eadd7046. 10.1126/science.add7046. [PubMed: 37824663] 

56. Chen J, Stork T, Kang Y, Nardone KAM, Auer F, Farrell RJ, Jay TR, Heo D, Sheehan A, Paton 
C, et al. (2024). Astrocyte growth is driven by the Tre1/S1pr1 phospholipid-binding G protein-
coupled receptor. Neuron 112, 93–112.e110. 10.1016/j.neuron.2023.11.008. [PubMed: 38096817] 

57. Aten S, Kiyoshi CM, Arzola EP, Patterson JA, Taylor AT, Du Y, Guiher AM, Philip M, Camacho 
EG, Mediratta D, et al. (2022). Ultrastructural view of astrocyte arborization, astrocyte-astrocyte 
and astrocyte-synapse contacts, intracellular vesicle-like structures, and mitochondrial network. 
Progress in neurobiology 213, 102264. 10.1016/j.pneurobio.2022.102264. [PubMed: 35283239] 

58. Chamberlain BL, and Ahmari SE (2021). Animal Models for OCD Research. Current topics in 

behavioral neurosciences 49, 55–96. 10.1007/7854_2020_196. [PubMed: 33763819] 
59. Monteiro P, and Feng G (2016). Learning From Animal Models of Obsessive-Compulsive 

Disorder. Biological psychiatry 79, 7–16. 10.1016/j.biopsych.2015.04.020. [PubMed: 26037910] 

60. Chen J, Stork T, Kang Y, Nardone KAM, Auer F, Farrell RJ, Jay TR, Heo D, Sheehan A, 

Paton C, et al. (2023). Astrocyte growth is driven by the Tre1/S1pr1 phospholipid-binding G 
protein-coupled receptor. Neuron. 10.1016/j.neuron.2023.11.008.

61. Züchner S, Wendland JR, Ashley-Koch AE, Collins AL, Tran-Viet KN, Quinn K, Timpano KC, 
Cuccaro ML, Pericak-Vance MA, Steffens DC, et al. (2009). Multiple rare SAPAP3 missense 
variants in trichotillomania and OCD. Molecular psychiatry 14, 6–9. 10.1038/mp.2008.83. 
[PubMed: 19096451] 

62. Bienvenu OJ, Wang Y, Shugart YY, Welch JM, Grados MA, Fyer AJ, Rauch SL, McCracken 
JT, Rasmussen SA, Murphy DL, et al. (2009). Sapap3 and pathological grooming in humans: 
Results from the OCD collaborative genetics study. American journal of medical genetics. Part 
B, Neuropsychiatric genetics : the official publication of the International Society of Psychiatric 
Genetics 150b, 710–720. 10.1002/ajmg.b.30897.

63. Chen SK, Tvrdik P, Peden E, Cho S, Wu S, Spangrude G, and Capecchi MR (2010). Hematopoietic 

origin of pathological grooming in Hoxb8 mutant mice. Cell 141, 775–785. [PubMed: 20510925] 

64. Nagarajan N, Jones BW, West PJ, Marc RE, and Capecchi MR (2018). Corticostriatal circuit 
defects in Hoxb8 mutant mice. Molecular psychiatry 23, 1868–1877. 10.1038/mp.2017.180. 
[PubMed: 28948967] 

65. Mondragón-González SL, Schreiweis C, and Burguière E (2024). Closed-loop recruitment of 

striatal interneurons prevents compulsive-like grooming behaviors. Nature neuroscience. 10.1038/
s41593-024-01633-3.

66. Tong X, Ao Y, Faas GC, Nwaobi SE, Xu J, Haustein MD, Anderson MA, Mody I, Olsen ML, 

Sofroniew MV, and Khakh BS (2014). Astrocyte Kir4.1 ion channel deficits contribute to neuronal 
dysfunction in Huntington’s disease model mice. Nature neuroscience 17, 694–703. [PubMed: 
24686787] 

67. Oakley RH, Laporte SA, Holt JA, Barak LS, and Caron MG (1999). Association of beta-

arrestin with G protein-coupled receptors during clathrin-mediated endocytosis dictates the profile 
of receptor resensitization. The Journal of biological chemistry 274, 32248–32257. 10.1074/
jbc.274.45.32248. [PubMed: 10542263] 

68. Urban DJ, and Roth BL (2015). DREADDs (designer receptors exclusively activated by designer 
drugs): chemogenetic tools with therapeutic utility. Annu Rev Pharmacol Toxicol 55, 399–417. 
[PubMed: 25292433] 

69. Park YG, Sohn CH, Chen R, McCue M, Yun DH, Drummond GT, Ku T, Evans NB, Oak HC, Trieu 
W, et al. (2018). Protection of tissue physicochemical properties using polyfunctional crosslinkers. 
Nature biotechnology. 10.1038/nbt.4281.

Neuron. Author manuscript; available in PMC 2025 October 23.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Soto et al.

Page 20

Highlights

Astrocyte Gi-GPCR signaling corrects repetitive and anxiety behaviors in Sapap3 KO 
mice.

Gi-GPCR signaling restores astrocyte morphology via the actin cytoskeleton.

Astrocyte Gi-GPCR signaling rescues medium spiny neuron excitability.

Improved phenotypes are accompanied with restored astrocyte homeostatic functions.

Neuron. Author manuscript; available in PMC 2025 October 23.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Soto et al.

Page 21

Figure 1. Behavioral improvements following in vivo astrocyte hM4Di activation.
(A) (i) Hypergeometric analysis heat map shows enrichment of astrocyte territory-related 
genes within those associated with human brain disorders; analyzed from Ref13. * indicates 
FDR < 0.05. (ii) Heat map shows percentage of genes related to OCD and HD shared with 
mouse striatal scRNA-seq data; analyzed from Ref28. OPC, oligodendrocyte precursor cell. 
(iii) Cartoon showing predicted astrocyte morphology changes in OCD. (B) Map of Sapap3 
interacting astrocyte proteins related to the actin cytoskeleton and G-protein signaling; data 
from Ref 10. (C) Schematic showing the experimental design for wild-type (WT) and Sapap3 
KO mice. (D) WT and Sapap3 KO mice treated with in vivo hM4Di activation or control 
AAV. (E) Traces of elevated plus maze recordings for each experimental group. (F) (i) 
Graphs show grooming behavior metrics: area of open lesions, number of lesions, grooming 
bouts, and time spent grooming. (n = 11–12 mice per group, area of lesions: Kruskal-Wallis 
Test; number of lesions: Kruskal-Wallis Test; number of grooming bouts: Two-way ANOVA 

Neuron. Author manuscript; available in PMC 2025 October 23.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Soto et al.

Page 22

with Tukey’s post hoc test [hM4Di effect p-value = 0.06]; time spent grooming: Two-way 
ANOVA with Tukey’s post hoc test [hM4Di effect p-value = 0.02]). (ii) Graphs show 
anxiety-like behaviors by two metrics: time spent in the open arms of the elevated plus maze 
and time spent in the center of the open field. (n = 11–12 mice per group, time spent in 
open arms: Two-way ANOVA with Tukey’s post hoc test [hM4Di effect p-value = 3.5 × 
10−4]; time in center: Two-way ANOVA with Tukey’s post hoc test [hM4Di effect p-value 
= 0.01]). (G) as in (D) but for hM3Dq. (H) Elevated plus maze recordings as in (E) but for 
hM3Dq. (I) (i) Grooming behavior measurements as in (F) but for hM3Dq (n = 7–9 mice per 
group, area of lesions: Kruskal-Wallis Test; number of lesions: Kruskal-Wallis Test; number 
of grooming bouts: Two-way ANOVA with Tukey’s post hoc test [hM3Dq effect p-value = 
0.23]; time spent grooming: Two-way ANOVA with Tukey’s post hoc test [hM3Dq main 
effect p-value = 0.61]). (ii) Anxiety behavior measurements as in (F) but for hM3Dq (n = 
7–9 mice per group, time spent in open arms: Two-way ANOVA with Tukey’s post hoc 
test [hM3Dq effect p-value = 0.44]; time in center: Kruskal-Wallis Test). (J) Summary heat 
map of behavioral Z-scores in Sapap3 KO mice with each experimental treatment, or with 
astrocyte-selective Sapap3 AAV from Ref10. (K) Summary heat map shows percent recovery 
of behaviors for each astrocyte-specific treatment in Sapap3 KO mice versus Sapap3 KO 
mice treated with 10 mg/kg fluoxetine from Ref10. Data are represented as mean ± SEM.

Neuron. Author manuscript; available in PMC 2025 October 23.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Soto et al.

Page 23

Figure 2. Restoration of astrocyte morphology in Sapap3 KO mice with hM4Di DREADDs.
(A) Images showing LifeAct-GFP in WT and Sapap3 KO astrocytes with tdTomato control 
AAV or in vivo hM4Di activation. (B) Left, LifeAct GFP mean actin intensity as a function 
of distance from the soma. Points represent the mean intensity from 15 cells per group 
from 4 mice per group. (Two-way repeated ANOVA with Bonferroni post hoc test; P < 0.05 
at 20–40 μm). Right, graphs show the astrocyte actin territory area and the roundness of 
the actin signal (n = 15 astrocytes from 4 animals per group; Actin area: Kruskal-Wallis 
Test; Roundness: Kruskal-Wallis Test). (C) Summary of astrocyte morphology z-scores in 
each experimental treatment. Far right column shows morphology changes with astrocyte-
selective Sapap3 AAV from Ref10. (D) Images of single WT or Sapap3 KO astrocytes from 
each experimental group after fractal dimension (Df) analysis. Graph shows the Df. (n = 
15 astrocytes from 4 animals per group; Two-way ANOVA with Tukey’s post hoc test). (E) 
Left, zoom-in images of S100β+ striatal astrocytes in each experimental treatment group. 

Neuron. Author manuscript; available in PMC 2025 October 23.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Soto et al.

Page 24

Right, graphs show the S100β somata area and branch number (n = 3–4 mice per group; 
Two-way ANOVA with Tukey’s post hoc test). Data are represented as mean ± SEM.

Neuron. Author manuscript; available in PMC 2025 October 23.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Soto et al.

Page 25

Figure 3. Neuronal properties in Sapap3 KO mice following in vivo astrocyte hM4Di activation.
(A) Heat map of coronal brain sections show the total number of ΔFosB+ cells in WT and 
Sapap3 KO mice treated with in vivo hM4Di activation or control AAV. Heat maps show 
the number of ΔFosB+ cells in the striatum and two cortical areas. Blue and yellow heat 
maps depict the resultant z-scores. (B) Images of ΔFosB+ neurons in the striatum of each 
experimental group. Graphs show the percentage of NeuN+ neurons expressing ΔFosB. (n = 
8 FOVs from 4 mice per group; Two-way ANOVA with Tukey’s post hoc test). DStr, dorsal 
striatum; VStr, ventral striatum. (C) Images of ΔFosB and D1 receptor mRNA+ neurons in 
the striatum of each experimental group. Percentage of D1+ neurons with ΔFosB expression. 
(n = 8 FOVs from 4–5 mice per group; Two-way ANOVA with Tukey’s post hoc test). (D) 
ΔFosB and D2 receptor mRNA+ neurons in the striatum of each experimental group. Graphs 
show the percentage of D2+ neurons with ΔFosB expression (n = 8 FOVs from 4–5 mice per 
group; Kruskal-Wallis Test). (E) Percent of total D1+ and D2+ neurons expressing ΔFosB 

Neuron. Author manuscript; available in PMC 2025 October 23.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Soto et al.

Page 26

in the striatum of Sapap3 KO mice treated with in vivo hM4Di activation or control AAV. 
(n = 8 FOVs from 4–5 mice per group; Unpaired t-test). (F) Representative traces of MSN 
evoked action potentials from each experimental group. (G) Relationship of injected current 
to number of action potentials for WT mice treated with in vivo hM4Di activation or control 
AAV. (n = 16–18 cells from 5–7 mice per group; One-way ANOVA). (H) As in (G) but for 
Sapap3 KO mice (n = 12–23 cells from 4–8 mice per group; One-way ANOVA). (I) Number 
of evoked MSN action potentials at 480 pA current in each experimental group. (n = 12–23 
cells from 4–8 mice per group; Two-way ANOVA with Tukey’s post hoc test). (J) MSN 
rheobase in each experimental group. (n = 12–23 cells from 4–8 mice per group; Two-way 
ANOVA with Tukey’s post hoc test). Data are represented as mean ± SEM.

Neuron. Author manuscript; available in PMC 2025 October 23.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Soto et al.

Page 27

Figure 4. Molecular mechanisms following astrocytic hM4Di activation in Sapap3 KO mice.
(A) Summary of findings from Figs 1–3. (B) Top, differentially expressed proteins (DEPs) 
in Sapap3 KO mice versus WT (FDR < 0.05). Center, differentially expressed genes (DEGs) 
in Sapap3 KO mice versus WT (FDR < 0.05). Bottom, list of 31 genes/proteins that were 
differentially expressed in both proteomic and transcriptomic datasets. (C) (i) Significant 
molecular and biological function Enrichr gene ontology (GO) terms for the 209 DEPs 
in Sapap3 KO striata. (ii) PANTHER protein classifications of the 209 DEPs in Sapap3 
KO striata. (D) Left, Venn diagram shows the comparison of the 209 DEPs in Sapap3 KO 
striata and 271 DEPs (FDR < 0.05) when Sapap3 KO + hM4Di mice was compared to 
Sapap3 KO + tdTomato mice. Right, top heat map shows 43 reciprocally changed proteins 
upon activation of hM4Di in astrocytes. Bottom heat map shows the 20 most enriched and 
depleted proteins that were unique to the hM4Di DEP dataset. The color legend corresponds 
to both heat maps. (E) Top: significant GO terms for the 43 recovered proteins from (D). 

Neuron. Author manuscript; available in PMC 2025 October 23.

 
 
 
 
A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

Soto et al.

Page 28

Bottom: significant GO terms for the 228 hM4Di unique proteins from (D). (F) Left, Venn 
diagram of the 43 hM4Di recovered proteins and 2,869 DEGs (P-value < 0.05) in human 
caudate OCD subjects from Ref 12. Right, heat map shows possible shared reciprocal 
changes upon activation of astrocytic hM4Di from the 15 shared genes. (G) Left, Venn 
diagram of the 228 hM4Di unique proteins and 2,869 human DEGs from (F). Right, heat 
map of the 51 shared genes in human OCD and Sapap3 KO + hM4Di mice. (H) Cartoon 
of Gi-GPCR signaling pathways related to actin polymerization regulation. (I) List of highly 
expressed actin polymerization effectors in striatal astrocytes (FPKM > 10). Astrocyte 
specific RNA-seq was obtained from Ref 28. Heat map and color scale (i) shows the log2fold 
change at the mRNA level of the 38 effectors in human caudate of OCD subjects compared 
to controls. Heat map and color scale (ii) shows the mRNA abundance (FPKM) of the 
effectors in our astrocyte specific mouse RNA-seq datasets. Heat maps and color scale (iii) 
shows the log2fold change of the actin effectors in Sapap3 KO mice. Arrowheads show 
effectors that display similar direction changes in human and mouse and were recovered 
with hM4Di. (J) Dot plot shows baseline expression of striatal enriched Gαi-coupled protein 
receptors from mouse astrocyte data overlapped with human striatum single-nucleus RNA 
sequencing from Ref55. (K) Cartoon: in Sapap3 KO mice, decreased astrocyte morphology 
leads to decreased tissue homeostasis and decreased astrocyte-neuronal contacts. In Sapap3 
KO mice with astrocytic activation of hM4Di, tissue homeostasis is restored.

Neuron. Author manuscript; available in PMC 2025 October 23.

 
 
 
 
Soto et al.

Page 29

Key resources table

REAGENT or RESOURCE

SOURCE

IDENTIFIER

Antibodies

Rabbit anti-S100β

Rabbit anti-NeuN

Abcam

ab41548; RRID: AB_956280

Cell Signaling Technology

12943S, RRID: AB_2630395

Guinea pig anti-NeuN

Synaptic Systems

266004, RRID:AB_2619988

Chicken anti-GFP

Mouse anti-RFP

Rabbit anti-ΔFosB

Abcam

Rockland

ab13970, RRID:AB_300798

200-301-379, RRID:AB_2611063

Cell Signaling Technology

14695S, RRID: AB_2798577

Alexa Fluor 488 goat anti-chicken

Molecular Probes

A11039, RRID:AB_2534096

Alexa Fluor 546 goat anti-mouse

Molecular Probes

A11003, RRID:AB_2534071

Alexa Fluor 647 goat anti-rabbit

Molecular Probes

A21244, RRID:AB_2535812

Alexa Fluor 647 goat anti-mouse

Molecular Probes

A21235, RRID:AB_2535804

Alexa Fluor 488 goat anti-rabbit

Molecular Probes

A11008, RRID: AB_143165

Cy™3 AffiniPure™ Goat Anti-Guinea Pig IgG

Jackson Immunoresearch

106-165-003, RRID: AB_2337423

Bacterial and virus strains

AAV5 GfaABC1D tdTomato

Tong et al., 201470

UPenn Vector Core Cat#AV-5-PV3106 Addgene Vectors 
#44332-AAV5

AAV2/5 GfaABC1D hM3D-mCherry

Chai et al., 201732

Addgene Vectors #92284

AAV2/5 GfaABC1D hM4D-mCherry

Chai et al., 201732

Addgene Vectors #92286

AAV2/5 GfaABC1D LifeAct-GFP

Soto et al., 202312

Addgene Vectors #190199

Chemicals, peptides, and recombinant proteins

Deschloroclozapine (10 μg/kg)

Formalin, Buffered, 10%

Normal goat serum

Critical commercial assays

Multiplex RNAscope Kit (v.2)

Integrated co-detection workflow

Protease pre-treat 4 solution

Mm-Drd1-C3 D1 mRNA probe

Mm-Drd2-C3 D2 mRNA probe

RNAscope wash buffer

Opal 690 fluorophore

Deposited data

Tocris/Biotechne

Fisher Chemical

Vector Laboratories

ACDBio

ACDBio

ACDBio

ACDBio

ACDBio

ACDBio

7193

SF100-20

S-1000

323180

323110

322340

461901-C3

406501-C3

310091

Akoya Biosciences

FP2497001KT

Raw and normalized RNA-seq data

This manuscript

GEO: GSE247241

Raw and normalized mass spectrometry data

This manuscript

MassIVE: MSV000093698

Source data used to generate figures

This manuscript

SI File S3

Experimental models: Organisms/strains

Mouse: Sapap3-null (B6.129-Dlgap3tm1Gfng/J)

Jackson Laboratories

008733

Neuron. Author manuscript; available in PMC 2025 October 23.

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

 
 
 
 
Soto et al.

Page 30

REAGENT or RESOURCE

SOURCE

IDENTIFIER

Software and algorithms

OriginPro 2018

GraphPad Instat v3.0

pClamp 10.4

ClampFit 10.4

Fluoview FV3000 FV31S-SW

ImageJ v1.51h

Anymaze

limmaVoom

artMS v3.18

Other

Origin Lab Corporation

RRID:SCR_014212

Dotmatics

RRID:SCR_000306

Molecular Devices

RRID: SCR_011323

Molecular Devices

Olympus

NIH

N/A

N/A

RRID: SCR_003070

Stoelting Co.

N/A

Law et al., 201414

https://rdrr.io/bioc/limma/man/voom.html

Jimenez-Morales et al., 202335

https://bioconductor.org/packages/artMS

ΔFosB whole CNS mapping

LifeCanvas Technologies

https://lifecanvastech.com/

Neuron. Author manuscript; available in PMC 2025 October 23.

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t

A
u
t
h
o
r

M
a
n
u
s
c
r
i
p
t
